

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article">

<head>
   <title>PLOS Computational Biology: Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks</title>



<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9" />

<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="home" title="home" href="/home.action;jsessionid=A06D857AE008E3829F545704038156D3" />
<link rel="alternate" type="application/rss+xml"
  title="PLOS Computational Biology: New Articles"
  href="http://www.ploscompbiol.org/article/feed" />

<link rel="pingback" href="http://www.ploscompbiol.org/pingback" />


<!-- global_css.ftl -->
<link rel="stylesheet" type="text/css" media="screen" href="/css/dojo/dijit/themes/tundra/tundra.css?v=zwugcxLdpBWmb72qqlHgGyeNAGM" />
<link rel="stylesheet" type="text/css" media="all" href="/css/star_rating.css?v=Mm3HyaUPUahAJAq6xG5W3tdF7Hk" />
<link rel="stylesheet" type="text/css" media="all" href="/css/rating.css?v=AOT9rozE9sYVjGTdeiyxHf18Ykw" />
<link rel="stylesheet" type="text/css" media="all" href="/css/screen.css?v=KBNFqBmhD635U0NgiM5tf9dtxRE" />
<link rel="stylesheet" type="text/css" media="all" href="/css/iepc.css?v=FIjyR83BhSovq_5yNVk1Sih60AI" />
<link rel="stylesheet" type="text/css" media="all" href="/css/journal.css?v=SiREXztjNSUsOyrlkE9sv028Y+0" />
<link rel="stylesheet" type="text/css" media="all" href="/css/lightBox.css?v=sDgsUXamFoYAakEshdj0YWrZiiQ" />
<link rel="stylesheet" type="text/css" media="print" href="/css/print.css?v=0T+LTV7Gd0jtHEhqEvBfSrmHUGg" />

<!--chartbeat --> 
<script type="text/javascript">var _sf_startpt=(new Date()).getTime()</script>


<!--[if IE]><![endif]-->
<meta name="description" content="PLOS Computational Biology is an open-access" />

<meta name="keywords" content="plos computational biology" />

  <meta name="citation_publisher" content="Public Library of Science" />
  <meta name="citation_doi" content="10.1371/journal.pcbi.1002720" />
    <meta name="citation_title" content="Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks"/>
    <meta itemprop="name" content="Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks"/>

      <meta name="citation_author" content="Osbaldo Resendis-Antonio" />
          <meta name="citation_author_institution" content="Centro de Ciencias Genómicas-UNAM, Col. Chamilpa, Cuernavaca, Morelos, México" />
      <meta name="citation_author" content="Magdalena Hernández" />
      <meta name="citation_author" content="Yolanda Mora" />
      <meta name="citation_author" content="Sergio Encarnación" />
          <meta name="citation_author_institution" content="Centro de Ciencias Genómicas-UNAM, Col. Chamilpa, Cuernavaca, Morelos, México" />

    <meta name="citation_date" content="2012/10/11"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pcbi.1002720.pdf" />

    <meta name="citation_journal_title" content="PLOS Computational Biology" />
    <meta name="citation_firstpage" content="e1002720"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="8"/>
    <meta name="citation_issn" content="1553-7358"/>

    <meta name="citation_journal_abbrev" content="PLoS Comput Biol" />

      <meta name="citation_reference" content="citation_title=From molecular to modular cell biology; citation_volume=402; citation_number=1; citation_pages=C47-52; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Understanding network concepts in modules; citation_volume=1; citation_number=2; citation_pages=24; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Detecting hierarchical modularity in biological networks; citation_volume=541; citation_number=3; citation_pages=145-160; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Hierarchical organization of modularity in metabolic networks; citation_volume=297; citation_number=4; citation_pages=1551-1555; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Modular analysis of the transcriptional regulatory network of E. coli; citation_volume=21; citation_number=5; citation_pages=16-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=KEGG: kyoto encyclopedia of genes and genomes; citation_volume=28; citation_number=6; citation_pages=27-30; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=What is flux balance analysis?; citation_volume=28; citation_number=7; citation_pages=245-248; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=The application of flux balance analysis in systems biology; citation_volume=2; citation_number=8; citation_pages=372-382; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Applications of genome-scale metabolic reconstructions; citation_volume=5; citation_number=9; citation_pages=320; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Achieving optimal growth through product feedback inhibition in metabolism; citation_volume=6; citation_number=10; citation_pages=e1000802; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Systems Biology of Bacterial Nitrogen Fixation: High-throughput technology and its integrative description with constraint-based modeling; citation_volume=5; citation_number=11; citation_pages=120; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Capillary electrophoresis-mass spectrometry for metabolomics; citation_volume=358; citation_number=12; citation_pages=129-137; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative metabolome analysis using capillary electrophoresis mass spectrometry; citation_volume=2; citation_number=13; citation_pages=488-494; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Infection of soybean and pea nodules by Rhizobium spp. purine auxotrophs in the presence of 5-aminoimidazole-4-carboxamide riboside; citation_volume=176; citation_number=14; citation_pages=3286-3294; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Carbon Metabolism Enzymes of Rhizobium tropici Cultures and Bacteroids; citation_volume=60; citation_number=15; citation_pages=2339-2342; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Reconstruction of biochemical networks in microorganisms; citation_volume=7; citation_number=16; citation_pages=129-143; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The growing scope of applications of genome-scale metabolic reconstructions using Escherichia coli; citation_volume=26; citation_number=17; citation_pages=659-667; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Analysis of optimality in natural and perturbed metabolic networks; citation_volume=99; citation_number=18; citation_pages=15112-15117; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Metabolic reconstruction and modeling of nitrogen fixation in Rhizobium etli; citation_volume=3; citation_number=19; citation_pages=1887-1895; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evidence for dynamically organized modularity in the yeast protein-protein interaction network; citation_volume=430; citation_number=20; citation_pages=88-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A rapid one-step method for the isolation of bacteroids from root nodules of soybean plants, utilizing self-generating Percoll gradients; citation_volume=27; citation_number=21; citation_pages=491-495; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns; citation_volume=95; citation_number=22; citation_pages=14863-14868; citation_date=1998; " />



<!--
<rdf:RDF xmlns="http://web.resource.org/cc/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<Work rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720">
   <license rdf:resource="http://creativecommons.org/licenses/by/2.5/" />
</Work>
<License rdf:about="http://creativecommons.org/licenses/by/2.5/">
   <permits rdf:resource="http://web.resource.org/cc/Reproduction" />
   <permits rdf:resource="http://web.resource.org/cc/Distribution" />
   <requires rdf:resource="http://web.resource.org/cc/Notice" />
   <requires rdf:resource="http://web.resource.org/cc/Attribution" />
   <permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" />
</License>
<rdf:Description
     rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720"
     dc:identifier="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720"
     dc:title="PLOS Computational Biology: Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks"
       trackback:ping="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720/trackback;jsessionid=A06D857AE008E3829F545704038156D3"
     />
</rdf:RDF>

-->
</head>
<body>
<span property="dc:language" content="en"></span>
<!-- begin : container -->
<div id="container">
  <!-- begin : top banner external ad space -->
  <div id="topBanner">


<!-- begin : left banner slot -->
<div class="left">
  <iframe id='affe494b' name='affe494b'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=84&amp;cb=9405'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=affe494b&amp;cb=8280'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=84&amp;cb=7453&amp;n=affe494b'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : left banner slot -->
<!-- begin : right banner slot -->
<div class="right">
  <iframe id='af654bd5' name='af654bd5'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=85&amp;cb=7843'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=af654bd5&amp;cb=7833'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=85&amp;cb=221&amp;n=af654bd5'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : right banner slot -->

  </div>
  <!-- end : top banner external ad space -->

  <!-- begin : header -->
  <div id="hdr" class="login">

  <!-- begin : logo -->
  <div id="logo" title="PLOS Computational Biology"><a href="/home.action;jsessionid=A06D857AE008E3829F545704038156D3" title="PLOS Computational Biology"><span>PLOS Computational Biology</span></a></div>
  <!-- end : logo -->
  <!-- begin : user controls -->





  <div id="user">
    <div>
      <ul>
        <li><a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002720" class="feedback"><strong>Login</strong></a> | </li>
        <li><a href="https://register.plos.org:443/ambra-registration/register.action">Create Account</a> | </li>
        <li class="feedback"><a href="/feedbackCreate.action?page=%252Farticle%252FfetchArticle.action%253FarticleURI%253Dinfo%25253Adoi%25252F10.1371%25252Fjournal.pcbi.1002720" title="Send us your feedback">Feedback</a></li>
      </ul>
    </div>
  </div>


  <!-- end : user controls -->
  <!-- begin search links -->
  <ul id="links"><li class="browse"><a href="/article/browse.action;jsessionid=A06D857AE008E3829F545704038156D3?field=date" title="Browse Articles">Browse</a></li><li class="rss"><a href="/static/rssFeeds.action;jsessionid=A06D857AE008E3829F545704038156D3" title="RSS Feeds">RSS</a></li></ul>
  <!-- end : search links -->
  <!-- begin : dashboard -->
  <div id="db">
    <form name="searchForm" action="/search/simpleSearch.action;jsessionid=A06D857AE008E3829F545704038156D3" method="get">
      <input type="hidden" name="from" value="globalSimpleSearch"/>
      <input type="hidden" name="filterJournals" value="PLoSCompBiol">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="wrap"><input id="search" type="text" name="query" value="Search articles..." onfocus="if(this.value=='Search articles...')value='';" onblur="if(this.value=='')value='Search articles...';" class="searchField" alt="Search articles..."/></div>
        <input src="/images/search_btn1.gif" onclick="submit();" value="ftsearch" alt="SEARCH" tabindex="3" class="button" type="image" />
      </fieldset>
    </form>
    <form name="gasf" action="/search/advancedSearch.action;jsessionid=A06D857AE008E3829F545704038156D3" method="get">
<input type="hidden" name="noSearchFlag" value="true" id="noSearchFlag"/><input type="hidden" name="query" value="" id="query"/>    </form>
    <a id="advSearch" href="#" onclick="if(document.searchForm.query.value!='Search articles...')document.gasf.query.value=document.searchForm.query.value;document.gasf.submit();return false;">Advanced Search</a>
  </div>
  <!-- end : dashboard -->
  <!-- begin : navigation -->
  <ul id="nav">
    <li><a href="/home.action;jsessionid=A06D857AE008E3829F545704038156D3" tabindex="101">Home</a></li>
     <!-- This is a temporary action to link to th static toc page -->
    <li><a href="/static/browse.action;jsessionid=A06D857AE008E3829F545704038156D3" tabindex="102">Browse Articles</a>
        <ul>
          <li><a href="/article/browseIssue.action;jsessionid=A06D857AE008E3829F545704038156D3?field=issue">Current Issue</a></li> <!-- Assuming dynamic TOC is in place -->
          <li><a href="/article/browseVolume.action;jsessionid=A06D857AE008E3829F545704038156D3?field=volume">Journal Archive</a></li> 
          <li><a href="/article/browse.action;jsessionid=A06D857AE008E3829F545704038156D3?field=date">By Publication Date</a></li>
          <li><a href="/article/browse.action;jsessionid=A06D857AE008E3829F545704038156D3">By Subject</a></li>
          <li><a href="http://www.ploscollections.org/static/pcbiCollections.action">Collections</a></li>
        </ul>
    </li>
    <li><a href="/static/about.action;jsessionid=A06D857AE008E3829F545704038156D3" tabindex="103">About</a>
        <ul>
          <li><a href="/static/information.action;jsessionid=A06D857AE008E3829F545704038156D3">Journal Information</a></li>
          <li><a href="/static/edboard.action;jsessionid=A06D857AE008E3829F545704038156D3">Editorial Board</a></li>
          <li><a href="/static/eic.action;jsessionid=A06D857AE008E3829F545704038156D3">Editors-in-Chief</a></li>
          <li><a href="/static/almInfo.action;jsessionid=A06D857AE008E3829F545704038156D3">Article-Level Metrics</a></li>
          <li><a href="/static/license.action;jsessionid=A06D857AE008E3829F545704038156D3">Open-Access License</a></li>
          <li><a href="/static/contact.action;jsessionid=A06D857AE008E3829F545704038156D3">Contact Us</a></li>
        </ul>
      </li>
    <li><a href="/static/users.action;jsessionid=A06D857AE008E3829F545704038156D3" tabindex="104">For Readers</a>
        <ul>
          <li><a href="/static/commentGuidelines.action;jsessionid=A06D857AE008E3829F545704038156D3">Guidelines for Notes, Comments, and Corrections</a></li>
          <li><a href="/static/ratingGuidelines.action;jsessionid=A06D857AE008E3829F545704038156D3">Guidelines for Rating</a></li>
          <li><a href="/static/help.action;jsessionid=A06D857AE008E3829F545704038156D3">Help Using This Site</a></li>
          <li><a href="/static/downloads.action;jsessionid=A06D857AE008E3829F545704038156D3">Media Downloads</a></li>
          <li><a href="/static/sitemap.action;jsessionid=A06D857AE008E3829F545704038156D3">Site Map</a></li>
        </ul>
      </li>
    <li><a href="/static/authors.action;jsessionid=A06D857AE008E3829F545704038156D3" tabindex="105">For Authors and Reviewers</a>
        <ul>
          <li><a href="/static/policies.action;jsessionid=A06D857AE008E3829F545704038156D3">Editorial and Publishing Policies</a></li>
          <li><a href="/static/guidelines.action;jsessionid=A06D857AE008E3829F545704038156D3">Author Guidelines</a></li>
          <li><a href="/static/figureGuidelines.action;jsessionid=A06D857AE008E3829F545704038156D3">Figure and Table Guidelines</a></li>
          <li><a href="/static/checklist.action;jsessionid=A06D857AE008E3829F545704038156D3">Submit Your Paper</a></li>
          <li><a href="/static/reviewerGuidelines.action;jsessionid=A06D857AE008E3829F545704038156D3">Reviewer Guidelines</a></li>
	   <li><a href="http://www.plos.org/journals/index.php">Submit to Other PLOS Journals</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://www.plos.org" title="PLOS" tabindex="110" class="drop">PLOS.org</a>
        <ul>
          <li><a href="http://www.plos.org/oa/index.html" title="Open Access Statement">Open Access</a></li>
          <li><a href="http://www.plos.org/support/donate.php" title="Join PLOS: Show Your Support">Join PLOS</a></li>
          <li><a href="http://blogs.plos.org/" title="PLOS Blogs">PLOS Blogs</a></li>
	  <li><a href="http://www.zazzle.com/plos?rf=238434112415374154" title="PLOS Store">PLOS Store</a></li>
          <li><a href="http://currents.plos.org" title="PLOS Currents">PLOS Currents</a></li>
          <li><a href="http://www.plos.org/connect.html" title="PLOS.org | Stay Connected">Stay Connected</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://hubs.plos.org" tabindex="109">Hubs</a>
        <ul>
          <li><a href="http://hubs.plos.org/biodiversity" title="PLOS Hubs: Biodiversity">Biodiversity</a></li>
          <li><a href="http://clinicaltrials.ploshubs.org" title="PLOS Hub for Clinical Trials">Clinical Trials</a></li>
        </ul>
    </li>
      <li class="journalnav"><a href="http://www.plosjournals.org" tabindex="108">Journals</a>
        <ul>
          <li><a href="http://www.plosbiology.org/" title="PLOSBiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org/" title="PLOSMedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org/" title="PLOSCompBiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://www.plosgenetics.org/" title="PLOSGenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org/" title="PLOSPathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org/" title="PLOSONE.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org/" title="PLOSNTDs.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </li>
    </ul>
  <!-- end : navigation -->
  </div>
  <!-- end : header --> 



<!-- begin : main content -->
<div id="content" class="article" style="visibility:visible;">

<!-- begin : right hand column -->

<div id="rhc" xpathLocation="noDialog">

  <div id="download" class="rhcBox_type1">
    <div class="wrap">
      <ul>
        <li class="download icon"><strong>Download:</strong>
          <a href="/article/fetchObjectAttachment.action;jsessionid=A06D857AE008E3829F545704038156D3?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720&amp;representation=PDF" title="Download article PDF">PDF</a> |
          <a href="/article/citationList.action;jsessionid=A06D857AE008E3829F545704038156D3?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720" title="Download citations">Citation</a> |
          <a href="/article/fetchObjectAttachment.action;jsessionid=A06D857AE008E3829F545704038156D3?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720&amp;representation=XML" title="Download Article XML">XML</a>
        </li>
        <li class="print icon"><a href="#" onclick="window.print();return false;" title="Print Article"><strong>Print article</strong></a></li>
        <li class="reprint icon"><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=3&doi=10.1371/journal.pcbi.1002720&volume=&issue=&title=Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks&author_name=Osbaldo%20Resendis-Antonio%2C%20Magdalena%20Hern%C3%A1ndez%2C%20Yolanda%20Mora%2C%20Sergio%20Encarnaci%C3%B3n&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a> New &amp; improved!</li>
      </ul>
    </div>
  </div>

  <div id="issues" class="rhcBox_type2">
    <p><strong>Published in the</strong>
    <a href="http://www.ploscompbiol.org/article/browseIssue.action?issue=info%3Adoi%2F10.1371%2Fissue.pcbi.v08.i10" title="Browse the Open-Access Issue">October 2012 Issue of <em>PLOS Computational Biology</em></a></p>
  </div>

  <div id="impact" class="rhcBox_type2">
    <h6>Metrics <a href="/static/help.action;jsessionid=A06D857AE008E3829F545704038156D3#articleMetrics" class="replaced" id="info" title="More information"><span>info</span></a></h6>

    <div class="wrap">
      <img id="rhcDataSpinner" src="/images/loading_small.gif" class="loading" />
      <div id="totalDataInRHC"></div>
      <div id="pubDateInRHC"></div>
      <div id="relatedCitesInRHC"></div>
    </div>

    <div class="more clearfix"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" title="More information">More</a></div>
  </div>

  <div id="related" class="rhcBox_type2">
    <h6>Related Content</h6>
    <dl class="related">
      <dt>Related Articles on the Web</dt>
      <dd><a href="http://scholar.google.com/scholar?hl=en&lr=&q=related:http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002720&btnG=Search">Google Scholar</a></dd>
      <dd id="pubMedRelatedLIInRHC" style="display:none;"><a id="pubMedRelatedURLInRHC">PubMed</a></dd>
    </dl>
    <div class="more clearfix"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" title="More information">More</a></div>
  </div>

  <div id="share" class="rhcBox_type2">
    <h6>Share this Article <a href="/static/help.action;jsessionid=A06D857AE008E3829F545704038156D3#socialBookmarkLinks" class="replaced" id="info" title="More information"><span>info</span></a></h6>
    <ul>
      <li class="bookmarklets">

        <a href="http://www.reddit.com/submit?url=http://dx.plos.org/10.1371/journal.pcbi.1002720"
           target="_new"
           id="reddit">
          <img src="http://www.reddit.com/static/spreddit4.gif" alt="submit to reddit" border="0"/> </a>
        <g:plusone size="small" annotation="none" href=http://dx.plos.org/10.1371/journal.pcbi.1002720></g:plusone>
        <script type="text/javascript">
          (function () {
            var po = document.createElement('script');
            po.type = 'text/javascript';
            po.async = true;
            po.src = 'https://apis.google.com/js/plusone.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(po, s);
          })();
        </script>
        <a href="http://www.stumbleupon.com/submit?url=http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720" target="_new"> <img border=0 src="http://cdn.stumble-upon.com/images/16x16_su_solid.gif" alt="StumbleUpon" title="Add to StumbleUpon"></a>
        <script>function fbs_click() {u='http://dx.plos.org/10.1371/journal.pcbi.1002720';t='Functional%20Modules%2C%20Structural%20Topology%2C%20and%20Optimal%20Activity%20in%20Metabolic%20Networks';window.open('http://www.facebook.com/sharer.php?u='+encodeURIComponent(u)+'&t='+encodeURIComponent(t),'sharer','toolbar=0,status=0,width=626,height=436');return false;}</script><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002720" onclick="return fbs_click()" target="_blank"><img src="http://static.ak.fbcdn.net/images/share/facebook_share_icon.gif" alt="Facebook" title="Add to Facebook" /></a>       <!-- for mor info, see http://www.facebook.com/share_partners.php -->
        <script type="text/javascript">
          function bookmark_in_connotea(u) {
            a=false; x=window; e=x.encodeURIComponent; d=document;
            w=open('http://www.connotea.org/addpopup?continue=confirm&uri='+e(u),
                'add', 'width=600, height=400, scrollbars, resizable');
            void(x.setTimeout('w.focus()',200));
          }
        </script>
        <a style='cursor: pointer;' onclick='javascript:bookmark_in_connotea("http://dx.plos.org/10.1371/journal.pcbi.1002720");'><img src='/images/icon_connotea_16x16.gif' alt="Connotea" title="Add to Connotea"/></a>
        <a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002720&title=Functional%20Modules%2C%20Structural%20Topology%2C%20and%20Optimal%20Activity%20in%20Metabolic%20Networks" target="_new"><img src='/images/icon_citeulike_16x16.gif' alt="CiteULike" title="Add to CiteULike" /></a>
        <a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002720" title="Add to Mendeley" target="_new"><img src="/images/icon_mendeley_16x16.gif" alt="Bibliography"></a>
        <script type="text/javascript">
          var twtTitle  = 'Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks';
          var twtUrl    = 'http://dx.plos.org/10.1371/journal.pcbi.1002720';
          var twtTag    = '#PLOSCompBio:'
          var maxLength = 140 - (twtUrl.length + 1);
          if (twtTitle.length > maxLength) {
            twtTitle = twtTitle.substr(0, (maxLength - 17))+'...';  //truncates title to include space for twtTag and ellipsis
          }
          var twtLink = 'http://twitter.com/intent/tweet?text='+encodeURIComponent(twtTag + ' ' + twtTitle + ' ' + twtUrl);
          document.write('<a href="'+twtLink+'" target="_blank"><img src="/images/icon_twitter.gif" border="0" alt="Twitter icon" title="Tweet This!" /></a>');
        </script>

      </li>
      <li class="email icon"><a href="/article/emailArticle.action;jsessionid=A06D857AE008E3829F545704038156D3?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720" title="Email this article">Email this article</a></li>
    </ul>
  </div>
  <div id="articleTwitterStream" style="display:none">
    <script src="http://widgets.twimg.com/j/2/widget.js"></script>
    <script>
      new TWTR.Widget({
        version: 2,
        type: 'search',
        search: encodeURIComponent('10.1371/journal.pcbi.1002720'),
        interval: 6000,
        title: '',
        subject: 'From Twitter',
        width: 300,
        height: 300,
        theme: {
          shell: {
            background: '#667893',
            color: '#ffffff'
          },
          tweets: {
            background: '#ffffff',
            color: '#444444',
            links: '#006699'
          }
        },
        features: {
          scrollbar: true,
          loop: false,
          live: true,
          hashtags: true,
          timestamp: true,
          avatars: true,
          toptweets: true,
          behavior: 'all'
        }
      }).render().start();
    </script>
  </div>
</div>
<!-- end : right hand column -->
<form name="articleInfo" id="articleInfo" method="" action="">
<input type="hidden" name="isAuthor" value="true" />
<input type="hidden" name="authorIdList" value="" />
<input type="hidden" name="userIdList" value="" />
<input type="hidden" name="otherIdList" value="" />
<input type="hidden" name="annotationId" value="" />
<input type="hidden" name="isResearchArticle" value="true" />
</form>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.XML" value="106302"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.PDF" value="745775"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g001.TIF" value="1614708"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g001.PNG_I" value="52451"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g001.PNG_S" value="10969"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g001.PNG_M" value="102231"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g001.PNG_L" value="1007225"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g002.TIF" value="1565524"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g002.PNG_I" value="70780"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g002.PNG_S" value="18997"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g002.PNG_M" value="150891"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g002.PNG_L" value="820444"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.t001.TIF" value="154226"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.t001.PNG_I" value="7872"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.t001.PNG_S" value="2069"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.t001.PNG_M" value="17638"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.t001.PNG_L" value="38344"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g003.TIF" value="1561768"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g003.PNG_I" value="48958"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g003.PNG_S" value="12648"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g003.PNG_M" value="111994"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g003.PNG_L" value="914291"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g004.TIF" value="1682948"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g004.PNG_I" value="59846"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g004.PNG_S" value="12651"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g004.PNG_M" value="137854"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g004.PNG_L" value="1207318"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g005.TIF" value="1013524"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g005.PNG_I" value="38164"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g005.PNG_S" value="13391"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g005.PNG_M" value="71132"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.g005.PNG_L" value="421126"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e001.TIF" value="16258"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e001.PNG" value="6571"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e002.TIF" value="14324"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e002.PNG" value="6984"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e003.TIF" value="13270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e003.PNG" value="5017"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e004.TIF" value="4328"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e004.PNG" value="2013"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e005.TIF" value="996"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.e005.PNG" value="1005"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.s001.XLSX" value="69384"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.s002.XLSX" value="22805"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002720.s003.XLSX" value="47412"/>
<div id="articleContainer">

<span about="http://www.plos.org"  typeof="foaf:Organization" property="foaf:name" style="display:none; visibility:hidden;">PLOS</span>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader" xpathLocation="noSelect">
  <p>Open Access</p><p id="articleType">Research Article</p>
</div>

  <h1 xpathLocation="noSelect" property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks</h1>
  <div property="dc:description" datatype="" style="display:none; visibility:hidden;">Author SummaryBiological networks are an inherent concept in systems biology that is useful in elucidating how biological entities&mdash;as metabolites or proteins&mdash;work together in supporting specific phenotypes in microorganisms. Notably, topological analyses carried out over these networks have shown that modular organization is a ubiquitous property at different levels of biological organization, in such a way that modular organization may serve as an organizing principle governing the metabolic activity in microorganisms. With the aim of elucidating the relationship among functional modules, network topology, and optimal metabolic activity, here we present an integrative study that combines computational modeling and metabolome data for evaluation of the metabolic activity of the soil bacterium Rhizobium etli during symbiotic nitrogen fixation with Phaseolus vulgaris. As a result, we supply experimental and computational evidence supporting the concept that the optimal metabolic activity during this biological process is guided by modular structures in the metabolic network of R. etli. Even more fundamentally, we suggest that these biochemical modules interact among each other to ensure an optimal phenotype during nitrogen fixation. Finally, through the in silico analysis on the genome scale metabolic reconstruction for R.etli, we give some examples that suggest that these modular structures supporting nitrogen fixation are robust to external physiological conditions.</div>
  <span property="dc:date" content="2012-10-11" datatype="xsd:date" rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pcbi.1002720"></span>
  <span property="dc:subject" content="Biochemistry"></span>
  <span property="dc:subject" content="Computational Biology"></span>
  <span property="dc:subject" content="Computer Science"></span>
  <span property="dc:subject" content="Genetics and Genomics"></span>
<form action="">
  <input type="hidden" name="journalDisplayName" id="journalDisplayName" value="PLOS Computational Biology" />
  <input type="hidden" name="crossRefPageURL" id="crossRefPageURL" value="/article/crossref/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" />
  <input type="hidden" name="twitterPageURL" id="twitterPageURL" value="/article/twitter/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" />
  <input type="hidden" name="metricsTabURL" id="metricsTabURL" value="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" />
  <input type="hidden" name="doi" id="doi" value="info:doi/10.1371/journal.pcbi.1002720" />
  <input type="hidden" name="articleTitleUnformatted" id="articleTitleUnformatted" value="Functional%20Modules%2C%20Structural%20Topology%2C%20and%20Optimal%20Activity%20in%20Metabolic%20Networks" />
  <input type="hidden" name="articlePubDate" id="articlePubDate" value="1349938800000" />
</form>
<div class="horizontalTabs" xpathLocation="noSelect">
  <ul id="tabsContainer">
        <li id="article" class="active"><a href="/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720;jsessionid=A06D857AE008E3829F545704038156D3" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenuBox" xpathLocation="noSelect">
    <div id="articleMenu">
      <div class="wrap">
        <ul>
          <li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" onclick="toggleAnnotation(this, 'public'); return false;" title="Click to turn notes on/off">Hide notes</a></li>
          <li class="discuss icon">
              <a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002720">Make a general comment</a>
          </li>
        </ul>
        <div id="sectionNavTopBox" style="display:none;">
          <p><strong>Jump to</strong></p>
          <div id="sectionNavTop" class="tools"></div>
        </div>
      </div>
    </div>
      <div id="lightBox" xpathLocation="noDialog" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720')">
      </div>
  </div>




<p class="authors" xpathLocation="noSelect"><span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Osbaldo Resendis-Antonio</span></span><sup><a href="#cor1" class="fnoteref">*</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Magdalena Hernández</span></span>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Yolanda Mora</span></span>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Sergio Encarnación</span></span><sup><a href="#cor1" class="fnoteref">*</a></sup></p><p class="affiliations" xpathLocation="noSelect"><a name="aff1" id="aff1"></a>Centro de Ciencias Genómicas-UNAM, Col. Chamilpa, Cuernavaca, Morelos, México</p><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2 xpathLocation="noSelect">Abstract&nbsp;<a href="#top">Top</a></h2><p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]/p[1]">Modular organization in biological networks has been suggested as a natural mechanism by which a cell coordinates its metabolic strategies for evolving and responding to environmental perturbations. To understand how this occurs, there is a need for developing computational schemes that contribute to integration of genomic-scale information and assist investigators in formulating biological hypotheses in a quantitative and systematic fashion. In this work, we combined metabolome data and constraint-based modeling to elucidate the relationships among structural modules, functional organization, and the optimal metabolic phenotype of <em>Rhizobium etli</em>, a bacterium that fixes nitrogen in symbiosis with <em>Phaseolus vulgaris</em>. To experimentally characterize the metabolic phenotype of this microorganism, we obtained the metabolic profile of 220 metabolites at two physiological stages: under free-living conditions, and during nitrogen fixation with <em>P. vulgaris</em>. By integrating these data into a constraint-based model, we built a refined computational platform with the capability to survey the metabolic activity underlying nitrogen fixation in <em>R. etli</em>. Topological analysis of the metabolic reconstruction led us to identify modular structures with functional activities. Consistent with modular activity in metabolism, we found that most of the metabolites experimentally detected in each module simultaneously increased their relative abundances during nitrogen fixation. In this work, we explore the relationships among topology, biological function, and optimal activity in the metabolism of <em>R. etli</em> through an integrative analysis based on modeling and metabolome data. Our findings suggest that the metabolic activity during nitrogen fixation is supported by interacting structural modules that correlate with three functional classifications: nucleic acids, peptides, and lipids. More fundamentally, we supply evidence that such modular organization during functional nitrogen fixation is a robust property under different environmental conditions.</p>
</div><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]"><a id="abstract1" name="abstract1" toc="abstract1" title="Author Summary"></a>
<h2 xpathLocation="noSelect">Author Summary&nbsp;<a href="#top">Top</a></h2>
<p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]/p[1]">Biological networks are an inherent concept in systems biology that is useful in elucidating how biological entities—as metabolites or proteins—work together in supporting specific phenotypes in microorganisms. Notably, topological analyses carried out over these networks have shown that modular organization is a ubiquitous property at different levels of biological organization, in such a way that modular organization may serve as an organizing principle governing the metabolic activity in microorganisms. With the aim of elucidating the relationship among functional modules, network topology, and optimal metabolic activity, here we present an integrative study that combines computational modeling and metabolome data for evaluation of the metabolic activity of the soil bacterium <em>Rhizobium etli</em> during symbiotic nitrogen fixation with <em>Phaseolus vulgaris</em>. As a result, we supply experimental and computational evidence supporting the concept that the optimal metabolic activity during this biological process is guided by modular structures in the metabolic network of <em>R. etli</em>. Even more fundamentally, we suggest that these biochemical modules interact among each other to ensure an optimal phenotype during nitrogen fixation. Finally, through the <em>in silico</em> analysis on the genome scale metabolic reconstruction for <em>R.etli</em>, we give some examples that suggest that these modular structures supporting nitrogen fixation are robust to external physiological conditions.</p>
</div>


<div class="articleinfo" xpathLocation="noSelect"><p><strong>Citation: </strong>Resendis-Antonio O, Hernández M, Mora Y, Encarnación S (2012) Functional Modules, Structural Topology, and Optimal Activity in Metabolic Networks. PLoS Comput Biol 8(10):
          e1002720.
            doi:10.1371/journal.pcbi.1002720</p><p><strong>Editor: </strong>Markus W. Covert, 
        Stanford University, United States of America
      </p><p></p><p><strong>Received:</strong> March 29, 2012; <strong>Accepted:</strong> August 16, 2012; <strong>Published:</strong> October 11, 2012</p><p><strong>Copyright:</strong> ©  Resendis-Antonio et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>One of the authors (OR-A) thank the financial support from National Council of Science and Technology CONACyT-Mexico (grants 83461) and PAPIIT-DGAPA-UNAM (grant IN203809-3) for supporting this project. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p><p><a name="cor1"></a>* E-mail: <a href="mailto:resendis@ccg.unam.mx">resendis@ccg.unam.mx</a> (ORA); <a href="mailto:encarnac@ccg.unam.mx">encarnac@ccg.unam.mx</a> (SE)</p></div>





<div id="section1" xpathLocation="/article[1]/body[1]/sec[1]"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3 xpathLocation="noSelect">Introduction&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[1]/p[1]">With the advent of bioinformatics and high-throughput technologies, plentiful sources of information stored in databases are now available to help unveil how a variety of biological entities interact among each other and to elucidate how these interacting networks support phenotypic behaviors in microorganisms. Notably, a variety of studies accomplished with these networks have contributed to elucidation of some fundamental organizing principles by which the cell presumably regulates and coordinates its vital biological functions. Among these organizing properties, structural modularity is a systemic property that has been observed in a variety of biological networks, which range from genetic transcriptional regulation to metabolic activity <a href="#pcbi.1002720-Hartwell1">[1]</a>, <a href="#pcbi.1002720-Dong1">[2]</a>, <a href="#pcbi.1002720-Ravasz1">[3]</a>, <a href="#pcbi.1002720-Ravasz2">[4]</a>, <a href="#pcbi.1002720-ResendisAntonio1">[5]</a>. On an even more fundamental level, there is evidence that these modules inferred from network topology can be associated with functional modules; these latter are defined as a set of biological components (metabolites, proteins, or genes) that coordinately participate in accomplishing a specific biological function in the cell <a href="#pcbi.1002720-Kanehisa1">[6]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[2]">On the other hand, the idea that optimization principles guide metabolic activity in microorganisms has been an interesting hypothesis that, in combination with genome-scale metabolic reconstructions, has resulted in a successful framework for a systematic, quantitative, and predictive scheme in systems biology <a href="#pcbi.1002720-Orth1">[7]</a>. Briefly, the optimization problem in this context is reduced to identification of the metabolic flux along the network that ensures maximal production of a specific array of metabolites representing a specific phenotypic state in the microorganism. An optimal metabolic phenotype constitutes the basis of constraint-based modeling, and its application domain has been extended to a variety of organisms in the past few decades <a href="#pcbi.1002720-Gianchandani1">[8]</a>, <a href="#pcbi.1002720-Oberhardt1">[9]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[3]">In this context, an immediate and fundamental question emerges: how do these structural modules organize and coordinate among themselves to support an optimal metabolic phenotype in microorganisms?. Even though this enterprise is far from being solved, some remarkable advances have been reported in the field. For instance, a recent experimental and <em>in silico</em> study on a metabolic reconstruction for <em>Escherichia coli</em> supported the idea that feedback inhibition in metabolic units, called modules, constitutes a mechanism capable of inducing an optimal growth rate <a href="#pcbi.1002720-Goyal1">[10]</a>. Equally relevant, there are studies that have pointed out that modular organization on a genomic scale may be a natural strategy for coordinating the transcriptional and metabolic activities required for ensuring that cells evolve and efficiently respond to environmental perturbations <a href="#pcbi.1002720-Hartwell1">[1]</a>, <a href="#pcbi.1002720-ResendisAntonio1">[5]</a>. Despite these and other advances, the study of the principles governing the metabolic organization in cells is in its infancy, and additional discoveries are required for surveying how structural modules in a metabolic network link together to efficiently achieve their biological functions <a href="#pcbi.1002720-Goyal1">[10]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[4]">In this work, by using a systems biology description, we supply computational and experimental evidence that suggests that structural and functional modularities are robust properties when the cell operates under its optimal metabolic phenotype at different physiological conditions. To support our conclusions, we carried out an integrative study involving computational modeling and high-throughput sequencing technology for characterizing the metabolic activity of <em>Rhizobium etli</em> CFN42 during symbiotic nitrogen fixation in symbiotic association with <em>Phaseolus vulgaris</em> (bean plant). Here, this organism is our benchmark model, a decision that was favored based on the availability of 1) a computational description of its genome-scale metabolic reconstruction, 2) an integrative description among high-throughput technologies at nitrogen fixation stages, and 3) the valuable physiological knowledge already available that describes the metabolic activity during nitrogen fixation by this organism in symbiotic association with <em>P. vulgaris</em> <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[5]">Hence, to elucidate the metabolic activity of <em>R.etli</em> during nitrogen fixation, constraint-based modeling was applied on an updated version of the metabolic reconstruction for this organism (<em>iOR450</em>) <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. At present, the metabolic reconstruction of <em>R. etli</em> is an integrated network of 402 reactions involving the participation of 450 genes and 377 metabolites. Unlike our previous studies, here we used metabolome technology to experimentally support our <em>in silico</em> interpretations of <em>R.etli</em> metabolism under two physiological conditions: when it fixes nitrogen in symbiosis with <em>P. vulgaris</em>, and under free-living conditions with succinate and ammonium as carbon and nitrogen sources, respectively (see the <a href="#s4">Materials and Methods</a> section).Thus, by applying capillary electrophoresis and mass spectrometry (CE-MS) <a href="#pcbi.1002720-Soga1">[12]</a>, <a href="#pcbi.1002720-Soga2">[13]</a> to <em>R.etli</em> and its products, we report the relative abundances of 220 metabolites under these physiological conditions. To improve the interpretations obtained from the constraint-based modeling, we used metabolome data to identify metabolites with meaningful biological roles in bacterial nitrogen fixation. Consequently, this information was used to guide the reconstruction of a more proper objective function (OF) to computationally simulate this biological process. Next, with the results of our integrative analysis carried out between the constraint-based modeling and the metabolome data for <em>R. etli</em>, we concluded that the metabolic activity at optimal nitrogen fixation is supported by structural modules with well-defined functional activities. Furthermore, our <em>in silico</em> study let us show that those modular structures tend to be robust during changes in environmental conditions. Overall, our study supplies evidence that modular organization in metabolic networks is required for promoting an optimal metabolic phenotype in microorganisms.</p>
</div>

<div id="section2" xpathLocation="/article[1]/body[1]/sec[2]"><a id="s2" name="s2" toc="s2" title="Results"></a><h3 xpathLocation="noSelect">Results&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/title[1]">Metabolome profile in bacterial nitrogen fixation</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]">In addition to transcriptome and proteome approaches, metabolome technology represents a third complementary approach to characterize the phenotypic state of a microorganism, through quantitative and qualitative descriptions of its metabolite concentrations. In order to elucidate the metabolic capabilities and organizing properties of <em>R. etli</em> metabolism, we investigated the metabolome profiles of this bacterium in two physiological situations: during nitrogen fixation with <em>P.vulgaris</em> and under free-living conditions (see the <a href="#s4">Materials and Methods</a> section and reference <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>). Biological samples from each physiological condition were analyzed in triplicate using CE-MS technologies <a href="#pcbi.1002720-Soga1">[12]</a>, <a href="#pcbi.1002720-Soga2">[13]</a>, and the output was used to sense the relative abundance levels of metabolites under each physiological condition. Metabolome measurements were carried out through a facility service at Human Metabolome Technology Inc., Tsuruoka, Japan. Briefly, the samples were prepared following an experimental protocol supplied by Human Metabolome Technology Inc., and the ionic metabolites were separated through electric fields. Having separated the ionic metabolites by capillary electrophoresis the samples were subjected to spectrometric analysis, and their identities were selectively detected by monitoring ions over a large range of <em>m</em>/<em>z</em> values. Thus, spectrum profiles were compared in consultation with the Human Metabolome Technology database and normalized with respect to internal controls for uncovering and estimating their identities and metabolic abundances in both physiological conditions. The relative peak area observed by spectrometry and the average, standard deviation, and log-ratio obtained for each metabolite are reported in <a href="#pcbi.1002720.s001">Dataset S1</a> in the supporting information and visually depicted in <a href="#pcbi-1002720-g001">Figure 1</a>. Overall, high-throughput analysis led us to identify and characterize the abundances of 220 ionic metabolites (93 cations and 127 anions) under both physiological conditions. A biological analysis of these results led us to arrive at the following conclusions:</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]"><a name="pcbi-1002720-g001" id="pcbi-1002720-g001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.g001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g001');"><span>Figure 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/title[1]">Experimental metabolome profile.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/p[1]">Log-ratios of 220 metabolites detected under two physiological conditions for <em>R. etli</em>: bacteroids during nitrogen fixation (Bac) and free-living conditions (VL). Meanwhile, metabolites (shown in marine blue at the left side of the illustration) detected exclusively under free-living conditions are listed, and the metabolites denoted in red were exclusively detected during bacterial nitrogen fixation (listed on the right side). The rest of the metabolites were detected under both physiological conditions at different ratios (see the middle part of the figure).</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002720.g001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002720.g001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002720.g001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002720.g001.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<ol class="order">

<li xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/list[1]/list-item[1]">A previous analysis based on transcriptome and proteome data led us to identify certain proteins required for the synthesis of different amino acids in <em>R. etli</em> during nitrogen fixation (arginine, tyrosine, tryptophan, phenylalanine, and lysine) <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. In parallel with these findings, our current metabolome analysis led us to detect all the common amino acids, with the exception of cysteine; in this case we only identified its homologue, homocysteine. In light of these results, the participation of a variety of amino acids in <em>R. etli</em> during symbiotic nitrogen fixation with <em>P. vulgaris</em> is clear. The role of each amino acid and which of them are essential to carry out the biological processes are some avenues to be explored in the future.</li>

<li xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/list[1]/list-item[2]">With dicarboxylic acids as the carbon source, the TCA cycle would be expected to be active. As we reported previously, an integrative analysis using high-throughput technology (on the transcriptome and proteome levels) and constraint-based modeling suggested that the TCA cycle is active during nitrogen fixation <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. Consistent with this finding, five metabolites that participate in the TCA cycle were detected from <em>R. etli</em> (citric acid, 2-oxoglutaric acid, succinic acid, fumaric and malic acid). In our opinion, our new high-throughput data supply additional evidence that an operational TCA cycle supports bacterial nitrogen fixation.</li>

<li xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/list[1]/list-item[3]">Purine and pyrimidine pathways are important during nodulation processes, given that most purine or pyrimidine auxotrophs in <em>Rhizobiaceas</em> are ineffective in symbiotic nitrogen fixation because they elicit pseudo-nodules that are devoid of an infection thread <a href="#pcbi.1002720-Newman1">[14]</a>.In parallel, our earlier results indicated that an intense synthesis of macromolecules, such as purines and pyrimidines, is carried out in bacteria <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. In support of these physiological descriptions and with an emphasis on the importance of these molecules during the nitrogen fixation process, we identified the purines—GTP, guanine, adenosine, guanosine, ADP, GDP, allantoic acid, XMP, AMP, GMP, ATP, dAMP, dADP, dAMP,and dADP—and the pyrimidines—CDP,uridine, CMP, cytosine, UDP, UMP,dTMP,UTP,dTDP, CTP, and cytidine—through our metabolome study.</li>

<li xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/list[1]/list-item[4]">In a previous report, we suggested that besides gluconeogenesis, a fueling pathway based on pentoses may exist in bacteria during nitrogen fixation <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>.In agreement with this hypothesis, ribose-5-phosphate, fructose-1,6-diphosphate, ribulose-1,5-diphosphate, ribulose-5-phosphate, ribose-1-phosphate, sedoheptulose-7-phosphate, and fructose-6-phosphate metabolites were detected in the bacteroids. In fast-growing rhizobia, the pentose phosphate pathway in combination with the Entner-Doudoroff pathway are probably the major routes used for the metabolism of sugars <a href="#pcbi.1002720-Romanov1">[15]</a>. Thus, based on this observation, we hypothesize that other carbon sources, in addition to dicarboxylic acids, participate in bacterial nitrogen fixation. This hypothesis constitutes a perspective that should be experimentally verified in future works.</li>

</ol><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[2]">Undoubtedly, the above descriptions are valuable for elucidation of the metabolic landscape during bacterial nitrogen fixation in <em>R.etli</em>; however, in order to explore the relationships among network topology, functionality, and optimal metabolic phenotype, it is necessary to move toward an integrative, quantitative, and predictive description. To this end, metabolome data were used for improving and assessing the flux metabolic activity inferred from constraint-based modeling when applied on the genome-scale metabolic reconstruction of <em>R. etli</em>. Thus, from the set of 220 metabolites experimentally detected, we identified 119 of the 377 metabolites that participate in the metabolic reconstruction reported for <em>R. etli OR450</em> <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. This metabolic set covered 32.75% of the total number of metabolites participating in the metabolic reconstruction and constituted our experimental dataset for improving, supporting, and assessing the results that emerged from the constraint-based modeling.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/title[1]">Bridging metabolome data and constraint-based modeling results</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[1]">Constraint-based modeling is a paradigm in systems biology for exploring the metabolic phenotype in cells under specific environmental conditions and/or subject to genetic perturbations <a href="#pcbi.1002720-Feist1">[16]</a>, <a href="#pcbi.1002720-Feist2">[17]</a>, <a href="#pcbi.1002720-Segre1">[18]</a>. In particular, Flux Balance Analysis (FBA) has proven to be a useful computational tool for surveying the metabolic phenotype capacity in a microorganism and to simultaneously evaluate its coherent description with high-throughput technology <a href="#pcbi.1002720-Oberhardt1">[9]</a>, <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. Briefly, once the genome-scale metabolic reconstruction for an organism has been completed, FBA can be divided into two main steps: 1) the reconstruction of an OF that simulates a particular phenotypic state for the microorganism (for instance, the growth rate or maximal nitrogen fixation, to name two), and 2) the search for a flux distribution along the entire network that maximizes the selected OF when the system is at steady-state <a href="#pcbi.1002720-Orth1">[7]</a>. Among the variety of applications that can be tackled with this systems-level framework, bacterial nitrogen fixation carried out by <em>R.etli</em> is an example of how an integrative study using computational modeling and high-throughput technology can contribute to understanding, predicting, and elucidating the metabolic activity during this biological process <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. Based on these previous achievements, we selected this organism as our model system for surveying the relationships among structural organization, functionality, and the maximal phenotype during bacterial nitrogen fixation.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[2]">Before proceeding with our <em>in silico</em> analysis, we took advantage of the metabolome data to refine and improve a previous FBA carried out for this organism <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. Thus, given the metabolic reconstruction <em>iOR450</em>, we improved the FBA by suggesting a more proper OF, one capable of modeling bacterial nitrogen fixation in a more accurate and realistic fashion. Reconstruction of an OF is a crucial issue in constraint-based modeling for computationally mimicking the metabolic activity of a microorganism, in this case, bacterial nitrogen fixation. For this reason, an OF is mainly defined in terms of those metabolites whose permanent production is essential for sustaining bacterial nitrogen fixation <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>, <a href="#pcbi.1002720-ResendisAntonio3">[19]</a>. Currently, the OF for modeling bacterial nitrogen fixation has been entirely constructed in terms of a query in the scientific literature <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>, <a href="#pcbi.1002720-ResendisAntonio3">[19]</a>; however, the metabolome data depicted in <a href="#pcbi-1002720-g001">Figure 1</a> open the possibility for identifying new potential metabolites that were not included in previous analyses due to a lack of physiological evidence.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[3]">The issue described above is extremely important because it directly influences the quality of the <em>in silico</em> interpretations and the coherent description based on the combination with high-throughput data. In order to proceed with this improvement, our strategy was based on the argument that metabolites detected at higher concentrations at the symbiotic nitrogen fixation stage, compared with those observed under free-living conditions, may mirror their fundamental participation in sustaining a functional phenotype during the biological process. Hence, under this assumption, we identified those metabolites whose concentrations increased during nitrogen-fixing activity by applying a statistical test on the log-ratio data, as shown in <a href="#pcbi-1002720-g001">Figure 1</a> and <a href="#pcbi.1002720.s001">Dataset S1</a> in the supporting information. To reduce the probability of including false-positive results, we limited the selection of metabolites to only those that obeyed the following conditions: 1)the log-ratio increased by at least by 2-fold during nitrogen fixation, with a <em>p</em>-value&lt;0.01 (see the <a href="#s4">Materials and Methods</a> section); and 2)once the candidate metabolite was included in the OF, there existed at least a solution for the metabolic flux distribution that maximized the OF at steady state.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[4]">A graphical representation of those metabolites obeying the first criterion is shown in <a href="#pcbi-1002720-g002">Figure 2 (A)</a> (blue points). Consequently, by applying the second criterion over this metabolite subset, we identified nine metabolites that potentially have an important role in driving and supporting bacterial nitrogen fixation. Having identified candidate metabolites, we integrated them into the OF and applied linear optimization to identify the metabolic flux distribution that maximized the metabolism of nitrogen-fixing bacteria (see the <a href="#s4">Materials and Methods</a> section). The time line for the metabolic components, integrating the previous OF in bacterial nitrogen fixation for <em>R. etli</em> (<em>Z<sup>Fix</sup></em>),including the one obtained with the criteria described above, is summarized in <a href="#pcbi-1002720-g002">Figure 2 (B)</a> and the <a href="#s4">Materials and Methods</a> section. As a result, the OF reported in this study shows a significant number of components that were not included in previous versions, and among them are the following: 3-phospho-<span class="small-caps">d</span>-glycerate (3pg),2-oxoglutarate (akg), <span class="small-caps">l</span>-arginine (arg-L), <span class="small-caps">l</span>-aspartate (asp-L), citrate (cit), CMP (cmp), fumarate (fum), malate (mal-L), and <span class="small-caps">l</span>-tryptophan (trp-L)[see <a href="#pcbi-1002720-g002">Figure 2(B)</a>].</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]"><a name="pcbi-1002720-g002" id="pcbi-1002720-g002" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g002');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g002');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.g002&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g002');"><span>Figure 2. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/title[1]">Integrative description among <em>in silico</em> analysis and high-throughput data.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/p[1]">(A) Log-ratio and <em>p</em>-values plot. In blue we show those metabolites that obeyed two of three criteria: a log-ratio higher than 2, a <em>p</em>-value lower than 0.01,and its requirement for consideration as a metabolite with an important role in supporting bacterial nitrogen fixation. (B) Historical improvements in OFs. Since the first publication dealing with the systems biology description of the bacterial nitrogen fixation for <em>R. etli</em> <a href="#pcbi.1002720-ResendisAntonio3">[19]</a>, some improvements have been carried out for determining the OF, by identifying proper metabolites required for supporting this biological process (see <a href="#pcbi-1002720-g002">Figure 2</a>). At present, the suggested objective function is the result of a careful search of the published literature and an integrative description in the metabolome data. Under these experimental and physiological bases, our modeling located an OF slightly closer to the complex metabolism supporting bacterial nitrogen fixation, and consequently constraint-based modeling raised its accuracy and predictive scope. D) Comparative analysis between FBA and transcriptome and proteome data. A detailed comparison between computational predictions and high-throughput data was carried out for all 22 metabolic pathways defined in panel C. One can distinguish three possible situations: 1) genes (enzymes) that were predicted <em>in silico</em> but not monitored experimentally (CBM not HT; blue); 2) genes (enzymes) that were consistently observed in both schemes (CBM and HT; green); and 3) genes (enzymes) that were experimentally monitored but not predicted <em>in silico</em> (HT not CBM; red). As explained in the <a href="#s4">Materials and Methods</a> section, <em>η</em> is related to the fraction of genes (enzymes) that were consistently observed in both schemes and constitutes the backbone for our modeling assessment. Upper and lower bars at the right side of panel D indicate the fraction of proteins and genes that were predicted <em>in silico</em> and detected in proteome or transcriptome data.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002720.g002</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002720.g002/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g002/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002720.g002.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g002/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002720.g002.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[5]">To assess the consequences of this improvement and compare the results reported here with those obtained in our previous study, we performed FBA on the metabolic reconstruction for <em>R. etli</em> (<em>iOR450</em>), taking into account the new OF depicted in <a href="#pcbi-1002720-g002">Figure 2(B)</a>. To proceed with this comparative analysis, our simulation was carried out under equivalent conditions as those described in reference <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>, which can be summarized as follows:</p>


<ol class="order">

<li xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/list[1]/list-item[1]">The experimental data underlying this analysis were integrated on those genes up-regulated and the proteins detected during bacterial nitrogen fixation after 18 days of inoculation of <em>R. etli</em> with root plant of <em>P. vulgaris</em> (see the <a href="#s4">Materials and Methods</a> section). Overall, this set of high-throughput data was integrated based on 415 proteins and 689 up-regulated genes (see the details in reference <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>).</li>

<li xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/list[1]/list-item[2]">In analogy with a previous analysis, we selected 22 KEGG metabolic pathways to evaluate the concordance between the metabolic activity predicted <em>in silico</em> and that interpreted with the high-throughput technology. <a href="#pcbi-1002720-g002">Figure 2 (C)</a> enlists that set of metabolic pathways. Even though the KEGG database reports around 300 different metabolic pathways for <em>R. etli</em>, we selected only 22 as the core of our analysis because transcriptome and proteome data available for this organism have made evident the participation of the 22 pathways during bacterial nitrogen fixation <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. According to the KEGG database, these 22 metabolic pathways contain 652 genes for <em>R. etli</em>, of which around 54% were included in the metabolic reconstruction for <em>iOR450</em>. This set of genes and their corresponding enzymes constituted the central core for evaluating the coherence between <em>in silico</em> predictions and high-throughput data interpretations. Even though the <em>in silico</em> assessment relied on the activity of these 22 metabolic pathways, the FBA took into account all the reactions included in the entire metabolic reconstruction. We expect that this latter consideration is valuable for exploring and predicting the metabolic role that additional pathways may have on bacterial nitrogen fixation.</li>

<li xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/list[1]/list-item[3]">To quantify the concordance between the metabolic activity predicted <em>in silico</em> and that interpreted from high-throughput technology, we calculated the fraction of genes (<em>η<sup>Genes</sup></em>) and enzymes (<em>η<sup>Enzymes</sup></em>)over the 22 pathways that were predicted to be active by constraint-based modeling and simultaneously detected in the high-throughput data (see the <a href="#s4">Materials and Methods</a> section). The numerical values of these parameters, were defined in such a way that 1 represented the highest and 0 the lowest consistency score between the genes (or enzymes) predicted by both procedures.</li>

</ol><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[6]">In this context, FBA carried out for <em>iOR450</em> with the new OF led us to conclude that the consistency coefficients for genes (<em>η<sup>Genes</sup></em>)and proteins (<em>η<sup>Enzymes</sup></em>) were 0.61 and 0.71, respectively [see <a href="#pcbi-1002720-g002">Figure 2 (D)</a>]. Given that the new OF was based on the experimental metabolome profile, we argue that our <em>in silico</em> analysis moves towards a more accurate and improved description of metabolic activity during bacterial nitrogen fixation.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[7]">In order to evaluate the metabolic implications of these results and compare our results with previous reports carried out for bacterial nitrogen fixation by <em>R. etli</em>, we report in <a href="#pcbi-1002720-t001">Table 1</a> the consistency coefficients and the degrees of coverage obtained in each case. Here, the degree of coverage is defined as the fraction of genes (proteins), from the total number of genes (proteins) obtained from constraint-based modeling, that were detected by transcriptome or proteome technology (see the data reported in reference <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>). As <a href="#pcbi-1002720-t001">Table 1</a> shows, the values of the consistency coefficients for genes and proteins had slight variations in each case. However, as <a href="#pcbi-1002720-t001">Table 1</a> chronologically shows, the OF has systematically improved two variables: the total number of genes (proteins) obtained from constraint-based modeling, and the subset of genes (proteins) belonging to this group detected by transcriptome or proteome technologies. Thus, the OF reconstructed in this work led us to increase the degree of coverage compared with previous cases: 72 of 101 enzymes predicted <em>in silico</em> were experimentally identified with high-throughput technologies. In agreement with this finding, 187 genes were experimentally identified by high-throughput technologies from the 306 genes predicted in the <em>in silico</em> analysis. In general terms, we conclude that while the consistency coefficients slightly varied among them, the OF constructed here induced the activity of additional reactions for the biosynthesis of new metabolites in bacterial nitrogen fixation. We consider that the latter issue is a crucial step in moving toward a more accurate description for characterizing and understanding the metabolic activity supporting bacterial nitrogen fixation and eventually uncovering their fundamental organizational principles at this biological level.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/table-wrap[1]"><a name="pcbi-1002720-t001" id="pcbi-1002720-t001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.t001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.t001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.t001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/table-wrap[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.t001');"><span>Table 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/table-wrap[1]/caption[1]/title[1]">Comparative analysis between <em>in silico</em> results and high-throughput data for the three OFs representing bacterial nitrogen fixation in <em>R. etli</em>.</span></strong></p><span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002720.t001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002720.t001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.t001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002720.t001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.t001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002720.t001.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/title[1]">Structural modules and their functional compositions</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[1]">The predominance of a hierarchical organization in a metabolic network seems to have advantages that can be observed on short-term and longer-term time scales, because structural modularity has been suggested as a fundamental network property by which cells evolve and orchestrate their physiological responses <a href="#pcbi.1002720-Ravasz1">[3]</a>, <a href="#pcbi.1002720-Ravasz2">[4]</a>, <a href="#pcbi.1002720-Han1">[20]</a>. In order to elucidate the relation between structural modules and biological functionalities in the metabolic activity of <em>R. etli</em>, we focused first on identification of the structural modules in the genome-scale metabolic reconstruction for <em>iOR450</em> and, consecutively, analyzed the biological roles of their metabolic components.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[2]">In terms of the first issue, modular structures were identified by considering a pure topological criterion, which has been suggested as a useful method for surveying the organization in a biological network <a href="#pcbi.1002720-Ravasz2">[4]</a>, <a href="#pcbi.1002720-ResendisAntonio1">[5]</a>. Hence, as we explained in the <a href="#s4">Materials and Methods</a> section, we defined a metric of closeness for each pair of metabolites in the network by calculating the inverse square of the minimal path length between them. Consequently, by taking into account that the metabolites integrating a module are those whose numerical values of the metric tend to be similar, we classified those metabolites with a similar pattern of closeness through a hierarchical clustering analysis (see the <a href="#s4">Materials and Methods</a> section). As a result of this analysis, nine topological modules were identified over the entire metabolic reconstruction [see <a href="#pcbi-1002720-g003">Figure 3 (A) and (B)</a>].</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]"><a name="pcbi-1002720-g003" id="pcbi-1002720-g003" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g003');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g003');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.g003&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g003');"><span>Figure 3. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/title[1]">Modularity in the metabolic reconstruction for <em>R. etli</em>.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/p[1]">The structural modules identified in the entire metabolic reconstruction are shown in panels (A) and (B).The color bar on the right indicates the different modules identified by their topological closeness in their components. The functional compositions of each structural module are shown in (C). In panel (D) we present the relative increment (white) or decrement (black) on concentration for those metabolites experimentally detected along the nine functional modules.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002720.g003</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002720.g003/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g003/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002720.g003.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g003/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002720.g003.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[3]">Even though the previous finding supplied information about the metabolic organization underlying the reconstruction, it was necessary to carry out a functional analysis to determine how these modules can be linked and support a specific metabolic phenotype. To this end, we proceeded to characterize the biological role of the set of metabolites that comprised each topological module defined in <a href="#pcbi-1002720-g003">Figure 3(B)</a>. The functional composition of each module was determined by identifying the biological roles of their components, which in turn were defined according to the classification reported in the KEGG database <a href="#pcbi.1002720-Kanehisa1">[6]</a>. By considering only those metabolites appearing in the KEGG database classification, we noted that the identified metabolites fell into three main biological groups: nucleic acids, peptides, and lipids [see <a href="#pcbi-1002720-g003">Figure 3(C)</a>]. This functional analysis suggested that most of the modules were formed by a heterogeneous composition among the three biological functionalities described above; however, we noted that two modules were characterized by a well-defined functional activity, <em>i.e.</em>, production of nucleic acids and peptide groups [see modules one and seven in <a href="#pcbi-1002720-g003">Figure 3 (C)</a>].</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[4]">Intuitively, one can expect that the metabolites integrating a functional module can be characterized by their coordinate response to physiological changes in such a way that the relative concentrations of the metabolites comprising the modules be up or down-regulated in a coherent fashion under both physiological conditions. To qualitatively assess this assumption, we first identified those metabolites that were experimentally detected by high-throughput technology in each modular structure defined in <a href="#pcbi-1002720-g003">Figure 3(B)</a>. Consequently, in order to estimate the coordinate behavior of metabolites inside modules, we evaluated their relative changes under the two physiological conditions: nitrogen fixation versus free-living conditions. Notably, most of the metabolites experimentally detected suggested that components inside modules up-regulate their concentrations during bacterial nitrogen fixation compared to free-living conditions [see <a href="#pcbi-1002720-g003">Figure 3(D)</a>]. This finding supports the intuitive idea that coherent activity occurs inside the metabolic modules reported in <a href="#pcbi-1002720-g003">Figure 3 (B)</a>. However, converse to this global trend, we observed a few metabolites with an opposite behavior [see the black lines in <a href="#pcbi-1002720-g003">Figure 3 (D)</a>]. These components are potentially central metabolites with a specific regulatory control that is perhaps required for transforming the functional background during bacterial nitrogen fixation, a hypothesis to be evaluated in future studies. A list of metabolites integrating each of the structural modules, those metabolites that were experimentally detected by high-throughput technology, the ratio obtained under the two physiological conditions, and their functional classifications according to the KEGG database are depicted in <a href="#pcbi.1002720.s002">Dataset S2</a> in the supporting information.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/title[1]">Metabolic flux activity and functional modules during bacterial nitrogen fixation</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[1]">The metabolic profile in <a href="#pcbi-1002720-g001">Figure 1</a> represents valuable data for surveying how metabolites are organized during bacterial nitrogen fixation on a network scale. In particular, modular organization in biological systems has been suggested as a common property in biological networks at different biological levels <a href="#pcbi.1002720-Ravasz2">[4]</a>, <a href="#pcbi.1002720-ResendisAntonio1">[5]</a>. Here, as discussed above, we have supplied evidence that structural modules for the metabolic reconstruction of <em>R. etli</em> can be considered a potential organizing principle by which the bacterium orchestrates its physiological response during nitrogen fixation (see <a href="#pcbi-1002720-g003">Figure 3</a>). However, these modules were identified through a static description, in which physiological information associated with a specific environment was completely absent. In order to study how the topological modules described in <a href="#pcbi-1002720-g003">Figure 3</a> can work together to support an optimal phenotype for <em>R.etli</em>, we applied FBA to simulate the metabolic flux activity during bacterial nitrogen fixation. The aims in this section are 2-fold: 1) identification of the metabolic flux profile that acts during bacterial nitrogen fixation and to evaluate to what degree the components of the structural modules participate in supporting maximal nitrogen fixation, and 2) evaluation of the extent to which the metabolic array required for reaching a maximal phenotype for bacterial nitrogen fixation is robust under perturbations of the physiological conditions.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[2]">As we described above, data-driven reconstruction of the OF represents a significant contribution to simultaneously improving the predictive scope of constraint-based modeling and unveiling the structural organization for cell metabolism. Hence, we proceeded to apply FBA to the metabolic reconstruction for <em>R.etli</em> to obtain the flux metabolic distribution that maximizes the metabolome-driven OF described above. The solution was found by solving the linear optimization problem, which was subject to thermodynamic and enzymatic constraints, over the entire set of biochemical reactions in the metabolic reconstruction (see the <a href="#s4">Materials and Methods</a> section and <a href="#pcbi.1002720.s003">Dataset S3</a> in the supporting information). Having applied FBA, we identified those reactions required for optimizing nitrogen fixation and, with them, reconstructed a subnetwork involving only their corresponding substrates and products (see <a href="#s4">Materials and Methods</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[3]">To dissect the functional participation of the metabolites integrating this subnetwork, we took into account the modular classification previously defined in <a href="#pcbi-1002720-g003">Figure 3(B)</a> and <a href="#pcbi.1002720.s002">Dataset S2</a> in the supporting information. As expected, we identified the metabolites that potentially participate in support of bacterial nitrogen fixation, their biological roles, and their distributions along the modules (see <a href="#pcbi-1002720-g004">Figure 4 (A)</a> and the <a href="#s4">Materials and Methods</a> section).The metabolic properties inferred from this subnetwork were such that 47.96% (47 of 98) of the metabolites predicted by constraint-based modeling were experimentally detected inside the bacteroids for <em>R.etli</em>. Even though this percentage of alignment with computational modeling is relatively low, this finding represents a significant advance towards a more realistic method for the study of the metabolic activity of bacterial nitrogen fixation. As far as our knowledge extends, this metabolome study is the first performed for <em>Rhizobiaceas</em> in such a way that it defines a benchmark point for future improvements. According to the functional classification described in <a href="#pcbi-1002720-g003">Figure 3(A)</a> and in <a href="#pcbi.1002720.s002">Dataset S2</a> at supporting information, our <em>in silico</em> analysis suggested that nitrogen fixation requires a variety of metabolites, mostly peptides and lipids, belonging to diverse structural modules. Converse to the idea that all the metabolites in a specific module participate during this biological process, we found that a metabolic heterogeneity—in terms of both the number of components and module classification—is required for optimizing bacterial nitrogen fixation in <em>R.etli</em>.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]"><a name="pcbi-1002720-g004" id="pcbi-1002720-g004" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g004" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g004');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g004');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.g004&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g004" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g004');"><span>Figure 4. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/title[1]">Robust modular composition at different uptake rates of carbon sources.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/p[1]">Figure (A) represents the fraction of metabolites required to optimize bacterial nitrogen fixation induced from FBA. Region (B) denotes the phenotype phase plane when inositol and succinate uptake rates were varied from 0 to 20 nM/gDW/hr. The black and white regions indicate low and high nitrogen fixation activities, respectively. Figures (C–F) denote the topological structures for metabolism that ensure optimal nitrogen fixation in the bacteria under different conditions of succinate and inositol uptake rates. Overall, we selected 20 different physiological conditions, corresponding to the diagonal line in the phenotype phase plane that is shown in (B).</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002720.g004</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002720.g004/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g004/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002720.g004.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g004/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002720.g004.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[4]">Notably, the subnetwork depicted in <a href="#pcbi-1002720-g004">Figure 4(A)</a> shows a few metabolites that participate in the synthesis of nucleic acids (dark green dots). These findings are in qualitative agreement with the fact that <em>R. etli</em> bacteroids do not grow during symbiotic nitrogen fixation with <em>P. vulgaris</em> (bean plant). In order to distinguish the modules obtained from pure topological criteria from modules specific for a physiological condition, here after we denote these latter as functional modules. At this stage, a question that immediately emerges is to what extent the functional modules depicted in <a href="#pcbi-1002720-g004">Figure 4 (A)</a> are robust under external environmental perturbations, and also whether this network array can be used as a fingerprint to characterize a functional phenotype in bacterial nitrogen fixation.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/title[1]">Robustness of the metabolic profile underlying functional states</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[1]">For the purpose of evaluating whether the metabolic organization shown in <a href="#pcbi-1002720-g004">Figure 4(A)</a> can be used as a fingerprint to define the optimal phenotype for bacterial nitrogen fixation in <em>R. etli</em>, we explored to what extent this topological structure is robust during changes under external environmental conditions. To this end, we evaluated the robustness of this metabolic array through the systematic reduction of uptake rates of two carbon sources: succinate and inositol. While the plant supplies succinate to the bacteroid as the main carbon source, inositol is an internal metabolite that has been detected at high concentrations inside bacteroids in nodules. As has been experimentally verified, both metabolites are important components in the support of nitrogen fixation in <em>Rhizobiaceas</em> <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>. To explore how the availability of inositol and succinate alter the metabolic phenotype shown in <a href="#pcbi-1002720-g003">Figure 3(A)</a>, we constructed a phenotype phase plane over these carbon sources [see <a href="#pcbi-1002720-g004">Figure 4 (B)</a>]. In agreement with the physiological knowledge for <em>Rhizobiaceas</em>, we observed that according to the uptake rate, carbon sources in bacterial metabolism were reduced, and a reduction effect on nitrogen fixation was obtained [see the black region of low nitrogen fixation in <a href="#pcbi-1002720-g004">Figure 4(B)</a>]. To evaluate the topological changes at different points in the phenotype phase plane, <em>i.e.</em>, with different succinate and inositol uptake rates, we selected a subset of 20 points along the metabolic phase plane. As depicted by the diagonal line in <a href="#pcbi-1002720-g004">Figure 4(B)</a>, uptake rates for both carbon sources were selected such that their fluxes simultaneously decreased from 20 to 0 nmol of gDW/hr. Then, for each one of the selected uptake rate conditions, we graphically represented the metabolic subnetwork that resulted from FBA. As described in the <a href="#s4">Materials and Methods</a> section, these subnetworks were constructed by considering only those metabolites participating in the biochemical reactions required for optimization of the phenotype under defined conditions of succinate and inositol uptake rates [see <a href="#pcbi-1002720-g004">Figure 4 (C–F)</a>]. To quantify the topological variations that resulted under different external conditions, we defined an overlapping coefficient as the fraction of metabolites that overlap these networks (see the <a href="#s4">Materials and Methods</a> section). With the purpose of quantifying the differences among all pairs of subnetworks, we constructed an overlapping matrix whose calculated numerical entries indicated the overlapping coefficient for each pair of subnetworks obtained at different succinate and inositol uptake rates. The comparative analysis over the 20 points depicted in <a href="#pcbi-1002720-g004">Figure 4(B)</a> is shown in <a href="#pcbi-1002720-g005">Figure 5</a>.This study led us to conclude that the metabolic profile required for optimizing bacterial nitrogen fixation does not change in a significant way for a wide range of uptake rates for either carbon source [see <a href="#pcbi-1002720-g005">Figure 5 (A–B)</a>]. Hence, our study supplies evidence that, while limited carbon sources reduce the bacterial phenotype, the metabolic organization supporting bacterial nitrogen fixation tends to be robust at different succinate and inositol uptake rates. In other words, even though there was a significant reduction for a phenotype, the topology of the metabolic network did not change in a significant fashion. In light of these results, we concluded that functional nitrogen fixation, at low or high rates, seems to be achieved through a conservative metabolic profile involving those metabolites required for support of optimal nitrogen fixation under specific environmental conditions (see <a href="#pcbi-1002720-g005">Figure 5</a>). Notably, this property opens the possibility for the use of network topology for characterizing cellular phenotypes through a specific pattern of metabolites and potentially distinguishing functional from dysfunctional states associated with a specific biological system. This latter approach is an avenue to be addressed in the future.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/fig[1]"><a name="pcbi-1002720-g005" id="pcbi-1002720-g005" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g005" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g005');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g005');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.g005&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002720?imageURI=info:doi/10.1371/journal.pcbi.1002720.g005" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002720', 'info:doi/10.1371/journal.pcbi.1002720.g005');"><span>Figure 5. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/fig[1]/caption[1]/title[1]">Matrix of overlapping networks.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/fig[1]/caption[1]/p[1]">Figure (A) graphically depicts the degree of similitude between the two subnetworks represented in <a href="#pcbi-1002720-g004">Figure 4</a> (C–G). In this matrix, an entry located at row <em>i</em> and column <em>j</em> indicate the comparative analysis between any pair of metabolic subnetworks, <em>i</em> = 1.20 and <em>j</em> = 1.20. For each entry, white and black indicate the highest and smallest degree of similitude (overlapping) between two subnetworks, respectively. Intermediary values are represented in gray. The degree of overlapping obtained between the three subnetworks (indicated in red) with the rest of the subgraphs depicted in <a href="#pcbi-1002720-g004">Figure 4</a> are shown in (B). As panel (B) shows, most of the subgraphs are similar around its vicinity in the phase plane, and an overlap of 0.83 was obtained for subnetworks obtained with extreme points in fluxes in the phase plane.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002720.g005</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002720.g005/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g005/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002720.g005.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002720.g005/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002720.g005.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[2]">Converse to the naive idea that a reduced phenotype in bacterial nitrogen fixation is the consequence of a broken modularity in metabolism, our study supplies evidence that a reduced efficiency of a phenotype can be mainly a consequence of a decrement in flux activity along the network, but without a significant rupture of the functional modules defined in <a href="#pcbi-1002720-g004">Figure 4(A)</a>. In this contextual scheme, modular organization in metabolism seems to be a necessary condition for supporting not only the maximal but a suboptimal functional phenotype in bacterial nitrogen fixation (see <a href="#pcbi-1002720-g004">Figure 4</a>).</p>

</div>

<div id="section3" xpathLocation="/article[1]/body[1]/sec[3]"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3 xpathLocation="noSelect">Discussion&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[3]/p[1]">A description of the integration between high-throughput data and computational modeling is a central issue in systems biology that is required for moving toward a quantitative and predictive analysis of the metabolic activity in microorganisms. This enterprise has relevant implications, not only in solving practical issues in biotechnology but also in supplying schemes that contribute to unraveling the principles and mechanisms that regulate and organize living systems.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[2]">In this work, we have explored the relationships between three biological concepts in metabolic networks: structural modularity, biological functionality, and optimal (maximal) phenotype. Here, the relationships among these concepts were studied at a systems level for metabolism in bacterial nitrogen fixation, an important biological process participating in the balance of nitrogen in the biosphere <a href="#pcbi.1002720-ResendisAntonio3">[19]</a>. Hence, by using constraint-based modeling and measuring metabolome data under two physiological conditions, we explored the metabolic organization in <em>R.etli</em> bacteroids while they fix nitrogen in symbiotic association with <em>P. vulgaris</em> (bean plants).</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[3]">Unlike our previous study <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>, here we present a refined version of the OF used for simulating bacterial nitrogen fixation based on our taking into account metabolome data. When constraint-based modeling was applied in this new context, we concluded that 71% of the metabolic reactions predicted <em>in silico</em> were justified by the proteome and microarray data previously stored in the GEO database [see <a href="#pcbi-1002720-g002">Figure 2(D)</a> and the <a href="#s4">Materials and Methods</a> section].</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[4]">On the other hand, based on the metabolic reconstruction for <em>R.etli</em>, we identified nine structural modules in which some of the metabolites components fell in one of the following biological roles: nucleic acids, peptides or lipids (see <a href="#pcbi.1002720.s002">Dataset S2</a> in supporting information). Furthermore, by considering the metabolome profile of 220 metabolites under two physiological conditions, we supplied evidence that most of the metabolites integrating each one of these modules works in a coordinated fashion by increasing their metabolite concentrations at nitrogen fixation stages [see <a href="#pcbi-1002720-g003">Figure 3 (D)</a>]. This latter finding supports the idea that metabolites inside a module tend to respond in a coordinated fashion. This systems-level description supplies evidence that the diverse metabolites forming the modules participate to support an optimal phenotype in bacterial nitrogen fixation. Even more fundamentally, we note that the network representation for those arrays of metabolites associated with an optimal metabolic performance is robust under different external conditions [see <a href="#pcbi-1002720-g004">Figures 4</a> and <a href="#pcbi-1002720-g005">5</a>].</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[5]">In light of this study's findings, the main contributions can be summed up as follows:1) we have supplied evidence that a robust modular organization at the metabolic level underlies optimal bacterial nitrogen fixation for <em>R. etli</em>; 2) we have proposed how these functional modules interact together for supporting bacterial nitrogen fixation; and 3) given the robustness observed for these functional modules when there are physiological changes, we suggest that these can be used as fingerprints to associate the active topological structure in a network with optimal phenotypic behavior. Why the metabolic activity is so robust under environmental changes?, maybe a common explanation can be found in other complex systems, for instance the social organization in factories, where even though the production of cars can change as a consequence of external factors—as low or high demand—the hierarchical and modular organization among employees is still maintained for covering a specific demand in an efficient way.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[6]">Finally, even though a variety of avenues should be addressed in future research to enrich this study, we envision that this unified description will contribute to the design of experiments for evaluating the interrelation and the role that these modules have in supporting functional states in bacterial nitrogen fixation. This latter issue is fundamental for uncovering the principles by which the cell organizes its metabolism to support functional states in bacterial nitrogen fixation.</p>
</div>

<div id="section4" xpathLocation="/article[1]/body[1]/sec[4]"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3 xpathLocation="noSelect">Materials and Methods&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/title[1]">Bacterial strains, growth conditions, and plant experiments</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]">The bacterial strain used was <em>R. etli</em> CFN42 wild type. Culture media and growth conditions for <em>R. etli</em> and plant experiments were performed as previously described by our group in reference <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/title[1]">Constraint-based modeling</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/p[1]">Metabolic flux distribution supporting nitrogen fixation in <em>R.etli</em> was predicted <em>in silico</em> by using constraint-based modeling <a href="#pcbi.1002720-ResendisAntonio3">[19]</a>. Briefly, simulations were carried out by defining a mathematical function, called the OF, for computationally mimicking the metabolism of bacterial nitrogen fixation and identifying the flux distribution that maximizes it at a steady-state behavior. The OF, <em>Z<sup>Fix</sup></em>, is composed of some key compounds, which are classified in two groups: 1) metabolites essential for sustaining nitrogen fixation, and 2) metabolites that are imported or exported to the bacteroid and establish the symbiotic relationship between <em>R. etli</em> and the plant (these are denoted by the index [e]). Thus, we write the OF as follows:<br><a name="pcbi.1002720.e001" id="pcbi.1002720.e001"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.e001&amp;representation=PNG"></span><br>where glycogen, histidine, lysine, polyhydroxybutyrate, valine, alanine, aspartate, and ammonium are denoted as <em>glycogen</em>, <em>hist</em>[c], <em>lys</em>, <em>phb</em>[c], <em>val[c]</em>, <em>ala</em>[e], <em>asp</em>[e], and <em>nh4</em>[e], respectively. Similarly, <em>mal</em>, <em>trp</em>, <em>arg</em>, <em>cit</em>, <em>cmp</em>, <em>fum</em>, <em>3pg</em>, and <em>akg</em> denote malate, tryptophan, arginine, citrate, CMP, fumarate, 3-phospho-<span class="small-caps">d</span>-glycerate, and 2-oxoglutarate, respectively. All these metabolites are required to support an effective symbiotic nitrogen fixation, and their spatial location in the cytoplasm is indicated by the index [c]. For the purpose of obtaining a computational profile of metabolic fluxes, we assumed that the metabolic state of the bacteroid during nitrogen fixation is one that optimizes the OF, <em>Z<sup>Fix</sup></em>. This problem was mathematically solved as a linear optimization algorithm subject to enzymatic and thermodynamic constraints, <em>i.e.</em>,<br><a name="pcbi.1002720.e002" id="pcbi.1002720.e002"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.e002&amp;representation=PNG"></span><br>where <em>S<sub>i,j</sub></em> represents the stoichiometric coefficient of metabolite <em>i</em> participating in the <em>j</em>th reaction. Thermodynamic and enzymatic constraints in each metabolic reaction were characterized through the parameters <em>α<sub>j</sub></em> and <em>β<sub>j</sub></em> (for a detailed numerical description of these parameters, see the lower and upper bounds in <a href="#pcbi.1002720.s003">Dataset S3</a> in supporting information). For the sake of simplicity, all the coefficients (<em>c<sub>i</sub></em>) were chosen as a unit along all the analyses. Linear optimization programming was carried out using the Tomlab optimization package from Matlab.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/title[1]">Metabolome data and inference of the OF for bacterial nitrogen fixation: statistical test of metabolite relative expression levels</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/p[1]">Metabolome data supply valuable information to survey the metabolic phenotype associated with a biological process, and in turn, elucidate the components integrating the OF used in the constraint-based modeling. With the purpose of carrying out FBA in the metabolic reconstruction for <em>R.etli</em>, a more realistic OF for simulating bacterial nitrogen fixation was obtained by identifying those metabolites with a statistically significant change during bacteroid activity compared to levels under free-living conditions. To this end, we calculated the corresponding intensity log-ratio for each metabolite between nitrogen fixation and free-living conditions. Taking into account the three experimental replicates obtained for each physiological condition, a one-side statistical <em>t</em>-test was applied to determine those metabolites that significantly increased their relative quantity between the two physiological conditions (see <a href="#pcbi-1002720-g001">Figure 1</a>). Those metabolites with a <em>p-value</em> lower than 0.01 and with a ratio higher than 2 during nitrogen fixation were selected as potential candidates to be included in the OF, see region I in <a href="#pcbi-1002720-g002">Figure 2(A)</a>. Finally, these sets of metabolites were separately included in the OF and were accepted if their effect on nitrogen fixation activity was not reduced to one-third of the corresponding previous OF.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/title[1]">Transcriptome and proteome data</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/p[1]">To assess the predictive scope of the computational model, we downloaded transcriptome and proteome data so that we could integrate a set of genes whose presence and upregulation suggested an important role for sustaining nitrogen fixation in <em>R.etli</em>. Earlier characterizations of nodules formed after 18 days of nodulation with root plants under both experimental conditions were carried out by our group and previously stored in public databases. The complete dataset of the transcriptome analysis is freely available at GEO (<a href="http://www.ncbi.nlm.nih.gov/geo">http://www.ncbi.nlm.nih.gov/geo</a>), with accession numbers GPL10081 for the <em>R.etli</em> platform and GSE21638 for data on free-living and symbiotic forms. In addition, to enrich the set of genes required for model assessment, proteome data obtained for <em>R.etli</em> bacteroids were downloaded from the ProteomeCommons.org Tranche by using the following hash:BY/eCcVjwTWN1+m+2ArvJ0QVnesGx5Ekgd4​wUOASACfm/ueNl7YI3iLf4xz0lnGsepV5LkpMWOQ​OrZtjYExlNpQkIBcAAAAAAAABjA=  = .</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/title[1]">Measurement of ionic metabolites by using the CE-MS system</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[1]">Bacteria were grown in minimal medium and were harvested at the exponential phase. Free-living cells were collected by centrifugation and washed once with double-distilled water, and immediately 2 ml of methanol and the internal standards were added and the mixture was treated in an ultrasonic bath for 30 s. A 1.6-ml cell suspension was transferred to centrifuge tubes,1.6 ml of CHCl<sub>3</sub> plus 640 µl of milliQ water was added, and the mixture was vortexed and then centrifuged at 2,300×<em>g</em>at 4°C for 5 min. A 1.5-ml aliquot of the aqueous layer was filtered through a Millipore 5-kDa cutoff filter. Then, the sample was fully dried by using a centrifugal evaporator. The bacteroid metabolome extraction followed the same method, except by day 18 after inoculation with root plant, bacteroids were extracted in a self-Percoll gradient in accordance with methods previously described <a href="#pcbi.1002720-Reibach1">[21]</a>. Measurements of extracted metabolites were performed by using CE coupled with electrospray ionization–time-of-flight analysis and MS with electrophoresis buffer (solution ID H3302-1021; Human Metabolome Technologies Inc., Tsuruoka, Japan).</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/title[1]">Definition of the consistency coefficient</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[1]">To assess agreement between <em>in silico</em> predictions and interpretations of high-throughput data, we defined a consistency coefficient that quantified the fraction of genes predicted to be upregulated <em>in silico</em> and simultaneously detected or induced by proteome or transcriptome technologies (<em>η<sub>Gene</sub></em>). Simultaneously, we defined a consistency coefficient that quantified the fraction of active enzymes that were predicted by constraint-based modeling and identified by high-throughput technology (<em>η<sub>Enzyme</sub></em>). To proceed with this evaluation, <em>E <sup>j</sup><sub>kegg</sub></em>(<em>G <sup>j</sup><sub>kegg</sub></em>) was denoted as the set of enzymes (genes) that conformed to the <em>j</em>-esime metabolic pathways in the KEGG database, with <em>j</em> ranging from 1 to 22. Similarly, the set of enzymes (genes) that integrated the <em>i</em>-esime metabolic pathway in the reconstruction and the set of enzymes that were detected by high-throughput data were denoted <em>E <sup>j</sup><sub>Rec</sub></em>(<em>G <sup>j</sup><sub>Rec</sub></em>)and <em>E <sup>j</sup><sub>HT</sub></em>(<em>G <sup>j</sup><sub>HT</sub></em>), respectively. Finally, the set of enzymes obtained from constraint-based modeling were denoted <em>E <sup>j</sup><sub>iModel</sub></em> and<em>G <sup>j</sup><sub>iModel</sub></em>. In order to evaluate and create a proper frame of comparison between <em>in silico</em> predictions and high-throughput data, we defined the consistency coefficient as the fraction of enzymes (genes) that were actively predicted <em>in silico</em> and were identified by high-throughput technology as follows:<br><a name="pcbi.1002720.e003" id="pcbi.1002720.e003"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.e003&amp;representation=PNG"></span><br>This ratio ranged from 0 to 1 and constituted our central parameter to assess and quantify the agreement between the constraint-based modeling and high-throughput data. To assess the output from FBA, we selected 22 metabolic pathways and evaluated the degree of coherence between the flux activity predicted <em>in silico</em> with data for the proteome and transcriptome previously reported for <em>R.etli</em> during nitrogen fixation (see reference <a href="#pcbi.1002720-ResendisAntonio2">[11]</a>).</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[7]/title[1]">Network representation of the metabolic network</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[7]/p[1]">Topological analysis was accomplished by representing the set of biochemical reactions in the metabolic reconstruction as an undirected network. In this network, nodes represent the metabolites and edges indicate their participation in a specific metabolic reaction <a href="#pcbi.1002720-Ravasz2">[4]</a>. Thus, we linked the metabolites in the reactants to all the products in the biochemical reaction, and this procedure was repeated for all the reactions included in the metabolic reconstruction, to finally obtain the results shown in <a href="#pcbi-1002720-g003">Figure 3(B)</a>. In the case of the FBA, the subnetworks depicted in <a href="#pcbi-1002720-g004">Figure 4(A), (C–F)</a> were obtained when we applied these rules only to those metabolic reactions for which the flux obtained from FBA was different from 0. In order to analyze those metabolites with biological roles, the following metabolites were excluded from the entire network representation: pi[e], ala-L[e], accoa[e], glu-L[e], pi[c], fdp[c], nh4[e], nh3[c], nh3[e], h[e], ppi[c], o2[c], o2[e], nadph[c], nadp[c], nadh[c], nad[c], n2[c], n2[e], h2o2[c], h2o[c], h2o[e], h[c], fdred[c], fdox[c], fadh2[c], co2[c], co2[e], nh4[c], and fad[c]. A detailed list of the metabolic reconstruction <em>iOR450</em> used in this work, including the metabolic reactions and metabolite abbreviations, is provided in <a href="#pcbi.1002720.s003">Dataset S3</a> in supporting information.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[8]/title[1]">Module identifications</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[8]/p[1]">Modular composition along the entire metabolic reconstruction was obtained by a clustering analysis applied on a matrix whose entries represented the inverse squares of minimal path lengths for pairs. Specifically, the clustering analysis was performed by calculating the shortest path length between every pair of genes (<em>i.e</em>.,<em>d<sub>ij</sub></em> is the shortest path length between gene <em>i</em> and gene <em>j</em>). Next, we calculated the association function, defined as 1/<em>dij</em><sup>2</sup>, for all pairs of metabolites along the metabolic reconstruction. This parameter gives a measure of the closeness among genes, amplifying the parameter for pairs of metabolites with low path lengths and minimizing pairs of metabolites located at remote distances. With the purpose of identifying topological modules along the network, these sets of parameters were used as input to perform a hierarchical clustering <a href="#pcbi.1002720-Ravasz1">[3]</a>, <a href="#pcbi.1002720-Ravasz2">[4]</a>, <a href="#pcbi.1002720-ResendisAntonio1">[5]</a>, <a href="#pcbi.1002720-Eisen1">[22]</a>, see <a href="#pcbi-1002720-g002">Figure 2(A)</a>. A detailed description of the modules identified by this algorithm and the corresponding biological roles for some of the components are shown in <a href="#pcbi.1002720.s002">Dataset S2</a>. The clustering analysis was performed using a hierarchical agglomerative average-linkage clustering algorithm, considering Kendall'sτ value as the similarity metric.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[9]/title[1]">Overlapping coefficient among modules</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[9]/p[1]">In order to compare the similarities and differences that emerged from the functional modules obtained along the phase plane region depicted in <a href="#pcbi-1002720-g004">Figure 4</a>, we define the following coefficient:<br><a name="pcbi.1002720.e004" id="pcbi.1002720.e004"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.e004&amp;representation=PNG"></span><br>where <em>η<sup>i</sup></em> and <em>η<sup>j</sup></em> represent the number of metabolites that appeared in only module <em>i</em> or <em>j</em>, respectively, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002720.e005&amp;representation=PNG"></span> indicates the number of metabolites in common between modules <em>i</em> and <em>j</em>. Clearly, this coefficient ranges from 1 to 0,depending on the complete or null analogy between the <em>i</em> and <em>j</em> functional subnetworks.</p>

</div>

<div id="section5" xpathLocation="/article[1]/body[1]/sec[5]"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3 xpathLocation="noSelect">Supporting Information&nbsp;<a href="#top">Top</a></h3><a name="pcbi.1002720.s001" id="pcbi.1002720.s001"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002720.s001">Dataset S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[1]"><strong>Metabolome data.</strong> This file contains the metabolome data set obtained experimentally through CE-MS. The table shows those metabolites whose concentrations were differentially up-regulated and down-regulated during bacterial nitrogen fixation respect to free-living conditions. In addition, we report those metabolites experimentally detected only in one of the physiological states analyzed.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002720.s002" id="pcbi.1002720.s002"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002720.s002">Dataset S2. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[1]"><strong>Metabolites integrating each structural module.</strong> This table contains information of the metabolites forming each one of the nine structural modules identified by the topological criteria used in the main text.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002720.s003" id="pcbi.1002720.s003"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002720.s003">Dataset S3. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[1]"><strong>Genome-scale metabolic reconstruction for </strong><strong><em>Rhizobium etli</em></strong><strong> (</strong><strong><em>iOR450</em></strong><strong>).</strong> This file contains the entire set of biochemical reactions included in the metabolic reconstruction for <em>R. etli</em>. Overall, the metabolic reconstruction of <em>R. etli</em> is an integrated network of 402 reactions involving the participation of 450 genes and 377 metabolites.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[2]">(XLSX)</p>
</div>





<div xpathLocation="noSelect"><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3 xpathLocation="noSelect">Acknowledgments&nbsp;<a href="#top">Top</a></h3>
<p xpathLocation="/article[1]/back[1]/ack[1]/p[1]">OR-A appreciates the valuable comments, encourage and guidance from Prof. Bernhard Ø. Palsson during the progress of this study. Finally, the authors appreciate the valuable comments and suggestions from two anonymous referees during the review process.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3 xpathLocation="noSelect">Author Contributions&nbsp;<a href="#top">Top</a></h3><p xpathLocation="noSelect">Conceived and designed the experiments: ORA SE. Performed the experiments: ORA SE MH YM. Analyzed the data: ORA. Contributed reagents/materials/analysis tools: ORA SE MH YM. Wrote the paper: ORA.</p></div><div xpathLocation="noSelect"><a id="references" name="references" toc="references" title="References"></a><h3 xpathLocation="noSelect">References&nbsp;<a href="#top">Top</a></h3><ol class="references" xpathLocation="noSelect"><li xpathLocation="noSelect"><a name="pcbi.1002720-Hartwell1" id="pcbi.1002720-Hartwell1"></a>Hartwell LH, Hopfield JJ, Leibler S, Murray AW (1999) From molecular to modular cell biology. Nature 402: C47–52. <a class="find" href="/article/findArticle.action?citedArticleID=5872023">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Dong1" id="pcbi.1002720-Dong1"></a>Dong J, Horvath S (2007) Understanding network concepts in modules. BMC Syst Biol 1: 24. <a class="find" href="/article/findArticle.action?citedArticleID=5872025">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Ravasz1" id="pcbi.1002720-Ravasz1"></a>Ravasz E (2009) Detecting hierarchical modularity in biological networks. Methods Mol Biol 541: 145–160. <a class="find" href="/article/findArticle.action?citedArticleID=5872027">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Ravasz2" id="pcbi.1002720-Ravasz2"></a>Ravasz E, Somera AL, Mongru DA, Oltvai ZN, Barabasi AL (2002) Hierarchical organization of modularity in metabolic networks. Science 297: 1551–1555. <a class="find" href="/article/findArticle.action?citedArticleID=5872029">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-ResendisAntonio1" id="pcbi.1002720-ResendisAntonio1"></a>Resendis-Antonio O, Freyre-Gonzalez JA, Menchaca-Mendez R, Gutierrez-Rios RM, Martinez-Antonio A, et al. (2005) Modular analysis of the transcriptional regulatory network of E. coli. Trends Genet 21: 16–20.
        doi:
        <a href="http://dx.doi.org/10.1016/j.tig.2004.11.010">10.1016/j.tig.2004.11.010</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5872031">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Kanehisa1" id="pcbi.1002720-Kanehisa1"></a>Kanehisa M, Goto S (2000) KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res 28: 27–30. <a class="find" href="/article/findArticle.action?citedArticleID=5872033">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Orth1" id="pcbi.1002720-Orth1"></a>Orth JD, Thiele I, Palsson BO (2010) What is flux balance analysis? Nat Biotechnol 28: 245–248.
        doi:
        <a href="http://dx.doi.org/10.1038/nbt.1614">10.1038/nbt.1614</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5872035">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Gianchandani1" id="pcbi.1002720-Gianchandani1"></a>Gianchandani EP, Chavali AK, Papin JA (2010) The application of flux balance analysis in systems biology. Wiley Interdiscip Rev Syst Biol Med 2: 372–382.
        doi:
        <a href="http://dx.doi.org/10.1002/wsbm.60">10.1002/wsbm.60</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5872037">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Oberhardt1" id="pcbi.1002720-Oberhardt1"></a>Oberhardt MA, Palsson BO, Papin JA (2009) Applications of genome-scale metabolic reconstructions. Mol Syst Biol 5: 320. <a class="find" href="/article/findArticle.action?citedArticleID=5872039">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Goyal1" id="pcbi.1002720-Goyal1"></a>Goyal S, Yuan J, Chen T, Rabinowitz JD, Wingreen NS (2010) Achieving optimal growth through product feedback inhibition in metabolism. PLoS Comput Biol 6: e1000802. <a class="find" href="/article/findArticle.action?citedArticleID=5872041">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-ResendisAntonio2" id="pcbi.1002720-ResendisAntonio2"></a>Resendis-Antonio O, Hernandez M, Salazar E, Contreras S, Martinez Batallar G, et al. (2011) Systems Biology of Bacterial Nitrogen Fixation: High-throughput technology and its integrative description with constraint-based modeling. BMC Syst Biol 5: 120.
        doi:
        <a href="http://dx.doi.org/10.1186/1752-0509-5-120">10.1186/1752-0509-5-120</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5872043">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Soga1" id="pcbi.1002720-Soga1"></a>Soga T (2007) Capillary electrophoresis-mass spectrometry for metabolomics. Methods Mol Biol 358: 129–137. <a class="find" href="/article/findArticle.action?citedArticleID=5872045">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Soga2" id="pcbi.1002720-Soga2"></a>Soga T, Ohashi Y, Ueno Y, Naraoka H, Tomita M, et al. (2003) Quantitative metabolome analysis using capillary electrophoresis mass spectrometry. J Proteome Res 2: 488–494. <a class="find" href="/article/findArticle.action?citedArticleID=5872047">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Newman1" id="pcbi.1002720-Newman1"></a>Newman JD, Diebold RJ, Schultz BW, Noel KD (1994) Infection of soybean and pea nodules by Rhizobium spp. purine auxotrophs in the presence of 5-aminoimidazole-4-carboxamide riboside. J Bacteriol 176: 3286–3294. <a class="find" href="/article/findArticle.action?citedArticleID=5872049">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Romanov1" id="pcbi.1002720-Romanov1"></a>Romanov VI, Hernandez-Lucas I, Martinez-Romero E (1994) Carbon Metabolism Enzymes of Rhizobium tropici Cultures and Bacteroids. Appl Environ Microbiol 60: 2339–2342. <a class="find" href="/article/findArticle.action?citedArticleID=5872051">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Feist1" id="pcbi.1002720-Feist1"></a>Feist AM, Herrgard MJ, Thiele I, Reed JL, Palsson BO (2009) Reconstruction of biochemical networks in microorganisms. Nat Rev Microbiol 7: 129–143. <a class="find" href="/article/findArticle.action?citedArticleID=5872053">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Feist2" id="pcbi.1002720-Feist2"></a>Feist AM, Palsson BO (2008) The growing scope of applications of genome-scale metabolic reconstructions using Escherichia coli. Nat Biotechnol 26: 659–667. <a class="find" href="/article/findArticle.action?citedArticleID=5872055">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Segre1" id="pcbi.1002720-Segre1"></a>Segre D, Vitkup D, Church GM (2002) Analysis of optimality in natural and perturbed metabolic networks. Proc Natl Acad Sci U S A 99: 15112–15117. <a class="find" href="/article/findArticle.action?citedArticleID=5872057">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-ResendisAntonio3" id="pcbi.1002720-ResendisAntonio3"></a>Resendis-Antonio O, Reed JL, Encarnacion S, Collado-Vides J, Palsson BO (2007) Metabolic reconstruction and modeling of nitrogen fixation in Rhizobium etli. PLoS Comput Biol 3: 1887–1895. <a class="find" href="/article/findArticle.action?citedArticleID=5872059">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Han1" id="pcbi.1002720-Han1"></a>Han JD, Bertin N, Hao T, Goldberg DS, Berriz GF, et al. (2004) Evidence for dynamically organized modularity in the yeast protein-protein interaction network. Nature 430: 88–93. <a class="find" href="/article/findArticle.action?citedArticleID=5872061">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Reibach1" id="pcbi.1002720-Reibach1"></a>Reibach PH, Mask PL, Streeter JG (1981) A rapid one-step method for the isolation of bacteroids from root nodules of soybean plants, utilizing self-generating Percoll gradients. Can J Microbiol 27: 491–495. <a class="find" href="/article/findArticle.action?citedArticleID=5872063">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002720-Eisen1" id="pcbi.1002720-Eisen1"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A 95: 14863–14868. <a class="find" href="/article/findArticle.action?citedArticleID=5872065">
                      Find this article online
                    </a></li></ol></div>

</div></div>
<div style="display:none">
<div dojoType="ambra.widget.RegionalDialog" id="AnnotationDialog" style="padding:0;margin:0;">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Post Your Note (For Public Viewing)</span></h5>
      <div class="posting pane">
        <form name="createAnnotation" id="createAnnotation" method="post" action="">
          <input type="hidden" name="target" value="info:doi/10.1371/journal.pcbi.1002720" />
          <input type="hidden" name="startPath" value="" />
          <input type="hidden" name="startOffset" value="" />
          <input type="hidden" name="endPath" value="" />
          <input type="hidden" name="endOffset" value="" />
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="false" />
          <input type="hidden" name="noteType" id="noteType" value="" />
          <fieldset>
            <legend>Compose Your Note</legend>
            <span id="submitMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td>
                  <label for="cNoteType">This is a </label><select name="cNoteType" id="cNoteType"><option value="note">note</option><option value="correction">correction</option></select>
                  <span id="cdls" style="visibility:hidden;margin-left:0.3em; white-space:nowrap;"><a href="/static/commentGuidelines.action;jsessionid=A06D857AE008E3829F545704038156D3?target=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002720#corrections">What are corrections?</a></span>
                  <label for="cTitle"><span class="none">Enter your note title</span><!-- error message text <em>A title is required for all public notes</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your note title..." class="title" alt="Enter your note title..." />
                  <label for="cArea"><span class="none">Enter your note</span><!-- error message text <em>Please enter your note</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your note..." alt="Enter your note...">Enter your note...</textarea>
                  <input type="hidden" name="isPublic" value="true" />
                </td>
                <td>&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" checked="checked" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." alt="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td colspan="3" class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel"/>
                  <input type="button" value="Submit" title="Click to post your note publicly" id="btn_post" class="primary"/>
                </td>
              </tr>
              <tr>
                <td colspan="3">
                  <p>Notes and Corrections can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p>
                </td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
    <div class="tip" id="dTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialog" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="caTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      Please follow our <a href="/static/commentGuidelines.action;jsessionid=A06D857AE008E3829F545704038156D3">guidelines for notes and comments</a> and review our <a href="/static/competing.action;jsessionid=A06D857AE008E3829F545704038156D3">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
      <ul>
        <li>Remarks that could be interpreted as allegations of misconduct</li>
        <li>Unsupported assertions or statements</li>
        <li>Inflammatory or insulting language</li>
      </ul>
      <form name="contextActionForm" id="contextActionForm" class="clearfix buttons" method="post" action="">
        <input type="button" name="Continue" value="Continue" id="ContextActionDialogContinueButton" onmouseup="ambra.displayAnnotationContext.startComment(event);" title="Add a note to this text" class="primary"/>
        <input type="button" name="Cancel" value="Cancel" id="ContextActionDialogCancelButton" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);" title="Close this Window"/>
      </form>
    </div>
    <div class="tip" id="caTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogNotLogged" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canlTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You must be logged in to add a note to an article.
      You may log in by <a onmousedown="ambra.displayAnnotationContext.disconnect(event);" href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002720">clicking here</a> or <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">cancel this note</a>.
    </div>
    <div class="tip" id="canlTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canBDTipu"></div>
    <div class="contextActionContent">
      <h5 class="annotation icon"><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot annotate this area of the document. <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canBDTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadRangeSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canbrTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot create an annotation that spans different sections of the document; please adjust your selection.<br/>
      <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canbrTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialog" style="padding:0;margin:0;">
  <div class="dialog preview">
    <div class="tipu" id="cTipu"></div>
    <div class="btn close" id="btn_close" title="Click to close"><a title="Click to close">Close</a></div>
    <div id="cmtContainer" class="comment">
      <h6 id="viewCmtTitle"></h6>
      <div class="detail" id="viewCmtDetail"></div>
      <div class="contentwrap" id="viewComment"></div>
      <div class="contentwrap" id="viewCIStatement"></div>
      <div class="detail" id="viewLink">
        <!--<a href="#" class="commentary icon" title="Click to view full thread and respond">View all responses</a>
        <a href="#" class="respond tooltip" title="Click to respond to this posting">Respond to this</a>-->
      </div>
    </div>
    <div class="tip" id="cTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialogMultiple" style="padding:0;margin:0;">
  <div class="dialog multiple preview">
    <div class="tipu" id="mTipu"></div>
    <div class="btn close" id="btn_close_multi" title="Click to close"><a title="Click to close">Close</a></div>
    <ol id="multilist"></ol>
    <br/>
    <div id="multidetail"></div>
    <div class="tip" id="mTip"></div>
  </div>
</div>


<div dojoType="dijit.Dialog" id="Rating">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Rate This Article</span></h5>
      <div class="instructions">Please follow our <a  href="/static/ratingGuidelines.action;jsessionid=A06D857AE008E3829F545704038156D3">guidelines for rating</a> and review our <a href="/static/competing.action;jsessionid=A06D857AE008E3829F545704038156D3">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
        <ol>
          <li>Remarks that could be interpreted as allegations of misconduct</li>
          <li>Unsupported assertions or statements</li>
          <li>Inflammatory or insulting language</li>
        </ol>
      </div>
      <div class="posting pane">
        <form name="ratingForm" id="ratingForm" method="post" action="">
          <input type="hidden" name="articleURI" value="info:doi/10.1371/journal.pcbi.1002720" />  
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="" />
          <fieldset>
            <legend>Compose Your Annotation</legend>
            <span id="submitRatingMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td rowspan="2">
                  <label for="insight">Insight</label>
                  <ul class="star-rating rating edit" title="Rate insight" id="rateInsight">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Bland" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'insight', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Profound" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="insight" title="insight" value="" />
                  <label for="reliability">Reliability</label>
                  <ul class="star-rating rating edit" title="Rate reliability" id="rateReliability">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Tenuous" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'reliability', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Unassailable" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="reliability" title="reliability" value="" />
                  <label for="style">Style</label>
                  <ul class="star-rating rating edit" title="Rate style" id="rateStyle">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Crude" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'style', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Elegant" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="style" title="style" value="" />
                  <label for="cTitle"><span class="none">Enter your comment title</span><!-- error message text <em>A title is required for all public annotations</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your comment title..." class="title" alt="Enter your comment title..." />
                  <label for="cArea"><span class="none">Enter your comment</span><!-- error message text <em>Please enter your annotation</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your comment..." alt="Enter your comment...">Enter your comment...</textarea>
                </td>
                <td rowspan="2">&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." title="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel_rating"/>
                  <input type="button" value="Submit" title="Click to post your annotation publicly" id="btn_post_rating" class="primary"/>
                </td>
              </tr>
              <tr>
                  <td colspan="3"><p>Ratings can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p></td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
<div dojoType="ambra.widget.LoadingCycle" id="LoadingCycle" class="loadingCycler">
  <img src="/images/loading.gif" width="58" height="58" title="Loading..." />
</div>
<div dojoType="dijit.Dialog" id="LightBox">

    <!-- start main window wrapper-->
    <div class="figure-window-wrapper">

      <!--start film strip Wrapper-->
      <div class="figure-window-nav-wrapper">
        <div id="figure-window-nav"></div>
      </div><!--end figure-window-nav-wrapper-->

      <!-- start figure container-->
      <div class="figure-window-container">

        <!-- start/end figure viewer-->
        <div id="figure-window-viewer">
          <img id="figure-window-img" class="large"/>
        </div>

        <!-- start/end figure doi-->
        <div id="figure-window-doi"></div>

        <!--start figure title-box-->
        <div id="figure-window-title-box" class="figure-window-title-less">
          <span id="figure-window-title"></span>
          <div id="figure-window-more" class="figure-window-more" onclick="return ambra.lightBox.showMoreOrLess();"></div>
          <div id="figure-window-description"></div>
        </div><!--end figure title-box-->

        <!--start figure download-->
        <div class="figure-window-download">
          <ul>
            <li class="download icon"><strong>Download:</strong>
              <a id="figure-window-ppt" title="Click to download PowerPoint slide of this image">PowerPoint
                slide</a> |
              <a id="figure-window-large" title="Click to download a larger version of this image">larger image
                (<span id="figure-window-large-size"></span>)</a> |
              <a id="figure-window-tiff" title="Click to download a original version of this image">original image
                (<span id="figure-window-tiff-size"></span>)</a>
            </li>
          </ul>
        </div><!--end figure download-->

        <!--start/end of close, context, next and previous button-->
        <div class="figure-window-close" onclick="return ambra.lightBox.hide();"></div>
        <div class="figure-window-context" onclick="return ambra.lightBox.showInContext();"></div>
        <div id="figure-window-previous" onclick="return ambra.lightBox.showPrevious();"></div>
        <div id="figure-window-next" onclick="return ambra.lightBox.showNext();"></div>

      </div><!--end figure-window-container-->

    </div><!--end figure-window-wrapper-->

</div><!--end dijit.Dialog-->
</div>
</div>
<!-- end : main contents -->

</div>
<!-- end : container -->	

<!-- begin : footer -->
<div id="ftr">
  <p><span>All site content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License 2.5" tabindex="200">Creative Commons Attribution License</a>.</span></p>
  <ul>
    <li><a href="/static/privacy.action;jsessionid=A06D857AE008E3829F545704038156D3" title="PLOS Privacy Statement" tabindex="501">Privacy Statement</a></li>
    <li><a href="/static/terms.action;jsessionid=A06D857AE008E3829F545704038156D3" title="PLOS Terms of Use" tabindex="502">Terms of Use</a></li>
    <li><a href="http://www.plos.org/advertise/" title="Advertise With PLOS" tabindex="503">Advertise</a></li>
    <li><a href="http://www.plos.org/about/media-inquiries/" title="PLOS Embargo Policy" tabindex="504">Media Inquiries</a></li>
    <li><a href="http://www.plos.org/publish/order-reprints/" title="PLOS in Print" tabindex="505">PLOS in Print</a></li>
    <li><a href="/static/sitemap.action;jsessionid=A06D857AE008E3829F545704038156D3" title="Site Map" tabindex="506">Site Map</a></li>
    <li><a href="http://www.plos.org" title="PLOS.org" tabindex="507">PLOS.org</a></li>
  </ul>
  <div class="powered">
  <ul>
    <li><a href="/static/releaseNotes.action" title="Ambra | Release Notes">Ambra 2.4.9</a></li>
    <li>Managed Colocation provided by <a href="http://www.isc.org/" title="Internet Systems Consortium">Internet Systems Consortium</a>.</li>
  </ul>
  </div>
</div>
<!-- end : footer -->
<script type="text/javascript">
  var _namespace="";
    var loggedIn = false;

  var almHost = "http://alm.plos.org";
  var solrHost = "http://api.plos.org/search";
  var pubGetHost = "http://pubget.com/widgetizer/links_hash";
  var appContext = "";

  // Safari v3.1.1 "console.debug" issue (http://trac.dojotoolkit.org/ticket/6849) workaround
  if (/3[\.0-9]+ Safari/.test(navigator.appVersion)) {
    window.console = {
      origConsole: window.console,
      log: function(s){ this.origConsole.log(s); },
      info: function(s){ this.origConsole.info(s); },
      error: function(s){ this.origConsole.error(s); },
      warn: function(s){ this.origConsole.warn(s); }
    };
  }

  var djConfig = {
    // don't debug for IE - as dojo's firebug lite module is error prone in IE
    isDebug: false,
    parseOnLoad: true
  };

  var bamGAID = 'UA-338393-1';
  var bamGAVPV = location.pathname + location.search + location.hash;
</script>
<script type="text/javascript" src="/javascript/dojo/dojo/dojo.js?v=pzlwVCPPo+oJC12upPgJrJ5EG_k"></script>
<script type="text/javascript" src="/javascript/dojo/dojo/ambra.js?v=Y8JM0CVD9YwjC99M_jvbcRTiGhY"></script>
<script type="text/javascript" src="/javascript/init_global.js?v=fF03pfG3r+Nkj_czZ8ORg8o5sNM"></script>
<script type="text/javascript" src="/javascript/init_article.js?v=qzBD_a7nKbeMM2xNUtB3vckN+Zc"></script>
<script type="text/javascript" src="/javascript/init_ratings.js?v=seKykuS0X33CQGS7Lj+8FtBy5Uw"></script>
<script type="text/javascript" src="/javascript/init_lightBox.js?v=vJRLPy7RRX8Zp31Ogcm3i+TA4f8"></script>
<script type="text/javascript" src="/javascript/init_article_body.js?v=YgFei2xCi50CahwaPbSsxUxUsmw"></script>
<script type="text/javascript" src="/javascript/init_article_rhc.js?v=djtbTMOagorKE6ZPrkCwufAfWWI"></script>
<script type="text/javascript" src="/javascript/alm.js?v=1tH9JZ8uDtXspXssGJDr8O+jBOs"></script>
<script type="text/javascript" src="/javascript/reporting/articleViewsCumulative.js?v=RQ+gCrcAl0dBeLbmokxgQZTGYYA"></script>
<script type="text/javascript" src="/javascript/check_twitter.js?v=Hc0sH1W9xIu0qhqxWPD4c1u50K4"></script>
<script type="text/javascript" src="/javascript/init_googleAnalytics.js?v=yCD4h9PHGI0dsefmuWDlMn7pHog"></script>

</body>
</html>

