

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article">

<head>
   <title>PLOS Computational Biology: Random Field Model Reveals Structure of the Protein Recombinational Landscape</title>



<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9" />

<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="home" title="home" href="/home.action;jsessionid=42F26552E5BE8446B0C72379D8339313" />
<link rel="alternate" type="application/rss+xml"
  title="PLOS Computational Biology: New Articles"
  href="http://www.ploscompbiol.org/article/feed" />

<link rel="pingback" href="http://www.ploscompbiol.org/pingback" />


<!-- global_css.ftl -->
<link rel="stylesheet" type="text/css" media="screen" href="/css/dojo/dijit/themes/tundra/tundra.css?v=zwugcxLdpBWmb72qqlHgGyeNAGM" />
<link rel="stylesheet" type="text/css" media="all" href="/css/star_rating.css?v=Mm3HyaUPUahAJAq6xG5W3tdF7Hk" />
<link rel="stylesheet" type="text/css" media="all" href="/css/rating.css?v=AOT9rozE9sYVjGTdeiyxHf18Ykw" />
<link rel="stylesheet" type="text/css" media="all" href="/css/screen.css?v=KBNFqBmhD635U0NgiM5tf9dtxRE" />
<link rel="stylesheet" type="text/css" media="all" href="/css/iepc.css?v=FIjyR83BhSovq_5yNVk1Sih60AI" />
<link rel="stylesheet" type="text/css" media="all" href="/css/journal.css?v=SiREXztjNSUsOyrlkE9sv028Y+0" />
<link rel="stylesheet" type="text/css" media="all" href="/css/lightBox.css?v=sDgsUXamFoYAakEshdj0YWrZiiQ" />
<link rel="stylesheet" type="text/css" media="print" href="/css/print.css?v=0T+LTV7Gd0jtHEhqEvBfSrmHUGg" />

<!--chartbeat --> 
<script type="text/javascript">var _sf_startpt=(new Date()).getTime()</script>


<!--[if IE]><![endif]-->
<meta name="description" content="PLOS Computational Biology is an open-access" />

<meta name="keywords" content="plos computational biology" />

  <meta name="citation_publisher" content="Public Library of Science" />
  <meta name="citation_doi" content="10.1371/journal.pcbi.1002713" />
    <meta name="citation_title" content="Random Field Model Reveals Structure of the Protein Recombinational Landscape"/>
    <meta itemprop="name" content="Random Field Model Reveals Structure of the Protein Recombinational Landscape"/>

      <meta name="citation_author" content="Philip A. Romero" />
      <meta name="citation_author" content="Frances H. Arnold" />
          <meta name="citation_author_institution" content="Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, California, United States of America" />

    <meta name="citation_date" content="2012/10/4"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pcbi.1002713.pdf" />

    <meta name="citation_journal_title" content="PLOS Computational Biology" />
    <meta name="citation_firstpage" content="e1002713"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="8"/>
    <meta name="citation_issn" content="1553-7358"/>

    <meta name="citation_journal_abbrev" content="PLoS Comput Biol" />

      <meta name="citation_reference" content="citation_title=Why sex and recombination?; citation_volume=281; citation_number=1; citation_pages=1986-1990; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Resolving the paradox of sex and recombination; citation_volume=3; citation_number=2; citation_pages=252-261; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Intragenic recombination as a source of population genetic variability; citation_volume=106; citation_number=3; citation_pages=737-753; citation_date=1972; " />
      <meta name="citation_reference" content="citation_title=The Effect of Intragenic Recombination on the Number of Alleles in a Finite Population; citation_volume=88; citation_number=4; citation_pages=829-844; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Allelic Variation at the Level of Intragenic Recombination; citation_volume=89; citation_number=5; citation_pages=211-224; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=The extent and importance of intragenic recombination; citation_volume=1; citation_number=6; citation_pages=410-420; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=DNA shuffling of a family of genes from diverse species accelerates directed evolution; citation_volume=391; citation_number=7; citation_pages=288-291; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Engineering by homologous recombination: exploring sequence and function within a conserved fold; citation_volume=17; citation_number=8; citation_pages=454-459; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Protein building blocks preserved by recombination; citation_volume=9; citation_number=9; citation_pages=553-558; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Site-directed protein recombination as a shortest-path problem; citation_volume=17; citation_number=10; citation_pages=589-594; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Combinatorial recombination of gene fragments to construct a library of chimeras; citation_volume=26; citation_number=11; citation_pages=2.1-2.20; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Efficient Sampling of SCHEMA Chimera Families for Identification of Useful Sequence Elements; citation_number=12; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Protein tolerance to random amino acid change; citation_volume=101; citation_number=13; citation_pages=9205-9210; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Thermo-dynamic prediction of protein neutrality; citation_volume=102; citation_number=14; citation_pages=606-611; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Robustness-epistasis link shapes the fitness landscape of a randomly drifting protein; citation_volume=444; citation_number=15; citation_pages=929-932; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=On the conservative nature of intragenic recombination; citation_volume=102; citation_number=16; citation_pages=5380-5385; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A diverse family of thermostable cytochrome P450s created by recombination of stabilizing fragments; citation_volume=25; citation_number=17; citation_pages=1051-1056; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A family of thermostable fungal cellulases created by structure-guided recombination; citation_volume=106; citation_number=18; citation_pages=5610-5615; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Diversification of catalytic function in a synthetic family of chimeric cytochrome p450s; citation_volume=14; citation_number=19; citation_pages=269-278; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=SCHEMA-Designed Variants of Human Arginase I and II Reveal Sequence Elements Important to Stability and Catalysis; citation_volume=1; citation_number=20; citation_pages=221-228; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=A diverse set of family 48 bacterial cellulases created by structure-guided recombination; citation_number=21; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Efficient screening of fungal cellobiohydrolase class I enzymes for thermostabilizing sequence blocks by SCHEMA structure-guided recombination; citation_volume=23; citation_number=22; citation_pages=871-880; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=SCHEMA recombination of a fungal cellulase uncovers a single mutation that contributes markedly to stability; citation_volume=284; citation_number=23; citation_pages=26229-26233; citation_date=2009; " />
      <meta name="citation_reference" content="citation_number=24; " />
      <meta name="citation_reference" content="citation_number=25; " />
      <meta name="citation_reference" content="citation_number=26; " />
      <meta name="citation_reference" content="citation_title=Random Field Models For Fitness Landscapes; citation_volume=38; citation_number=27; citation_pages=435-478; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Towards a general theory of adaptive walks on rugged landscapes; citation_volume=128; citation_number=28; citation_pages=11-45; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Estimation of effective interresidue contact energies from protein crystal structures: quasi-chemical approximation; citation_volume=18; citation_number=29; citation_pages=534-552; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Folding Lennard-Jones proteins by a contact potential; citation_volume=37; citation_number=30; citation_pages=544-553; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Can a pairwise contact potential stabilize native protein folds against decoys obtained by threading?; citation_volume=38; citation_number=31; citation_pages=134-148; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Structure-Guided Recombination Creates an Artificial Family of Cytochromes P450; citation_volume=4; citation_number=32; citation_pages=e112; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Why are proteins marginally stable?; citation_volume=46; citation_number=33; citation_pages=105-109; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Library analysis of SCHEMA-guided protein recombination; citation_volume=12; citation_number=34; citation_pages=1686-1693; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=35; " />
      <meta name="citation_reference" content="citation_title=Additivity in both thermodynamic stability and thermal transition temperature for rubredoxin chimeras via hybrid native partitioning; citation_volume=13; citation_number=36; citation_pages=1153-1163; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=37; " />
      <meta name="citation_reference" content="citation_number=38; " />
      <meta name="citation_reference" content="citation_title=Adaptive landscapes and protein evolution; citation_volume=107; citation_number=39; citation_pages=1747-1751; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=How are model protein structures distributed in sequence space?; citation_volume=73; citation_number=40; citation_pages=2393-2403; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Simulating protein evolution in sequence and structure space; citation_volume=14; citation_number=41; citation_pages=202-207; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional proteins from a random-sequence library; citation_volume=410; citation_number=42; citation_pages=715-718; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Estimating the prevalence of protein sequences adopting functional enzyme folds; citation_volume=341; citation_number=43; citation_pages=1295-1315; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution favors protein mutational robustness in sufficiently large populations; citation_volume=5; citation_number=44; citation_pages=29; citation_date=2007; " />
      <meta name="citation_reference" content="citation_number=45; " />
      <meta name="citation_reference" content="citation_title=Mapping recombination hotspots in human phosphoglucomutase (PGM1); citation_volume=8; citation_number=46; citation_pages=1699-1706; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Recombination as a mechanism for sporadic mutation in the surfactant protein-C gene; citation_volume=43; citation_number=47; citation_pages=443-450; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Effects of intra-gene fitness interactions on the benefit of sexual recombination; citation_volume=34; citation_number=48; citation_pages=560-561; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome structure and the benefit of sex; citation_volume=65; citation_number=49; citation_pages=523-536; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Structure-guided SCHEMA recombination of distantly related beta-lactamases; citation_volume=19; citation_number=50; citation_pages=563-570; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The use of confidence or fiducial limits illustrated in the case of the binomial; citation_volume=26; citation_number=51; citation_pages=404-413; citation_date=1934; " />
      <meta name="citation_reference" content="citation_title=Accurate approximations for posterior moments and marginal densities; citation_volume=81; citation_number=52; citation_pages=82-86; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=SCOP: a structural classification of proteins database for the investigation of sequences and structures; citation_volume=247; citation_number=53; citation_pages=536-40; citation_date=1995; " />



<!--
<rdf:RDF xmlns="http://web.resource.org/cc/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<Work rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713">
   <license rdf:resource="http://creativecommons.org/licenses/by/2.5/" />
</Work>
<License rdf:about="http://creativecommons.org/licenses/by/2.5/">
   <permits rdf:resource="http://web.resource.org/cc/Reproduction" />
   <permits rdf:resource="http://web.resource.org/cc/Distribution" />
   <requires rdf:resource="http://web.resource.org/cc/Notice" />
   <requires rdf:resource="http://web.resource.org/cc/Attribution" />
   <permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" />
</License>
<rdf:Description
     rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713"
     dc:identifier="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713"
     dc:title="PLOS Computational Biology: Random Field Model Reveals Structure of the Protein Recombinational Landscape"
       trackback:ping="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713/trackback;jsessionid=42F26552E5BE8446B0C72379D8339313"
     />
</rdf:RDF>

-->
</head>
<body>
<span property="dc:language" content="en"></span>
<!-- begin : container -->
<div id="container">
  <!-- begin : top banner external ad space -->
  <div id="topBanner">


<!-- begin : left banner slot -->
<div class="left">
  <iframe id='affe494b' name='affe494b'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=84&amp;cb=5377'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=affe494b&amp;cb=9102'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=84&amp;cb=5033&amp;n=affe494b'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : left banner slot -->
<!-- begin : right banner slot -->
<div class="right">
  <iframe id='af654bd5' name='af654bd5'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=85&amp;cb=6374'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=af654bd5&amp;cb=7805'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=85&amp;cb=5738&amp;n=af654bd5'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : right banner slot -->

  </div>
  <!-- end : top banner external ad space -->

  <!-- begin : header -->
  <div id="hdr" class="login">

  <!-- begin : logo -->
  <div id="logo" title="PLOS Computational Biology"><a href="/home.action;jsessionid=42F26552E5BE8446B0C72379D8339313" title="PLOS Computational Biology"><span>PLOS Computational Biology</span></a></div>
  <!-- end : logo -->
  <!-- begin : user controls -->





  <div id="user">
    <div>
      <ul>
        <li><a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002713" class="feedback"><strong>Login</strong></a> | </li>
        <li><a href="https://register.plos.org:443/ambra-registration/register.action">Create Account</a> | </li>
        <li class="feedback"><a href="/feedbackCreate.action?page=%252Farticle%252FfetchArticle.action%253FarticleURI%253Dinfo%25253Adoi%25252F10.1371%25252Fjournal.pcbi.1002713" title="Send us your feedback">Feedback</a></li>
      </ul>
    </div>
  </div>


  <!-- end : user controls -->
  <!-- begin search links -->
  <ul id="links"><li class="browse"><a href="/article/browse.action;jsessionid=42F26552E5BE8446B0C72379D8339313?field=date" title="Browse Articles">Browse</a></li><li class="rss"><a href="/static/rssFeeds.action;jsessionid=42F26552E5BE8446B0C72379D8339313" title="RSS Feeds">RSS</a></li></ul>
  <!-- end : search links -->
  <!-- begin : dashboard -->
  <div id="db">
    <form name="searchForm" action="/search/simpleSearch.action;jsessionid=42F26552E5BE8446B0C72379D8339313" method="get">
      <input type="hidden" name="from" value="globalSimpleSearch"/>
      <input type="hidden" name="filterJournals" value="PLoSCompBiol">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="wrap"><input id="search" type="text" name="query" value="Search articles..." onfocus="if(this.value=='Search articles...')value='';" onblur="if(this.value=='')value='Search articles...';" class="searchField" alt="Search articles..."/></div>
        <input src="/images/search_btn1.gif" onclick="submit();" value="ftsearch" alt="SEARCH" tabindex="3" class="button" type="image" />
      </fieldset>
    </form>
    <form name="gasf" action="/search/advancedSearch.action;jsessionid=42F26552E5BE8446B0C72379D8339313" method="get">
<input type="hidden" name="noSearchFlag" value="true" id="noSearchFlag"/><input type="hidden" name="query" value="" id="query"/>    </form>
    <a id="advSearch" href="#" onclick="if(document.searchForm.query.value!='Search articles...')document.gasf.query.value=document.searchForm.query.value;document.gasf.submit();return false;">Advanced Search</a>
  </div>
  <!-- end : dashboard -->
  <!-- begin : navigation -->
  <ul id="nav">
    <li><a href="/home.action;jsessionid=42F26552E5BE8446B0C72379D8339313" tabindex="101">Home</a></li>
     <!-- This is a temporary action to link to th static toc page -->
    <li><a href="/static/browse.action;jsessionid=42F26552E5BE8446B0C72379D8339313" tabindex="102">Browse Articles</a>
        <ul>
          <li><a href="/article/browseIssue.action;jsessionid=42F26552E5BE8446B0C72379D8339313?field=issue">Current Issue</a></li> <!-- Assuming dynamic TOC is in place -->
          <li><a href="/article/browseVolume.action;jsessionid=42F26552E5BE8446B0C72379D8339313?field=volume">Journal Archive</a></li> 
          <li><a href="/article/browse.action;jsessionid=42F26552E5BE8446B0C72379D8339313?field=date">By Publication Date</a></li>
          <li><a href="/article/browse.action;jsessionid=42F26552E5BE8446B0C72379D8339313">By Subject</a></li>
          <li><a href="http://www.ploscollections.org/static/pcbiCollections.action">Collections</a></li>
        </ul>
    </li>
    <li><a href="/static/about.action;jsessionid=42F26552E5BE8446B0C72379D8339313" tabindex="103">About</a>
        <ul>
          <li><a href="/static/information.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Journal Information</a></li>
          <li><a href="/static/edboard.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Editorial Board</a></li>
          <li><a href="/static/eic.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Editors-in-Chief</a></li>
          <li><a href="/static/almInfo.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Article-Level Metrics</a></li>
          <li><a href="/static/license.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Open-Access License</a></li>
          <li><a href="/static/contact.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Contact Us</a></li>
        </ul>
      </li>
    <li><a href="/static/users.action;jsessionid=42F26552E5BE8446B0C72379D8339313" tabindex="104">For Readers</a>
        <ul>
          <li><a href="/static/commentGuidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Guidelines for Notes, Comments, and Corrections</a></li>
          <li><a href="/static/ratingGuidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Guidelines for Rating</a></li>
          <li><a href="/static/help.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Help Using This Site</a></li>
          <li><a href="/static/downloads.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Media Downloads</a></li>
          <li><a href="/static/sitemap.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Site Map</a></li>
        </ul>
      </li>
    <li><a href="/static/authors.action;jsessionid=42F26552E5BE8446B0C72379D8339313" tabindex="105">For Authors and Reviewers</a>
        <ul>
          <li><a href="/static/policies.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Editorial and Publishing Policies</a></li>
          <li><a href="/static/guidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Author Guidelines</a></li>
          <li><a href="/static/figureGuidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Figure and Table Guidelines</a></li>
          <li><a href="/static/checklist.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Submit Your Paper</a></li>
          <li><a href="/static/reviewerGuidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313">Reviewer Guidelines</a></li>
	   <li><a href="http://www.plos.org/journals/index.php">Submit to Other PLOS Journals</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://www.plos.org" title="PLOS" tabindex="110" class="drop">PLOS.org</a>
        <ul>
          <li><a href="http://www.plos.org/oa/index.html" title="Open Access Statement">Open Access</a></li>
          <li><a href="http://www.plos.org/support/donate.php" title="Join PLOS: Show Your Support">Join PLOS</a></li>
          <li><a href="http://blogs.plos.org/" title="PLOS Blogs">PLOS Blogs</a></li>
	  <li><a href="http://www.zazzle.com/plos?rf=238434112415374154" title="PLOS Store">PLOS Store</a></li>
          <li><a href="http://currents.plos.org" title="PLOS Currents">PLOS Currents</a></li>
          <li><a href="http://www.plos.org/connect.html" title="PLOS.org | Stay Connected">Stay Connected</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://hubs.plos.org" tabindex="109">Hubs</a>
        <ul>
          <li><a href="http://hubs.plos.org/biodiversity" title="PLOS Hubs: Biodiversity">Biodiversity</a></li>
          <li><a href="http://clinicaltrials.ploshubs.org" title="PLOS Hub for Clinical Trials">Clinical Trials</a></li>
        </ul>
    </li>
      <li class="journalnav"><a href="http://www.plosjournals.org" tabindex="108">Journals</a>
        <ul>
          <li><a href="http://www.plosbiology.org/" title="PLOSBiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org/" title="PLOSMedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org/" title="PLOSCompBiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://www.plosgenetics.org/" title="PLOSGenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org/" title="PLOSPathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org/" title="PLOSONE.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org/" title="PLOSNTDs.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </li>
    </ul>
  <!-- end : navigation -->
  </div>
  <!-- end : header --> 



<!-- begin : main content -->
<div id="content" class="article" style="visibility:visible;">

<!-- begin : right hand column -->

<div id="rhc" xpathLocation="noDialog">

  <div id="download" class="rhcBox_type1">
    <div class="wrap">
      <ul>
        <li class="download icon"><strong>Download:</strong>
          <a href="/article/fetchObjectAttachment.action;jsessionid=42F26552E5BE8446B0C72379D8339313?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713&amp;representation=PDF" title="Download article PDF">PDF</a> |
          <a href="/article/citationList.action;jsessionid=42F26552E5BE8446B0C72379D8339313?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713" title="Download citations">Citation</a> |
          <a href="/article/fetchObjectAttachment.action;jsessionid=42F26552E5BE8446B0C72379D8339313?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713&amp;representation=XML" title="Download Article XML">XML</a>
        </li>
        <li class="print icon"><a href="#" onclick="window.print();return false;" title="Print Article"><strong>Print article</strong></a></li>
        <li class="reprint icon"><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=3&doi=10.1371/journal.pcbi.1002713&volume=&issue=&title=Random Field Model Reveals Structure of the Protein Recombinational Landscape&author_name=Philip%20A.%20Romero%2C%20Frances%20H.%20Arnold&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a> New &amp; improved!</li>
      </ul>
    </div>
  </div>

  <div id="issues" class="rhcBox_type2">
    <p><strong>Published in the</strong>
    <a href="http://www.ploscompbiol.org/article/browseIssue.action?issue=info%3Adoi%2F10.1371%2Fissue.pcbi.v08.i10" title="Browse the Open-Access Issue">October 2012 Issue of <em>PLOS Computational Biology</em></a></p>
  </div>

  <div id="impact" class="rhcBox_type2">
    <h6>Metrics <a href="/static/help.action;jsessionid=42F26552E5BE8446B0C72379D8339313#articleMetrics" class="replaced" id="info" title="More information"><span>info</span></a></h6>

    <div class="wrap">
      <img id="rhcDataSpinner" src="/images/loading_small.gif" class="loading" />
      <div id="totalDataInRHC"></div>
      <div id="pubDateInRHC"></div>
      <div id="relatedCitesInRHC"></div>
    </div>

    <div class="more clearfix"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" title="More information">More</a></div>
  </div>

  <div id="related" class="rhcBox_type2">
    <h6>Related Content</h6>
    <dl class="related">
      <dt>Related Articles on the Web</dt>
      <dd><a href="http://scholar.google.com/scholar?hl=en&lr=&q=related:http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002713&btnG=Search">Google Scholar</a></dd>
      <dd id="pubMedRelatedLIInRHC" style="display:none;"><a id="pubMedRelatedURLInRHC">PubMed</a></dd>
    </dl>
    <div class="more clearfix"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" title="More information">More</a></div>
  </div>

  <div id="share" class="rhcBox_type2">
    <h6>Share this Article <a href="/static/help.action;jsessionid=42F26552E5BE8446B0C72379D8339313#socialBookmarkLinks" class="replaced" id="info" title="More information"><span>info</span></a></h6>
    <ul>
      <li class="bookmarklets">

        <a href="http://www.reddit.com/submit?url=http://dx.plos.org/10.1371/journal.pcbi.1002713"
           target="_new"
           id="reddit">
          <img src="http://www.reddit.com/static/spreddit4.gif" alt="submit to reddit" border="0"/> </a>
        <g:plusone size="small" annotation="none" href=http://dx.plos.org/10.1371/journal.pcbi.1002713></g:plusone>
        <script type="text/javascript">
          (function () {
            var po = document.createElement('script');
            po.type = 'text/javascript';
            po.async = true;
            po.src = 'https://apis.google.com/js/plusone.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(po, s);
          })();
        </script>
        <a href="http://www.stumbleupon.com/submit?url=http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713" target="_new"> <img border=0 src="http://cdn.stumble-upon.com/images/16x16_su_solid.gif" alt="StumbleUpon" title="Add to StumbleUpon"></a>
        <script>function fbs_click() {u='http://dx.plos.org/10.1371/journal.pcbi.1002713';t='Random%20Field%20Model%20Reveals%20Structure%20of%20the%20Protein%20Recombinational%20Landscape';window.open('http://www.facebook.com/sharer.php?u='+encodeURIComponent(u)+'&t='+encodeURIComponent(t),'sharer','toolbar=0,status=0,width=626,height=436');return false;}</script><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002713" onclick="return fbs_click()" target="_blank"><img src="http://static.ak.fbcdn.net/images/share/facebook_share_icon.gif" alt="Facebook" title="Add to Facebook" /></a>       <!-- for mor info, see http://www.facebook.com/share_partners.php -->
        <script type="text/javascript">
          function bookmark_in_connotea(u) {
            a=false; x=window; e=x.encodeURIComponent; d=document;
            w=open('http://www.connotea.org/addpopup?continue=confirm&uri='+e(u),
                'add', 'width=600, height=400, scrollbars, resizable');
            void(x.setTimeout('w.focus()',200));
          }
        </script>
        <a style='cursor: pointer;' onclick='javascript:bookmark_in_connotea("http://dx.plos.org/10.1371/journal.pcbi.1002713");'><img src='/images/icon_connotea_16x16.gif' alt="Connotea" title="Add to Connotea"/></a>
        <a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002713&title=Random%20Field%20Model%20Reveals%20Structure%20of%20the%20Protein%20Recombinational%20Landscape" target="_new"><img src='/images/icon_citeulike_16x16.gif' alt="CiteULike" title="Add to CiteULike" /></a>
        <a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002713" title="Add to Mendeley" target="_new"><img src="/images/icon_mendeley_16x16.gif" alt="Bibliography"></a>
        <script type="text/javascript">
          var twtTitle  = 'Random Field Model Reveals Structure of the Protein Recombinational Landscape';
          var twtUrl    = 'http://dx.plos.org/10.1371/journal.pcbi.1002713';
          var twtTag    = '#PLOSCompBio:'
          var maxLength = 140 - (twtUrl.length + 1);
          if (twtTitle.length > maxLength) {
            twtTitle = twtTitle.substr(0, (maxLength - 17))+'...';  //truncates title to include space for twtTag and ellipsis
          }
          var twtLink = 'http://twitter.com/intent/tweet?text='+encodeURIComponent(twtTag + ' ' + twtTitle + ' ' + twtUrl);
          document.write('<a href="'+twtLink+'" target="_blank"><img src="/images/icon_twitter.gif" border="0" alt="Twitter icon" title="Tweet This!" /></a>');
        </script>

      </li>
      <li class="email icon"><a href="/article/emailArticle.action;jsessionid=42F26552E5BE8446B0C72379D8339313?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713" title="Email this article">Email this article</a></li>
    </ul>
  </div>
  <div id="articleTwitterStream" style="display:none">
    <script src="http://widgets.twimg.com/j/2/widget.js"></script>
    <script>
      new TWTR.Widget({
        version: 2,
        type: 'search',
        search: encodeURIComponent('10.1371/journal.pcbi.1002713'),
        interval: 6000,
        title: '',
        subject: 'From Twitter',
        width: 300,
        height: 300,
        theme: {
          shell: {
            background: '#667893',
            color: '#ffffff'
          },
          tweets: {
            background: '#ffffff',
            color: '#444444',
            links: '#006699'
          }
        },
        features: {
          scrollbar: true,
          loop: false,
          live: true,
          hashtags: true,
          timestamp: true,
          avatars: true,
          toptweets: true,
          behavior: 'all'
        }
      }).render().start();
    </script>
  </div>
</div>
<!-- end : right hand column -->
<form name="articleInfo" id="articleInfo" method="" action="">
<input type="hidden" name="isAuthor" value="true" />
<input type="hidden" name="authorIdList" value="" />
<input type="hidden" name="userIdList" value="" />
<input type="hidden" name="otherIdList" value="" />
<input type="hidden" name="annotationId" value="" />
<input type="hidden" name="isResearchArticle" value="true" />
</form>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.XML" value="125515"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.PDF" value="829597"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e001.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e001.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e002.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e002.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e003.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e003.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.t001.TIF" value="482890"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.t001.PNG_I" value="16346"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.t001.PNG_S" value="6959"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.t001.PNG_M" value="31862"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.t001.PNG_L" value="184084"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e004.TIF" value="534"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e004.PNG" value="490"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e005.TIF" value="476"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e005.PNG" value="452"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e006.TIF" value="1726"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e006.PNG" value="982"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e007.TIF" value="792"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e007.PNG" value="603"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e008.TIF" value="574"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e008.PNG" value="493"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e009.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e009.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e010.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e010.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e011.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e011.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e012.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e012.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e013.TIF" value="476"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e013.PNG" value="452"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e014.TIF" value="434"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e014.PNG" value="428"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e015.TIF" value="942"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e015.PNG" value="661"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e016.TIF" value="462"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e016.PNG" value="444"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e017.TIF" value="4808"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e017.PNG" value="2519"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e018.TIF" value="774"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e018.PNG" value="609"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e019.TIF" value="434"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e019.PNG" value="435"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e020.TIF" value="838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e020.PNG" value="635"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e021.TIF" value="434"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e021.PNG" value="435"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e022.TIF" value="1004"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e022.PNG" value="712"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e023.TIF" value="1048"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e023.PNG" value="727"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e024.TIF" value="434"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e024.PNG" value="435"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e025.TIF" value="462"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e025.PNG" value="444"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e026.TIF" value="774"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e026.PNG" value="609"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e027.TIF" value="838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e027.PNG" value="635"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e028.TIF" value="688"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e028.PNG" value="571"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e029.TIF" value="756"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e029.PNG" value="592"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e030.TIF" value="1852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e030.PNG" value="1114"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e031.TIF" value="2036"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e031.PNG" value="1199"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e032.TIF" value="462"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e032.PNG" value="444"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e033.TIF" value="4700"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e033.PNG" value="2477"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e034.TIF" value="850"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e034.PNG" value="658"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e035.TIF" value="2160"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e035.PNG" value="1259"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e036.TIF" value="462"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e036.PNG" value="444"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e037.TIF" value="5308"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e037.PNG" value="2767"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e038.TIF" value="8284"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e038.PNG" value="4139"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g001.TIF" value="1124584"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g001.PNG_I" value="187928"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g001.PNG_S" value="28575"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g001.PNG_M" value="314227"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g001.PNG_L" value="931540"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e039.TIF" value="700"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e039.PNG" value="566"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e040.TIF" value="828"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e040.PNG" value="643"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e041.TIF" value="768"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e041.PNG" value="604"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e042.TIF" value="882"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e042.PNG" value="664"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e043.TIF" value="820"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e043.PNG" value="612"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e044.TIF" value="2904"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e044.PNG" value="1637"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e045.TIF" value="2982"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e045.PNG" value="1697"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e046.TIF" value="2868"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e046.PNG" value="1623"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e047.TIF" value="2954"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e047.PNG" value="1688"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e048.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e048.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e049.TIF" value="434"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e049.PNG" value="428"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e050.TIF" value="434"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e050.PNG" value="428"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e051.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e051.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e052.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e052.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e053.TIF" value="3158"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e053.PNG" value="1812"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e054.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e054.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e055.TIF" value="654"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e055.PNG" value="537"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e056.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e056.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e057.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e057.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e058.TIF" value="476"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e058.PNG" value="452"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e059.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e059.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g002.TIF" value="199084"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g002.PNG_I" value="51092"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g002.PNG_S" value="13351"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g002.PNG_M" value="38539"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g002.PNG_L" value="126108"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e060.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e060.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e061.TIF" value="2374"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e061.PNG" value="1358"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e062.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e062.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e063.TIF" value="554"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e063.PNG" value="497"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e064.TIF" value="534"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e064.PNG" value="490"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e065.TIF" value="476"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e065.PNG" value="452"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e066.TIF" value="518"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e066.PNG" value="476"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e067.TIF" value="1200"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e067.PNG" value="806"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e068.TIF" value="554"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e068.PNG" value="497"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e069.TIF" value="3188"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e069.PNG" value="1727"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e070.TIF" value="4430"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e070.PNG" value="2300"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e071.TIF" value="554"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e071.PNG" value="497"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e072.TIF" value="5634"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e072.PNG" value="2639"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e073.TIF" value="798"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e073.PNG" value="600"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e074.TIF" value="774"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e074.PNG" value="609"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e075.TIF" value="838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e075.PNG" value="635"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e076.TIF" value="8862"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e076.PNG" value="4257"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e077.TIF" value="724"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e077.PNG" value="577"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e078.TIF" value="888"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e078.PNG" value="649"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e079.TIF" value="462"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e079.PNG" value="444"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e080.TIF" value="13102"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e080.PNG" value="5700"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e081.TIF" value="1608"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e081.PNG" value="961"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e082.TIF" value="1790"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e082.PNG" value="1055"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e083.TIF" value="2238"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e083.PNG" value="1277"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e084.TIF" value="1448"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e084.PNG" value="887"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e085.TIF" value="1274"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e085.PNG" value="812"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e086.TIF" value="8614"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e086.PNG" value="3664"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e087.TIF" value="1214"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e087.PNG" value="798"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e088.TIF" value="934"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e088.PNG" value="668"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e089.TIF" value="730"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e089.PNG" value="577"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e090.TIF" value="808"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e090.PNG" value="635"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e091.TIF" value="1250"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e091.PNG" value="832"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e092.TIF" value="1842"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e092.PNG" value="1078"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e093.TIF" value="1218"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e093.PNG" value="813"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e094.TIF" value="1392"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e094.PNG" value="893"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e095.TIF" value="1118"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e095.PNG" value="740"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g003.TIF" value="85508"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g003.PNG_I" value="15746"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g003.PNG_S" value="7940"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g003.PNG_M" value="27057"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.g003.PNG_L" value="48381"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e096.TIF" value="1214"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e096.PNG" value="798"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e097.TIF" value="1778"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e097.PNG" value="1044"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e098.TIF" value="1778"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e098.PNG" value="1044"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e099.TIF" value="1778"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e099.PNG" value="1044"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e100.TIF" value="1214"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e100.PNG" value="798"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e101.TIF" value="1214"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e101.PNG" value="798"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e102.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e102.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e103.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e103.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e104.TIF" value="554"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e104.PNG" value="497"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e105.TIF" value="1838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e105.PNG" value="1099"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e106.TIF" value="574"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e106.PNG" value="493"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e107.TIF" value="1818"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e107.PNG" value="1117"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e108.TIF" value="1276"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e108.PNG" value="814"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e109.TIF" value="1276"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e109.PNG" value="814"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e110.TIF" value="1274"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e110.PNG" value="812"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e111.TIF" value="3732"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e111.PNG" value="1881"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e112.TIF" value="1088"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e112.PNG" value="724"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e113.TIF" value="1088"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e113.PNG" value="724"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e114.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e114.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e115.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e115.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e116.TIF" value="434"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e116.PNG" value="428"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e117.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e117.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e118.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e118.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e119.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e119.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e120.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e120.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e121.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e121.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e122.TIF" value="574"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e122.PNG" value="493"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e123.TIF" value="772"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e123.PNG" value="599"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e124.TIF" value="700"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e124.PNG" value="566"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e125.TIF" value="828"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e125.PNG" value="643"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e126.TIF" value="768"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e126.PNG" value="604"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e127.TIF" value="882"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e127.PNG" value="664"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e128.TIF" value="774"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e128.PNG" value="609"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e129.TIF" value="838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e129.PNG" value="635"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e130.TIF" value="12960"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e130.PNG" value="5770"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e131.TIF" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e131.PNG" value="482"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e132.TIF" value="774"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e132.PNG" value="609"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e133.TIF" value="614"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e133.PNG" value="526"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e134.TIF" value="838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e134.PNG" value="635"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e135.TIF" value="686"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e135.PNG" value="567"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e136.TIF" value="1004"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e136.PNG" value="712"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e137.TIF" value="1048"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e137.PNG" value="727"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e138.TIF" value="596"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e138.PNG" value="517"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e139.TIF" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e139.PNG" value="482"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e140.TIF" value="9320"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e140.PNG" value="4442"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e141.TIF" value="442"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e141.PNG" value="436"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e142.TIF" value="3172"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e142.PNG" value="1775"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e143.TIF" value="632"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e143.PNG" value="534"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e144.TIF" value="462"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e144.PNG" value="444"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e145.TIF" value="872"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e145.PNG" value="648"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e146.TIF" value="1004"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e146.PNG" value="712"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e147.TIF" value="928"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e147.PNG" value="670"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e148.TIF" value="1048"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e148.PNG" value="727"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e149.TIF" value="13988"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e149.PNG" value="6576"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e150.TIF" value="852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e150.PNG" value="635"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e151.TIF" value="890"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e151.PNG" value="651"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e152.TIF" value="708"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e152.PNG" value="570"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e153.TIF" value="638"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e153.PNG" value="511"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e154.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e154.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e155.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e155.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e156.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e156.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e157.TIF" value="1888"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e157.PNG" value="1114"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e158.TIF" value="1382"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e158.PNG" value="891"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e159.TIF" value="1400"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e159.PNG" value="891"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e160.TIF" value="1606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e160.PNG" value="987"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.s001.TIF" value="748144"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e161.TIF" value="2312"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e161.PNG" value="1337"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e162.TIF" value="530"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e162.PNG" value="477"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e163.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e163.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.s002.PDF" value="99949"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e164.TIF" value="1296"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e164.PNG" value="826"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e165.TIF" value="974"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002713.e165.PNG" value="702"/>
<div id="articleContainer">

<span about="http://www.plos.org"  typeof="foaf:Organization" property="foaf:name" style="display:none; visibility:hidden;">PLOS</span>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader" xpathLocation="noSelect">
  <p>Open Access</p><p id="articleType">Research Article</p>
</div>

  <h1 xpathLocation="noSelect" property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">Random Field Model Reveals Structure of the Protein Recombinational Landscape</h1>
  <div property="dc:description" datatype="" style="display:none; visibility:hidden;">Author SummaryMutation and recombination are the primary sources of genetic variation in evolving populations. The relative benefit of these two diversification mechanisms and how they complement each other has been a long-standing question in evolutionary biology. While it is clear what types of genetic diversity these two mechanisms can create, a significant challenge is relating these sequence changes to changes in fitness. The fitness landscape, which describes this mapping from genotype to phenotype, is extraordinarily complex and defined over an incomprehensibly large space of sequences. Here, we develop a model of the landscape that relies not on the details of this mapping, but rather on the statistical relationships between sequences. By studying the expected values of landscape properties, we can gain insights into the structure of the landscape that are independent of the details of how genotype dictates phenotype. We use this random field model to understand how recombination explores a functionally enriched and diverse subset of protein sequence space.</div>
  <span property="dc:date" content="2012-10-04" datatype="xsd:date" rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pcbi.1002713"></span>
  <span property="dc:subject" content="Computational Biology"></span>
  <span property="dc:subject" content="Evolutionary Biology"></span>
<form action="">
  <input type="hidden" name="journalDisplayName" id="journalDisplayName" value="PLOS Computational Biology" />
  <input type="hidden" name="crossRefPageURL" id="crossRefPageURL" value="/article/crossref/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" />
  <input type="hidden" name="twitterPageURL" id="twitterPageURL" value="/article/twitter/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" />
  <input type="hidden" name="metricsTabURL" id="metricsTabURL" value="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" />
  <input type="hidden" name="doi" id="doi" value="info:doi/10.1371/journal.pcbi.1002713" />
  <input type="hidden" name="articleTitleUnformatted" id="articleTitleUnformatted" value="Random%20Field%20Model%20Reveals%20Structure%20of%20the%20Protein%20Recombinational%20Landscape" />
  <input type="hidden" name="articlePubDate" id="articlePubDate" value="1349334000000" />
</form>
<div class="horizontalTabs" xpathLocation="noSelect">
  <ul id="tabsContainer">
        <li id="article" class="active"><a href="/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713;jsessionid=42F26552E5BE8446B0C72379D8339313" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenuBox" xpathLocation="noSelect">
    <div id="articleMenu">
      <div class="wrap">
        <ul>
          <li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" onclick="toggleAnnotation(this, 'public'); return false;" title="Click to turn notes on/off">Hide notes</a></li>
          <li class="discuss icon">
              <a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002713">Make a general comment</a>
          </li>
        </ul>
        <div id="sectionNavTopBox" style="display:none;">
          <p><strong>Jump to</strong></p>
          <div id="sectionNavTop" class="tools"></div>
        </div>
      </div>
    </div>
      <div id="lightBox" xpathLocation="noDialog" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713')">
      </div>
  </div>




<p class="authors" xpathLocation="noSelect"><span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Philip A. Romero</span></span>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Frances H. Arnold</span></span><sup><a href="#cor1" class="fnoteref">*</a></sup></p><p class="affiliations" xpathLocation="noSelect"><a name="aff1" id="aff1"></a>Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, California, United States of America</p><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2 xpathLocation="noSelect">Abstract&nbsp;<a href="#top">Top</a></h2><p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]/p[1]">We are interested in how intragenic recombination contributes to the evolution of proteins and how this mechanism complements and enhances the diversity generated by random mutation. Experiments have revealed that proteins are highly tolerant to recombination with homologous sequences (mutation by recombination is conservative); more surprisingly, they have also shown that homologous sequence fragments make largely additive contributions to biophysical properties such as stability. Here, we develop a random field model to describe the statistical features of the subset of protein space accessible by recombination, which we refer to as the recombinational landscape. This model shows quantitative agreement with experimental results compiled from eight libraries of proteins that were generated by recombining gene fragments from homologous proteins. The model reveals a recombinational landscape that is highly enriched in functional sequences, with properties dominated by a large-scale additive structure. It also quantifies the relative contributions of parent sequence identity, crossover locations, and protein fold to the tolerance of proteins to recombination. Intragenic recombination explores a unique subset of sequence space that promotes rapid molecular diversification and functional adaptation.</p>
</div><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]"><a id="abstract1" name="abstract1" toc="abstract1" title="Author Summary"></a>
<h2 xpathLocation="noSelect">Author Summary&nbsp;<a href="#top">Top</a></h2>
<p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]/p[1]">Mutation and recombination are the primary sources of genetic variation in evolving populations. The relative benefit of these two diversification mechanisms and how they complement each other has been a long-standing question in evolutionary biology. While it is clear what types of genetic diversity these two mechanisms can create, a significant challenge is relating these sequence changes to changes in fitness. The fitness landscape, which describes this mapping from genotype to phenotype, is extraordinarily complex and defined over an incomprehensibly large space of sequences. Here, we develop a model of the landscape that relies not on the details of this mapping, but rather on the statistical relationships between sequences. By studying the expected values of landscape properties, we can gain insights into the structure of the landscape that are independent of the details of how genotype dictates phenotype. We use this random field model to understand how recombination explores a functionally enriched and diverse subset of protein sequence space.</p>
</div>


<div class="articleinfo" xpathLocation="noSelect"><p><strong>Citation: </strong>Romero PA, Arnold FH (2012) Random Field Model Reveals Structure of the Protein Recombinational Landscape. PLoS Comput Biol 8(10):
          e1002713.
            doi:10.1371/journal.pcbi.1002713</p><p><strong>Editor: </strong>Peter Joseph Artymiuk, 
        University of Sheffield, United Kingdom
      </p><p></p><p><strong>Received:</strong> June 5, 2012; <strong>Accepted:</strong> August 3, 2012; <strong>Published:</strong> October 4, 2012</p><p><strong>Copyright:</strong> ©  Romero, Arnold. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors acknowledge support from the National Institutes of Health, ARRA (grant R01 GM068664) for funding the theoretical and P450 chimera work, and the U.S. Army Research Office, Institute for Collaborative Biotechnologies (grant W911NF-09-D-0001) for funding design and construction of the cellulase libraries. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p><p><a name="cor1"></a>* E-mail: <a href="mailto:frances@cheme.caltech.edu">frances@cheme.caltech.edu</a></p></div>





<div id="section1" xpathLocation="/article[1]/body[1]/sec[1]"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3 xpathLocation="noSelect">Introduction&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[1]/p[1]">The ubiquity of sex and recombination suggests a significant role in evolution, yet their benefit is still debated <a href="#pcbi.1002713-Barton1">[1]</a>, <a href="#pcbi.1002713-Otto1">[2]</a>. Intragenic recombination events generate chimeric genes, which are believed to make important contributions to allelic diversity in natural populations <a href="#pcbi.1002713-Watt1">[3]</a>–<a href="#pcbi.1002713-DeSilva1">[6]</a>. Laboratory experiments clearly demonstrate the benefits of recombining homologous proteins: intragenic recombination generates new proteins that are functionally diverse while still having a high probability of folding properly and functioning <a href="#pcbi.1002713-Crameri1">[7]</a>, <a href="#pcbi.1002713-Carbone1">[8]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[2]">We have developed techniques for the design, construction, and characterization of libraries of chimeric proteins generated by site-directed recombination of homologous sequences <a href="#pcbi.1002713-Voigt1">[9]</a>–<a href="#pcbi.1002713-Heinzelman1">[12]</a>. Briefly, libraries are designed (i.e. crossover sites are selected) to minimize the number of novel residue contacts that are generated upon recombination (we call this number ‘SCHEMA disruption’), which tend to be deleterious to protein function. The sequence fragments chosen this way are then shuffled to generate a combinatorial library of chimeric proteins. The resulting proteins have no random point mutations; all the ‘mutations’ are homologous, that is, to amino acids that already appear in at least one of the parent sequences. These libraries can be used to explore the nature of recombination, without the high levels of random mutations typically found in protein libraries made by DNA shuffling <a href="#pcbi.1002713-Crameri1">[7]</a> and other, similar methods for homologous recombination <em>in vitro</em>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[3]">To date, this laboratory has constructed and tested eight such recombination libraries consisting of chimeric bacterial <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e001&amp;representation=PNG"></span>-lactamases (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e002&amp;representation=PNG"></span>lac13 and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e003&amp;representation=PNG"></span>lac), bacterial cytochrome P450s (P450), fungal family 5 cellulases (Cel5), bacterial family 48 cellulases (Cel48), fungal class I cellobiohydrolases (CBHI), fungal class II cellobiohydrolases (CBHII), and human arginases (Arg) (<a href="#pcbi-1002713-t001">Table 1</a>). Each library, which typically consists of thousands of new sequences, provides a glimpse of the protein fitness landscape that is accessible by recombination, which we refer to as the recombinational landscape. Since every member of the library can be generated by recombining other members, the genetic diversity in these libraries has similarities to that of a sexually reproducing population.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[1]/table-wrap[1]"><a name="pcbi-1002713-t001" id="pcbi-1002713-t001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.t001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.t001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.t001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[1]/table-wrap[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.t001');"><span>Table 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[1]/table-wrap[1]/caption[1]/title[1]">Summary of eight protein recombination libraries.</span></strong></p><span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002713.t001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002713.t001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.t001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002713.t001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.t001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002713.t001.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[1]/p[4]">Studies of these libraries have highlighted the enrichment of functional sequences in the recombinational landscape: SCHEMA-designed libraries contain a significant proportion (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e014&amp;representation=PNG"></span>20–50%) of functional sequences, despite having a high average mutation level (i.e. average distance of a chimera sequence from its closest parent). For comparison, random mutation libraries with the same number of mutations are estimated to contain 10–20 orders of magnitude fewer functional sequences <a href="#pcbi.1002713-Guo1">[13]</a>–<a href="#pcbi.1002713-Bershtein1">[15]</a>. Whereas random mutations cause the probability a sequence remains functional to decrease exponentially, mutation by recombination always moves towards other functional sequences and is therefore significantly more conservative <a href="#pcbi.1002713-Drummond1">[16]</a>. For this reason, intragenic recombination effectively explores functional ridges through a protein sequence space that is mostly nonfunctional.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[5]">These libraries have also revealed significant variation in thermostability <a href="#pcbi.1002713-Li1">[17]</a>, <a href="#pcbi.1002713-Heinzelman2">[18]</a> and other properties <a href="#pcbi.1002713-Landwehr1">[19]</a>–<a href="#pcbi.1002713-Smith1">[21]</a> within the recombinational landscape. We observed that most of this variation can be explained by additive effects <a href="#pcbi.1002713-Li1">[17]</a>, <a href="#pcbi.1002713-Heinzelman2">[18]</a>, <a href="#pcbi.1002713-Romero1">[20]</a>–<a href="#pcbi.1002713-Heinzelman3">[22]</a>. That is, the stability, for example, of a chimeric protein can be expressed as the sum of contributions from each of its sequence fragments. This additivity can be used to efficiently engineer highly optimized chimeric proteins for a variety of applications <a href="#pcbi.1002713-Li1">[17]</a>, <a href="#pcbi.1002713-Romero1">[20]</a>, <a href="#pcbi.1002713-Heinzelman3">[22]</a>, <a href="#pcbi.1002713-Heinzelman4">[23]</a>. The additive structure, or lack of epistasis, within the recombinational landscape may provide an abundance of adaptive pathways for natural protein evolution.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[6]">We would like to understand the features of the recombinational landscape that contribute to its extreme enrichment in functional sequences and its additive structure. Since the details of the protein recombinational landscape are unknown, we develop a random field model which captures its statistical properties. Random field models are effective at describing statistical features of uncertain, spatially-organized functions, with applications ranging from geostatistics to image analysis <a href="#pcbi.1002713-Adler1">[24]</a>–<a href="#pcbi.1002713-Li2">[26]</a>. This versatile class of models has also been used to describe fitness landscapes <a href="#pcbi.1002713-Stadler1">[27]</a>, the best known example being Kauffman's <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e015&amp;representation=PNG"></span>-model <a href="#pcbi.1002713-Kauffman1">[28]</a>. Our random field model for the recombinational landscape uses a physics-inspired energy function to describe the sequence-fitness relationship and is parametrized with experimental data. Using this model, we derive approximations for the fraction of functional sequences within a recombination library and the degree of landscape additivity, and we relate these quantities back to experimental observations. We discuss how the structure of the recombinational landscape contributes to the utility of recombination in evolution.</p>
</div>

<div id="section2" xpathLocation="/article[1]/body[1]/sec[2]"><a id="s2" name="s2" toc="s2" title="Results/Discussion"></a><h3 xpathLocation="noSelect">Results/Discussion&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/title[1]">Random field model of the protein recombinational landscape</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]">We use a pairwise, residue-level energy function to describe the large number of intramolecular interactions that stabilize protein structures (<a href="#pcbi-1002713-g001">Figure 1</a>). Such simplified contact potentials have been used in the past for protein folding simulations and structure prediction <a href="#pcbi.1002713-Miyazawa1">[29]</a>–<a href="#pcbi.1002713-Vendruscolo1">[31]</a>. Assuming a fixed structure (set of residue-residue contacts), the energy of any sequence is the sum of energy terms associated with the sequence's specific residue combinations at every pair of contacting residues. For chimeric proteins we distinguish between two types of contacts: parental (P) contacts, which are residue pairs observed in at least one of the parents, and novel (N) contacts, which are not (<a href="#pcbi-1002713-g001">Figure 1</a>). With this model, the energy of any chimeric protein <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e016&amp;representation=PNG"></span> is given by summing the contact energies<br><a name="pcbi.1002713.e017" id="pcbi.1002713.e017"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e017&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]/disp-formula[1]/label[1]">(1)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e018&amp;representation=PNG"></span> is the energy term associated with parental contact <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e019&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e020&amp;representation=PNG"></span> is the energy term associated with novel contact <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e021&amp;representation=PNG"></span>, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e022&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e023&amp;representation=PNG"></span> are binary variables which indicate the specific residue pairs for each contact <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e024&amp;representation=PNG"></span> in chimeric protein <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e025&amp;representation=PNG"></span>. Since the specific energy values of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e026&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e027&amp;representation=PNG"></span> are unknown, we define the independent and identically distributed random numbers <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e028&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e029&amp;representation=PNG"></span>, distributed with means and variances<br><a name="pcbi.1002713.e030" id="pcbi.1002713.e030"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e030&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]/disp-formula[2]/label[1]">(2)</span></span><br><br><a name="pcbi.1002713.e031" id="pcbi.1002713.e031"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e031&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]/disp-formula[3]/label[1]">(3)</span></span><br>Substituting these random variables into <a href="#pcbi.1002713.e017">equation 1</a> defines a random energy function associated with any chimeric protein <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e032&amp;representation=PNG"></span><br><a name="pcbi.1002713.e033" id="pcbi.1002713.e033"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e033&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]/disp-formula[4]/label[1]">(4)</span></span><br>This random energy function is defined over the parental subspace <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e034&amp;representation=PNG"></span>, the set of all sequences that can be generated by recombining the parent sequences, which specifies the random field<br><a name="pcbi.1002713.e035" id="pcbi.1002713.e035"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e035&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]/disp-formula[5]/label[1]">(5)</span></span><br>The expected value of the random field at chimeric protein <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e036&amp;representation=PNG"></span> is<br><a name="pcbi.1002713.e037" id="pcbi.1002713.e037"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e037&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]/disp-formula[6]/label[1]">(6)</span></span><br>and the covariance between any two sequences is<br><a name="pcbi.1002713.e038" id="pcbi.1002713.e038"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e038&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]/disp-formula[7]/label[1]">(7)</span></span><br>Importantly, this covariance structure expresses how pairs of sequences are related and captures our intuitive notion of protein similarity: proteins with similar sequences have similar structures and therefore similar properties. This random field model provides a statistical description of the recombinational landscape.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]"><a name="pcbi-1002713-g001" id="pcbi-1002713-g001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.g001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g001');"><span>Figure 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/title[1]">Contact model of protein recombination.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/p[1]">When homologous proteins are recombined, structural fragments are acquired from different parents (colored red and blue). Here, lines illustrate contacts between positions that contain residues within 4.5 Å and that are not conserved in the parent sequence alignment. When these nonconserved contacts span structural fragments (i.e., between red and blue) they generate novel (N) interactions that are not observed in either parent. All other contacts, including those between conserved positions (not shown) and those within parental fragments (red-red or blue-blue), provide parental (P) interactions that are found in at least one of the parent structures.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002713.g001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002713.g001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.g001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002713.g001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.g001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002713.g001.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[2]">To parametrize the random field model, we must determine the mean energy <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e039&amp;representation=PNG"></span> and variance <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e040&amp;representation=PNG"></span> of parental contacts and the equivalent parameters <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e041&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e042&amp;representation=PNG"></span> for novel contacts. Using a large binary functional status data set from a library made by recombining three bacterial cytochrome P450 enzyme heme domains <a href="#pcbi.1002713-Otey1">[32]</a>, these four parameters were estimated by maximizing a marginalized likelihood function (see <a href="#s3">Methods</a>). If we assume the functional status depends on a sequence's Gibbs free energy difference from the nonfunctional state, these estimated parameters can be interpreted as Gibbs free energy differences in <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e043&amp;representation=PNG"></span> units because the two-state Boltzmann distribution is identical to the logistic likelihood function. As expected, parental contacts are slightly stabilizing (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e044&amp;representation=PNG"></span>), novel contacts are significantly destabilizing (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e045&amp;representation=PNG"></span>), and both classes of contacts show significant variation relative to their means (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e046&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e047&amp;representation=PNG"></span>). Estimating these parameters on recombination data from other protein families yields qualitatively similar relationships (<a href="#pcbi.1002713.s001">Figure S1</a>). This is not surprising, considering that most proteins are marginally stable <a href="#pcbi.1002713-Taverna1">[33]</a> and mutations (novel contacts) tend to be deleterious to protein function <a href="#pcbi.1002713-Guo1">[13]</a>–<a href="#pcbi.1002713-Bershtein1">[15]</a>. In the following sections, we use this parametrized random field model to interpret experimental observations from protein recombination libraries.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/title[1]">Effect of homologous substitutions on protein function</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[1]">Previously, we compared the effects of random versus homologous amino acid substitutions <a href="#pcbi.1002713-Drummond1">[16]</a>. Whereas the fraction of functional sequences declines exponentially with increasing random mutations <a href="#pcbi.1002713-Guo1">[13]</a>, <a href="#pcbi.1002713-Bloom1">[14]</a>, that fraction varies log-parabolically with the number of substitutions taken from another functional parent. For two parents, the log-parabolic behavior appears because accumulating homologous substitutions must eventually convert one functional parent sequence into another functional parent sequence. Random mutagenesis of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e048&amp;representation=PNG"></span>-lactamase indicated a probability that a single random mutation will preserve function (neutrality) of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e049&amp;representation=PNG"></span>0.54, whereas recombination experiments on the same enzyme indicated the probability a homologous substitution will preserve function (recombinational tolerance) was <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e050&amp;representation=PNG"></span>0.79 <a href="#pcbi.1002713-Drummond1">[16]</a>. A recombinational tolerance significantly larger than the neutrality indicates that homologous substitutions tend to be more conservative than random ones. Here, we evaluate the effects of homologous substitutions using the random field model and compare the results to this previous analysis.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[2]">Analyzing a library of chimeric <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e051&amp;representation=PNG"></span>-lactamases (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e052&amp;representation=PNG"></span>lac13) <a href="#pcbi.1002713-Meyer1">[34]</a>, the probability of functioning for each chimera was estimated by evaluating the logistic function <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e053&amp;representation=PNG"></span> at the expected value of the random field (<a href="#pcbi.1002713.e037">equation 6</a>). These probabilities were averaged within 15 groups of chimeras binned by their number of homologous substitutions. The same analysis was also performed on simulated random substitutions, where a novel contact was any residue pair not present in the two <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e054&amp;representation=PNG"></span>-lactamase parents. With two parents, at least 18/19 random mutations will result in non-parental amino acids and therefore novel interactions with any contacting residues. The random field model results show excellent agreement with the experimental results of substitutions generated by recombination and randomly (<a href="#pcbi-1002713-g002">Figure 2<em>A</em></a>). As observed previously, the fraction of functional sequences undergoes a steep exponential decline with random mutations, while functionality displays a log-parabolic dependence on homologous substitutions.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]"><a name="pcbi-1002713-g002" id="pcbi-1002713-g002" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g002');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g002');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.g002&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g002');"><span>Figure 2. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/title[1]">Effect of homologous substitutions on the fraction of functional sequences in a library of chimeric <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e055&amp;representation=PNG"></span>-lactamases.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/p[1]">(<em>A</em>) The random field model agrees well with experimental data on random and homologous substitutions in <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e056&amp;representation=PNG"></span>-lactamase <a href="#pcbi.1002713-Drummond1">[16]</a>. The parabolic curve displays the effect of homologous substitutions, and the error bars represent the 95% confidence intervals of the fraction of correctly constructed chimeras (see <a href="#s3">Methods</a>). The steep exponential curves (and inset) show the effect of random mutations, and the error bars represent one standard error. (<em>B</em>) As parent sequence identity decreases, the homologous substitution curves stretch to higher levels of mutation and lower fraction functional. Shown are the substitution curves for the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e057&amp;representation=PNG"></span>lac13 library (crossover locations and contacts) averaged over 100 random parent sequences with sequence identity ranging from 20–80%. (<em>C</em>) As the number of crossovers <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e058&amp;representation=PNG"></span> decreases, the homologous substitution curve shifts towards a higher fraction functional. Shown are the substitution curves for the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e059&amp;representation=PNG"></span>lac13 library (parents and contacts) averaged over 100 random crossover locations with the number of crossovers varying from 6 to 27. The random homologous substitution curve was generated by averaging over 100 randomly sampled sequences at each level of mutation.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002713.g002</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002713.g002/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.g002/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002713.g002.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.g002/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002713.g002.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[3]">With the random field model, we can now explore how key recombination parameters, such as parent sequence identity or the number of sequence crossovers, influence the shape of the recombination curve shown in <a href="#pcbi-1002713-g002">Figure 2<em>A</em></a>. As the sequence identity shared by the parents decreases, the curve stretches to a higher level of mutation (more mutations are possible for a fixed sequence length) and to a lower fraction functional (<a href="#pcbi-1002713-g002">Figure 2<em>B</em></a>), as was shown previously using lattice protein simulations <a href="#pcbi.1002713-Drummond1">[16]</a>. Here we see that homologous substitutions from more-distant parents tend to be more deleterious to protein function than substitutions from less-distant proteins. This happens because distant proteins are more likely to have their contact networks composed of different residues, and these networks are therefore less compatible when recombined. We also see that as the number of sequence crossovers decreases, the log-parabolic recombination curve shifts towards a higher fraction functional (<a href="#pcbi-1002713-g002">Figure 2<em>C</em></a>), necessarily approaching a flat line when there are no crossovers. This happens because each crossover event creates opportunities to generate deleterious interactions. This improvement to the previous analysis allows us to see how recombinational tolerance depends on the number of sequence crossovers. To estimate the effects of homologous amino acid substitutions independent of the number of crossovers, we sampled random homologous substitutions and calculated the average probability of folding at each level of mutation (<a href="#pcbi-1002713-g002">Figure 2<em>C</em></a>). The effects of random homologous substitutions still follow the log-parabolic curve, although this curve dips over five orders of magnitude lower than the curve obtained from the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e060&amp;representation=PNG"></span>-lactamase library experiments <a href="#pcbi.1002713-Meyer1">[34]</a>. Fitting the log-parabolic equation <a href="#pcbi.1002713-Drummond1">[16]</a>, we estimate the recombinational tolerance of random homologous substitutions to be <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e061&amp;representation=PNG"></span>. The recombinational tolerance is still significantly greater than the neutrality (0.54), but to a lesser degree than previously estimated.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[4]">The only difference between random homologous substitutions and those generated by recombination (<a href="#pcbi-1002713-g002">Figure 2<em>C</em></a>) is how the mutations are distributed throughout the sequence and structure. Random homologous substitutions are distributed uniformly throughout the sequence, while those generated by recombination occur in contiguous stretches of sequence. By making mutations in groups, recombination preserves many local interactions. From this analysis, we propose an updated model for the conservative nature of intragenic recombination which includes contributions from homologous substitutions (as shown previously) as well as groups of coevolved residues that vary simultaneously. The latter effect is expected to be particularly important in natural evolution, where the number of intragenic crossover events per generation is likely to be small.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[5]">It is interesting that the random field model for the recombinational landscape also works reasonably well to describe the effects of random mutations. Random mutations frequently result in a non-parental amino acid and therefore cause deleterious novel interactions with all contacting residues. This simplified model recapitulates the exponential decline in functional sequences that was observed upon random mutagenesis of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e062&amp;representation=PNG"></span>-lactamase (<a href="#pcbi-1002713-g002">Figure 2<em>A</em></a>) and in other mutational studies <a href="#pcbi.1002713-Guo1">[13]</a>–<a href="#pcbi.1002713-Bershtein1">[15]</a>. In addition, this model trivially captures the well-known fact that surface mutations tend to be less deleterious than mutations in the protein core, because core residues tend to have many more contacts. With a single model to explain the effects of both random and homologous substitutions, we can understand their differences in terms of residue contacts. The number of deleterious contacts generated by a homologous substitution is less than or equal to the number generated by a random mutation at the same position, with equality rarely being achieved. This is consistent with the explanation that homologous substitutions are conservative because they have been previously selected to be compatible with the protein fold <a href="#pcbi.1002713-Drummond1">[16]</a>.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/title[1]">Effect of intragenic recombination across protein families</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[1]">The factors that determine a particular protein family's tolerance to recombination are unknown. <a href="#pcbi-1002713-t001">Table 1</a> reports the fraction of functional sequences in eight recombination libraries, representing protein families of different functions, sizes, and fold classes. Seven of these libraries were designed with the intent of maximizing the fraction of functional sequences, yet there is significant variation (2–3 fold) in this fraction between libraries. While some of this variation is likely due to experimental differences in classifying functional versus nonfunctional sequences for different enzymes, we expect a significant proportion of this variation to arise from differences in parent fold, parent sequence identity, and the specific crossover locations chosen in the library design. Using the random field model, we derive an approximation for the expected value of the fraction of functional sequences in a recombination library and use this to understand how these factors contribute to a protein's tolerance to recombination.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[2]">Consider a recombination library <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e063&amp;representation=PNG"></span> generated by recombining sequence fragments from <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e064&amp;representation=PNG"></span> parental sequences at <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e065&amp;representation=PNG"></span> crossover sites. We refer to the sequence fragments between crossover sites as ‘blocks’; therefore the library is composed of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e066&amp;representation=PNG"></span> sequence blocks (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e067&amp;representation=PNG"></span>). All sequence fragments in these libraries are roughly the same length, and therefore, with the random field model, we can assume that each fragment's energetic contribution is an independently and identically distributed Gaussian random variable. With this assumption, the distribution of sequence energies within recombination library <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e068&amp;representation=PNG"></span> is Gaussian and can be described by its mean<br><a name="pcbi.1002713.e069" id="pcbi.1002713.e069"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e069&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[2]/disp-formula[1]/label[1]">(8)</span></span><br>and variance<br><a name="pcbi.1002713.e070" id="pcbi.1002713.e070"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e070&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[2]/disp-formula[2]/label[1]">(9)</span></span><br>The fraction of functional sequences within library <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e071&amp;representation=PNG"></span> is given by evaluating the Gaussian cumulative distribution function at zero<br><a name="pcbi.1002713.e072" id="pcbi.1002713.e072"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e072&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[2]/disp-formula[3]/label[1]">(10)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e073&amp;representation=PNG"></span> is the error function.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[3]">Since the specific energy terms that shape the recombinational landscape are unknown, we use the random field model to calculate the expected value of the fraction of functional sequences by integrating over all possible energy terms <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e074&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e075&amp;representation=PNG"></span>. The expected value of the library mean is given by<br><a name="pcbi.1002713.e076" id="pcbi.1002713.e076"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e076&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[3]/disp-formula[1]/label[1]">(11)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e077&amp;representation=PNG"></span> is the total number of contacts and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e078&amp;representation=PNG"></span> is the number of novel contacts in chimera <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e079&amp;representation=PNG"></span>. The expected value of the library variance is given by<br><a name="pcbi.1002713.e080" id="pcbi.1002713.e080"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e080&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[3]/disp-formula[2]/label[1]">(12)</span></span><br>More specific details of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e081&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e082&amp;representation=PNG"></span>, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e083&amp;representation=PNG"></span> are given in <a href="#pcbi.1002713.s002">Text S1</a>. With these two expectations, the expected value of the fraction of functional sequences can be approximated with a leading-order Taylor series expansion about <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e084&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e085&amp;representation=PNG"></span><br><a name="pcbi.1002713.e086" id="pcbi.1002713.e086"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e086&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[3]/disp-formula[3]/label[1]">(13)</span></span><br></p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[4]">The expected value of the fraction of functional sequences within a library <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e087&amp;representation=PNG"></span> shows quantitative agreement with the experimentally determined values, as shown in <a href="#pcbi-1002713-g003">Figure 3<em>A</em></a>. With the random field model, both parental and novel contacts contribute to the distribution of sequence energies within a recombination library and therefore to the fraction of functional sequences. The deleterious novel contacts dictate the mean energy of the library (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e088&amp;representation=PNG"></span>), while parental contacts, which typically outnumber novel contacts 50–100-fold, dominate the variance (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e089&amp;representation=PNG"></span>). This suggests recombination events can cause loss of function by two independent mechanisms: (1) by introducing new deleterious interactions between sequence fragments, or (2) by introducing sequence fragments that already contain deleterious interactions.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]"><a name="pcbi-1002713-g003" id="pcbi-1002713-g003" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g003');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g003');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.g003&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002713?imageURI=info:doi/10.1371/journal.pcbi.1002713.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002713', 'info:doi/10.1371/journal.pcbi.1002713.g003');"><span>Figure 3. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/title[1]">Comparison between library properties and their expected values within the random field model.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/p[1]">Note diagonal lines represent <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e090&amp;representation=PNG"></span>. (<em>A</em>) The random field's expected fraction of functional sequences shows quantitative agreement with experimental results (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e091&amp;representation=PNG"></span> with <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e092&amp;representation=PNG"></span>). Error bars represent the binomial 95% confidence intervals calculated using the Clopper-Pearson method <a href="#pcbi.1002713-Clopper1">[51]</a>. (<em>B</em>) The expected additivity agrees well with experimentally determined values (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e093&amp;representation=PNG"></span> with <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e094&amp;representation=PNG"></span>). While the small data set limits the statistical significance of this correlation, all <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e095&amp;representation=PNG"></span>s are large and within the ranges that are observed experimentally.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002713.g003</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002713.g003/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.g003/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002713.g003.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002713.g003/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002713.g003.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[5]">To better understand the variation in the fraction of functional sequences in the different recombination libraries, we sampled random libraries, calculated <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e096&amp;representation=PNG"></span>, and estimated the contribution from protein fold, specific breakpoints, and parent sequence identity. For each protein fold, we sampled 100 random two-parent sequence alignments with sequence identity ranging from 10–90%, and for each of these alignments we sampled 100 random 7-crossover libraries, for a total of 90,000 libraries. A three-way analysis of variance shows the protein fold (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e097&amp;representation=PNG"></span>), specific breakpoints (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e098&amp;representation=PNG"></span>), and parent sequence identity (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e099&amp;representation=PNG"></span>) all make significant contributions to the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e100&amp;representation=PNG"></span>. Estimating the variance components, we find parent sequence identity to be the main determinant of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e101&amp;representation=PNG"></span> (contributing 92% of the variance), followed by specific crossover locations (4%), and protein fold (2%). This strong dependence on parent sequence identity is the result of the approximately exponential increase in the number of (deleterious) novel contacts as parent sequences diverge.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[6]">Interestingly, the parent sequence identity also dictates the mechanism of chimeric protein inactivation. When the parent sequence identity is low, most of the nonfunctional chimeric proteins are the result of new deleterious interactions between sequence fragments. However, when the parent sequence identity is high, nonfunctional sequences are usually the result of inheriting sequence fragments which already contain deleterious interactions. This is consistent with the observation of high mutual information between a chimeric protein's functional status and its number of novel contacts for the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e102&amp;representation=PNG"></span>-lactamase library (low parent sequence identity) and the low mutual information observed for the P450 library (high parent sequence identity) <a href="#pcbi.1002713-Endelman2">[35]</a>. In the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e103&amp;representation=PNG"></span>-lactamase library, the number of new interactions between fragments (novel contacts) is predictive of the functional status of chimeras. However, in the P450 library, the number of novel contacts is not predictive, suggesting other mechanisms must be responsible for chimera inactivation (i.e. acquisition of deleterious sequence fragments).</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/title[1]">Additive structure of the recombinational landscape</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[1]">Perhaps the most surprising finding from protein recombination experiments has been the additive structure of the recombinational landscape <a href="#pcbi.1002713-Li1">[17]</a>, <a href="#pcbi.1002713-Romero1">[20]</a>, <a href="#pcbi.1002713-Heinzelman3">[22]</a>, <a href="#pcbi.1002713-Heinzelman4">[23]</a>, <a href="#pcbi.1002713-LeMaster1">[36]</a>. Linear models are able to explain a majority of variation in stability as well as some other properties, suggesting that sequence elements make largely independent, additive contributions to a protein's overall properties. In quantitative genetics, this is referred to as additive genetic variance, which according to Fisher's fundamental theorem of natural selection determines a population's response to selection <a href="#pcbi.1002713-Fisher1">[37]</a>, <a href="#pcbi.1002713-Charlesworth1">[38]</a>. Additive landscapes are easier for evolving populations to climb because they are not stymied by rugged, epistatic features. This additivity has been especially useful for engineering optimized chimeric proteins in the laboratory, because a small sampling of sequences provides sufficient information to make accurate predictions across the entire library <a href="#pcbi.1002713-Li1">[17]</a>, <a href="#pcbi.1002713-Heinzelman3">[22]</a>, <a href="#pcbi.1002713-Heinzelman4">[23]</a>. Here, we use the random field model to understand the origin of the additive structure within the recombinational landscape.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[2]">Within the recombination library <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e104&amp;representation=PNG"></span> described in the previous section, the total variance can be partitioned into additive and epistatic components (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e105&amp;representation=PNG"></span>). We define the landscape's additivity <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e106&amp;representation=PNG"></span> as the fraction of the total variance that is explained by additive effects<br><a name="pcbi.1002713.e107" id="pcbi.1002713.e107"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e107&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[2]/disp-formula[1]/label[1]">(14)</span></span><br>This dimensionless quantity, which ranges from 0 to 1, describes the smoothness of the landscape and is inversely related to the landscape ‘ruggedness’ defined in <a href="#pcbi.1002713-Carneiro1">[39]</a>. For four of the recombination libraries, there are sufficient data to calculate the additivity of the thermostability landscape (see <a href="#s3">Methods</a>). These results are presented in <a href="#pcbi-1002713-t001">Table 1</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[3]">The additive variation can be understood by considering how individual mutations contribute to variation in the library. A mutation that occurs at a position with a fixed structural context, such as a mutation within a structural fragment inherited from one parent or a mutation surrounded by conserved positions, will always have the same effect throughout the library and therefore contributes entirely to additive variation. However, a mutation can have different effects in different sequences if it occurs at a position whose environment varies. The effects of these mutations can only be expressed with an epistatic (non-additive) model, but their additive contribution can be found by averaging their effects over all structural environments within the library. An additive energy function can be defined by accounting for purely additive and averaged epistatic effects (<a href="#pcbi.1002713.s002">Text S1</a>). This additive energy can be used to calculate the expected value of a library's additive variance <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e108&amp;representation=PNG"></span> using the same equations as the total variance (previous section). With this, the expected value of the additivity can be approximated with a Taylor series expansion about <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e109&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e110&amp;representation=PNG"></span><br><a name="pcbi.1002713.e111" id="pcbi.1002713.e111"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e111&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[3]/disp-formula[1]/label[1]">(15)</span></span><br></p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[4]">The expected value of the landscape additivity <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e112&amp;representation=PNG"></span> shows close agreement with the experimentally determined values (<a href="#pcbi-1002713-g003">Figure 3<em>B</em></a>). While the correlation is not statistically significant, due to the limited data, all the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e113&amp;representation=PNG"></span>s are large and within the experimentally observed ranges. In addition, the four uncharacterized libraries also have large expected additivities (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e114&amp;representation=PNG"></span>lac13 = 0.44, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e115&amp;representation=PNG"></span>lac = 0.67, Cel5 = 0.65, Arg = 0.82), suggesting this additive structure within the recombinational landscape may be quite general. Despite being generated by a purely pairwise energy function, which is by definition epistatic, a majority of the variation within these recombination libraries can be explained by additive effects. This surprising result can be attributed to two factors: sequence conservation among the parents and the partitioning of interactions into structural modules. Epistatic interactions that are conserved among the parents will not contribute to the variation of any property within the library, and those interactions involving one conserved position will only contribute to additive variation. Epistatic interactions that are partitioned into structural modules will vary together, and therefore contribute to only additive variation. Of the thousands of contacting residues within a chimeric protein, only a small fraction (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e116&amp;representation=PNG"></span>5%) actually contribute to epistatic variation.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[5]">The additivity exhibited by the random field model does not hold for chimeric proteins that adopt alternate structures (as described by a contact map). For example, nonfunctional sequences, which account for a significant proportion of chimeras, will clearly not display additivity in properties involving protein function. For many properties, such as thermostability (retention of function at elevated temperatures), where we have observed additivity, the experimental measurements require the chimeras be enzymatically active, which greatly increases the likelihood that they will adopt the same or very similar structures. The subset of sequences that adopt the same structure is referred to as a neutral network <a href="#pcbi.1002713-BornbergBauer1">[40]</a>, <a href="#pcbi.1002713-Xia1">[41]</a>, and this may define the domain of additivity within the recombinational landscape.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/title[1]">Summary and conclusions</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[1]">By using a statistical description of the protein recombinational landscape, we can study the behavior of an astronomical number of sequences–insight which could not be obtained experimentally or even by analyzing homology-based structural models. A probabilistic contact potential was used to specify the mean energy of individual chimeric proteins and how the energy of any sequence is expected to co-vary with others (<a href="#pcbi.1002713.e037">equations 6</a> and <a href="#pcbi.1002713.e038">7</a>), defining a multivariate probability distribution over all sequences accessible by recombination. While this random field model provides little information about specific sequences, it does reveal the large-scale structure of the recombinational landscape, which we used here to interpret experimental results from past recombination libraries. Within this random field, the expected values of various library properties show excellent agreement with experimental values across multiple protein families. This striking correspondence may arise because a library's properties depend on a large number of interactions, and the cumulative effects of these interactions converge toward the expected value.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[2]">The random field model was used to study the enrichment of functional sequences within the recombinational landscape. As shown previously, we found the tolerance of proteins to recombination to be influenced by the conservative effects of homologous substitutions, which have been previously selected to be compatible with the protein fold <a href="#pcbi.1002713-Drummond1">[16]</a>. However, a more significant contribution comes from groups of coevolved residues varying together. This is especially relevant for understanding natural evolution, where the number of crossover events is relatively low. Evaluating the random field model across protein families, we found parent sequence identity to be the primary determinant for tolerance to recombination, while the specific crossover locations and parent fold make statistically significant, but minor contributions.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[3]">Using the random field model, we explored the origins of the additive structure of the recombinational landscape. Both sequence conservation among the parents and the partitioning of epistatic interactions into structural modules make significant contributions to this additivity. The results presented here are for a random field that describes a protein's free energy difference between the functional and non-functional states, which is closely related to protein stability. However, this additivity is generally true for any landscape that is generated from local interactions (including higher order), because sequence conservation and structural partitioning will still be present. This suggests the additivity may apply to numerous biophysical quantities such as binding affinity or substrate specificity.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[4]">Previous studies of protein fitness landscapes have highlighted the abundance of nonfunctional sequences <a href="#pcbi.1002713-Keefe1">[42]</a>, <a href="#pcbi.1002713-Axe1">[43]</a> and neutral sequence changes <a href="#pcbi.1002713-Guo1">[13]</a>, <a href="#pcbi.1002713-Bloom1">[14]</a>, <a href="#pcbi.1002713-Bloom2">[44]</a>, suggesting a surface which is mostly flat and filled with holes <a href="#pcbi.1002713-Gavrilets1">[45]</a>. In contrast to this full landscape, the recombinational landscape contains orders of magnitude fewer ‘holes’ (non-functional sequences). The functional variation displayed within recombination libraries reveals the large-scale structure of the recombinational landscape, which arises from the cumulative effects of multiple mutations. In addition, most of this functional variation can be explained by additive effects, and additive variation determines a population's response to selection <a href="#pcbi.1002713-Fisher1">[37]</a>, <a href="#pcbi.1002713-Charlesworth1">[38]</a> These results were observed in SCHEMA-designed libraries, which tend to be optimized for both functional sequences and additivity. This emphasizes the evolutionary preference for some crossover sites over others, which could explain the presence of recombination hotspots in natural genes <a href="#pcbi.1002713-DeSilva1">[6]</a>, <a href="#pcbi.1002713-Yip1">[46]</a>, <a href="#pcbi.1002713-McBee1">[47]</a>. The picture of the recombinational landscape that emerges from the random field model is a surface enriched in functional sequences, which can display locally-epistatic behavior but still has an overall additive structure.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[5]">The evolutionary benefit of intragenic recombination may arise because mutation and recombination effectively traverse different landscapes <a href="#pcbi.1002713-Watson1">[48]</a>. While climbing the landscape by point mutations, evolution encounters a large number of nonfunctional sequences in addition to epistatic landscape features. In contrast, recombination explores sequences which are much more likely to be functional, in a landscape with an abundance of adaptive pathways. Recombination can provide faster adaptation than point mutation because it generates functional sequences with a large number of substitutions. Recombination may also find sequences that are inaccessible by adaptive point mutation, by simultaneously incorporating multiple coupled mutations, essentially ‘jumping over’ epistatic landscape features. A similar effect was recently described for recombination at the genome level <a href="#pcbi.1002713-Watson2">[49]</a>, where the authors describe how landscapes arising from high epistasis within genes and no epistasis between genes strongly favors recombination. Running simulations on these ‘modular’ landscapes, the authors found recombination to provide an efficient route to genotypes that were inaccessible by point mutation.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[6]">Intragenic recombination is a powerful molecular diversification mechanism. The ubiquity of intragenic recombination in nature and experimental evidence from protein recombination libraries show that it provides distinct advantages over point mutation. In naturally evolving populations, these two genetic variation mechanisms work together. Mutation provides new diversity, while recombination efficiently sorts through the large combinatorial space of existing diversity. A better understanding of how to balance mutation and recombination could assist in engineering highly optimized proteins.</p>

</div>

<div id="section3" xpathLocation="/article[1]/body[1]/sec[3]"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3 xpathLocation="noSelect">Methods&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/title[1]">Compiling the chimeric protein data set</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[1]">Since multiple structures have been solved for each protein family tested, we decided to use all available structures to generate the residue-residue contact map. The contact map for each library was determined by identifying all protein chains within the Protein Data Bank that share at least 50% sequence identity with any parent. Also included were three unpublished P450 structures, for a total of 88 <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e117&amp;representation=PNG"></span>lac13, 173 <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e118&amp;representation=PNG"></span>lac, 91 P450, 39 CBHI, 24 CBHII, 6 Cel5, 21 Cel48, and 143 arginase chains. For each chain, a residue pair was considered contacting if they contained any heavy atoms within 4.5 Å. The final contact map for each library is composed of residue pairs that are contacting in more than 50% of all chains. We believe this ‘averaged’ residue-reside contact map should provide a more complete description of the protein family's fold, but the use of any single structure does not change the results presented above.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[2]">The number of functional and nonfunctional chimeric proteins was retrieved from previously published results: <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e119&amp;representation=PNG"></span>lac13 <a href="#pcbi.1002713-Meyer1">[34]</a>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e120&amp;representation=PNG"></span>lac <a href="#pcbi.1002713-Meyer2">[50]</a>, P450 <a href="#pcbi.1002713-Otey1">[32]</a>, CBHI <a href="#pcbi.1002713-Heinzelman3">[22]</a>, CBHII <a href="#pcbi.1002713-Heinzelman4">[23]</a>, Cel5 (unpublished data), Cel48 <a href="#pcbi.1002713-Smith1">[21]</a>, Arg <a href="#pcbi.1002713-Romero1">[20]</a>. The fraction of functional chimeras was estimated using maximum likelihood, and 95% confidence intervals were calculated using the Clopper-Pearson method <a href="#pcbi.1002713-Clopper1">[51]</a>. We could not accurately estimate the fraction of functional sequences for the CBHI library due to the extreme bias in chimera sampling <a href="#pcbi.1002713-Heinzelman3">[22]</a>. The results from the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e121&amp;representation=PNG"></span>lac13 library were reanalyzed to account for library construction errors (see below).</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[3]">The additivity of the P450, CBHI, CBHII, and Cel48 libraries was calculated using published thermostability data <a href="#pcbi.1002713-Li1">[17]</a>, <a href="#pcbi.1002713-Smith1">[21]</a>–<a href="#pcbi.1002713-Heinzelman4">[23]</a>. For each library, a block-based linear regression model <a href="#pcbi.1002713-Li1">[17]</a> was parametrized on all the available data. The resulting predictions are unbiased, so the total variance can be partitioned into explained and residual components. The ratio of the explained variance to total variance is the additivity <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e122&amp;representation=PNG"></span>, and in this case is identical to the regression model's coefficient of determination <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e123&amp;representation=PNG"></span>.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/title[1]">Estimation of parental and novel contact parameters</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[1]">Given a data set which maps contact information to binary functional status, we want to estimate the mean energy <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e124&amp;representation=PNG"></span> and variance <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e125&amp;representation=PNG"></span> of parental contacts and the mean energy <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e126&amp;representation=PNG"></span> and variance <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e127&amp;representation=PNG"></span> for novel contacts. The true energy terms <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e128&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e129&amp;representation=PNG"></span> can be integrated out to give the marginalized likelihood function<br><a name="pcbi.1002713.e130" id="pcbi.1002713.e130"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e130&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[1]/disp-formula[1]/label[1]">(16)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e131&amp;representation=PNG"></span> is the binary functional status and for notational simplicity all parental energy terms <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e132&amp;representation=PNG"></span> are combined in the vector <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e133&amp;representation=PNG"></span>, all novel energy terms <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e134&amp;representation=PNG"></span> are combined in the vector <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e135&amp;representation=PNG"></span>, and all binary indicator variables (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e136&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e137&amp;representation=PNG"></span>) are combined into the matrix <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e138&amp;representation=PNG"></span>. The mean and variance of parental and novel contacts can be estimated by maximizing this marginalized likelihood function.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[2]">Since <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e139&amp;representation=PNG"></span> is composed of binary data, we assume that it is generated from a Bernoulli process whose proportion is determined by the energy of a sequence. With this assumption, the first term in the integrand is given by the logistic likelihood function<br><a name="pcbi.1002713.e140" id="pcbi.1002713.e140"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e140&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[2]/disp-formula[1]/label[1]">(17)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e141&amp;representation=PNG"></span> is the logistic sigmoid function given by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e142&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e143&amp;representation=PNG"></span> is the binary functional status of chimera <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e144&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e145&amp;representation=PNG"></span> is a vector containing all <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e146&amp;representation=PNG"></span>, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e147&amp;representation=PNG"></span> is a vector containing all <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e148&amp;representation=PNG"></span>. Assuming the energy components are Gaussian distributed, the second and third terms of the integrand are given by multivariate Gaussian distributions. Since the integral in <a href="#pcbi.1002713.e130">equation 16</a> is analytically intractable, we can approximate it using Laplace's method <a href="#pcbi.1002713-Tierney1">[52]</a>. First we approximate the integrand with a multivariate Gaussian about a stationary point and then we evaluate the Gaussian integral to yield<br><a name="pcbi.1002713.e149" id="pcbi.1002713.e149"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e149&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[2]/disp-formula[2]/label[1]">(18)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e150&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e151&amp;representation=PNG"></span> are the stationary points, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e152&amp;representation=PNG"></span> is the fixed number of contacts, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e153&amp;representation=PNG"></span> is the Hessian matrix evaluated at the stationary points. The stationary points were found using Newton's method and the marginalized likelihood function was maximized using the Nelder-Mead method.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[3]/title[1]">Reanalyzing <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e154&amp;representation=PNG"></span>-lactamase data to account for library construction errors</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[3]/p[1]">The 13-crossover <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e155&amp;representation=PNG"></span>-lactamase library (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e156&amp;representation=PNG"></span>lac13) was assembled from synthetic fragments and had a significant number of construction errors <a href="#pcbi.1002713-Meyer1">[34]</a>. Sequencing of unselected chimeric genes found 9 of 13 to have frame shift mutations <a href="#pcbi.1002713-Drummond1">[16]</a>, which almost certainly result in inactive proteins. Since a majority of frame shifts are incorporated at the PCR step during library construction, it is likely these errors are present throughout all constructed chimeras <a href="#pcbi.1002713-Farrow1">[11]</a>. The maximum likelihood estimate for the proportion of correctly constructed chimeras is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e157&amp;representation=PNG"></span>, with 95% confidence intervals between 0.09 and 0.61 using the Clopper-Pearson interval <a href="#pcbi.1002713-Clopper1">[51]</a>. The sequencing data indicate there may be one to three sequence fragments (chimera blocks) that contain frameshift mutations. Assuming all frame shifts cause inactivation and exhaustive library coverage (over twelvefold sampling), the fraction of functional chimeras can be estimated as the number of functional chimeras divided by the number of correctly constructed chimeras. With these assumptions, we estimate the fraction of functional sequences to be <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e158&amp;representation=PNG"></span> with 95% confidence intervals between <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e159&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e160&amp;representation=PNG"></span> The same modification can be performed on chimeras binned by the number of homologous substitutions (<a href="#pcbi-1002713-g002">Figure 2<em>A</em></a>) because the construction errors display little relation to the level of mutation.</p>

</div>

<div id="section4" xpathLocation="/article[1]/body[1]/sec[4]"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3 xpathLocation="noSelect">Supporting Information&nbsp;<a href="#top">Top</a></h3><a name="pcbi.1002713.s001" id="pcbi.1002713.s001"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002713.s001">Figure S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[4]/supplementary-material[1]/caption[1]/p[1]"><strong>Estimation of contact parameters on other recombination libraries.</strong> The parental and novel contact parameters (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e161&amp;representation=PNG"></span>) were estimated on four binary functional status data sets. The number of sequences in each data set are indicated in the plot titles. The estimated parameters are reported as the mean <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e162&amp;representation=PNG"></span> one standard deviation, and the associated Gaussian probability density functions are plotted. The two largest data sets (P450 and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e163&amp;representation=PNG"></span>-lactamase) give very similar parameter estimates, while all data sets provide the same qualitative relationships among parameters. Within all four parameter sets, we see the mean of parental contacts is slightly favorable and novel contacts are significantly deleterious. The means of these two distributions are separated by approximately one standard deviation, indicating it is relatively common for parental contacts to be as deleterious as novel contacts, and vice versa.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/supplementary-material[1]/caption[1]/p[2]">(TIFF)</p>
<a name="pcbi.1002713.s002" id="pcbi.1002713.s002"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002713.s002">Text S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[4]/supplementary-material[2]/caption[1]/p[1]"><strong>Derivation of a library's expected variance and a chimera's additive energy component.</strong> A detailed description of how the expected value of a library's variance <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e164&amp;representation=PNG"></span> and the additive component of a chimera's energy <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002713.e165&amp;representation=PNG"></span> are calculated.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/supplementary-material[2]/caption[1]/p[2]">(PDF)</p>
</div>





<div xpathLocation="noSelect"><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3 xpathLocation="noSelect">Acknowledgments&nbsp;<a href="#top">Top</a></h3>
<p xpathLocation="/article[1]/back[1]/ack[1]/p[1]">We thank Zhen-Gang Wang and M. Shum for helpful discussions, and D. A. Drummond for feedback on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3 xpathLocation="noSelect">Author Contributions&nbsp;<a href="#top">Top</a></h3><p xpathLocation="noSelect">Analyzed the data: PAR FHA. Wrote the paper: PAR FHA.</p></div><div xpathLocation="noSelect"><a id="references" name="references" toc="references" title="References"></a><h3 xpathLocation="noSelect">References&nbsp;<a href="#top">Top</a></h3><ol class="references" xpathLocation="noSelect"><li xpathLocation="noSelect"><a name="pcbi.1002713-Barton1" id="pcbi.1002713-Barton1"></a>Barton NH, Charlesworth B (1998) Why sex and recombination? Science 281: 1986–1990. <a class="find" href="/article/findArticle.action?citedArticleID=5813057">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Otto1" id="pcbi.1002713-Otto1"></a>Otto SP, Lenormand T (2002) Resolving the paradox of sex and recombination. Nat Rev Genet 3: 252–261. <a class="find" href="/article/findArticle.action?citedArticleID=5813059">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Watt1" id="pcbi.1002713-Watt1"></a>Watt WB (1972) Intragenic recombination as a source of population genetic variability. Amer Nat 106: 737–753. <a class="find" href="/article/findArticle.action?citedArticleID=5813061">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Strobeck1" id="pcbi.1002713-Strobeck1"></a>Strobeck C, Morgan K (1978) The Effect of Intragenic Recombination on the Number of Alleles in a Finite Population. Genetics 88: 829–844. <a class="find" href="/article/findArticle.action?citedArticleID=5813063">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Freeling1" id="pcbi.1002713-Freeling1"></a>Freeling M (1978) Allelic Variation at the Level of Intragenic Recombination. Genetics 89: 211–224. <a class="find" href="/article/findArticle.action?citedArticleID=5813065">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-DeSilva1" id="pcbi.1002713-DeSilva1"></a>De Silva E, Kelley LA, Stumpf MPH (2004) The extent and importance of intragenic recombination. Hum Genomics 1: 410–420. <a class="find" href="/article/findArticle.action?citedArticleID=5813067">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Crameri1" id="pcbi.1002713-Crameri1"></a>Crameri A, Raillard SA, Bermudez E, Stemmer WPC (1998) DNA shuffling of a family of genes from diverse species accelerates directed evolution. Nature 391: 288–291. <a class="find" href="/article/findArticle.action?citedArticleID=5813069">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Carbone1" id="pcbi.1002713-Carbone1"></a>Carbone M, Arnold FH (2007) Engineering by homologous recombination: exploring sequence and function within a conserved fold. Curr Opin Struct Biol 17: 454–459. <a class="find" href="/article/findArticle.action?citedArticleID=5813071">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Voigt1" id="pcbi.1002713-Voigt1"></a>Voigt CA, Martinez C, Wang ZG, Mayo SL, Arnold FH (2002) Protein building blocks preserved by recombination. Nat Struct Biol 9: 553–558. <a class="find" href="/article/findArticle.action?citedArticleID=5813073">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Endelman1" id="pcbi.1002713-Endelman1"></a>Endelman JB, Silberg JJ, Wang ZG, Arnold FH (2004) Site-directed protein recombination as a shortest-path problem. Protein Eng Des Sel 17: 589–594. <a class="find" href="/article/findArticle.action?citedArticleID=5813075">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Farrow1" id="pcbi.1002713-Farrow1"></a>Farrow MF, Arnold FH (2010) Combinatorial recombination of gene fragments to construct a library of chimeras. Curr Prot Prot Sci 26: 2.1–2.20. <a class="find" href="/article/findArticle.action?citedArticleID=5813077">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Heinzelman1" id="pcbi.1002713-Heinzelman1"></a>Heinzelman P, Romero PA, Arnold FH (2012) Efficient Sampling of SCHEMA Chimera Families for Identification of Useful Sequence Elements. Method Enzymol In press. <a class="find" href="/article/findArticle.action?citedArticleID=5813079">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Guo1" id="pcbi.1002713-Guo1"></a>Guo HH, Choe J, Loeb LA (2004) Protein tolerance to random amino acid change. Proc Natl Acad Sci U S A 101: 9205–9210. <a class="find" href="/article/findArticle.action?citedArticleID=5813081">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Bloom1" id="pcbi.1002713-Bloom1"></a>Bloom JD, Silberg JJ, Wilke CO, Drummond DA, Adami C, et al. (2005) Thermo-dynamic prediction of protein neutrality. Proc Natl Acad Sci U S A 102: 606–611. <a class="find" href="/article/findArticle.action?citedArticleID=5813083">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Bershtein1" id="pcbi.1002713-Bershtein1"></a>Bershtein S, Segal M, Bekerman R, Tokuriki N, Tawfik DS (2006) Robustness-epistasis link shapes the fitness landscape of a randomly drifting protein. Nature 444: 929–932. <a class="find" href="/article/findArticle.action?citedArticleID=5813085">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Drummond1" id="pcbi.1002713-Drummond1"></a>Drummond DA, Silberg JJ, Meyer MM, Wilke CO, Arnold FH (2005) On the conservative nature of intragenic recombination. Proc Natl Acad Sci U S A 102: 5380–5385. <a class="find" href="/article/findArticle.action?citedArticleID=5813087">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Li1" id="pcbi.1002713-Li1"></a>Li Y, Drummond DA, Sawayama AM, Snow CD, Bloom JD, et al. (2007) A diverse family of thermostable cytochrome P450s created by recombination of stabilizing fragments. Nat Biotechnol 25: 1051–1056. <a class="find" href="/article/findArticle.action?citedArticleID=5813089">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Heinzelman2" id="pcbi.1002713-Heinzelman2"></a>Heinzelman P, Snow CD, Wu I, Nguyen C, Villalobos A, et al. (2009) A family of thermostable fungal cellulases created by structure-guided recombination. Proc Natl Acad Sci U S A 106: 5610–5615. <a class="find" href="/article/findArticle.action?citedArticleID=5813091">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Landwehr1" id="pcbi.1002713-Landwehr1"></a>Landwehr M, Carbone M, Otey CR, Li Y, Arnold FH (2007) Diversification of catalytic function in a synthetic family of chimeric cytochrome p450s. Chem Biol 14: 269–278. <a class="find" href="/article/findArticle.action?citedArticleID=5813093">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Romero1" id="pcbi.1002713-Romero1"></a>Romero PA, Stone E, Lamb C, Chantranupong L, Krause A, et al. (2012) SCHEMA-Designed Variants of Human Arginase I and II Reveal Sequence Elements Important to Stability and Catalysis. ACS Synth Biol 1: 221–228. <a class="find" href="/article/findArticle.action?citedArticleID=5813095">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Smith1" id="pcbi.1002713-Smith1"></a>Smith MA, Rentmeister A, Snow CD, Wu T, Farrow MF, et al. (2012) A diverse set of family 48 bacterial cellulases created by structure-guided recombination. FEBS J In press. <a class="find" href="/article/findArticle.action?citedArticleID=5813097">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Heinzelman3" id="pcbi.1002713-Heinzelman3"></a>Heinzelman P, Komor R, Kannan A, Romero PA, Yu X, et al. (2010) Efficient screening of fungal cellobiohydrolase class I enzymes for thermostabilizing sequence blocks by SCHEMA structure-guided recombination. Protein Eng Des Sel 23: 871–880. <a class="find" href="/article/findArticle.action?citedArticleID=5813099">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Heinzelman4" id="pcbi.1002713-Heinzelman4"></a>Heinzelman P, Snow CD, Smith MA, Yu X, Kannan A, et al. (2009) SCHEMA recombination of a fungal cellulase uncovers a single mutation that contributes markedly to stability. J Biol Chem 284: 26229–26233. <a class="find" href="/article/findArticle.action?citedArticleID=5813101">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Adler1" id="pcbi.1002713-Adler1"></a>Adler RJ (1981) The Geometry of Random Fields. 1st edition. Chichester: Wiley &amp; Sons. </li><li xpathLocation="noSelect"><a name="pcbi.1002713-Stein1" id="pcbi.1002713-Stein1"></a>Stein ML (1999) Interpolation of Spatial Data: Some Theory for Kriging. 1st edition. New York: Springer. 247 p. </li><li xpathLocation="noSelect"><a name="pcbi.1002713-Li2" id="pcbi.1002713-Li2"></a>Li SZ (2009) Markov Random Field Modeling in Image Analysis. 3rd edition. London: Springer. 362 p. </li><li xpathLocation="noSelect"><a name="pcbi.1002713-Stadler1" id="pcbi.1002713-Stadler1"></a>Stadler PF, Happel R (1999) Random Field Models For Fitness Landscapes. J Math Biol 38: 435–478. <a class="find" href="/article/findArticle.action?citedArticleID=5813109">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Kauffman1" id="pcbi.1002713-Kauffman1"></a>Kauffman SA, Levin SA (1987) Towards a general theory of adaptive walks on rugged landscapes. J Theor Biol 128: 11–45. <a class="find" href="/article/findArticle.action?citedArticleID=5813111">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Miyazawa1" id="pcbi.1002713-Miyazawa1"></a>Miyazawa S, Jernigan RL (1985) Estimation of effective interresidue contact energies from protein crystal structures: quasi-chemical approximation. Macromolecules 18: 534–552. <a class="find" href="/article/findArticle.action?citedArticleID=5813113">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Clementi1" id="pcbi.1002713-Clementi1"></a>Clementi C, Vendruscolo M, Maritan A, Domany E (1999) Folding Lennard-Jones proteins by a contact potential. Proteins 37: 544–553. <a class="find" href="/article/findArticle.action?citedArticleID=5813115">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Vendruscolo1" id="pcbi.1002713-Vendruscolo1"></a>Vendruscolo M, Najmanovich R, Domany E (2000) Can a pairwise contact potential stabilize native protein folds against decoys obtained by threading? Proteins 38: 134–148. <a class="find" href="/article/findArticle.action?citedArticleID=5813117">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Otey1" id="pcbi.1002713-Otey1"></a>Otey CR, Landwehr M, Endelman JB, Hiraga K, Bloom JD, et al. (2006) Structure-Guided Recombination Creates an Artificial Family of Cytochromes P450. PLoS Biol 4: e112. <a class="find" href="/article/findArticle.action?citedArticleID=5813119">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Taverna1" id="pcbi.1002713-Taverna1"></a>Taverna DM, Goldstein RA (2002) Why are proteins marginally stable? Proteins 46: 105–109. <a class="find" href="/article/findArticle.action?citedArticleID=5813121">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Meyer1" id="pcbi.1002713-Meyer1"></a>Meyer MM, Silberg JJ, Voigt CA, Endelman JB, Mayo SL, et al. (2003) Library analysis of SCHEMA-guided protein recombination. Protein Sci 12: 1686–1693. <a class="find" href="/article/findArticle.action?citedArticleID=5813123">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Endelman2" id="pcbi.1002713-Endelman2"></a>Endelman JB (2005) Design and analysis of combinatorial protein libraries created by site-directed recombination [PhD dissertation]. Pasadena (California): Department of Bioengineering, California Institute of Technology. 122 p. </li><li xpathLocation="noSelect"><a name="pcbi.1002713-LeMaster1" id="pcbi.1002713-LeMaster1"></a>LeMaster DM, Hernández G (2005) Additivity in both thermodynamic stability and thermal transition temperature for rubredoxin chimeras via hybrid native partitioning. Structure 13: 1153–1163. <a class="find" href="/article/findArticle.action?citedArticleID=5813127">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Fisher1" id="pcbi.1002713-Fisher1"></a>Fisher RA (1930) The Genetical Theory of Natural Selection. Oxford: Clarendon. </li><li xpathLocation="noSelect"><a name="pcbi.1002713-Charlesworth1" id="pcbi.1002713-Charlesworth1"></a>Charlesworth B, Charlesworth D (2010) Elements of evolutionary genetics. Roberts and Co. Publishers, 734 p. </li><li xpathLocation="noSelect"><a name="pcbi.1002713-Carneiro1" id="pcbi.1002713-Carneiro1"></a>Carneiro M, Hartl DL (2010) Adaptive landscapes and protein evolution. Proc Natl Acad Sci U S A 107: 1747–1751. <a class="find" href="/article/findArticle.action?citedArticleID=5813133">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-BornbergBauer1" id="pcbi.1002713-BornbergBauer1"></a>Bornberg-Bauer E (1997) How are model protein structures distributed in sequence space? Biophys J 73: 2393–2403. <a class="find" href="/article/findArticle.action?citedArticleID=5813135">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Xia1" id="pcbi.1002713-Xia1"></a>Xia Y, Levitt M (2004) Simulating protein evolution in sequence and structure space. Curr Opin Struct Biol 14: 202–207. <a class="find" href="/article/findArticle.action?citedArticleID=5813137">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Keefe1" id="pcbi.1002713-Keefe1"></a>Keefe AD, Szostak JW (2001) Functional proteins from a random-sequence library. Nature 410: 715–718. <a class="find" href="/article/findArticle.action?citedArticleID=5813139">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Axe1" id="pcbi.1002713-Axe1"></a>Axe DD (2004) Estimating the prevalence of protein sequences adopting functional enzyme folds. J Mol Biol 341: 1295–1315. <a class="find" href="/article/findArticle.action?citedArticleID=5813141">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Bloom2" id="pcbi.1002713-Bloom2"></a>Bloom JD, Lu Z, Chen D, Raval A, Venturelli OS, et al. (2007) Evolution favors protein mutational robustness in sufficiently large populations. BMC Biol 5: 29. <a class="find" href="/article/findArticle.action?citedArticleID=5813143">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Gavrilets1" id="pcbi.1002713-Gavrilets1"></a>Gavrilets S (2004) Fitness Landscapes and the Origin of Species. Princeton, NJ: Princeton University Press. 476 p. </li><li xpathLocation="noSelect"><a name="pcbi.1002713-Yip1" id="pcbi.1002713-Yip1"></a>Yip SP, Lovegrove JU, Rana NA, Hopkinson DA, Whitehouse DB (1999) Mapping recombination hotspots in human phosphoglucomutase (PGM1). Hum Mol Gen 8: 1699–1706. <a class="find" href="/article/findArticle.action?citedArticleID=5813147">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-McBee1" id="pcbi.1002713-McBee1"></a>McBee AD, Wegner DJ, Carlson CS, Wambach JA, Yang P, et al. (2008) Recombination as a mechanism for sporadic mutation in the surfactant protein-C gene. Pediatr Pulm 43: 443–450. <a class="find" href="/article/findArticle.action?citedArticleID=5813149">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Watson1" id="pcbi.1002713-Watson1"></a>Watson RA, Weinreich D, Wakeley J (2006) Effects of intra-gene fitness interactions on the benefit of sexual recombination. Biochem Soc Trans 34: 560–561. <a class="find" href="/article/findArticle.action?citedArticleID=5813151">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Watson2" id="pcbi.1002713-Watson2"></a>Watson RA, Weinreich DM, Wakeley J (2011) Genome structure and the benefit of sex. Evolution 65: 523–536. <a class="find" href="/article/findArticle.action?citedArticleID=5813153">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Meyer2" id="pcbi.1002713-Meyer2"></a>Meyer MM, Hochrein L, Arnold FH (2006) Structure-guided SCHEMA recombination of distantly related beta-lactamases. Protein Eng Des Sel 19: 563–570. <a class="find" href="/article/findArticle.action?citedArticleID=5813155">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Clopper1" id="pcbi.1002713-Clopper1"></a>Clopper CJ, Pearson ES (1934) The use of confidence or fiducial limits illustrated in the case of the binomial. Biometrika 26: 404–413. <a class="find" href="/article/findArticle.action?citedArticleID=5813157">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Tierney1" id="pcbi.1002713-Tierney1"></a>Tierney L, Kadane JB (1986) Accurate approximations for posterior moments and marginal densities. J Amer Statist Assoc 81: 82–86. <a class="find" href="/article/findArticle.action?citedArticleID=5813159">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002713-Murzin1" id="pcbi.1002713-Murzin1"></a>Murzin AG, Brenner SE, Hubbard T, Chothia C (1995) SCOP: a structural classification of proteins database for the investigation of sequences and structures. J Mol Biol 247: 536–40.
        doi:
        <a href="http://dx.doi.org/10.1016/S0022-2836(05)80134-2">10.1016/S0022-2836(05)80134-2</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5813161">
                      Find this article online
                    </a></li></ol></div>

</div></div>
<div style="display:none">
<div dojoType="ambra.widget.RegionalDialog" id="AnnotationDialog" style="padding:0;margin:0;">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Post Your Note (For Public Viewing)</span></h5>
      <div class="posting pane">
        <form name="createAnnotation" id="createAnnotation" method="post" action="">
          <input type="hidden" name="target" value="info:doi/10.1371/journal.pcbi.1002713" />
          <input type="hidden" name="startPath" value="" />
          <input type="hidden" name="startOffset" value="" />
          <input type="hidden" name="endPath" value="" />
          <input type="hidden" name="endOffset" value="" />
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="false" />
          <input type="hidden" name="noteType" id="noteType" value="" />
          <fieldset>
            <legend>Compose Your Note</legend>
            <span id="submitMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td>
                  <label for="cNoteType">This is a </label><select name="cNoteType" id="cNoteType"><option value="note">note</option><option value="correction">correction</option></select>
                  <span id="cdls" style="visibility:hidden;margin-left:0.3em; white-space:nowrap;"><a href="/static/commentGuidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313?target=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002713#corrections">What are corrections?</a></span>
                  <label for="cTitle"><span class="none">Enter your note title</span><!-- error message text <em>A title is required for all public notes</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your note title..." class="title" alt="Enter your note title..." />
                  <label for="cArea"><span class="none">Enter your note</span><!-- error message text <em>Please enter your note</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your note..." alt="Enter your note...">Enter your note...</textarea>
                  <input type="hidden" name="isPublic" value="true" />
                </td>
                <td>&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" checked="checked" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." alt="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td colspan="3" class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel"/>
                  <input type="button" value="Submit" title="Click to post your note publicly" id="btn_post" class="primary"/>
                </td>
              </tr>
              <tr>
                <td colspan="3">
                  <p>Notes and Corrections can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p>
                </td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
    <div class="tip" id="dTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialog" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="caTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      Please follow our <a href="/static/commentGuidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313">guidelines for notes and comments</a> and review our <a href="/static/competing.action;jsessionid=42F26552E5BE8446B0C72379D8339313">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
      <ul>
        <li>Remarks that could be interpreted as allegations of misconduct</li>
        <li>Unsupported assertions or statements</li>
        <li>Inflammatory or insulting language</li>
      </ul>
      <form name="contextActionForm" id="contextActionForm" class="clearfix buttons" method="post" action="">
        <input type="button" name="Continue" value="Continue" id="ContextActionDialogContinueButton" onmouseup="ambra.displayAnnotationContext.startComment(event);" title="Add a note to this text" class="primary"/>
        <input type="button" name="Cancel" value="Cancel" id="ContextActionDialogCancelButton" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);" title="Close this Window"/>
      </form>
    </div>
    <div class="tip" id="caTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogNotLogged" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canlTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You must be logged in to add a note to an article.
      You may log in by <a onmousedown="ambra.displayAnnotationContext.disconnect(event);" href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002713">clicking here</a> or <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">cancel this note</a>.
    </div>
    <div class="tip" id="canlTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canBDTipu"></div>
    <div class="contextActionContent">
      <h5 class="annotation icon"><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot annotate this area of the document. <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canBDTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadRangeSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canbrTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot create an annotation that spans different sections of the document; please adjust your selection.<br/>
      <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canbrTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialog" style="padding:0;margin:0;">
  <div class="dialog preview">
    <div class="tipu" id="cTipu"></div>
    <div class="btn close" id="btn_close" title="Click to close"><a title="Click to close">Close</a></div>
    <div id="cmtContainer" class="comment">
      <h6 id="viewCmtTitle"></h6>
      <div class="detail" id="viewCmtDetail"></div>
      <div class="contentwrap" id="viewComment"></div>
      <div class="contentwrap" id="viewCIStatement"></div>
      <div class="detail" id="viewLink">
        <!--<a href="#" class="commentary icon" title="Click to view full thread and respond">View all responses</a>
        <a href="#" class="respond tooltip" title="Click to respond to this posting">Respond to this</a>-->
      </div>
    </div>
    <div class="tip" id="cTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialogMultiple" style="padding:0;margin:0;">
  <div class="dialog multiple preview">
    <div class="tipu" id="mTipu"></div>
    <div class="btn close" id="btn_close_multi" title="Click to close"><a title="Click to close">Close</a></div>
    <ol id="multilist"></ol>
    <br/>
    <div id="multidetail"></div>
    <div class="tip" id="mTip"></div>
  </div>
</div>


<div dojoType="dijit.Dialog" id="Rating">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Rate This Article</span></h5>
      <div class="instructions">Please follow our <a  href="/static/ratingGuidelines.action;jsessionid=42F26552E5BE8446B0C72379D8339313">guidelines for rating</a> and review our <a href="/static/competing.action;jsessionid=42F26552E5BE8446B0C72379D8339313">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
        <ol>
          <li>Remarks that could be interpreted as allegations of misconduct</li>
          <li>Unsupported assertions or statements</li>
          <li>Inflammatory or insulting language</li>
        </ol>
      </div>
      <div class="posting pane">
        <form name="ratingForm" id="ratingForm" method="post" action="">
          <input type="hidden" name="articleURI" value="info:doi/10.1371/journal.pcbi.1002713" />  
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="" />
          <fieldset>
            <legend>Compose Your Annotation</legend>
            <span id="submitRatingMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td rowspan="2">
                  <label for="insight">Insight</label>
                  <ul class="star-rating rating edit" title="Rate insight" id="rateInsight">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Bland" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'insight', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Profound" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="insight" title="insight" value="" />
                  <label for="reliability">Reliability</label>
                  <ul class="star-rating rating edit" title="Rate reliability" id="rateReliability">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Tenuous" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'reliability', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Unassailable" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="reliability" title="reliability" value="" />
                  <label for="style">Style</label>
                  <ul class="star-rating rating edit" title="Rate style" id="rateStyle">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Crude" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'style', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Elegant" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="style" title="style" value="" />
                  <label for="cTitle"><span class="none">Enter your comment title</span><!-- error message text <em>A title is required for all public annotations</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your comment title..." class="title" alt="Enter your comment title..." />
                  <label for="cArea"><span class="none">Enter your comment</span><!-- error message text <em>Please enter your annotation</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your comment..." alt="Enter your comment...">Enter your comment...</textarea>
                </td>
                <td rowspan="2">&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." title="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel_rating"/>
                  <input type="button" value="Submit" title="Click to post your annotation publicly" id="btn_post_rating" class="primary"/>
                </td>
              </tr>
              <tr>
                  <td colspan="3"><p>Ratings can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p></td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
<div dojoType="ambra.widget.LoadingCycle" id="LoadingCycle" class="loadingCycler">
  <img src="/images/loading.gif" width="58" height="58" title="Loading..." />
</div>
<div dojoType="dijit.Dialog" id="LightBox">

    <!-- start main window wrapper-->
    <div class="figure-window-wrapper">

      <!--start film strip Wrapper-->
      <div class="figure-window-nav-wrapper">
        <div id="figure-window-nav"></div>
      </div><!--end figure-window-nav-wrapper-->

      <!-- start figure container-->
      <div class="figure-window-container">

        <!-- start/end figure viewer-->
        <div id="figure-window-viewer">
          <img id="figure-window-img" class="large"/>
        </div>

        <!-- start/end figure doi-->
        <div id="figure-window-doi"></div>

        <!--start figure title-box-->
        <div id="figure-window-title-box" class="figure-window-title-less">
          <span id="figure-window-title"></span>
          <div id="figure-window-more" class="figure-window-more" onclick="return ambra.lightBox.showMoreOrLess();"></div>
          <div id="figure-window-description"></div>
        </div><!--end figure title-box-->

        <!--start figure download-->
        <div class="figure-window-download">
          <ul>
            <li class="download icon"><strong>Download:</strong>
              <a id="figure-window-ppt" title="Click to download PowerPoint slide of this image">PowerPoint
                slide</a> |
              <a id="figure-window-large" title="Click to download a larger version of this image">larger image
                (<span id="figure-window-large-size"></span>)</a> |
              <a id="figure-window-tiff" title="Click to download a original version of this image">original image
                (<span id="figure-window-tiff-size"></span>)</a>
            </li>
          </ul>
        </div><!--end figure download-->

        <!--start/end of close, context, next and previous button-->
        <div class="figure-window-close" onclick="return ambra.lightBox.hide();"></div>
        <div class="figure-window-context" onclick="return ambra.lightBox.showInContext();"></div>
        <div id="figure-window-previous" onclick="return ambra.lightBox.showPrevious();"></div>
        <div id="figure-window-next" onclick="return ambra.lightBox.showNext();"></div>

      </div><!--end figure-window-container-->

    </div><!--end figure-window-wrapper-->

</div><!--end dijit.Dialog-->
</div>
</div>
<!-- end : main contents -->

</div>
<!-- end : container -->	

<!-- begin : footer -->
<div id="ftr">
  <p><span>All site content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License 2.5" tabindex="200">Creative Commons Attribution License</a>.</span></p>
  <ul>
    <li><a href="/static/privacy.action;jsessionid=42F26552E5BE8446B0C72379D8339313" title="PLOS Privacy Statement" tabindex="501">Privacy Statement</a></li>
    <li><a href="/static/terms.action;jsessionid=42F26552E5BE8446B0C72379D8339313" title="PLOS Terms of Use" tabindex="502">Terms of Use</a></li>
    <li><a href="http://www.plos.org/advertise/" title="Advertise With PLOS" tabindex="503">Advertise</a></li>
    <li><a href="http://www.plos.org/about/media-inquiries/" title="PLOS Embargo Policy" tabindex="504">Media Inquiries</a></li>
    <li><a href="http://www.plos.org/publish/order-reprints/" title="PLOS in Print" tabindex="505">PLOS in Print</a></li>
    <li><a href="/static/sitemap.action;jsessionid=42F26552E5BE8446B0C72379D8339313" title="Site Map" tabindex="506">Site Map</a></li>
    <li><a href="http://www.plos.org" title="PLOS.org" tabindex="507">PLOS.org</a></li>
  </ul>
  <div class="powered">
  <ul>
    <li><a href="/static/releaseNotes.action" title="Ambra | Release Notes">Ambra 2.4.9</a></li>
    <li>Managed Colocation provided by <a href="http://www.isc.org/" title="Internet Systems Consortium">Internet Systems Consortium</a>.</li>
  </ul>
  </div>
</div>
<!-- end : footer -->
<script type="text/javascript">
  var _namespace="";
    var loggedIn = false;

  var almHost = "http://alm.plos.org";
  var solrHost = "http://api.plos.org/search";
  var pubGetHost = "http://pubget.com/widgetizer/links_hash";
  var appContext = "";

  // Safari v3.1.1 "console.debug" issue (http://trac.dojotoolkit.org/ticket/6849) workaround
  if (/3[\.0-9]+ Safari/.test(navigator.appVersion)) {
    window.console = {
      origConsole: window.console,
      log: function(s){ this.origConsole.log(s); },
      info: function(s){ this.origConsole.info(s); },
      error: function(s){ this.origConsole.error(s); },
      warn: function(s){ this.origConsole.warn(s); }
    };
  }

  var djConfig = {
    // don't debug for IE - as dojo's firebug lite module is error prone in IE
    isDebug: false,
    parseOnLoad: true
  };

  var bamGAID = 'UA-338393-1';
  var bamGAVPV = location.pathname + location.search + location.hash;
</script>
<script type="text/javascript" src="/javascript/dojo/dojo/dojo.js?v=pzlwVCPPo+oJC12upPgJrJ5EG_k"></script>
<script type="text/javascript" src="/javascript/dojo/dojo/ambra.js?v=Y8JM0CVD9YwjC99M_jvbcRTiGhY"></script>
<script type="text/javascript" src="/javascript/init_global.js?v=fF03pfG3r+Nkj_czZ8ORg8o5sNM"></script>
<script type="text/javascript" src="/javascript/init_article.js?v=qzBD_a7nKbeMM2xNUtB3vckN+Zc"></script>
<script type="text/javascript" src="/javascript/init_ratings.js?v=seKykuS0X33CQGS7Lj+8FtBy5Uw"></script>
<script type="text/javascript" src="/javascript/init_lightBox.js?v=vJRLPy7RRX8Zp31Ogcm3i+TA4f8"></script>
<script type="text/javascript" src="/javascript/init_article_body.js?v=YgFei2xCi50CahwaPbSsxUxUsmw"></script>
<script type="text/javascript" src="/javascript/init_article_rhc.js?v=djtbTMOagorKE6ZPrkCwufAfWWI"></script>
<script type="text/javascript" src="/javascript/alm.js?v=1tH9JZ8uDtXspXssGJDr8O+jBOs"></script>
<script type="text/javascript" src="/javascript/reporting/articleViewsCumulative.js?v=RQ+gCrcAl0dBeLbmokxgQZTGYYA"></script>
<script type="text/javascript" src="/javascript/check_twitter.js?v=Hc0sH1W9xIu0qhqxWPD4c1u50K4"></script>
<script type="text/javascript" src="/javascript/init_googleAnalytics.js?v=yCD4h9PHGI0dsefmuWDlMn7pHog"></script>

</body>
</html>

