

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article">

<head>
   <title>PLOS Computational Biology: Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay</title>



<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9" />

<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="home" title="home" href="/home.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" />
<link rel="alternate" type="application/rss+xml"
  title="PLOS Computational Biology: New Articles"
  href="http://www.ploscompbiol.org/article/feed" />

<link rel="pingback" href="http://www.ploscompbiol.org/pingback" />


<!-- global_css.ftl -->
<link rel="stylesheet" type="text/css" media="screen" href="/css/dojo/dijit/themes/tundra/tundra.css?v=zwugcxLdpBWmb72qqlHgGyeNAGM" />
<link rel="stylesheet" type="text/css" media="all" href="/css/star_rating.css?v=Mm3HyaUPUahAJAq6xG5W3tdF7Hk" />
<link rel="stylesheet" type="text/css" media="all" href="/css/rating.css?v=AOT9rozE9sYVjGTdeiyxHf18Ykw" />
<link rel="stylesheet" type="text/css" media="all" href="/css/screen.css?v=KBNFqBmhD635U0NgiM5tf9dtxRE" />
<link rel="stylesheet" type="text/css" media="all" href="/css/iepc.css?v=FIjyR83BhSovq_5yNVk1Sih60AI" />
<link rel="stylesheet" type="text/css" media="all" href="/css/journal.css?v=SiREXztjNSUsOyrlkE9sv028Y+0" />
<link rel="stylesheet" type="text/css" media="all" href="/css/lightBox.css?v=sDgsUXamFoYAakEshdj0YWrZiiQ" />
<link rel="stylesheet" type="text/css" media="print" href="/css/print.css?v=0T+LTV7Gd0jtHEhqEvBfSrmHUGg" />

<!--chartbeat --> 
<script type="text/javascript">var _sf_startpt=(new Date()).getTime()</script>


<!--[if IE]><![endif]-->
<meta name="description" content="PLOS Computational Biology is an open-access" />

<meta name="keywords" content="plos computational biology" />

  <meta name="citation_publisher" content="Public Library of Science" />
  <meta name="citation_doi" content="10.1371/journal.pcbi.1002772" />
    <meta name="citation_title" content="Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay"/>
    <meta itemprop="name" content="Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay"/>

      <meta name="citation_author" content="Filippo Cacace" />
          <meta name="citation_author_institution" content="Università Campus Biomedico, Rome, Italy" />
      <meta name="citation_author" content="Paola Paci" />
          <meta name="citation_author_institution" content="Università Campus Biomedico, Rome, Italy" />
          <meta name="citation_author_institution" content="CNR-Institute of Systems Analysis and Computer Science (IASI), BioMathLab, Rome, Italy" />
      <meta name="citation_author" content="Valerio Cusimano" />
          <meta name="citation_author_institution" content="Università Campus Biomedico, Rome, Italy" />
      <meta name="citation_author" content="Alfredo Germani" />
          <meta name="citation_author_institution" content="Dipartimento di Ingegneria e Scienze dell'Informazione e Matematica (DISIM), Università de L'Aquila, L'Aquila, Italy" />
      <meta name="citation_author" content="Lorenzo Farina" />
          <meta name="citation_author_institution" content="Dipartimento di Informatica e Sistemistica, Sapienza Università di Roma, Rome, Italy" />

    <meta name="citation_date" content="2012/11/8"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pcbi.1002772.pdf" />

    <meta name="citation_journal_title" content="PLOS Computational Biology" />
    <meta name="citation_firstpage" content="e1002772"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="8"/>
    <meta name="citation_issn" content="1553-7358"/>

    <meta name="citation_journal_abbrev" content="PLoS Comput Biol" />

      <meta name="citation_reference" content="citation_title=The highways and byways of mrna decay; citation_volume=8; citation_number=1; citation_pages=113-126; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Dynamic profiling of mrna turnover reveals gene-specific and system-wide regulation of mrna decay; citation_volume=22; citation_number=2; citation_pages=2787-2795; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=The global dynamics of rna stability orchestrates responses to cellular activation; citation_volume=8; citation_number=3; citation_pages=95; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Extensive association of functionally and cytotopically related mrnas with puf family rna-binding proteins in yeast; citation_volume=2; citation_number=4; citation_pages=342-354; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Diverse rna-binding proteins interact with functionally related sets of rnas, suggesting extensive regulatory system; citation_volume=6; citation_number=5; citation_pages=e255; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The many pathways of rna degradation; citation_volume=136; citation_number=6; citation_pages=763-776; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A review of methods to monitor the modulation of mrna stability: a novel approach to drug discovery and therapeutic intervention; citation_volume=15; citation_number=7; citation_pages=609-622; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of mrna decay rates and their determinants in arabidopsis thaliana; citation_volume=19; citation_number=8; citation_pages=3418-3436; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Database for mrna half-life of 19977 genes obtained by dna microarray analysis of pluripotent and differentiating mouse embryonic cells; citation_volume=16; citation_number=9; citation_pages=45-58; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Transient transcriptional re- sponses to stress are generated by opposing effects of mrna production and degradation; citation_volume=4; citation_number=10; citation_pages=223; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Precision and functional specificity in mrna decay; citation_volume=99; citation_number=11; citation_pages=5860-5865; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of mrna stability using transcription inhibitors and microarrays reveals posttranscriptional control of ribosome biogenesis factors; citation_volume=25; citation_number=12; citation_pages=5534-5547; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Methods to determine mrna half-life in saccharomyces cerevisiae; citation_volume=448; citation_number=13; citation_pages=267-284; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Metabolic labeling of rna uncovers principles of rna production and degradation dynamics in mammalian cells; citation_volume=29; citation_number=14; citation_pages=436-442; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Genomic run-on evaluates transcription rates for all yeasts genes and identifies gene regulatory mechanisms; citation_volume=15; citation_number=15; citation_pages=303-313; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The transcriptome of the intraery- throcytic developmental cycle of Plasmodium falciparum; citation_volume=I; citation_number=16; citation_pages=E5; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Whole-genome analysis of mrna decay in Plasmodium falci- parum reveals a global lengthening of mrna half-life during the intra-erythrocytic developmental cycle; citation_volume=8; citation_number=17; citation_pages=R134; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The forkhead transcription factor hcm1 regulates chromosome segregation genes and fills the s-phase gap in the transcriptional circuitry of the cell cycle; citation_volume=20; citation_number=18; citation_pages=2266-2278; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global control of cell-cycle tran- scription by coupled cdk and network oscillators; citation_volume=453; citation_number=19; citation_pages=944-947; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes; citation_volume=310; citation_number=20; citation_pages=1152-1158; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=mrna stability in mammalian cells; citation_volume=59; citation_number=21; citation_pages=423-450; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Major role for mrna stability in shaping the kinetics of gene induction; citation_volume=11; citation_number=22; citation_pages=259; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Embedding mrna stability in corre-lation analysis of time-series gene expression data; citation_volume=4; citation_number=23; citation_pages=e1000141; citation_date=2008; " />
      <meta name="citation_reference" content="citation_number=24; " />
      <meta name="citation_reference" content="citation_title=A new approach to linear filtering and prediction problems; citation_volume=82; citation_number=25; citation_pages=35-45; citation_date=1960; " />
      <meta name="citation_reference" content="citation_title=Gorilla: A tool for discovery and visualization of enriched go terms in ranked gene lists; citation_volume=2009; citation_number=26; citation_pages=10-48; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Progression of the cell cycle through mitosis leads to abortion of nascent transcripts; citation_volume=67; citation_number=27; citation_pages=303-310; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Regulatory mechanisms and networks couple the different phases of gene expression; citation_volume=27; citation_number=28; citation_pages=316-322; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Comparison of computa-tional methods for the identification of cell cycle-regulated genes; citation_volume=21; citation_number=29; citation_pages=1164-1171; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Continuous representations of time-series gene expression data; citation_volume=10; citation_number=30; citation_pages=341-356; citation_date=2003; " />



<!--
<rdf:RDF xmlns="http://web.resource.org/cc/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<Work rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772">
   <license rdf:resource="http://creativecommons.org/licenses/by/2.5/" />
</Work>
<License rdf:about="http://creativecommons.org/licenses/by/2.5/">
   <permits rdf:resource="http://web.resource.org/cc/Reproduction" />
   <permits rdf:resource="http://web.resource.org/cc/Distribution" />
   <requires rdf:resource="http://web.resource.org/cc/Notice" />
   <requires rdf:resource="http://web.resource.org/cc/Attribution" />
   <permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" />
</License>
<rdf:Description
     rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772"
     dc:identifier="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772"
     dc:title="PLOS Computational Biology: Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay"
       trackback:ping="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772/trackback;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1"
     />
</rdf:RDF>

-->
</head>
<body>
<span property="dc:language" content="en"></span>
<!-- begin : container -->
<div id="container">
  <!-- begin : top banner external ad space -->
  <div id="topBanner">


<!-- begin : left banner slot -->
<div class="left">
  <iframe id='affe494b' name='affe494b'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=84&amp;cb=2571'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=affe494b&amp;cb=4785'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=84&amp;cb=6424&amp;n=affe494b'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : left banner slot -->
<!-- begin : right banner slot -->
<div class="right">
  <iframe id='af654bd5' name='af654bd5'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=85&amp;cb=538'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=af654bd5&amp;cb=8164'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=85&amp;cb=6291&amp;n=af654bd5'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : right banner slot -->

  </div>
  <!-- end : top banner external ad space -->

  <!-- begin : header -->
  <div id="hdr" class="login">

  <!-- begin : logo -->
  <div id="logo" title="PLOS Computational Biology"><a href="/home.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" title="PLOS Computational Biology"><span>PLOS Computational Biology</span></a></div>
  <!-- end : logo -->
  <!-- begin : user controls -->





  <div id="user">
    <div>
      <ul>
        <li><a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002772" class="feedback"><strong>Login</strong></a> | </li>
        <li><a href="https://register.plos.org:443/ambra-registration/register.action">Create Account</a> | </li>
        <li class="feedback"><a href="/feedbackCreate.action?page=%252Farticle%252FfetchArticle.action%253FarticleURI%253Dinfo%25253Adoi%25252F10.1371%25252Fjournal.pcbi.1002772" title="Send us your feedback">Feedback</a></li>
      </ul>
    </div>
  </div>


  <!-- end : user controls -->
  <!-- begin search links -->
  <ul id="links"><li class="browse"><a href="/article/browse.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?field=date" title="Browse Articles">Browse</a></li><li class="rss"><a href="/static/rssFeeds.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" title="RSS Feeds">RSS</a></li></ul>
  <!-- end : search links -->
  <!-- begin : dashboard -->
  <div id="db">
    <form name="searchForm" action="/search/simpleSearch.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" method="get">
      <input type="hidden" name="from" value="globalSimpleSearch"/>
      <input type="hidden" name="filterJournals" value="PLoSCompBiol">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="wrap"><input id="search" type="text" name="query" value="Search articles..." onfocus="if(this.value=='Search articles...')value='';" onblur="if(this.value=='')value='Search articles...';" class="searchField" alt="Search articles..."/></div>
        <input src="/images/search_btn1.gif" onclick="submit();" value="ftsearch" alt="SEARCH" tabindex="3" class="button" type="image" />
      </fieldset>
    </form>
    <form name="gasf" action="/search/advancedSearch.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" method="get">
<input type="hidden" name="noSearchFlag" value="true" id="noSearchFlag"/><input type="hidden" name="query" value="" id="query"/>    </form>
    <a id="advSearch" href="#" onclick="if(document.searchForm.query.value!='Search articles...')document.gasf.query.value=document.searchForm.query.value;document.gasf.submit();return false;">Advanced Search</a>
  </div>
  <!-- end : dashboard -->
  <!-- begin : navigation -->
  <ul id="nav">
    <li><a href="/home.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" tabindex="101">Home</a></li>
     <!-- This is a temporary action to link to th static toc page -->
    <li><a href="/static/browse.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" tabindex="102">Browse Articles</a>
        <ul>
          <li><a href="/article/browseIssue.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?field=issue">Current Issue</a></li> <!-- Assuming dynamic TOC is in place -->
          <li><a href="/article/browseVolume.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?field=volume">Journal Archive</a></li> 
          <li><a href="/article/browse.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?field=date">By Publication Date</a></li>
          <li><a href="/article/browse.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">By Subject</a></li>
          <li><a href="http://www.ploscollections.org/static/pcbiCollections.action">Collections</a></li>
        </ul>
    </li>
    <li><a href="/static/about.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" tabindex="103">About</a>
        <ul>
          <li><a href="/static/information.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Journal Information</a></li>
          <li><a href="/static/edboard.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Editorial Board</a></li>
          <li><a href="/static/eic.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Editors-in-Chief</a></li>
          <li><a href="/static/almInfo.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Article-Level Metrics</a></li>
          <li><a href="/static/license.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Open-Access License</a></li>
          <li><a href="/static/contact.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Contact Us</a></li>
        </ul>
      </li>
    <li><a href="/static/users.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" tabindex="104">For Readers</a>
        <ul>
          <li><a href="/static/commentGuidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Guidelines for Notes, Comments, and Corrections</a></li>
          <li><a href="/static/ratingGuidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Guidelines for Rating</a></li>
          <li><a href="/static/help.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Help Using This Site</a></li>
          <li><a href="/static/downloads.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Media Downloads</a></li>
          <li><a href="/static/sitemap.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Site Map</a></li>
        </ul>
      </li>
    <li><a href="/static/authors.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" tabindex="105">For Authors and Reviewers</a>
        <ul>
          <li><a href="/static/policies.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Editorial and Publishing Policies</a></li>
          <li><a href="/static/guidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Author Guidelines</a></li>
          <li><a href="/static/figureGuidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Figure and Table Guidelines</a></li>
          <li><a href="/static/checklist.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Submit Your Paper</a></li>
          <li><a href="/static/reviewerGuidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">Reviewer Guidelines</a></li>
	   <li><a href="http://www.plos.org/journals/index.php">Submit to Other PLOS Journals</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://www.plos.org" title="PLOS" tabindex="110" class="drop">PLOS.org</a>
        <ul>
          <li><a href="http://www.plos.org/oa/index.html" title="Open Access Statement">Open Access</a></li>
          <li><a href="http://www.plos.org/support/donate.php" title="Join PLOS: Show Your Support">Join PLOS</a></li>
          <li><a href="http://blogs.plos.org/" title="PLOS Blogs">PLOS Blogs</a></li>
	  <li><a href="http://www.zazzle.com/plos?rf=238434112415374154" title="PLOS Store">PLOS Store</a></li>
          <li><a href="http://currents.plos.org" title="PLOS Currents">PLOS Currents</a></li>
          <li><a href="http://www.plos.org/connect.html" title="PLOS.org | Stay Connected">Stay Connected</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://hubs.plos.org" tabindex="109">Hubs</a>
        <ul>
          <li><a href="http://hubs.plos.org/biodiversity" title="PLOS Hubs: Biodiversity">Biodiversity</a></li>
          <li><a href="http://clinicaltrials.ploshubs.org" title="PLOS Hub for Clinical Trials">Clinical Trials</a></li>
        </ul>
    </li>
      <li class="journalnav"><a href="http://www.plosjournals.org" tabindex="108">Journals</a>
        <ul>
          <li><a href="http://www.plosbiology.org/" title="PLOSBiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org/" title="PLOSMedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org/" title="PLOSCompBiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://www.plosgenetics.org/" title="PLOSGenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org/" title="PLOSPathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org/" title="PLOSONE.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org/" title="PLOSNTDs.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </li>
    </ul>
  <!-- end : navigation -->
  </div>
  <!-- end : header --> 



<!-- begin : main content -->
<div id="content" class="article" style="visibility:visible;">

<!-- begin : right hand column -->

<div id="rhc" xpathLocation="noDialog">

  <div id="download" class="rhcBox_type1">
    <div class="wrap">
      <ul>
        <li class="download icon"><strong>Download:</strong>
          <a href="/article/fetchObjectAttachment.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772&amp;representation=PDF" title="Download article PDF">PDF</a> |
          <a href="/article/citationList.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772" title="Download citations">Citation</a> |
          <a href="/article/fetchObjectAttachment.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772&amp;representation=XML" title="Download Article XML">XML</a>
        </li>
        <li class="print icon"><a href="#" onclick="window.print();return false;" title="Print Article"><strong>Print article</strong></a></li>
        <li class="reprint icon"><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=3&doi=10.1371/journal.pcbi.1002772&volume=&issue=&title=Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay&author_name=Filippo%20Cacace%2C%20Paola%20Paci%2C%20Valerio%20Cusimano%2C%20Alfredo%20Germani%2C%20Lorenzo%20Farina&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a> New &amp; improved!</li>
      </ul>
    </div>
  </div>

  <div id="issues" class="rhcBox_type2">
    <p><strong>Published in the</strong>
    <a href="http://www.ploscompbiol.org/article/browseIssue.action?issue=info%3Adoi%2F10.1371%2Fissue.pcbi.v08.i11" title="Browse the Open-Access Issue">November 2012 Issue of <em>PLOS Computational Biology</em></a></p>
  </div>

  <div id="impact" class="rhcBox_type2">
    <h6>Metrics <a href="/static/help.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1#articleMetrics" class="replaced" id="info" title="More information"><span>info</span></a></h6>

    <div class="wrap">
      <img id="rhcDataSpinner" src="/images/loading_small.gif" class="loading" />
      <div id="totalDataInRHC"></div>
      <div id="pubDateInRHC"></div>
      <div id="relatedCitesInRHC"></div>
    </div>

    <div class="more clearfix"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" title="More information">More</a></div>
  </div>

  <div id="related" class="rhcBox_type2">
    <h6>Related Content</h6>
    <dl class="related">
      <dt>Related Articles on the Web</dt>
      <dd><a href="http://scholar.google.com/scholar?hl=en&lr=&q=related:http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002772&btnG=Search">Google Scholar</a></dd>
      <dd id="pubMedRelatedLIInRHC" style="display:none;"><a id="pubMedRelatedURLInRHC">PubMed</a></dd>
    </dl>
    <div class="more clearfix"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" title="More information">More</a></div>
  </div>

  <div id="share" class="rhcBox_type2">
    <h6>Share this Article <a href="/static/help.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1#socialBookmarkLinks" class="replaced" id="info" title="More information"><span>info</span></a></h6>
    <ul>
      <li class="bookmarklets">

        <a href="http://www.reddit.com/submit?url=http://dx.plos.org/10.1371/journal.pcbi.1002772"
           target="_new"
           id="reddit">
          <img src="http://www.reddit.com/static/spreddit4.gif" alt="submit to reddit" border="0"/> </a>
        <g:plusone size="small" annotation="none" href=http://dx.plos.org/10.1371/journal.pcbi.1002772></g:plusone>
        <script type="text/javascript">
          (function () {
            var po = document.createElement('script');
            po.type = 'text/javascript';
            po.async = true;
            po.src = 'https://apis.google.com/js/plusone.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(po, s);
          })();
        </script>
        <a href="http://www.stumbleupon.com/submit?url=http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772" target="_new"> <img border=0 src="http://cdn.stumble-upon.com/images/16x16_su_solid.gif" alt="StumbleUpon" title="Add to StumbleUpon"></a>
        <script>function fbs_click() {u='http://dx.plos.org/10.1371/journal.pcbi.1002772';t='Stochastic%20Modeling%20of%20Expression%20Kinetics%20Identifies%20Messenger%20Half-Lives%20and%20Reveals%20Sequential%20Waves%20of%20Co-ordinated%20Transcription%20and%20Decay';window.open('http://www.facebook.com/sharer.php?u='+encodeURIComponent(u)+'&t='+encodeURIComponent(t),'sharer','toolbar=0,status=0,width=626,height=436');return false;}</script><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002772" onclick="return fbs_click()" target="_blank"><img src="http://static.ak.fbcdn.net/images/share/facebook_share_icon.gif" alt="Facebook" title="Add to Facebook" /></a>       <!-- for mor info, see http://www.facebook.com/share_partners.php -->
        <script type="text/javascript">
          function bookmark_in_connotea(u) {
            a=false; x=window; e=x.encodeURIComponent; d=document;
            w=open('http://www.connotea.org/addpopup?continue=confirm&uri='+e(u),
                'add', 'width=600, height=400, scrollbars, resizable');
            void(x.setTimeout('w.focus()',200));
          }
        </script>
        <a style='cursor: pointer;' onclick='javascript:bookmark_in_connotea("http://dx.plos.org/10.1371/journal.pcbi.1002772");'><img src='/images/icon_connotea_16x16.gif' alt="Connotea" title="Add to Connotea"/></a>
        <a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002772&title=Stochastic%20Modeling%20of%20Expression%20Kinetics%20Identifies%20Messenger%20Half-Lives%20and%20Reveals%20Sequential%20Waves%20of%20Co-ordinated%20Transcription%20and%20Decay" target="_new"><img src='/images/icon_citeulike_16x16.gif' alt="CiteULike" title="Add to CiteULike" /></a>
        <a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002772" title="Add to Mendeley" target="_new"><img src="/images/icon_mendeley_16x16.gif" alt="Bibliography"></a>
        <script type="text/javascript">
          var twtTitle  = 'Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay';
          var twtUrl    = 'http://dx.plos.org/10.1371/journal.pcbi.1002772';
          var twtTag    = '#PLOSCompBio:'
          var maxLength = 140 - (twtUrl.length + 1);
          if (twtTitle.length > maxLength) {
            twtTitle = twtTitle.substr(0, (maxLength - 17))+'...';  //truncates title to include space for twtTag and ellipsis
          }
          var twtLink = 'http://twitter.com/intent/tweet?text='+encodeURIComponent(twtTag + ' ' + twtTitle + ' ' + twtUrl);
          document.write('<a href="'+twtLink+'" target="_blank"><img src="/images/icon_twitter.gif" border="0" alt="Twitter icon" title="Tweet This!" /></a>');
        </script>

      </li>
      <li class="email icon"><a href="/article/emailArticle.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772" title="Email this article">Email this article</a></li>
    </ul>
  </div>
  <div id="articleTwitterStream" style="display:none">
    <script src="http://widgets.twimg.com/j/2/widget.js"></script>
    <script>
      new TWTR.Widget({
        version: 2,
        type: 'search',
        search: encodeURIComponent('10.1371/journal.pcbi.1002772'),
        interval: 6000,
        title: '',
        subject: 'From Twitter',
        width: 300,
        height: 300,
        theme: {
          shell: {
            background: '#667893',
            color: '#ffffff'
          },
          tweets: {
            background: '#ffffff',
            color: '#444444',
            links: '#006699'
          }
        },
        features: {
          scrollbar: true,
          loop: false,
          live: true,
          hashtags: true,
          timestamp: true,
          avatars: true,
          toptweets: true,
          behavior: 'all'
        }
      }).render().start();
    </script>
  </div>
</div>
<!-- end : right hand column -->
<form name="articleInfo" id="articleInfo" method="" action="">
<input type="hidden" name="isAuthor" value="true" />
<input type="hidden" name="authorIdList" value="" />
<input type="hidden" name="userIdList" value="" />
<input type="hidden" name="otherIdList" value="" />
<input type="hidden" name="annotationId" value="" />
<input type="hidden" name="isResearchArticle" value="true" />
</form>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.XML" value="136898"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.PDF" value="1542423"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e001.TIF" value="1266"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e001.PNG" value="856"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e002.TIF" value="1244"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e002.PNG" value="809"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g001.TIF" value="351766"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g001.PNG_I" value="36523"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g001.PNG_S" value="12487"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g001.PNG_M" value="74360"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g001.PNG_L" value="206976"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t001.TIF" value="192118"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t001.PNG_I" value="7851"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t001.PNG_S" value="2064"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t001.PNG_M" value="18603"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t001.PNG_L" value="42309"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e003.TIF" value="1256"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e003.PNG" value="836"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e004.TIF" value="1626"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e004.PNG" value="1018"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e005.TIF" value="3720"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e005.PNG" value="1969"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e006.TIF" value="580"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e006.PNG" value="524"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e007.TIF" value="908"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e007.PNG" value="682"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e008.TIF" value="2376"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e008.PNG" value="1387"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e009.TIF" value="1028"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e009.PNG" value="738"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e010.TIF" value="1112"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e010.PNG" value="778"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e011.TIF" value="580"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e011.PNG" value="524"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e012.TIF" value="1028"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e012.PNG" value="738"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e013.TIF" value="1112"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e013.PNG" value="778"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e014.TIF" value="1038"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e014.PNG" value="749"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e015.TIF" value="1552"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e015.PNG" value="994"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e016.TIF" value="2212"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e016.PNG" value="1346"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e017.TIF" value="1564"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e017.PNG" value="1010"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e018.TIF" value="1020"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e018.PNG" value="706"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e019.TIF" value="1086"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e019.PNG" value="750"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g002.TIF" value="496606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g002.PNG_I" value="27857"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g002.PNG_S" value="9572"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g002.PNG_M" value="62673"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g002.PNG_L" value="298456"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t002.TIF" value="194198"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t002.PNG_I" value="8498"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t002.PNG_S" value="2107"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t002.PNG_M" value="20176"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.t002.PNG_L" value="45026"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e020.TIF" value="562"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e020.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e021.TIF" value="560"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e021.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e022.TIF" value="8684"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e022.PNG" value="4008"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e023.TIF" value="1198"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e023.PNG" value="810"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e024.TIF" value="1204"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e024.PNG" value="814"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e025.TIF" value="562"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e025.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e026.TIF" value="560"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e026.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e027.TIF" value="1248"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e027.PNG" value="839"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e028.TIF" value="1270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e028.PNG" value="852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e029.TIF" value="754"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e029.PNG" value="606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e030.TIF" value="748"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e030.PNG" value="607"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e031.TIF" value="562"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e031.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e032.TIF" value="560"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e032.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e033.TIF" value="1248"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e033.PNG" value="839"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e034.TIF" value="1270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e034.PNG" value="852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e035.TIF" value="6638"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e035.PNG" value="3293"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e036.TIF" value="1026"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e036.PNG" value="733"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e037.TIF" value="722"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e037.PNG" value="599"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e038.TIF" value="748"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e038.PNG" value="604"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e039.TIF" value="1224"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e039.PNG" value="816"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e040.TIF" value="1224"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e040.PNG" value="823"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e041.TIF" value="2440"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e041.PNG" value="1446"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e042.TIF" value="2518"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e042.PNG" value="1479"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e043.TIF" value="1248"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e043.PNG" value="839"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e044.TIF" value="1270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e044.PNG" value="852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e045.TIF" value="736"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e045.PNG" value="590"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e046.TIF" value="1680"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e046.PNG" value="1029"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e047.TIF" value="1528"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e047.PNG" value="982"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e048.TIF" value="1248"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e048.PNG" value="839"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e049.TIF" value="1270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e049.PNG" value="852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e050.TIF" value="838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e050.PNG" value="632"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e051.TIF" value="1680"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e051.PNG" value="1029"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e052.TIF" value="1590"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e052.PNG" value="997"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e053.TIF" value="1248"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e053.PNG" value="839"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e054.TIF" value="1270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e054.PNG" value="852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e055.TIF" value="828"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e055.PNG" value="643"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e056.TIF" value="1098"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e056.PNG" value="775"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e057.TIF" value="1122"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e057.PNG" value="775"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e058.TIF" value="684"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e058.PNG" value="563"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e059.TIF" value="858"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e059.PNG" value="658"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e060.TIF" value="1026"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e060.PNG" value="733"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e061.TIF" value="754"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e061.PNG" value="606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e062.TIF" value="748"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e062.PNG" value="607"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e063.TIF" value="722"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e063.PNG" value="599"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e064.TIF" value="748"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e064.PNG" value="604"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e065.TIF" value="684"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e065.PNG" value="563"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e066.TIF" value="580"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e066.PNG" value="524"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e067.TIF" value="1098"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e067.PNG" value="761"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e068.TIF" value="1584"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e068.PNG" value="998"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e069.TIF" value="1068"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e069.PNG" value="728"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e070.TIF" value="1020"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e070.PNG" value="709"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e071.TIF" value="1220"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e071.PNG" value="826"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g003.TIF" value="281366"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g003.PNG_I" value="48243"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g003.PNG_S" value="17290"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g003.PNG_M" value="100270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g003.PNG_L" value="180593"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e072.TIF" value="752"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e072.PNG" value="598"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e073.TIF" value="752"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e073.PNG" value="598"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e074.TIF" value="1096"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e074.PNG" value="771"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e075.TIF" value="1544"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e075.PNG" value="959"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e076.TIF" value="1256"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e076.PNG" value="858"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e077.TIF" value="1096"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e077.PNG" value="771"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g004.TIF" value="1224906"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g004.PNG_I" value="59375"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g004.PNG_S" value="15344"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g004.PNG_M" value="139341"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g004.PNG_L" value="825657"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e078.TIF" value="1284"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e078.PNG" value="853"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e079.TIF" value="1268"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e079.PNG" value="854"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e080.TIF" value="1224"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e080.PNG" value="842"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e081.TIF" value="1578"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e081.PNG" value="971"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e082.TIF" value="1052"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e082.PNG" value="724"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e083.TIF" value="1020"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e083.PNG" value="715"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g005.TIF" value="1031006"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g005.PNG_I" value="58113"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g005.PNG_S" value="16413"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g005.PNG_M" value="125877"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g005.PNG_L" value="638649"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e084.TIF" value="1720"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e084.PNG" value="1055"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e085.TIF" value="1466"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e085.PNG" value="953"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e086.TIF" value="1748"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e086.PNG" value="1053"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e087.TIF" value="1468"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e087.PNG" value="944"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e088.TIF" value="1726"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e088.PNG" value="1046"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g006.TIF" value="1146388"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g006.PNG_I" value="65322"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g006.PNG_S" value="18334"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g006.PNG_M" value="124851"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g006.PNG_L" value="560986"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g007.TIF" value="907194"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g007.PNG_I" value="43805"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g007.PNG_S" value="10679"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g007.PNG_M" value="100913"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g007.PNG_L" value="583890"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g008.TIF" value="2068022"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g008.PNG_I" value="95012"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g008.PNG_S" value="22569"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g008.PNG_M" value="235913"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.g008.PNG_L" value="1602055"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e089.TIF" value="892"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e089.PNG" value="663"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e090.TIF" value="1128"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e090.PNG" value="785"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e091.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e091.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e092.TIF" value="446"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e092.PNG" value="454"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e093.TIF" value="1128"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e093.PNG" value="785"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e094.TIF" value="1154"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e094.PNG" value="796"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e095.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e095.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e096.TIF" value="3242"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e096.PNG" value="1866"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e097.TIF" value="686"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e097.PNG" value="586"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e098.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e098.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e099.TIF" value="698"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e099.PNG" value="582"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e100.TIF" value="2196"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e100.PNG" value="1287"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e101.TIF" value="1200"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e101.PNG" value="817"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e102.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e102.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e103.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e103.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e104.TIF" value="1774"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e104.PNG" value="1061"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e105.TIF" value="1160"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e105.PNG" value="786"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e106.TIF" value="3446"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e106.PNG" value="1917"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e107.TIF" value="746"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e107.PNG" value="605"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e108.TIF" value="756"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e108.PNG" value="606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e109.TIF" value="1026"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e109.PNG" value="733"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e110.TIF" value="1230"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e110.PNG" value="825"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e111.TIF" value="3720"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e111.PNG" value="2007"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e112.TIF" value="670"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e112.PNG" value="568"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e113.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e113.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e114.TIF" value="1102"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e114.PNG" value="769"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e115.TIF" value="724"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e115.PNG" value="593"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e116.TIF" value="1026"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e116.PNG" value="733"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e117.TIF" value="2192"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e117.PNG" value="1291"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e118.TIF" value="1192"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e118.PNG" value="818"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e119.TIF" value="4300"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e119.PNG" value="2237"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e120.TIF" value="562"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e120.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e121.TIF" value="560"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e121.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e122.TIF" value="22324"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e122.PNG" value="10127"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e123.TIF" value="7338"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e123.PNG" value="3547"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e124.TIF" value="22246"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e124.PNG" value="8214"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e125.TIF" value="11424"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e125.PNG" value="4377"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e126.TIF" value="6012"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e126.PNG" value="2619"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e127.TIF" value="588"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e127.PNG" value="529"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e128.TIF" value="580"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e128.PNG" value="524"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e129.TIF" value="1076"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e129.PNG" value="765"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e130.TIF" value="580"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e130.PNG" value="524"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e131.TIF" value="826"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e131.PNG" value="638"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e132.TIF" value="7834"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e132.PNG" value="3710"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e133.TIF" value="9136"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e133.PNG" value="4389"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e134.TIF" value="1496"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e134.PNG" value="969"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e135.TIF" value="668"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e135.PNG" value="556"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e136.TIF" value="4334"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e136.PNG" value="2153"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e137.TIF" value="686"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e137.PNG" value="586"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e138.TIF" value="670"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e138.PNG" value="568"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e139.TIF" value="728"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e139.PNG" value="598"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e140.TIF" value="1058"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e140.PNG" value="754"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e141.TIF" value="620"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e141.PNG" value="550"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e142.TIF" value="682"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e142.PNG" value="578"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e143.TIF" value="1198"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e143.PNG" value="810"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e144.TIF" value="1204"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e144.PNG" value="814"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e145.TIF" value="1026"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e145.PNG" value="733"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e146.TIF" value="6938"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e146.PNG" value="3490"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e147.TIF" value="1424"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e147.PNG" value="913"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e148.TIF" value="540"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e148.PNG" value="500"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e149.TIF" value="540"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e149.PNG" value="500"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e150.TIF" value="3606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e150.PNG" value="2001"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e151.TIF" value="1212"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e151.PNG" value="821"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e152.TIF" value="3450"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e152.PNG" value="1914"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e153.TIF" value="540"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e153.PNG" value="500"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e154.TIF" value="7868"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e154.PNG" value="3950"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e155.TIF" value="500"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e155.PNG" value="476"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e156.TIF" value="2196"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e156.PNG" value="1287"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e157.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e157.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e158.TIF" value="684"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e158.PNG" value="563"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e159.TIF" value="1042"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e159.PNG" value="740"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e160.TIF" value="686"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e160.PNG" value="586"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e161.TIF" value="754"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e161.PNG" value="606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e162.TIF" value="1042"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e162.PNG" value="740"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e163.TIF" value="732"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e163.PNG" value="594"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e164.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e164.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e165.TIF" value="562"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e165.PNG" value="512"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e166.TIF" value="732"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e166.PNG" value="594"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e167.TIF" value="2150"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e167.PNG" value="1290"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e168.TIF" value="2204"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e168.PNG" value="1314"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e169.TIF" value="732"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e169.PNG" value="594"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e170.TIF" value="1432"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e170.PNG" value="917"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e171.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e171.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e172.TIF" value="1484"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e172.PNG" value="935"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e173.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e173.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e174.TIF" value="796"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e174.PNG" value="634"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e175.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e175.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e176.TIF" value="816"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e176.PNG" value="643"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e177.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e177.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e178.TIF" value="744"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e178.PNG" value="568"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e179.TIF" value="11114"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e179.PNG" value="5255"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e180.TIF" value="7870"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e180.PNG" value="3898"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e181.TIF" value="7576"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e181.PNG" value="3720"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e182.TIF" value="732"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e182.PNG" value="594"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e183.TIF" value="848"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e183.PNG" value="642"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e184.TIF" value="880"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e184.PNG" value="667"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e185.TIF" value="974"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e185.PNG" value="709"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e186.TIF" value="1956"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e186.PNG" value="1159"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e187.TIF" value="3214"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e187.PNG" value="1717"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e188.TIF" value="848"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e188.PNG" value="642"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e189.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e189.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e190.TIF" value="862"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e190.PNG" value="658"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e191.TIF" value="874"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e191.PNG" value="662"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e192.TIF" value="3718"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e192.PNG" value="1894"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e193.TIF" value="2936"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e193.PNG" value="1659"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e194.TIF" value="600"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e194.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e195.TIF" value="890"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e195.PNG" value="666"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e196.TIF" value="600"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e196.PNG" value="536"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e197.TIF" value="862"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e197.PNG" value="658"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e198.TIF" value="874"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e198.PNG" value="662"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e199.TIF" value="5380"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e199.PNG" value="2711"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e200.TIF" value="756"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e200.PNG" value="609"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e201.TIF" value="458"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e201.PNG" value="459"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e202.TIF" value="862"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e202.PNG" value="658"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e203.TIF" value="874"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e203.PNG" value="662"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e204.TIF" value="976"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e204.PNG" value="720"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e205.TIF" value="968"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e205.PNG" value="720"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e206.TIF" value="8606"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e206.PNG" value="4075"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e207.TIF" value="812"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e207.PNG" value="627"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e208.TIF" value="792"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e208.PNG" value="630"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e209.TIF" value="862"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e209.PNG" value="658"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e210.TIF" value="874"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e210.PNG" value="662"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e211.TIF" value="756"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e211.PNG" value="609"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e212.TIF" value="4864"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e212.PNG" value="2526"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e213.TIF" value="878"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e213.PNG" value="652"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e214.TIF" value="926"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e214.PNG" value="675"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e215.TIF" value="904"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e215.PNG" value="671"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e216.TIF" value="676"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e216.PNG" value="565"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e217.TIF" value="686"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e217.PNG" value="580"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e218.TIF" value="690"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e218.PNG" value="585"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e219.TIF" value="1248"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e219.PNG" value="839"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e220.TIF" value="1270"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e220.PNG" value="852"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e221.TIF" value="690"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e221.PNG" value="585"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e222.TIF" value="978"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e222.PNG" value="699"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e223.TIF" value="568"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e223.PNG" value="514"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e224.TIF" value="464"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e224.PNG" value="460"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e225.TIF" value="942"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e225.PNG" value="705"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e226.TIF" value="826"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e226.PNG" value="651"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e227.TIF" value="976"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e227.PNG" value="717"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e228.TIF" value="826"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e228.PNG" value="651"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e229.TIF" value="1112"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e229.PNG" value="775"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e230.TIF" value="822"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e230.PNG" value="653"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e231.TIF" value="1112"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e231.PNG" value="775"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e232.TIF" value="816"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.e232.PNG" value="633"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s001.TIF" value="439454"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s002.TIF" value="399023"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s003.TIF" value="190913"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s004.TIF" value="552888"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s005.TIF" value="612861"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s006.TIF" value="1575302"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s007.XLSX" value="26601"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s008.XLSX" value="26826"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s009.XLSX" value="26812"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s010.XLSX" value="26869"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s011.XLSX" value="41341"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s012.XLSX" value="42028"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s013.XLSX" value="21201"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002772.s014.PDF" value="90674"/>
<div id="articleContainer">

<span about="http://www.plos.org"  typeof="foaf:Organization" property="foaf:name" style="display:none; visibility:hidden;">PLOS</span>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader" xpathLocation="noSelect">
  <p>Open Access</p><p id="articleType">Research Article</p>
</div>

  <h1 xpathLocation="noSelect" property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay</h1>
  <div property="dc:description" datatype="" style="display:none; visibility:hidden;">Author SummaryThe amount of a given transcript in a cell is determined by a fine tuned balance of production and degradation in a complex regulatory network. Regulation of transcription controls when transcription occurs and how much mRNA is created, whereas regulation of degradation controls the rate at which messengers are destroyed. The latter mechanism has recently gained attention due to the increasing evidence of its key role in the overall co-ordination of gene expression. A long lifetime of mRNA enables a cell to produce more proteins from that mRNA. By contrast, a short lifetime rapidly alters protein levels in response to changing needs. Measuring mRNA stability is a complex and expensive experiment and, given the condition-specific response of the degradation pathway, it would be desirable to take advantage of the large variety of expression experiments stored in public databases. To this end, we developed a stochastic model to infer each specific mRNA lifetime from gene expression data. Predictions were validated using malaria data. We then applied our methodology to the reproductive and metabolic cycle of budding yeast and found, in all cases, the presence of a general principle for the proper coordination of transcription and degradation machinery.</div>
  <span property="dc:date" content="2012-11-08" datatype="xsd:date" rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pcbi.1002772"></span>
  <span property="dc:subject" content="Computational Biology"></span>
<form action="">
  <input type="hidden" name="journalDisplayName" id="journalDisplayName" value="PLOS Computational Biology" />
  <input type="hidden" name="crossRefPageURL" id="crossRefPageURL" value="/article/crossref/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" />
  <input type="hidden" name="twitterPageURL" id="twitterPageURL" value="/article/twitter/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" />
  <input type="hidden" name="metricsTabURL" id="metricsTabURL" value="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" />
  <input type="hidden" name="doi" id="doi" value="info:doi/10.1371/journal.pcbi.1002772" />
  <input type="hidden" name="articleTitleUnformatted" id="articleTitleUnformatted" value="Stochastic%20Modeling%20of%20Expression%20Kinetics%20Identifies%20Messenger%20Half-Lives%20and%20Reveals%20Sequential%20Waves%20of%20Co-ordinated%20Transcription%20and%20Decay" />
  <input type="hidden" name="articlePubDate" id="articlePubDate" value="1352361600000" />
</form>
<div class="horizontalTabs" xpathLocation="noSelect">
  <ul id="tabsContainer">
        <li id="article" class="active"><a href="/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenuBox" xpathLocation="noSelect">
    <div id="articleMenu">
      <div class="wrap">
        <ul>
          <li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" onclick="toggleAnnotation(this, 'public'); return false;" title="Click to turn notes on/off">Hide notes</a></li>
          <li class="discuss icon">
              <a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002772">Make a general comment</a>
          </li>
        </ul>
        <div id="sectionNavTopBox" style="display:none;">
          <p><strong>Jump to</strong></p>
          <div id="sectionNavTop" class="tools"></div>
        </div>
      </div>
    </div>
      <div id="lightBox" xpathLocation="noDialog" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772')">
      </div>
  </div>




<p class="authors" xpathLocation="noSelect"><span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Filippo Cacace</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#equal-contrib">#</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Paola Paci</span></span><sup><a href="#aff1">1</a></sup><sup>,</sup><sup><a href="#aff2">2</a></sup><sup><a href="#equal-contrib">#</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Valerio Cusimano</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Alfredo Germani</span></span><sup><a href="#aff3">3</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Lorenzo Farina</span></span><sup><a href="#aff4">4</a></sup></p><p class="affiliations" xpathLocation="noSelect"><a name="aff1" id="aff1"></a><strong>1</strong> Università Campus Biomedico, Rome, Italy, <a name="aff2" id="aff2"></a><strong>2</strong> CNR-Institute of Systems Analysis and Computer Science (IASI), BioMathLab, Rome, Italy, <a name="aff3" id="aff3"></a><strong>3</strong> Dipartimento di Ingegneria e Scienze dell'Informazione e Matematica (DISIM), Università de L'Aquila, L'Aquila, Italy, <a name="aff4" id="aff4"></a><strong>4</strong> Dipartimento di Informatica e Sistemistica, Sapienza Università di Roma, Rome, Italy</p><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2 xpathLocation="noSelect">Abstract&nbsp;<a href="#top">Top</a></h2><p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]/p[1]">The transcriptome in a cell is finely regulated by a large number of molecular mechanisms able to control the balance between mRNA production and degradation. Recent experimental findings have evidenced that fine and specific regulation of degradation is needed for proper orchestration of a global cell response to environmental conditions. We developed a computational technique based on stochastic modeling, to infer condition-specific individual mRNA half-lives directly from gene expression time-courses. Predictions from our method were validated by experimentally measured mRNA decay rates during the intraerythrocytic developmental cycle of <em>Plasmodium falciparum</em>. We then applied our methodology to publicly available data on the reproductive and metabolic cycle of budding yeast. Strikingly, our analysis revealed, in all cases, the presence of periodic changes in decay rates of sequentially induced genes and co-ordination strategies between transcription and degradation, thus suggesting a general principle for the proper coordination of transcription and degradation machinery in response to internal and/or external stimuli.</p>
</div><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]"><a id="abstract1" name="abstract1" toc="abstract1" title="Author Summary"></a>
<h2 xpathLocation="noSelect">Author Summary&nbsp;<a href="#top">Top</a></h2>
<p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]/p[1]">The amount of a given transcript in a cell is determined by a fine tuned balance of production and degradation in a complex regulatory network. Regulation of transcription controls when transcription occurs and how much mRNA is created, whereas regulation of degradation controls the rate at which messengers are destroyed. The latter mechanism has recently gained attention due to the increasing evidence of its key role in the overall co-ordination of gene expression. A long lifetime of mRNA enables a cell to produce more proteins from that mRNA. By contrast, a short lifetime rapidly alters protein levels in response to changing needs. Measuring mRNA stability is a complex and expensive experiment and, given the condition-specific response of the degradation pathway, it would be desirable to take advantage of the large variety of expression experiments stored in public databases. To this end, we developed a stochastic model to infer each specific mRNA lifetime from gene expression data. Predictions were validated using malaria data. We then applied our methodology to the reproductive and metabolic cycle of budding yeast and found, in all cases, the presence of a general principle for the proper coordination of transcription and degradation machinery.</p>
</div>


<div class="articleinfo" xpathLocation="noSelect"><p><strong>Citation: </strong>Cacace F, Paci P, Cusimano V, Germani A, Farina L (2012) Stochastic Modeling of Expression Kinetics Identifies Messenger Half-Lives and Reveals Sequential Waves of Co-ordinated Transcription and Decay. PLoS Comput Biol 8(11):
          e1002772.
            doi:10.1371/journal.pcbi.1002772</p><p><strong>Editor: </strong>Teresa M. Przytycka, 
        National Center for Biotechnology Information (NCBI), United States of America
      </p><p></p><p><strong>Received:</strong> May 5, 2012; <strong>Accepted:</strong> September 23, 2012; <strong>Published:</strong> November 8, 2012</p><p><strong>Copyright:</strong> © 2012 Cacace et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors are partially supported by The Epigenomics Flagship Project (Progetto Bandiera Epigenomica) EPIGEN funded by Italian Ministry of Education, University and Research (MIUR) and the National Research Council of Italy (CNR). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p><p><a name="cor1"></a>* E-mail: <a href="mailto:paola.paci@iasi.cnr.it">paola.paci@iasi.cnr.it</a></p><p><a name="equal-contrib"></a>#
              These authors contributed equally to this work.
          </p></div>





<div id="section1" xpathLocation="/article[1]/body[1]/sec[1]"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3 xpathLocation="noSelect">Introduction&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[1]/p[1]">Appropriate and timely changes in gene expression are essential for cell life. The transcriptome is finely regulated by a large number of molecular mechanisms able to adjust the balance between mRNA production and degradation. Every aspect of transcript life is subject to elaborate control but, traditionally, the focus of the research has been on transcriptional regulation <a href="#pcbi.1002772-Garneau1">[1]</a>. However, whereas mRNA abundance results from the dynamic interplay between transcription and degradation, the speed by which cells can adjust their mRNA levels is critically dependent on the rate of mRNA turnover <a href="#pcbi.1002772-Munchel1">[2]</a>. As a result, small changes in mRNA stability may dramatically drive rapid variations of transcript abundance. Efforts to understand the underlying principles of mRNA decay and transcription co-ordination are very important since the balance between transcription and decay influences most, if not all, the cell responses to endogenous and exogenous signals <a href="#pcbi.1002772-Keene1">[3]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[2]">The current widespread interest in this topic has been fostered by the finding of specific regulatory mechanisms of mRNA stability such as, for example, RNA binding proteins <a href="#pcbi.1002772-Gerber1">[4]</a>, <a href="#pcbi.1002772-Hogan1">[5]</a> and small RNAs <a href="#pcbi.1002772-Houseley1">[6]</a>. Regulation of transcript stability cannot be considered a simple “disposal system” but a sophisticated tool for the proper orchestration of the global cell response to internal and external stimuli <a href="#pcbi.1002772-Houseley1">[6]</a>. Remarkably, a key role of mRNA stability has been reported in cancer, inflammatory diseases and Alzheimer's <a href="#pcbi.1002772-Cheneval1">[7]</a>. In recent years there has been a surge in empirical studies that measured, on a genome-wide scale in a variety of environmental conditions, messenger half-lives of many organisms, including plants <a href="#pcbi.1002772-Narsai1">[8]</a> mammals <a href="#pcbi.1002772-Sharova1">[9]</a> and fungi <a href="#pcbi.1002772-Munchel1">[2]</a>. The discovery of such new regulatory layer has clarified that, in order to obtain a clear picture of the underlying regulatory machinery, it is necessary to complement the traditional time-course experiment measuring the cell transcriptional response under certain conditions far from steady state with decay rates data under the <em>same</em> condition <a href="#pcbi.1002772-Shalem1">[10]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[3]">Experimental procedures for the evaluation of mRNA decay rates are based on measuring gene expression upon inhibition of transcription <a href="#pcbi.1002772-Wang1">[11]</a>–<a href="#pcbi.1002772-Coller1">[13]</a> or on pulse-chase RNA labeling protocols <a href="#pcbi.1002772-Munchel1">[2]</a>, <a href="#pcbi.1002772-Coller1">[13]</a>–<a href="#pcbi.1002772-GarciaMartinez1">[15]</a>. Such protocols are very critical (see <a href="#pcbi-1002772-g001">Figure 1</a> for a comparison among different studies summarized in <a href="#pcbi-1002772-t001">Table 1</a>), since, for instance, transcriptional shut-off blocks growth and has a profound effect on cellular physiology, as well as on mRNA metabolism <a href="#pcbi.1002772-Munchel1">[2]</a>. In fact, Wang <em>et al.</em> <a href="#pcbi.1002772-Wang1">[11]</a> and Grigull <em>et al.</em> <a href="#pcbi.1002772-Grigull1">[12]</a> datasets show a low value of the Pearson correlation (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e001&amp;representation=PNG"></span>), and no correlation at all can be found (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e002&amp;representation=PNG"></span>) between Munchel <em>et al.</em> <a href="#pcbi.1002772-Munchel1">[2]</a> and Wang <em>et al.</em> <a href="#pcbi.1002772-Wang1">[11]</a> datasets (see <a href="#pcbi-1002772-g001">Figure 1A</a> and <a href="#pcbi-1002772-g001">Figure 1B</a> respectively). Despite the same experimental conditions (asynchronous growth), the two half-life independent measurements obtained by Wang <em>et al.</em> and Munchel <em>et al.</em> are uncorrelated, probably due to differences in the shut off protocol (pulse chase for <a href="#pcbi.1002772-Munchel1">[2]</a> and thermal inactivation for <a href="#pcbi.1002772-Wang1">[11]</a>), whereas Grigull <em>et al.</em> and Wang <em>et al.</em> appears significantly correlated, probably due to the same shut off protocol used (thermal inactivation).</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[1]/fig[1]"><a name="pcbi-1002772-g001" id="pcbi-1002772-g001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g001');"><span>Figure 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[1]/fig[1]/caption[1]/title[1]">Basic comparison statistics among the yeast <em>S. cerevisiae</em> mRNA half-lives during asynchronous growth measured by three independent laboratories.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[1]/fig[1]/caption[1]/p[1]">Three genome-wide studies considered are: Grigull <em>et al.</em> <a href="#pcbi.1002772-Grigull1">[12]</a>, Wang <em>et al.</em> <a href="#pcbi.1002772-Wang1">[11]</a> and Munchel <em>et al.</em> <a href="#pcbi.1002772-Munchel1">[2]</a>. (A) Scatterplot of Wang <em>et al.</em> and Grigull <em>et al.</em> datasets; (B) Scatterplot of Munchel <em>et al.</em> and Wang <em>et al.</em> datasets.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g001.TIF"></span> TIFF)
                  </a></li></ul></div></div><div class="figure" xpathLocation="/article[1]/body[1]/sec[1]/table-wrap[1]"><a name="pcbi-1002772-t001" id="pcbi-1002772-t001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.t001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.t001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.t001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[1]/table-wrap[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.t001');"><span>Table 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[1]/table-wrap[1]/caption[1]/title[1]">Half-lives experimental measurements.</span></strong></p><span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.t001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.t001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.t001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.t001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.t001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.t001.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[1]/p[4]">It has been shown that genes having the same biological function <a href="#pcbi.1002772-Wang1">[11]</a>, <a href="#pcbi.1002772-Grigull1">[12]</a> are likely to share similar half-life values. Consistently, by averaging using functional groups, we found an increase in correlation between Wang <em>et al.</em> and Grigull <em>et al.</em> datasets (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e003&amp;representation=PNG"></span>, see Supplementary <a href="#pcbi.1002772.s001">Figure S1A</a>), and still no correlation between Munchel <em>et al.</em> and Wang <em>et al.</em> datasets (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e004&amp;representation=PNG"></span>, see Supplementary <a href="#pcbi.1002772.s001">Figure S1B</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[5]">Here, we developed a stochastic computational model of the expression kinetics to identify condition-specific mRNA stabilities which makes use only of experimental mRNA time profiles. We also assumed that degradation rates are gene-specific but approximately constant over the experiment time course. Predictions of our algorithm, termed DRAGON (Decay RAtes from Gene expressiON), were validated on experimental mRNA abundance <a href="#pcbi.1002772-Bozdech1">[16]</a> and turnover <a href="#pcbi.1002772-Shock1">[17]</a> data, both collected during the Intraerythrocytic Developmental Cycle (IDC) of <em>Plasmodium falciparum</em>. The estimations were in line with the experimental measurements. Remarkably, the DRAGON estimated half-lives were consistent with the finding of a peculiar pattern of mean half-life values along the wave of sequentially induced genes in subsequent stages of <em>P. falciparum</em> development. We also applied our methodology to public time-series datasets for which half-lives data, under the same experimental conditions, have not been experimentally measured. In particular we focused on budding yeast reproductive <a href="#pcbi.1002772-Pramila1">[18]</a>, <a href="#pcbi.1002772-Orlando1">[19]</a> and metabolic cycle data <a href="#pcbi.1002772-Tu1">[20]</a>. In fact, for the yeast <em>Saccharomyces cerevisiae</em>, only half-life data under asynchronous growth are publicly available <a href="#pcbi.1002772-Munchel1">[2]</a>, <a href="#pcbi.1002772-Wang1">[11]</a>, <a href="#pcbi.1002772-Grigull1">[12]</a>. Our study showed the presence of the same periodic pattern of mean half-life values in all datasets, thus suggesting that such behavior may be a general feature, not limited to the <em>Plasmodium falciparum</em> IDC.</p>

<h4 xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/title[1]">mRNA kinetics and half-life</h4>
<p xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/p[1]">Experimental evidence suggest that the majority of mRNAs are degraded with a first-order decay rate <a href="#pcbi.1002772-Munchel1">[2]</a>, <a href="#pcbi.1002772-Ross1">[21]</a>. This allows to characterize mRNA disappearance time profiles by a first-order rate equation<br><a name="pcbi.1002772.e005" id="pcbi.1002772.e005"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e005&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/p[1]/disp-formula[1]/label[1]">(1)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e006&amp;representation=PNG"></span> is the decay rate (or half-life <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e007&amp;representation=PNG"></span>, with <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e008&amp;representation=PNG"></span>), <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e009&amp;representation=PNG"></span> is the mRNA concentration and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e010&amp;representation=PNG"></span> is the promoter activity (the rate of production of new mRNAs). It is worth noting that, the degradation rate <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e011&amp;representation=PNG"></span> cannot be estimated from the concentration time profile <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e012&amp;representation=PNG"></span> for a single gene, since the term <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e013&amp;representation=PNG"></span> is not usually available in the typical time-course microarray experiment. The measurement of the promoter activity time profile would require additional experiments (such as those described in <a href="#pcbi.1002772-GarciaMartinez1">[15]</a>) but, in this paper, we will assume that only mRNA abundance time-series data are available. At steady-state <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e014&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e015&amp;representation=PNG"></span> are constant so that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e016&amp;representation=PNG"></span> and, consequently, we obtain<br><a name="pcbi.1002772.e017" id="pcbi.1002772.e017"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e017&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/p[1]/disp-formula[2]/label[1]">(2)</span></span><br>From the above equation, it is clear that at steady-state an increase (decrease) in mRNA concentration can be produced either by an increase (decrease) of transcription or by a decreased (increased) value of the decay rate: the two regulatory strategies have therefore an equivalent outcome. As a result, from steady-states measurements, it is hopeless to reveal the relative contribution of transcription and degradation and, most importantly, their co-ordinated activity as well. By contrast, the whole kinetics of induction and relaxation, as measured by time-courses experiments, depends on the degradation and production rate in different ways: increasing (decreasing) the production rate results in a proportionally increased (decreased) mRNA abundance, whereas the rise time (<em>i.e.</em> the time required for the response to rise from <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e018&amp;representation=PNG"></span> to <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e019&amp;representation=PNG"></span> of its final value) is not affected. Increasing the decay rate results in a faster rise time both in the induction and relaxation phases, whereas a decrease results in slower rise time <a href="#pcbi.1002772-Shalem1">[10]</a>. This key point is illustrated in <a href="#pcbi-1002772-g002">Figure 2A</a> and in Supplementary <a href="#pcbi.1002772.s002">Figures S2A and S2B</a>.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/fig[1]"><a name="pcbi-1002772-g002" id="pcbi-1002772-g002" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g002');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g002');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g002&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g002');"><span>Figure 2. </span></a> <span xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/fig[1]/caption[1]/title[1]">Kinetics of gene induction.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/fig[1]/caption[1]/p[1]">Panel A shows <em>in silico</em> experiments to illustrate some basic features of gene induction kinetics. The “ON” and “OFF” regions correspond to the turning “ON” or “OFF” of the promoter activity. (A) Induction kinetic of transcripts having the same steady-state concentration, but different half-lives and synthesis rates reaching the same steady state value. The time profile plotted in red corresponds to an unstable transcript and displays a fast induction and relaxation profile. By contrast, the blue one has an higher half-life value, resulting in a slower response. (B) Cascade of transcription factors resulting in waves of sequentially induced genes. The timing of expression peaks is modulated by transcriptional serial regulation. (C) Sequentially induced genes generated by a single transcription factor and a stability “gradient”. The timing of expression peaks is modulated by post-transcriptional regulation. Early induced genes are those with a low half-life value, late induced ones are those with an high half-life value.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g002</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g002/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g002/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g002.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g002/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g002.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/p[2]">Another important consequence of half-life specificity is the regulation of the timing of gene induction, as pointed out by Elkon <em>et al.</em> <a href="#pcbi.1002772-Elkon1">[22]</a>. In fact, an expression wave, <em>i.e.</em> the sequential activation of genes, is usually interpreted as resulting from the corresponding activation of a multi-step transcription factors cascade (as illustrated in <a href="#pcbi-1002772-g002">Figure 2B</a>). Whereas such mechanism is certainly very important, there is also an alternative way to obtain an expression wave by means of a “stability gradient”. As illustrated in <a href="#pcbi-1002772-g002">Figure 2C</a>, a single transcription factor may initiate transcription of a set of target genes and their peak of induction can be modulated by a stability “gradient”, <em>i.e.</em> by specifically adjusted decay rates. More precisely, early induced genes would have short half-lives and late responding genes would have long half-lives. Clearly, both mechanism may well act in cells, thus generating a wide spectrum of responses.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/p[3]">Time-courses are a very common design for microarray analysis, which allows researchers to follow the dynamics of the cellular response to perturbations <a href="#pcbi.1002772-Elkon1">[22]</a>. Such data are available for a very large number of experimental conditions and organisms: only the Stanford Microarray Database includes to date 1545 time course data sets. Among the examples later illustrated in the paper, it is worth mentioning the genome-wide gene expression time-series obtained during the reproductive cell cycle <a href="#pcbi.1002772-Pramila1">[18]</a>, <a href="#pcbi.1002772-Orlando1">[19]</a>, the metabolic cycle <a href="#pcbi.1002772-Tu1">[20]</a> and the <em>P. falciparum</em> IDC <a href="#pcbi.1002772-Bozdech1">[16]</a>. The time-series datasets used in this paper are summarized in <a href="#pcbi-1002772-t002">Table 2</a>.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/table-wrap[1]"><a name="pcbi-1002772-t002" id="pcbi-1002772-t002" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.t002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.t002');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.t002');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.t002&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/table-wrap[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.t002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.t002');"><span>Table 2. </span></a> <span xpathLocation="/article[1]/body[1]/sec[1]/sec[1]/table-wrap[1]/caption[1]/title[1]">Gene expression time-series experimental measurements.</span></strong></p><span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.t002</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.t002/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.t002/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.t002.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.t002/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.t002.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<h4 xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/title[1]">DRAGON–an algorithm for half-life estimation</h4>
<p xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[1]">The goal of the DRAGON methodology is to derive a robust estimate of each mRNA species half-life starting from all available gene expression pairs. The rationale for the algorithm mainly draws on properties of pairs exhibiting a certain degree of common promoter activity (as in <a href="#pcbi.1002772-Farina1">[23]</a>). Besides, DRAGON infers common promoter activity using a statistical model that simulates both gene-specific and common effects.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[2]">The rate of change of mRNA concentration for a generic pair of genes, say gene <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e020&amp;representation=PNG"></span> and gene <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e021&amp;representation=PNG"></span>, is:<br><a name="pcbi.1002772.e022" id="pcbi.1002772.e022"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e022&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[2]/disp-formula[1]/label[1]">(3)</span></span><br>where the symbols <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e023&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e024&amp;representation=PNG"></span> represent the mRNA time profiles of the gene pair <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e025&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e026&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e027&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e028&amp;representation=PNG"></span> are the promoters activity, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e029&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e030&amp;representation=PNG"></span> are the degradation rate of mRNA of gene <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e031&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e032&amp;representation=PNG"></span>, respectively. The terms <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e033&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e034&amp;representation=PNG"></span> are not known since we considered the case in which only mRNA abundance is measured.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[3]">We modeled promoter activities as the sum of two terms, the first one common to the pair and the other one specific for each gene:<br><a name="pcbi.1002772.e035" id="pcbi.1002772.e035"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e035&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[3]/disp-formula[1]/label[1]">(4)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e036&amp;representation=PNG"></span> is the common part, scaled by constants <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e037&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e038&amp;representation=PNG"></span>, whereas <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e039&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e040&amp;representation=PNG"></span> are gene specific independent stochastic processes with zero mean, that is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e041&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e042&amp;representation=PNG"></span>. <a href="#pcbi.1002772.e022">Equations (3)</a>–<a href="#pcbi.1002772.e035">(4)</a> encompass the case of:</p>


<ol class="roman-lower">

<li xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/list[1]/list-item[1]"><span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e043&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e044&amp;representation=PNG"></span> fully correlated (correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e045&amp;representation=PNG"></span>) for which <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e046&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e047&amp;representation=PNG"></span></li>

<li xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/list[1]/list-item[2]"><span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e048&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e049&amp;representation=PNG"></span> partially correlated (correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e050&amp;representation=PNG"></span>) for which <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e051&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e052&amp;representation=PNG"></span></li>

<li xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/list[1]/list-item[3]"><span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e053&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e054&amp;representation=PNG"></span> un-correlated (correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e055&amp;representation=PNG"></span>) for which either <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e056&amp;representation=PNG"></span> or <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e057&amp;representation=PNG"></span>.</li>

</ol><p xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[4]"><a href="#pcbi.1002772.e022">Equations (3)</a>–<a href="#pcbi.1002772.e035">(4)</a> can therefore be written for all available gene pairs; thus, for a set of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e058&amp;representation=PNG"></span> genes, we have <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e059&amp;representation=PNG"></span> pairs to analyze. For each gene pair DRAGON provides an estimate of the time profile of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e060&amp;representation=PNG"></span>, of all the parameters, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e061&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e062&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e063&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e064&amp;representation=PNG"></span>, and the covariance matrix of the stochastic processes. For each gene we therefore have <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e065&amp;representation=PNG"></span> estimates of the decay rate <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e066&amp;representation=PNG"></span>, one for each pair containing that gene.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[5]">Notice that <a href="#pcbi.1002772.e022">equations (3)</a>–<a href="#pcbi.1002772.e035">(4)</a> yield a couple of linear stochastic differential equations. Since measurements of mRNA concentrations are available only at given time points, it is necessary to transform (3)–(4) in a couple of discrete stochastic equations. The exact discretization of (3)–(4) is possible since they are linear <a href="#pcbi.1002772-Kailath1">[24]</a>. The <em>Kalman filter</em> <a href="#pcbi.1002772-Kalman1">[25]</a> is used on the resulting discrete equations and a maximum likelihood algorithm is exploited to generate the best possible estimate of the parameters.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/sec[2]/p[6]">A complete description of the mathematical model and of the discretization and parameters estimation procedure is given in the paragraph <em>Stochastic modeling of expression kinetics and Kalman filtering</em> of the <a href="#s4"><em>Materials and Methods</em></a> section.</p>

</div>

<div id="section2" xpathLocation="/article[1]/body[1]/sec[2]"><a id="s2" name="s2" toc="s2" title="Results"></a><h3 xpathLocation="noSelect">Results&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/title[1]">Performance evaluation on malaria IDC experimental data</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]">The IDC is characterized by four morphologic stages: ring, trophozoite, schizont and late schizont. The cycle begins with the red cells invasion by merozoites followed by a remodeling of the host cell in the ring stage <a href="#pcbi.1002772-Bozdech1">[16]</a>. The merozoites then develop into trophozoites. During the schizont stage, after a period of growth, the trophozoite undergoes an asexual dividing process and the parasite is ready for the next round of invasion by new merozoites (late schizont phase).</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[2]">Bozdech <em>et al.</em> <a href="#pcbi.1002772-Bozdech1">[16]</a>, using microarrays, measured genome-wide mRNA abundance profiles across 48 h during one cycle of <em>P. falciparum</em> IDC, collecting one sample per hour. Later on, Shock <em>et al.</em> <a href="#pcbi.1002772-Shock1">[17]</a> measured mRNA half-lives of 2774 transcripts of the IDC using chemical inhibitors to reach transcriptional shut-off.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[3]">The simultaneous availability of gene expression and decay data during the same biological process (IDC) represents a natural test bed for the validation of the DRAGON algorithm. Therefore, we applied DRAGON on Bozdech <em>et al.</em> dataset to obtain mRNA stability estimations (provided in Supplementary <a href="#pcbi.1002772.s012">Table S6</a>) to be compared with Shock <em>et al.</em> measurements for performance evaluation. The resulting Pearson correlation between <em>in vitro</em> and <em>in silico</em> measures is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e067&amp;representation=PNG"></span> (P value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e068&amp;representation=PNG"></span>), and the first principal component accounts for <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e069&amp;representation=PNG"></span> of the variability, thus showing a good performance for DRAGON algorithm (see <a href="#pcbi-1002772-g003">Figure 3</a>). However, since gene expression and decay data have been measured by different groups, we can speculate that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e070&amp;representation=PNG"></span> of unexplained variability may be partly due to inherent biological variability and to transcriptional inhibition stress. As further analysis, we computed average mRNA half-lives in both studies for functional categories (see Supplementary <a href="#pcbi.1002772.s003">Figure S3</a>). We found that the two studies are in better accordance when half-lives are averaged for all genes within any given functional category (Pearson correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e071&amp;representation=PNG"></span>).</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]"><a name="pcbi-1002772-g003" id="pcbi-1002772-g003" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g003');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g003');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g003&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g003');"><span>Figure 3. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/title[1]">DRAGON algorithm validation using <em>P. falciparum</em> IDC data.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/p[1]">Scatterplot of mRNA half-lives for 616 genes estimated by DRAGON <em>versus</em> experimentally measured by Shock <em>et al.</em> <a href="#pcbi.1002772-Shock1">[17]</a>.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g003</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g003/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g003/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g003.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g003/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g003.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[4]">Remarkably, Shock <em>et al.</em> in <a href="#pcbi.1002772-Shock1">[17]</a>, found progressive stage-dependent average increases in mRNA stability and suggested such phenomenon to be a major determinant of mRNA accumulation (see <a href="#pcbi-1002772-g004">Figure 4A</a>). The same feature is also found using DRAGON estimated half-lives (see <a href="#pcbi-1002772-g004">Figure 4B</a>). To investigate in further detail the behavior of average half-life of genes sequentially induced during IDC, we computed for each gene the time point corresponding to its peak of expression (see the <em>Data processing</em> paragraph of the <a href="#s4"><em>Materials and Methods</em></a> section for details) and selected <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e072&amp;representation=PNG"></span> groups of genes having peak of expression at each hourly time points over the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e073&amp;representation=PNG"></span> hours monitored by Shock <em>et al.</em> For each gene group we computed half-lives mean and standard deviation and found a high correlation with the corresponding curve obtained using experimental data (Pearson correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e074&amp;representation=PNG"></span>, P value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e075&amp;representation=PNG"></span>; see <a href="#pcbi-1002772-g004">Figure 4C</a>). Early responding genes are characterized by high instability, whereas late responders are more stable, as also reported by Elkon <em>et al.</em> in <a href="#pcbi.1002772-Elkon1">[22]</a> when studying mammalian cells. A possible explanation for the presence of stable mRNAs at the schizont stage, suggested by Shock <em>et al.</em>, is that it may be important for the merozoite to receive a carefully regulated “starting package”, that would allow rapid activation of the IDC following the next round of invasion <a href="#pcbi.1002772-Shock1">[17]</a>. By contrast, the initial low mRNA stability values may be an indication of the fast dynamic remodeling after merozoite invasion <a href="#pcbi.1002772-Shock1">[17]</a>. To evaluate the probability of obtaining such behavior by chance, we randomized the gene expression matrix and used DRAGON to estimate half-lives (see <a href="#pcbi-1002772-g004">Figure 4D</a>). Consistently, the estimation of half-lives using random data does not produce any correlation with experimental data (Pearson correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e076&amp;representation=PNG"></span>).</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]"><a name="pcbi-1002772-g004" id="pcbi-1002772-g004" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g004" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g004');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g004');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g004&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g004" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g004');"><span>Figure 4. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]/caption[1]/title[1]">Periodic behavior of average half-lives of sequentially induced genes in <em>P. falciparum</em> IDC.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]/caption[1]/p[1]">(A) Histograms of mRNA half-lives for genes induced at each stage of the <em>P. falciparum</em> IDC as experimentally measured by Shock <em>et al.</em> and (B) estimated by DRAGON algorithm. The inset panels show mean and standard deviation of half-lives during each stage. Both studies show an increase of average transcript stabilities of sequentially induced genes during <em>P. falciparum</em> IDC. (C) Average experimental and estimated half-life values (red and blue dots, respectively) corresponding to genes having the same expression peak timing, indicated on the x-axis. Standard deviations are drawn as pale blue and pale red stripes. The two curves both show a maximum peak of average half-life value during the schizont stage and a minimal value during the ring stage. A sharp increase of average half-life occurs during the trophozoite stage. The Person correlation between experimental and DRAGON estimated curve is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e077&amp;representation=PNG"></span>, thus showing a good agreement between the two studies. (D) Effect of randomizing the gene expression matrix on DRAGON estimated half-lives.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g004</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g004/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g004/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g004.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g004/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g004.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/title[1]">Half-lives estimation during reproductive cycle in <em>S. cerevisiae</em></h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[1]">Gene expression during yeast cell cycle has been recently measured by Pramila <em>et al.</em> <a href="#pcbi.1002772-Pramila1">[18]</a> using alpha-factor synchronization and by Orlando <em>et al.</em> <a href="#pcbi.1002772-Orlando1">[19]</a> using centrifugal elutriation for synchronization. We obtained a high consistency of DRAGON estimations using data for 569 transcripts over replicate datasets (Pearson correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e078&amp;representation=PNG"></span> for Pramila <em>et al.</em> dataset and Pearson correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e079&amp;representation=PNG"></span> for the Orlando <em>et al.</em> dataset; see <a href="#pcbi-1002772-g005">Figure 5A–B</a>). The larger variability in half-lives estimations may be explained by the inconsistencies between replicate time-series in the Pramila <em>et al.</em> dataset with respect to the Orlando <em>et al.</em> dataset (see <a href="#pcbi-1002772-g005">Figure 5C</a>). All half-lives estimations obtained with the DRAGON algorithm are provided in Supplementary <a href="#pcbi.1002772.s007">Tables S1</a> and <a href="#pcbi.1002772.s008">S2</a> (Pramila) and in Supplementary <a href="#pcbi.1002772.s009">Tables S3</a> and <a href="#pcbi.1002772.s010">S4</a> (Orlando). Notwithstanding significant differences in synchronization procedures, we also found a high correlation of DRAGON half-lives estimations over the two datasets (Pearson correlation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e080&amp;representation=PNG"></span>, P value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e081&amp;representation=PNG"></span>; see <a href="#pcbi-1002772-g005">Figure 5D</a>) where the first principal component accounts for <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e082&amp;representation=PNG"></span> of the overall variability. We can speculate that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e083&amp;representation=PNG"></span> of unexplained variability may be partly due to the different synchronization methods used. In fact, Orlando <em>et al.</em> obtained a cell cycle duration of about 2 hours, 8 samples per cycle <a href="#pcbi.1002772-Orlando1">[19]</a>, whereas Pramila <em>et al.</em> obtained a cell cycle duration of about 1 hour, 12 samples per cycle. Consistently, most of the transcripts during the slower cycle display higher half-lives when compared to the fastest cycle (see <a href="#pcbi-1002772-g005">Figure 5D</a>).</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]"><a name="pcbi-1002772-g005" id="pcbi-1002772-g005" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g005" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g005');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g005');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g005&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g005" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g005');"><span>Figure 5. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/title[1]">DRAGON performance over 569 cell cycle regulated genes from the Pramila and Orlando datasets.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/p[1]">(A) Scatterplot of the DRAGON estimated half-lives using two replicates taken from Pramila <em>et al.</em> dataset <a href="#pcbi.1002772-Pramila1">[18]</a> (denoted by alpha30 and alpha38). (B) Scatterplot of the DRAGON estimated half-lives using two replicates taken from Orlando <em>et al.</em> dataset <a href="#pcbi.1002772-Orlando1">[19]</a> (denoted by Orlando wtr1 and Orlando wtr2). (C) Histograms of the Pearson correlation values between time series relative to each gene in two replicates. Orlando <em>et al.</em> dataset shows a better consistency between replicates with respect to the Pramila <em>et al.</em> dataset. (D) Scatterplot of the DRAGON estimated half-lives using the Pramila <em>et al.</em> <a href="#pcbi.1002772-Pramila1">[18]</a> and the Orlando <em>et al.</em> dataset <a href="#pcbi.1002772-Orlando1">[19]</a>. The half-lives obtained by replicate datasets have been averaged. The half-lives estimated using Orlando dataset show slightly higher values with respect to those obtained using Pramila dataset, as shown by the deviation from the bisector line (dashed blue line). This is consistent with the slower cell cycle in Orlando experiment (2 hours) compared to that of Pramila experiment (1 hour).</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g005</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g005/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g005/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g005.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g005/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g005.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/title[1]">GO annotations of genes with extreme half-lives in <em>S. cerevisiae</em></h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[1]">In this paragraph we briefly discuss functional annotations (done using GOrilla software <a href="#pcbi.1002772-Eden1">[26]</a>) of novel predicted half-lives provided by DRAGON algorithm using yeast reproductive and metabolic cycle time series. For the yeast cell cycle we normalized the half-life log-distribution (Z-score), for each dataset, and then computed the geometric mean to obtain a single half-life value for each gene. Notably, the averaging has the effect of reducing the impact of the different synchronization stress response. The list of half-lives normalized values (geometric mean value equal to 1) for common genes to all datasets is provided as Supplementary <a href="#pcbi.1002772.s013">Table S7</a> in the <em>Half-life estimation</em> paragraph of the <a href="#s4"><em>Materials and Methods</em></a> section.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[2]">Unstable genes are enriched with replication fork complex (p-value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e084&amp;representation=PNG"></span>) and stable genes (histones HA1-2,HB1-2) are enriched with nucleosome (p-value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e085&amp;representation=PNG"></span>). This is consistent with the need of producing a large number of histones during DNA replication process so that stable histone mRNAs contribute to a higher translation efficiency. Moreover, DNA replication timing requires first the formation of the replication fork, then the production of the needed histones for chromatin assembling: such temporal sequence of events is consistent with a rapid turnover of the replication complex genes and a slow turnover of the histone genes (see Supplementary <a href="#pcbi.1002772.s004">Figure S4</a>). Among unstable genes we also found the G1/S transition cyclins and among stable ones we found G2/M transition cyclins (see Supplementary <a href="#pcbi.1002772.s005">Figure S5</a>). In this case, the temporal sequence of events is the progression of the cell cycle from DNA replication to mitosis.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[3]">For the yeast metabolic cycle (half-lives estimations using DRAGON algorithm are provided in Supplementary <a href="#pcbi.1002772.s011">Table S5</a>) we found many stable mRNA species involved in the organic acid and arginine metabolism and protein catabolic processes. Among unstable messengers we found genes involved in DNA repair (p-value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e086&amp;representation=PNG"></span>), DNA metabolism (p-value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e087&amp;representation=PNG"></span>) and chromatin silencing (p-value <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e088&amp;representation=PNG"></span>).</p>

</div>

<div id="section3" xpathLocation="/article[1]/body[1]/sec[3]"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3 xpathLocation="noSelect">Discussion&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/title[1]">Periodic behavior of average half-lives of sequentially induced genes</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[1]">The increasing pattern of average half-life found during <em>P. falciparum</em> IDC (shown in <a href="#pcbi-1002772-g004">Figure 4A</a>) motivated us to investigate whether a periodicity could be found also in other cyclical biological processes. We focused on the reproductive cell cycle and the metabolic cycle in <em>Saccharomyces cerevisiae</em>, for which high resolution time series measurements are available on public repositories (see <a href="#pcbi-1002772-t002">Table 2</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[2]">To study if a periodic pattern of average half-life of sequentially induced genes exists along the cell cycle progression, for each gene we computed the time points at which maximal expression is attained (see the <em>Data processing</em> paragraph of the <a href="#s4"><em>Materials and Methods</em></a> section for details). Thus, we obtained, for each time point, the list of genes having expression peak value at that time and computed the corresponding mean and variance of DRAGON estimated half-lives values. Indeed, we found a cyclic behavior along sequentially induced genes in both datasets (see <a href="#pcbi-1002772-g006">Figure 6A</a> for the Pramila <em>et al.</em> dataset and <a href="#pcbi-1002772-g006">Figure 6B</a> for the Orlando <em>et al.</em> dataset). Synchronization methods, cell cycle duration and number of samples are different between the two cited studies, but, reassuringly, the phases of the cell cycle at which mean half-life is minimal or maximal is consistent. In fact, for both datasets we observed a cyclical increase of mean half-life from G1 phase to M phase and a subsequent decrease back to G1. The figure clarifies that the minimal mean half-life is reached at the G1/S transition, whereas the maximal value correspond to the M/G2 phase for both cycles and datasets. The latter is consistent with the observation that, in higher eukaryotes, mitosis is accompanied by global repression of nuclear RNA synthesis <a href="#pcbi.1002772-Shermoen1">[27]</a>, indicating that mRNAs must be stable to be inherited from daughter cells.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/fig[1]"><a name="pcbi-1002772-g006" id="pcbi-1002772-g006" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g006" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g006');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g006');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g006&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g006" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g006');"><span>Figure 6. </span></a> <span xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/fig[1]/caption[1]/title[1]">Periodic behavior of average half-life during the reproductive and metabolic cycle.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/fig[1]/caption[1]/p[1]">Average DRAGON estimated messenger half-life values corresponding to genes having the same expression peak timing, indicated on the x-axis (dark blue points). Standard deviations are drawn as pale blue stripes. (A) yeast cell cycle using Pramila dataset, (B) yeast cell cycle using Orlando dataset and (C) yeast metabolic cycle. Strikingly, in all datasets the maximal average half-life is attained for genes induced during G2/M phase, including the metabolic cycle. The minimal average half-life, in all datasets, is attained at the G1/S transition phase. The bar charts show mean and standard deviation of half-lives during each stage.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g006</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g006/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g006/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g006.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g006/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g006.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[3]">The yeast metabolic cycle has been recently studied by Tu <em>et al.</em> <a href="#pcbi.1002772-Tu1">[20]</a> using a continuous culture system, after a brief starvation period, the culture spontaneously began persistent respiratory cycles of about 5 hours. In the same study, a genome-wide microarray gene expression measurement was performed. Samples were taken every 25 minutes for 3 consecutive cycles. Using DRAGON algorithm we estimated half-lives using data of 1043 transcripts. Surprisingly, also in this case we found a cyclical pattern for mean half-life of sequentially induced genes. The maximum peak is located at the RC phase and the minimum peak located at RB phase (see <a href="#pcbi-1002772-g006">Figure 6C</a>).</p>


<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/title[1]">Integrated analysis–sequential waves of co-ordinated transcription and decay</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[1]">Recently, the appearance of a number of studies has revealed the fundamental role of stability regulation in shaping appropriate cell response <a href="#pcbi.1002772-Garneau1">[1]</a>. A key point has been recently addressed by Shalem <em>et al.</em> <a href="#pcbi.1002772-Shalem1">[10]</a>, who have shown the dynamic co-ordinated interplay between transcription and degradation. They have found in yeast two basic regulatory strategies in response to stress. More precisely, they measured changes of mRNA abundance and decay rates in a yeast population subjected to oxidative and DNA damage stress. By grouping genes according to the time point at which the maximal (minimal) fold change is attained and combining normalized (mean and variance) mRNA abundance and decay rate data, they constructed a “stability <em>versus</em> folding” (SF) diagram where change in mRNA stability relative to a reference state (mean value in our case) is plotted against the maximal fold change. Using yeast expression time-course data obtained in response to an oxidative stress and a DNA damage, they were able to reveal two different strategies: a) a “counteracting regulation” strategy (see <a href="#pcbi-1002772-g007">Figure 7A</a>), characterized by genes in which an increase (decrease) in degradation rates counteracts a increase (decrease) in mRNA abundance, <em>i.e.</em> repressed genes are stabilized and induced genes are destabilized; b) a “synergistic regulation” strategy (see <a href="#pcbi-1002772-g007">Figure 7B</a>), characterized by genes in which an increase (decrease) in degradation rates is associated with an decrease (increase) in mRNA abundance, <em>i.e.</em> induced genes are stabilized and repressed genes are destabilized.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[1]"><a name="pcbi-1002772-g007" id="pcbi-1002772-g007" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g007" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g007');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g007');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g007&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g007" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g007');"><span>Figure 7. </span></a> <span xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[1]/caption[1]/title[1]">Illustration of the <em>counteracting</em> and <em>synergistic</em> regulatory strategy.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[1]/caption[1]/p[1]">The stability/folding diagrams (SF), introduced by Shalem <em>et al.</em> in <a href="#pcbi.1002772-Shalem1">[10]</a>, show the change in mRNA stability relative to the average value plotted against the maximal fold change. (A) Counteracting strategy (negative correlation): induced genes are destabilized and repressed genes are stabilized. (B) Synergistic strategy (positive correlation): induced genes are stabilized and repressed genes are destabilized.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g007</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g007/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g007/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g007.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g007/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g007.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[2]">Shalem <em>et al.</em> also found that, progressing from early time points forward, the negative correlation (counteracting) was replaced with a positive correlation (synergistic). Such co-ordination strategy may permit crosstalk between different steps of mRNA biogenesis, providing a mechanism to control the order and timing of events <a href="#pcbi.1002772-Dahan1">[28]</a>. The work of Shalem <em>et al.</em> has shown the importance of combining expression data with decay rates under the same experimental condition to reveal the underlying strategy of co-ordination of the two “regulatory arms”, namely transcription and degradation. Uncovering such relationships is certainly a fundamental task, since the underlying reciprocal influences between mRNA production and degradation are largely unexplored <a href="#pcbi.1002772-Shalem1">[10]</a>. The DRAGON algorithm, by estimating half-lives directly from gene expression data under specific conditions, allows the computational integration of mRNA abundance and decay rates data, making this powerful combined analysis possible when experimentally measured half-lives are not available.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[3]">We computed SF diagrams for <em>P. falciparum</em> IDC, yeast cell cycle (Pramila <em>et al.</em> dataset) and metabolic cycle (shown in <a href="#pcbi-1002772-g008">Figure 8</a>). In panels A,C and E each blue dot corresponds to a Pearson correlation of the SF diagram at the peak time point indicated on the x-axis, for the three datasets. In panels B,D and F the SF diagrams corresponding to the correlation values indicated by the arrows in panels A,C and E are displayed. The arrows point to maximal negative (red dots in panels B,D,F and red arrows in panels A,C,E) and maximal positive correlation values (green dots in panels B,D,F and green arrows in panels A,C,E).</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[2]"><a name="pcbi-1002772-g008" id="pcbi-1002772-g008" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g008" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g008');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g008');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.g008&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[2]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002772?imageURI=info:doi/10.1371/journal.pcbi.1002772.g008" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002772', 'info:doi/10.1371/journal.pcbi.1002772.g008');"><span>Figure 8. </span></a> <span xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[2]/caption[1]/title[1]">Temporal progress on the stability/folding (SF) diagrams.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/fig[2]/caption[1]/p[1]">In all cases under study, we found a progressive shift from an inverse (counteracting) to a direct (synergistic) relationship. (A) <em>Plasmodium falciparum</em> IDC, (C) yeast cell cycle (Pramila <em>et al.</em> dataset) and (E) yeast metabolic cycle. In panels A,C and E each blue dot corresponds to a Pearson correlation of the SF diagram at the peak time point indicated on the x-axis, for the three datasets. In panels B,D and F the SF diagrams corresponding to the correlation values indicated by arrows in panels A,C and E are displayed. The arrows point to maximal negative (red dots in panels B,D,F and red arrows in panels A,C,E) and maximal positive correlation values (green dots in panels B,D,F and corresponding green arrows in panels A,C,E).</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002772.g008</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002772.g008/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g008/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002772.g008.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002772.g008/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002772.g008.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[4]">Strikingly, in all cases we reached the same conclusions of Shalem <em>et al.</em>, namely we found that early induced genes show counteracting regulation, whereas late induced genes show a synergistic regulation.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[3]/title[1]">Advantages and disadvantages of the method</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[3]/p[1]">The main advantage of the DRAGON algorithm consists in the estimation of the mRNA half-lives directly from gene expression time-course during condition-specific experiments. Moreover it estimates the correlation among promoter activities between pairs of genes. Another advantage of the algorithm lies in its robustness. Specifically, we observed that even if the accuracy of the absolute values of the estimated half-lives can be influenced by many factors (such as the number of points in the time series, the accuracy of the measurements, the time interval between samples, the choice of the thresholds for the outliers, etc.), the ranking of half-lives is insensitive to the factors mentioned above.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[3]/p[2]">The main disadvantages are the following: DRAGON can work only with time-series under the same experimental condition and cannot handle steady-state values under different conditions. As a general rule a reliable estimate requires at least 10–12 time samples, <em>i.e.</em> a number significantly larger than the number of parameters to be estimated (this rule is not obviously always applicable as the required number of points depends strongly on the signal to noise ratio) and a sampling time not larger than the expected average half-life. If no information is available about the correlation of promoter activities, as a rule of thumb, a set of at least 50–100 time series must be processed together in order to have reliable half-lives estimates. One basic hypothesis is that the half-life of a transcript is approximately constant during the time course of the experiment, thus a substantial change of its value would yield an unreliable estimate. These problems can be handled by performing more measurements using a shorter sample time, or by considering moving time windows. The computational overhead can be significant: for a sample of 1000 time series there are <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e089&amp;representation=PNG"></span> pairs to analyze, requiring a computation of about 150 hours on a medium-speed single-processor machine capable of analyzing 2 pairs per second.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[4]/title[1]">Conclusion</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[4]/p[1]">Our analysis supports and strengthens Shalem <em>et al.</em> conclusions about the coordination of transcriptional and mRNA degradation in the cell in response to stress. We have demonstrated that during periodic processes, such as the <em>P. falciparum</em> IDC, the reproductive cell cycle and the metabolic cycle, the alternative interplays between changes in mRNA stability and changes in mRNA abundance are activated by periodically switching from a counteracting to a synergistic regulation. In light of these results, the classical vision of periodic processes as the result of serial transcription factor sequential activation, should be re-considered from a broader point of view by including post-transcriptional regulation and coordination.</p>

</div>

<div id="section4" xpathLocation="/article[1]/body[1]/sec[4]"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3 xpathLocation="noSelect">Materials and Methods&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/title[1]">Stochastic modeling of expression kinetics and Kalman filtering</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]">We defined as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e090&amp;representation=PNG"></span> the time profile of the expression of gene <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e091&amp;representation=PNG"></span> at time <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e092&amp;representation=PNG"></span>. The underlying conservation equation simply stems from the observation that the rate of change of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e093&amp;representation=PNG"></span> with time, <em>i.e.</em> its time derivative <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e094&amp;representation=PNG"></span>, must equal the difference between the production and degradation term. Based on experimental evidence <a href="#pcbi.1002772-Munchel1">[2]</a>, the degradation is well described by a first order term. The dynamics of the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e095&amp;representation=PNG"></span>-th transcript is therefore described by<br><a name="pcbi.1002772.e096" id="pcbi.1002772.e096"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e096&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]/disp-formula[1]/label[1]">(5)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e097&amp;representation=PNG"></span> is the mRNA decay rate of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e098&amp;representation=PNG"></span>-th messenger. This value is linked to the <em>half-life</em> <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e099&amp;representation=PNG"></span> of the transcript by the relation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e100&amp;representation=PNG"></span>. <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e101&amp;representation=PNG"></span> is the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e102&amp;representation=PNG"></span>-th gene promoter activity regulated by transcription factors. Such regulation occurs by triggering or suppressing the transcription of the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e103&amp;representation=PNG"></span>-th gene, thus we have <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e104&amp;representation=PNG"></span>. Moreover, the observed measure <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e105&amp;representation=PNG"></span> is also a noisy time-series, thus we have<br><a name="pcbi.1002772.e106" id="pcbi.1002772.e106"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e106&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]/disp-formula[2]/label[1]">(6)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e107&amp;representation=PNG"></span> is the standard deviation of measurements white noise <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e108&amp;representation=PNG"></span> (see supplementary material <a href="#pcbi.1002772.s014">Text S1</a> for an example of the identification procedure). We considered a generic pair of expression time profiles characterized by the presence of two terms: a stochastically correlated promoter activity <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e109&amp;representation=PNG"></span> and a gene-specific term <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e110&amp;representation=PNG"></span>. We then considered the case:<br><a name="pcbi.1002772.e111" id="pcbi.1002772.e111"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e111&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]/disp-formula[3]/label[1]">(7)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e112&amp;representation=PNG"></span> is a scaling factor accounting for the relative contribution to the overall promoter activity regarding gene <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e113&amp;representation=PNG"></span>. The term <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e114&amp;representation=PNG"></span> models the part of the promoter activity which is not common to the pair. We model this part by means of a noise term, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e115&amp;representation=PNG"></span> which is assumed to be a white noise. The common part <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e116&amp;representation=PNG"></span> is modeled as a Wiener process:<br><a name="pcbi.1002772.e117" id="pcbi.1002772.e117"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e117&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]/disp-formula[4]/label[1]">(8)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e118&amp;representation=PNG"></span> is white noise. Thus <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e119&amp;representation=PNG"></span>. The complete mathematical dynamic model for two transcripts <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e120&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e121&amp;representation=PNG"></span>, together with their respective measurement equations, is<br><a name="pcbi.1002772.e122" id="pcbi.1002772.e122"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e122&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]/disp-formula[5]/label[1]">(9)</span></span><br>We can rewrite the linear dynamic system (9) using a compact matrix notation<br><a name="pcbi.1002772.e123" id="pcbi.1002772.e123"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e123&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]/disp-formula[6]/label[1]">(10)</span></span><br>where<br><a name="pcbi.1002772.e124" id="pcbi.1002772.e124"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e124&amp;representation=PNG"></span><br>and<br><a name="pcbi.1002772.e125" id="pcbi.1002772.e125"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e125&amp;representation=PNG"></span><br><br><a name="pcbi.1002772.e126" id="pcbi.1002772.e126"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e126&amp;representation=PNG"></span><br>Since the dynamic system (10) is linear, it can be exactly discretized (see <a href="#pcbi.1002772-Kailath1">[24]</a>) for a given time interval <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e127&amp;representation=PNG"></span>, corresponding to the time interval between two consecutive measurements. The <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e128&amp;representation=PNG"></span>-th measurements corresponds to <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e129&amp;representation=PNG"></span>, thus in the discretized system we can use <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e130&amp;representation=PNG"></span> in place of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e131&amp;representation=PNG"></span>, to keep the notation simple.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[2]">The solution of the linear dynamic system (10) is<br><a name="pcbi.1002772.e132" id="pcbi.1002772.e132"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e132&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[2]/disp-formula[1]/label[1]">(11)</span></span><br>and its discretized form is<br><a name="pcbi.1002772.e133" id="pcbi.1002772.e133"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e133&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[2]/disp-formula[2]/label[1]">(12)</span></span><br>where<br><a name="pcbi.1002772.e134" id="pcbi.1002772.e134"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e134&amp;representation=PNG"></span><br>and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e135&amp;representation=PNG"></span> is the covariance matrix defined by<br><a name="pcbi.1002772.e136" id="pcbi.1002772.e136"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e136&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[2]/disp-formula[4]/label[1]">(13)</span></span><br>The unknown parameters of the model to be estimated are <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e137&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e138&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e139&amp;representation=PNG"></span>, with <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e140&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e141&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e142&amp;representation=PNG"></span>. The <em>state variables</em> of the system are <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e143&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e144&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e145&amp;representation=PNG"></span>. For each given choice of the parameters we used the Kalman filter <a href="#pcbi.1002772-Kalman1">[25]</a> to estimate of the state variables.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[3]">The Kalman filter equation uses a feedback control strategy. It contains a <em>prediction term</em> for projecting forward (in time) the current state to obtain the <em>a priori</em> estimate, and a correcting term for incorporating a new measurement into the <em>a priori</em> estimate to obtain an improved <em>a posteriori</em> estimate<br><a name="pcbi.1002772.e146" id="pcbi.1002772.e146"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e146&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[3]/disp-formula[1]/label[1]">(14)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e147&amp;representation=PNG"></span> is the prediction Kalman gain that depends on the parameters <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e148&amp;representation=PNG"></span> of the stochastic equation.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[4]">For each choice of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e149&amp;representation=PNG"></span> we run the Kalman filter. A probability value is associated to the resulting estimation. These values measures the probability that the current parametrization of the model generates the measured time series. Denoting by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e150&amp;representation=PNG"></span> the <em>innovation</em> of the stochastic process, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e151&amp;representation=PNG"></span> is a sequence of independent gaussian random variables with covariance <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e152&amp;representation=PNG"></span>. The optimal set <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e153&amp;representation=PNG"></span> of parameters if therefore chosen according to a <em>maximum likelihood</em> criterion as the choice corresponding to the maximum of the <em>a priori</em> probability density of the innovation sequence. This corresponds to the minimum of the likelihood function<br><a name="pcbi.1002772.e154" id="pcbi.1002772.e154"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e154&amp;representation=PNG"></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e155&amp;representation=PNG"></span> is the number of samples. We are interested in the half life <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e156&amp;representation=PNG"></span> of the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e157&amp;representation=PNG"></span>-th messenger. To use all the available information and make the method robust with respect to measurement and estimation errors, we have designed the following algorithm (see Supplementary <a href="#pcbi.1002772.s006">Figure S6</a>):</p>


<ol class="order">

<li xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[1]">Given a set of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e158&amp;representation=PNG"></span> mRNA time profiles, perform the maximum likelihood estimation for <em>every</em> pair <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e159&amp;representation=PNG"></span> and compute the corresponding <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e160&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e161&amp;representation=PNG"></span>.</li>

<li xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[2]">For each pair <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e162&amp;representation=PNG"></span> compute the ratio matrix <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e163&amp;representation=PNG"></span> whose elements are the ratios between the half-lives of gene <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e164&amp;representation=PNG"></span> and gene <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e165&amp;representation=PNG"></span>. The matrix <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e166&amp;representation=PNG"></span> is generally not symmetric due to the presence of outliers and numerical sensitivity. Thus we defined the ratio estimation by row as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e167&amp;representation=PNG"></span> and by column as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e168&amp;representation=PNG"></span>. The matrix <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e169&amp;representation=PNG"></span> contains all the ratios <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e170&amp;representation=PNG"></span> on the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e171&amp;representation=PNG"></span>-th row, and all the ratios <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e172&amp;representation=PNG"></span> on the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e173&amp;representation=PNG"></span>-th column. Let us denote <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e174&amp;representation=PNG"></span> the sum of the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e175&amp;representation=PNG"></span>th row, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e176&amp;representation=PNG"></span> the sum of the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e177&amp;representation=PNG"></span> column, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e178&amp;representation=PNG"></span> the <em>mean</em> operator, that is,<br><a name="pcbi.1002772.e179" id="pcbi.1002772.e179"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e179&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[2]/p[1]/disp-formula[1]/label[1]">(15)</span></span><br><br><a name="pcbi.1002772.e180" id="pcbi.1002772.e180"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e180&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[2]/p[1]/disp-formula[2]/label[1]">(16)</span></span><br><br><a name="pcbi.1002772.e181" id="pcbi.1002772.e181"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e181&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[2]/p[1]/disp-formula[3]/label[1]">(17)</span></span><br></li>

<li xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[3]">Given <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e182&amp;representation=PNG"></span>, delete outliers to obtain a final matrix <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e183&amp;representation=PNG"></span>. First, compute the probability density (using a smoothing kernel approach) of all the entries <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e184&amp;representation=PNG"></span> and delete those values below a probability of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e185&amp;representation=PNG"></span> of occurring in the distribution. Second, since ideally <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e186&amp;representation=PNG"></span>, we considered as outliers those pairs such that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e187&amp;representation=PNG"></span>.</li>

<li xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[4]">On the resulting <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e188&amp;representation=PNG"></span> matrix compute for each transcript <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e189&amp;representation=PNG"></span> two estimates of its half-life <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e190&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e191&amp;representation=PNG"></span>, using <a href="#pcbi.1002772.e179">equations (15)</a>, <a href="#pcbi.1002772.e180">(16)</a>and <a href="#pcbi.1002772.e181">(17)</a>. We obtained<br><a name="pcbi.1002772.e192" id="pcbi.1002772.e192"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e192&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[4]/p[1]/disp-formula[1]/label[1]">(18)</span></span><br><br><a name="pcbi.1002772.e193" id="pcbi.1002772.e193"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e193&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[4]/p[1]/disp-formula[2]/label[1]">(19)</span></span><br>This computation requires the value of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e194&amp;representation=PNG"></span>. When this value is known for the group of transcripts under analysis the measured value can be used. Otherwise, letting <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e195&amp;representation=PNG"></span> one can obtain half-life values that are relative to the average half-life of the group. However, we have followed a third approach. All the results reported in this paper have been obtained by replacing <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e196&amp;representation=PNG"></span> with the geometric mean of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e197&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e198&amp;representation=PNG"></span>, that is<br><a name="pcbi.1002772.e199" id="pcbi.1002772.e199"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e199&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[4]/p[1]/disp-formula[3]/label[1]">(20)</span></span><br>The final estimate of the half-life <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e200&amp;representation=PNG"></span> for the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e201&amp;representation=PNG"></span>-th gene is computed as the weighted average of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e202&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e203&amp;representation=PNG"></span> using as weights the respective variances <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e204&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e205&amp;representation=PNG"></span> as follows<br><a name="pcbi.1002772.e206" id="pcbi.1002772.e206"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e206&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[4]/p[1]/disp-formula[4]/label[1]">(21)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e207&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e208&amp;representation=PNG"></span> are the standard deviation of the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e209&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e210&amp;representation=PNG"></span>, respectively.</li>

<li xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/list[1]/list-item[5]">We considered as a quality index for each estimated half-life <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e211&amp;representation=PNG"></span> the following:<br><a name="pcbi.1002772.e212" id="pcbi.1002772.e212"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e212&amp;representation=PNG"></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e213&amp;representation=PNG"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e214&amp;representation=PNG"></span> are noise variances of the discrete system (12) and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e215&amp;representation=PNG"></span> (see <a href="#pcbi.1002772.e136">equation (13)</a>) is the mutual covariance of the state noise between time series <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e216&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e217&amp;representation=PNG"></span>. Thus, high values of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e218&amp;representation=PNG"></span> imply the presence of a correlation between <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e219&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e220&amp;representation=PNG"></span> in <a href="#pcbi.1002772.e122">equation (9)</a>. We removed the half-lives having a <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e221&amp;representation=PNG"></span> value smaller than the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e222&amp;representation=PNG"></span> percentile of its distribution.</li>

</ol>

<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/title[1]">Datasets</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/p[1]">Public experimental data used throughout the paper are described in <a href="#pcbi-1002772-t001">Table 1</a> (experimental half-lives measurements) and in <a href="#pcbi-1002772-t002">Table 2</a> (gene expression time series). Pramila <em>et al.</em> in <a href="#pcbi.1002772-Pramila1">[18]</a> and Orlando <em>et al.</em> in <a href="#pcbi.1002772-Orlando1">[19]</a> experimentally measured genome-wide gene expression data during the reproductive cell cycle. We considered the ranking provided by the combined test developed by de Lichtenberg <em>et al.</em> <a href="#pcbi.1002772-deLichtenberg1">[29]</a> for each replicate for the two datasets and, among the list of 1000 genes with highest ranking, we selected those common to all datasets. We ended up with a list of 569 genes that we used for half-life estimation. Tu <em>et al.</em> in <a href="#pcbi.1002772-Tu1">[20]</a> experimentally measured genome-wide gene expression data during the metabolic cell cycle. We selected 1000 genes with the best periodicity score according to <a href="#pcbi.1002772-Tu1">[20]</a>. Of the 1000 genes, DRAGON estimated half-lives are 939.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/title[1]">Data processing</h4>
<h5 xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/sec[1]/title[1]">Half-lives determination of genes induced during each stage of <em>P. falciparum</em> IDC.</h5><p xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/sec[1]/p[1]">Shock <em>et al.</em> in <a href="#pcbi.1002772-Shock1">[17]</a> experimentally measured genome-wide values of decay rates for each gene in each of the four stages of the IDC. To obtain a single half-life value for each messenger, we performed a k-means clustering of microarray gene expression data <a href="#pcbi.1002772-Bozdech1">[16]</a> by considering 5 stages (according to <a href="#pcbi.1002772-Bozdech1">[16]</a>: early ring, ring, trophozoite, schizont and late schizont). Then, we merged the early ring cluster with the ring cluster to obtain the same stages as in Shock <em>et al.</em>. Among the 4488 genes in <a href="#pcbi.1002772-Bozdech1">[16]</a> we chose 1000 genes with the best periodicity score (power signal/power total ratio) according to <a href="#pcbi.1002772-Bozdech1">[16]</a>. Of the 1000 genes, DRAGON estimated half-lives are 967, available experimental half-lives are 675. Both data are available over a set of 616 genes.</p>



<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/title[1]">Expression peak timing estimation</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/p[1]">To estimate peak timing, for a given noisy gene expression time profile, we preliminary performed the smoothing algorithm presented by Bar-Joseph <em>et al.</em> in <a href="#pcbi.1002772-BarJoseph1">[30]</a>. The algorithm employs two parameters: grid <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e223&amp;representation=PNG"></span> (number of spline curves) and classes <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e224&amp;representation=PNG"></span> (number of classes to use for clustering). In particular, for Pramila datasets we used <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e225&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e226&amp;representation=PNG"></span>, for Orlando datasets we used <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e227&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e228&amp;representation=PNG"></span>, for Tu dataset we used <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e229&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e230&amp;representation=PNG"></span>, for Malaria dataset we used <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e231&amp;representation=PNG"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002772.e232&amp;representation=PNG"></span>.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/title[1]">Half-life estimations</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[1]">DRAGON estimated half-lives are provided as supplementary materials <a href="#pcbi.1002772.s007">Tables S1</a>, <a href="#pcbi.1002772.s008">S2</a>, <a href="#pcbi.1002772.s009">S3</a>, <a href="#pcbi.1002772.s010">S4</a>, <a href="#pcbi.1002772.s011">S5</a>, <a href="#pcbi.1002772.s012">S6</a>, <a href="#pcbi.1002772.s012">S7</a>, described <em>Supporting Information</em> section.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[2]">Matlab code will be provided upon request.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[3]">Additional data and information can be found at web site <a href="http://www.dis.uniroma1.it/~farina/dragon">http://www.dis.uniroma1.it/~farina/drago​n</a>.</p>

</div>

<div id="section5" xpathLocation="/article[1]/body[1]/sec[5]"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3 xpathLocation="noSelect">Supporting Information&nbsp;<a href="#top">Top</a></h3><a name="pcbi.1002772.s001" id="pcbi.1002772.s001"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s001">Figure S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[1]"><strong>Functional categories analysis in yeast </strong><strong><em>S. cerevisiae</em></strong><strong> during asynchronous growth measured by three laboratories.</strong> Three genome-wide studies are considered: Grigull <em>et al.</em>, Wang <em>et al.</em> and Munchel <em>et al.</em> (A) Average mRNA half-lives in both studies Wang <em>et al.</em> and the Grigull <em>et al.</em> datasets for 111 functional categories from the yeast GO Biological Process database (<a href="http://www.geneontology.org">http://www.geneontology.org</a>) that are represented in the set of 2863 transcripts by 5 or more members. (B) compare, in the same way, the Munchel <em>et al.</em> and the Wang <em>et al.</em> datasets.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[2]">(TIF)</p>
<a name="pcbi.1002772.s002" id="pcbi.1002772.s002"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s002">Figure S2. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[1]"><strong>Kinetics of gene induction.</strong> Panels A–B show <em>in silico</em> experiments to illustrate some basic features of gene induction kinetics. The reference time profile with unity steady-state is plotted in black. The “ON” and “OFF” regions correspond to the turning “ON” or “OFF” of the promoter activity. (A) Induction kinetic of transcripts having the same half-life value and, as a consequence, the same speed of response. The higher (or lower) steady-state value of the red and blue time profiles is due only to an increased (or decreased) transcription rate. (B) Induction kinetic of transcripts having different half-lives. The time profile plotted in red corresponds to an unstable transcript. It has a faster induction and relaxation profile but a lower steady-state value. By contrast, the blue one has an higher half-life value, resulting in a higher steady state value but a slower response. The example illustrates that, to obtain both a fast response and an high steady-state value, the regulatory strategy must destabilize transcriptionally up-regulated genes.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[2]">(TIF)</p>
<a name="pcbi.1002772.s003" id="pcbi.1002772.s003"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s003">Figure S3. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[1]"><strong>Functional categories analysis for DRAGON estimations using </strong><strong><em>P. falciparum</em></strong><strong> IDC data.</strong> Average mRNA half-lives in both studies, DRAGON iestimations <em>versus</em> and experimentally measured by Shock <em>et al.</em> half-lives, for 12 functional categories from the <em>P. falciparum</em> GO annotation database (<a href="http://www.geneontology.org">http://www.geneontology.org</a>) that are represented in the set of 616 transcripts by 5 or more members.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[2]">(TIF)</p>
<a name="pcbi.1002772.s004" id="pcbi.1002772.s004"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s004">Figure S4. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[1]"><strong>GO annotations of genes with extreme half-lives in </strong><strong><em>S. cerevisiae</em></strong> DNA replication timing requires first the formation of the replication fork, then the production of the needed histones for chromatin assembling: such temporal sequence of events is consistent with a rapid turnover of the replication complex genes and a slow turnover of the histone genes.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[2]">(TIF)</p>
<a name="pcbi.1002772.s005" id="pcbi.1002772.s005"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s005">Figure S5. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[1]"><strong>GO annotations of genes with extreme half-lives in </strong><strong><em>S. cerevisiae</em></strong> Among unstable genes we also found the G1/S transition cyclins and among stable ones we found G2/M transition cyclins. In this case, the temporal sequence of events is the progression of the cell cycle from DNA replication to mitosis.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[2]">(TIF)</p>
<a name="pcbi.1002772.s006" id="pcbi.1002772.s006"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s006">Figure S6. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[6]/caption[1]/p[1]">
            <strong>DRAGON algorithm pipeline.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[6]/caption[1]/p[2]">(TIF)</p>
<a name="pcbi.1002772.s007" id="pcbi.1002772.s007"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s007">Table S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[7]/caption[1]/p[1]">
            <strong>DRAGON estimated half-lives using alpha30 dataset.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[7]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002772.s008" id="pcbi.1002772.s008"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s008">Table S2. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[8]/caption[1]/p[1]">
            <strong>DRAGON estimated half-lives using alpha38 dataset.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[8]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002772.s009" id="pcbi.1002772.s009"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s009">Table S3. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[9]/caption[1]/p[1]">
            <strong>DRAGON estimated half-lives using Orlando replicate 1 dataset.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[9]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002772.s010" id="pcbi.1002772.s010"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s010">Table S4. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[10]/caption[1]/p[1]">
            <strong>DRAGON estimated half-lives using Orlando replicate 2 dataset.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[10]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002772.s011" id="pcbi.1002772.s011"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s011">Table S5. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[11]/caption[1]/p[1]">
            <strong>DRAGON estimated half-lives using metabolic cycle dataset.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[11]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002772.s012" id="pcbi.1002772.s012"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s012">Table S6. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[12]/caption[1]/p[1]">
            <strong>DRAGON estimated half-lives using </strong>
            <strong>
              <em>P. falciparum</em>
            </strong>
            <strong> dataset.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[12]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002772.s013" id="pcbi.1002772.s013"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s013">Table S7. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[13]/caption[1]/p[1]">
            <strong>DRAGON estimated normalized half-lives using all yeast datasets.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[13]/caption[1]/p[2]">(XLSX)</p>
<a name="pcbi.1002772.s014" id="pcbi.1002772.s014"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002772.s014">Text S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[14]/caption[1]/p[1]">
            <strong>Example of parameter estimation through the Kalman filter.</strong>
          </p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[14]/caption[1]/p[2]">(PDF)</p>
</div>





<div xpathLocation="noSelect"><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3 xpathLocation="noSelect">Acknowledgments&nbsp;<a href="#top">Top</a></h3>
<p xpathLocation="/article[1]/back[1]/ack[1]/p[1]">We would like to thank Prof. Pino Macino and Dr. Teresa Colombo for helpful suggestions on the manuscript and the “Consorzio interuniversitario per le Applicazioni di Supercalcolo Per Università e Ricerca” (CASPUR) for providing computing resources and Dr. Alessandro Federico for support.</p>
<p xpathLocation="/article[1]/back[1]/ack[1]/p[2]">The authors are partially supported by The Epigenomics Flagship Project (Progetto Bandiera Epigenomica) EPIGEN funded by Italian Ministry of Education, University and Research (MIUR) and the National Research Council of Italy (CNR).</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3 xpathLocation="noSelect">Author Contributions&nbsp;<a href="#top">Top</a></h3><p xpathLocation="noSelect">Conceived and designed the experiments: LF AG. Performed the experiments: FC PP VC. Analyzed the data: PP LF. Wrote the paper: FC PP AG LF. Made the figures: LF.</p></div><div xpathLocation="noSelect"><a id="references" name="references" toc="references" title="References"></a><h3 xpathLocation="noSelect">References&nbsp;<a href="#top">Top</a></h3><ol class="references" xpathLocation="noSelect"><li xpathLocation="noSelect"><a name="pcbi.1002772-Garneau1" id="pcbi.1002772-Garneau1"></a>Garneau N, Wilusz J, Wilusz C (2007) The highways and byways of mrna decay. Nat Rev Mol Cell Bio 8: 113–126. <a class="find" href="/article/findArticle.action?citedArticleID=6077229">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Munchel1" id="pcbi.1002772-Munchel1"></a>Munchel S, Shultzaberger R, Takizawa N, Weis K (2011) Dynamic profiling of mrna turnover reveals gene-specific and system-wide regulation of mrna decay. Mol Biol Cell 22: 2787–2795.
        doi:
        <a href="http://dx.doi.org/10.1091/mbc.E11-01-0028">10.1091/mbc.E11-01-0028</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=6077231">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Keene1" id="pcbi.1002772-Keene1"></a>Keene J (2010) The global dynamics of rna stability orchestrates responses to cellular activation. BMC Biol 8: 95.
        doi:
        <a href="http://dx.doi.org/10.1186/1741-7007-8-95">10.1186/1741-7007-8-95</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=6077233">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Gerber1" id="pcbi.1002772-Gerber1"></a>Gerber A, Herschlag D, Brown P (2004) Extensive association of functionally and cytotopically related mrnas with puf family rna-binding proteins in yeast. PLoS Biol 2: 342–354. <a class="find" href="/article/findArticle.action?citedArticleID=6077235">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Hogan1" id="pcbi.1002772-Hogan1"></a>Hogan D, Riordan D, Gerber A, Herschlag D, Brown P (2008) Diverse rna-binding proteins interact with functionally related sets of rnas, suggesting extensive regulatory system. PLoS Biol 6: e255. <a class="find" href="/article/findArticle.action?citedArticleID=6077237">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Houseley1" id="pcbi.1002772-Houseley1"></a>Houseley J, Tollervay D (2009) The many pathways of rna degradation. Cell 136: 763–776. <a class="find" href="/article/findArticle.action?citedArticleID=6077239">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Cheneval1" id="pcbi.1002772-Cheneval1"></a>Cheneval D, Kastelic T, Fuerst P, Parker C (2010) A review of methods to monitor the modulation of mrna stability: a novel approach to drug discovery and therapeutic intervention. J Biomol Screen 15: 609–622. <a class="find" href="/article/findArticle.action?citedArticleID=6077241">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Narsai1" id="pcbi.1002772-Narsai1"></a>Narsai R, Howell K, Millar A, O'Toole N, Small I, et al. (2007) Genome-wide analysis of mrna decay rates and their determinants in arabidopsis thaliana. Plant Cell 19: 3418–3436. <a class="find" href="/article/findArticle.action?citedArticleID=6077243">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Sharova1" id="pcbi.1002772-Sharova1"></a>Sharova L, Sharov A, Nedorezov T, Piao Y, Shaik N, et al. (2009) Database for mrna half-life of 19977 genes obtained by dna microarray analysis of pluripotent and differentiating mouse embryonic cells. DNA Res 16: 45–58. <a class="find" href="/article/findArticle.action?citedArticleID=6077245">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Shalem1" id="pcbi.1002772-Shalem1"></a>Shalem O, Dahan O, Martinez M, Furman I, Segal E, et al. (2008) Transient transcriptional re- sponses to stress are generated by opposing effects of mrna production and degradation. Mol Syst Biol 4: 223. <a class="find" href="/article/findArticle.action?citedArticleID=6077247">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Wang1" id="pcbi.1002772-Wang1"></a>Wang Y, Liu C, Storey J, Tibshirani R, Hershlag D, et al. (2002) Precision and functional specificity in mrna decay. Proc Natl Acad Sci U S A 99: 5860–5865. <a class="find" href="/article/findArticle.action?citedArticleID=6077249">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Grigull1" id="pcbi.1002772-Grigull1"></a>Grigull J, Mnaimneh S, Pootoolal J, Robinson M, Hughes T (2004) Genome-wide analysis of mrna stability using transcription inhibitors and microarrays reveals posttranscriptional control of ribosome biogenesis factors. Mol Cell Biol 25: 5534–5547. <a class="find" href="/article/findArticle.action?citedArticleID=6077251">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Coller1" id="pcbi.1002772-Coller1"></a>Coller J (2008) Methods to determine mrna half-life in saccharomyces cerevisiae. Methods Enzymol 448: 267–284. <a class="find" href="/article/findArticle.action?citedArticleID=6077253">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Rabani1" id="pcbi.1002772-Rabani1"></a>Rabani M, Levin J, Fan L, Adiconis X, Raychowdhury R, et al. (2011) Metabolic labeling of rna uncovers principles of rna production and degradation dynamics in mammalian cells. Nature Biotechnol 29: 436–442. <a class="find" href="/article/findArticle.action?citedArticleID=6077255">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-GarciaMartinez1" id="pcbi.1002772-GarciaMartinez1"></a>Garcia-Martinez J, Aranda A, Perez-Ortin J (2004) Genomic run-on evaluates transcription rates for all yeasts genes and identifies gene regulatory mechanisms. Mol Cell 15: 303–313.
        doi:
        <a href="http://dx.doi.org/10.1016/j.molcel.2004.06.004">10.1016/j.molcel.2004.06.004</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=6077257">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Bozdech1" id="pcbi.1002772-Bozdech1"></a>Bozdech Z, Llinas M, Pulliam B, Wong E, Zhu J, et al. (2003) The transcriptome of the intraery- throcytic developmental cycle of <em>Plasmodium falciparum</em>. PLoS Biol I: E5. <a class="find" href="/article/findArticle.action?citedArticleID=6077259">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Shock1" id="pcbi.1002772-Shock1"></a>Shock J, Fischer K, DeRisi J (2007) Whole-genome analysis of mrna decay in <em>Plasmodium falci- parum</em> reveals a global lengthening of mrna half-life during the intra-erythrocytic developmental cycle. Genome Biol 8: R134. <a class="find" href="/article/findArticle.action?citedArticleID=6077261">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Pramila1" id="pcbi.1002772-Pramila1"></a>Pramila T, Wu W, Miles S, Noble W, Breeden L (2006) The forkhead transcription factor hcm1 regulates chromosome segregation genes and fills the s-phase gap in the transcriptional circuitry of the cell cycle. Genes Dev 20: 2266–2278.
        doi:
        <a href="http://dx.doi.org/10.1101/gad.1450606">10.1101/gad.1450606</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=6077263">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Orlando1" id="pcbi.1002772-Orlando1"></a>Orlando D, Lin C, Bernard A, Wang J, Socolar J, et al. (2008) Global control of cell-cycle tran- scription by coupled cdk and network oscillators. Nature 453: 944–947. <a class="find" href="/article/findArticle.action?citedArticleID=6077265">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Tu1" id="pcbi.1002772-Tu1"></a>Tu B, Kudlicki A, Rowicka M, McNight S (2005) Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes. Science 310: 1152–1158. <a class="find" href="/article/findArticle.action?citedArticleID=6077267">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Ross1" id="pcbi.1002772-Ross1"></a>Ross J (1995) mrna stability in mammalian cells. Microbiol Rev 59: 423–450. <a class="find" href="/article/findArticle.action?citedArticleID=6077269">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Elkon1" id="pcbi.1002772-Elkon1"></a>Elkon R, Zlotorynski E, Zeller K, Agami R (2010) Major role for mrna stability in shaping the kinetics of gene induction. BMC Genomics 11: 259.
        doi:
        <a href="http://dx.doi.org/10.1186/1471-2164-11-259">10.1186/1471-2164-11-259</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=6077271">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Farina1" id="pcbi.1002772-Farina1"></a>Farina L, Santis AD, Salvucci S, Morelli G, Ruberti I (2008) Embedding mrna stability in corre-lation analysis of time-series gene expression data. PLoS Comput Biol 4: e1000141. <a class="find" href="/article/findArticle.action?citedArticleID=6077273">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Kailath1" id="pcbi.1002772-Kailath1"></a>Kailath T (1980) Linear systems. Prentice-Hall. 682 p. </li><li xpathLocation="noSelect"><a name="pcbi.1002772-Kalman1" id="pcbi.1002772-Kalman1"></a>Kalman R (1960) A new approach to linear filtering and prediction problems. Transactions of the ASME–Journal of Basic Engineering 82: 35–45. <a class="find" href="/article/findArticle.action?citedArticleID=6077277">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Eden1" id="pcbi.1002772-Eden1"></a>Eden E, Navon R, Steinfeld I, Lipson D, Yakhini Z (2009) Gorilla: A tool for discovery and visualization of enriched go terms in ranked gene lists. BMC Bioinformatics 2009: 10–48. <a class="find" href="/article/findArticle.action?citedArticleID=6077279">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Shermoen1" id="pcbi.1002772-Shermoen1"></a>Shermoen A, O'Farrell P (1991) Progression of the cell cycle through mitosis leads to abortion of nascent transcripts. Cell 67: 303–310. <a class="find" href="/article/findArticle.action?citedArticleID=6077281">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-Dahan1" id="pcbi.1002772-Dahan1"></a>Dahan O, Gingold H, Pilpel Y (2011) Regulatory mechanisms and networks couple the different phases of gene expression. Trends Genet 27: 316–322.
        doi:
        <a href="http://dx.doi.org/10.1016/j.tig.2011.05.008">10.1016/j.tig.2011.05.008</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=6077283">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-deLichtenberg1" id="pcbi.1002772-deLichtenberg1"></a>de Lichtenberg U, Jensen L, Fausboll A, Jensen T, Bork P, et al. (2004) Comparison of computa-tional methods for the identification of cell cycle-regulated genes. Bioinformatics 21: 1164–1171. <a class="find" href="/article/findArticle.action?citedArticleID=6077285">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002772-BarJoseph1" id="pcbi.1002772-BarJoseph1"></a>Bar-Joseph Z, Gerber G, Gifford D, Jaakkola T, Simon I (2003) Continuous representations of time-series gene expression data. J Comput Biol 10: 341–356. <a class="find" href="/article/findArticle.action?citedArticleID=6077287">
                      Find this article online
                    </a></li></ol></div>

</div></div>
<div style="display:none">
<div dojoType="ambra.widget.RegionalDialog" id="AnnotationDialog" style="padding:0;margin:0;">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Post Your Note (For Public Viewing)</span></h5>
      <div class="posting pane">
        <form name="createAnnotation" id="createAnnotation" method="post" action="">
          <input type="hidden" name="target" value="info:doi/10.1371/journal.pcbi.1002772" />
          <input type="hidden" name="startPath" value="" />
          <input type="hidden" name="startOffset" value="" />
          <input type="hidden" name="endPath" value="" />
          <input type="hidden" name="endOffset" value="" />
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="false" />
          <input type="hidden" name="noteType" id="noteType" value="" />
          <fieldset>
            <legend>Compose Your Note</legend>
            <span id="submitMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td>
                  <label for="cNoteType">This is a </label><select name="cNoteType" id="cNoteType"><option value="note">note</option><option value="correction">correction</option></select>
                  <span id="cdls" style="visibility:hidden;margin-left:0.3em; white-space:nowrap;"><a href="/static/commentGuidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1?target=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002772#corrections">What are corrections?</a></span>
                  <label for="cTitle"><span class="none">Enter your note title</span><!-- error message text <em>A title is required for all public notes</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your note title..." class="title" alt="Enter your note title..." />
                  <label for="cArea"><span class="none">Enter your note</span><!-- error message text <em>Please enter your note</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your note..." alt="Enter your note...">Enter your note...</textarea>
                  <input type="hidden" name="isPublic" value="true" />
                </td>
                <td>&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" checked="checked" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." alt="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td colspan="3" class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel"/>
                  <input type="button" value="Submit" title="Click to post your note publicly" id="btn_post" class="primary"/>
                </td>
              </tr>
              <tr>
                <td colspan="3">
                  <p>Notes and Corrections can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p>
                </td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
    <div class="tip" id="dTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialog" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="caTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      Please follow our <a href="/static/commentGuidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">guidelines for notes and comments</a> and review our <a href="/static/competing.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
      <ul>
        <li>Remarks that could be interpreted as allegations of misconduct</li>
        <li>Unsupported assertions or statements</li>
        <li>Inflammatory or insulting language</li>
      </ul>
      <form name="contextActionForm" id="contextActionForm" class="clearfix buttons" method="post" action="">
        <input type="button" name="Continue" value="Continue" id="ContextActionDialogContinueButton" onmouseup="ambra.displayAnnotationContext.startComment(event);" title="Add a note to this text" class="primary"/>
        <input type="button" name="Cancel" value="Cancel" id="ContextActionDialogCancelButton" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);" title="Close this Window"/>
      </form>
    </div>
    <div class="tip" id="caTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogNotLogged" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canlTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You must be logged in to add a note to an article.
      You may log in by <a onmousedown="ambra.displayAnnotationContext.disconnect(event);" href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002772">clicking here</a> or <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">cancel this note</a>.
    </div>
    <div class="tip" id="canlTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canBDTipu"></div>
    <div class="contextActionContent">
      <h5 class="annotation icon"><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot annotate this area of the document. <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canBDTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadRangeSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canbrTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot create an annotation that spans different sections of the document; please adjust your selection.<br/>
      <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canbrTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialog" style="padding:0;margin:0;">
  <div class="dialog preview">
    <div class="tipu" id="cTipu"></div>
    <div class="btn close" id="btn_close" title="Click to close"><a title="Click to close">Close</a></div>
    <div id="cmtContainer" class="comment">
      <h6 id="viewCmtTitle"></h6>
      <div class="detail" id="viewCmtDetail"></div>
      <div class="contentwrap" id="viewComment"></div>
      <div class="contentwrap" id="viewCIStatement"></div>
      <div class="detail" id="viewLink">
        <!--<a href="#" class="commentary icon" title="Click to view full thread and respond">View all responses</a>
        <a href="#" class="respond tooltip" title="Click to respond to this posting">Respond to this</a>-->
      </div>
    </div>
    <div class="tip" id="cTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialogMultiple" style="padding:0;margin:0;">
  <div class="dialog multiple preview">
    <div class="tipu" id="mTipu"></div>
    <div class="btn close" id="btn_close_multi" title="Click to close"><a title="Click to close">Close</a></div>
    <ol id="multilist"></ol>
    <br/>
    <div id="multidetail"></div>
    <div class="tip" id="mTip"></div>
  </div>
</div>


<div dojoType="dijit.Dialog" id="Rating">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Rate This Article</span></h5>
      <div class="instructions">Please follow our <a  href="/static/ratingGuidelines.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">guidelines for rating</a> and review our <a href="/static/competing.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
        <ol>
          <li>Remarks that could be interpreted as allegations of misconduct</li>
          <li>Unsupported assertions or statements</li>
          <li>Inflammatory or insulting language</li>
        </ol>
      </div>
      <div class="posting pane">
        <form name="ratingForm" id="ratingForm" method="post" action="">
          <input type="hidden" name="articleURI" value="info:doi/10.1371/journal.pcbi.1002772" />  
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="" />
          <fieldset>
            <legend>Compose Your Annotation</legend>
            <span id="submitRatingMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td rowspan="2">
                  <label for="insight">Insight</label>
                  <ul class="star-rating rating edit" title="Rate insight" id="rateInsight">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Bland" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'insight', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Profound" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="insight" title="insight" value="" />
                  <label for="reliability">Reliability</label>
                  <ul class="star-rating rating edit" title="Rate reliability" id="rateReliability">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Tenuous" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'reliability', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Unassailable" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="reliability" title="reliability" value="" />
                  <label for="style">Style</label>
                  <ul class="star-rating rating edit" title="Rate style" id="rateStyle">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Crude" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'style', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Elegant" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="style" title="style" value="" />
                  <label for="cTitle"><span class="none">Enter your comment title</span><!-- error message text <em>A title is required for all public annotations</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your comment title..." class="title" alt="Enter your comment title..." />
                  <label for="cArea"><span class="none">Enter your comment</span><!-- error message text <em>Please enter your annotation</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your comment..." alt="Enter your comment...">Enter your comment...</textarea>
                </td>
                <td rowspan="2">&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." title="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel_rating"/>
                  <input type="button" value="Submit" title="Click to post your annotation publicly" id="btn_post_rating" class="primary"/>
                </td>
              </tr>
              <tr>
                  <td colspan="3"><p>Ratings can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p></td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
<div dojoType="ambra.widget.LoadingCycle" id="LoadingCycle" class="loadingCycler">
  <img src="/images/loading.gif" width="58" height="58" title="Loading..." />
</div>
<div dojoType="dijit.Dialog" id="LightBox">

    <!-- start main window wrapper-->
    <div class="figure-window-wrapper">

      <!--start film strip Wrapper-->
      <div class="figure-window-nav-wrapper">
        <div id="figure-window-nav"></div>
      </div><!--end figure-window-nav-wrapper-->

      <!-- start figure container-->
      <div class="figure-window-container">

        <!-- start/end figure viewer-->
        <div id="figure-window-viewer">
          <img id="figure-window-img" class="large"/>
        </div>

        <!-- start/end figure doi-->
        <div id="figure-window-doi"></div>

        <!--start figure title-box-->
        <div id="figure-window-title-box" class="figure-window-title-less">
          <span id="figure-window-title"></span>
          <div id="figure-window-more" class="figure-window-more" onclick="return ambra.lightBox.showMoreOrLess();"></div>
          <div id="figure-window-description"></div>
        </div><!--end figure title-box-->

        <!--start figure download-->
        <div class="figure-window-download">
          <ul>
            <li class="download icon"><strong>Download:</strong>
              <a id="figure-window-ppt" title="Click to download PowerPoint slide of this image">PowerPoint
                slide</a> |
              <a id="figure-window-large" title="Click to download a larger version of this image">larger image
                (<span id="figure-window-large-size"></span>)</a> |
              <a id="figure-window-tiff" title="Click to download a original version of this image">original image
                (<span id="figure-window-tiff-size"></span>)</a>
            </li>
          </ul>
        </div><!--end figure download-->

        <!--start/end of close, context, next and previous button-->
        <div class="figure-window-close" onclick="return ambra.lightBox.hide();"></div>
        <div class="figure-window-context" onclick="return ambra.lightBox.showInContext();"></div>
        <div id="figure-window-previous" onclick="return ambra.lightBox.showPrevious();"></div>
        <div id="figure-window-next" onclick="return ambra.lightBox.showNext();"></div>

      </div><!--end figure-window-container-->

    </div><!--end figure-window-wrapper-->

</div><!--end dijit.Dialog-->
</div>
</div>
<!-- end : main contents -->

</div>
<!-- end : container -->	

<!-- begin : footer -->
<div id="ftr">
  <p><span>All site content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License 2.5" tabindex="200">Creative Commons Attribution License</a>.</span></p>
  <ul>
    <li><a href="/static/privacy.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" title="PLOS Privacy Statement" tabindex="501">Privacy Statement</a></li>
    <li><a href="/static/terms.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" title="PLOS Terms of Use" tabindex="502">Terms of Use</a></li>
    <li><a href="http://www.plos.org/advertise/" title="Advertise With PLOS" tabindex="503">Advertise</a></li>
    <li><a href="http://www.plos.org/about/media-inquiries/" title="PLOS Embargo Policy" tabindex="504">Media Inquiries</a></li>
    <li><a href="http://www.plos.org/publish/order-reprints/" title="PLOS in Print" tabindex="505">PLOS in Print</a></li>
    <li><a href="/static/sitemap.action;jsessionid=AB0054C73D7EEC1CB5DDE465F5D1BAD1" title="Site Map" tabindex="506">Site Map</a></li>
    <li><a href="http://www.plos.org" title="PLOS.org" tabindex="507">PLOS.org</a></li>
  </ul>
  <div class="powered">
  <ul>
    <li><a href="/static/releaseNotes.action" title="Ambra | Release Notes">Ambra 2.4.9</a></li>
    <li>Managed Colocation provided by <a href="http://www.isc.org/" title="Internet Systems Consortium">Internet Systems Consortium</a>.</li>
  </ul>
  </div>
</div>
<!-- end : footer -->
<script type="text/javascript">
  var _namespace="";
    var loggedIn = false;

  var almHost = "http://alm.plos.org";
  var solrHost = "http://api.plos.org/search";
  var pubGetHost = "http://pubget.com/widgetizer/links_hash";
  var appContext = "";

  // Safari v3.1.1 "console.debug" issue (http://trac.dojotoolkit.org/ticket/6849) workaround
  if (/3[\.0-9]+ Safari/.test(navigator.appVersion)) {
    window.console = {
      origConsole: window.console,
      log: function(s){ this.origConsole.log(s); },
      info: function(s){ this.origConsole.info(s); },
      error: function(s){ this.origConsole.error(s); },
      warn: function(s){ this.origConsole.warn(s); }
    };
  }

  var djConfig = {
    // don't debug for IE - as dojo's firebug lite module is error prone in IE
    isDebug: false,
    parseOnLoad: true
  };

  var bamGAID = 'UA-338393-1';
  var bamGAVPV = location.pathname + location.search + location.hash;
</script>
<script type="text/javascript" src="/javascript/dojo/dojo/dojo.js?v=pzlwVCPPo+oJC12upPgJrJ5EG_k"></script>
<script type="text/javascript" src="/javascript/dojo/dojo/ambra.js?v=Y8JM0CVD9YwjC99M_jvbcRTiGhY"></script>
<script type="text/javascript" src="/javascript/init_global.js?v=fF03pfG3r+Nkj_czZ8ORg8o5sNM"></script>
<script type="text/javascript" src="/javascript/init_article.js?v=qzBD_a7nKbeMM2xNUtB3vckN+Zc"></script>
<script type="text/javascript" src="/javascript/init_ratings.js?v=seKykuS0X33CQGS7Lj+8FtBy5Uw"></script>
<script type="text/javascript" src="/javascript/init_lightBox.js?v=vJRLPy7RRX8Zp31Ogcm3i+TA4f8"></script>
<script type="text/javascript" src="/javascript/init_article_body.js?v=YgFei2xCi50CahwaPbSsxUxUsmw"></script>
<script type="text/javascript" src="/javascript/init_article_rhc.js?v=djtbTMOagorKE6ZPrkCwufAfWWI"></script>
<script type="text/javascript" src="/javascript/alm.js?v=1tH9JZ8uDtXspXssGJDr8O+jBOs"></script>
<script type="text/javascript" src="/javascript/reporting/articleViewsCumulative.js?v=RQ+gCrcAl0dBeLbmokxgQZTGYYA"></script>
<script type="text/javascript" src="/javascript/check_twitter.js?v=Hc0sH1W9xIu0qhqxWPD4c1u50K4"></script>
<script type="text/javascript" src="/javascript/init_googleAnalytics.js?v=yCD4h9PHGI0dsefmuWDlMn7pHog"></script>

</body>
</html>

