

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article">

<head>
   <title>PLOS Computational Biology: Confidence-based Somatic Mutation Evaluation and Prioritization</title>



<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9" />

<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="home" title="home" href="/home.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" />
<link rel="alternate" type="application/rss+xml"
  title="PLOS Computational Biology: New Articles"
  href="http://www.ploscompbiol.org/article/feed" />

<link rel="pingback" href="http://www.ploscompbiol.org/pingback" />


<!-- global_css.ftl -->
<link rel="stylesheet" type="text/css" media="screen" href="/css/dojo/dijit/themes/tundra/tundra.css?v=zwugcxLdpBWmb72qqlHgGyeNAGM" />
<link rel="stylesheet" type="text/css" media="all" href="/css/star_rating.css?v=Mm3HyaUPUahAJAq6xG5W3tdF7Hk" />
<link rel="stylesheet" type="text/css" media="all" href="/css/rating.css?v=AOT9rozE9sYVjGTdeiyxHf18Ykw" />
<link rel="stylesheet" type="text/css" media="all" href="/css/screen.css?v=KBNFqBmhD635U0NgiM5tf9dtxRE" />
<link rel="stylesheet" type="text/css" media="all" href="/css/iepc.css?v=FIjyR83BhSovq_5yNVk1Sih60AI" />
<link rel="stylesheet" type="text/css" media="all" href="/css/journal.css?v=SiREXztjNSUsOyrlkE9sv028Y+0" />
<link rel="stylesheet" type="text/css" media="all" href="/css/lightBox.css?v=sDgsUXamFoYAakEshdj0YWrZiiQ" />
<link rel="stylesheet" type="text/css" media="print" href="/css/print.css?v=0T+LTV7Gd0jtHEhqEvBfSrmHUGg" />

<!--chartbeat --> 
<script type="text/javascript">var _sf_startpt=(new Date()).getTime()</script>


<!--[if IE]><![endif]-->
<meta name="description" content="PLOS Computational Biology is an open-access" />

<meta name="keywords" content="plos computational biology" />

  <meta name="citation_publisher" content="Public Library of Science" />
  <meta name="citation_doi" content="10.1371/journal.pcbi.1002714" />
    <meta name="citation_title" content="Confidence-based Somatic Mutation Evaluation and Prioritization"/>
    <meta itemprop="name" content="Confidence-based Somatic Mutation Evaluation and Prioritization"/>

      <meta name="citation_author" content="Martin Löwer" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
      <meta name="citation_author" content="Bernhard Y. Renard" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
          <meta name="citation_author_institution" content="Research Group Bioinformatics (NG 4), Robert Koch-Institut, Berlin, Germany" />
      <meta name="citation_author" content="Jos de Graaf" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
      <meta name="citation_author" content="Meike Wagner" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
          <meta name="citation_author_institution" content="Department of Internal Medicine III, Division of Translational and Experimental Oncology, University Medical Center, Johannes Gutenberg University, Mainz, Germany" />
      <meta name="citation_author" content="Claudia Paret" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
      <meta name="citation_author" content="Christoph Kneip" />
          <meta name="citation_author_institution" content="Theracode GmbH, Mainz, Germany" />
      <meta name="citation_author" content="Özlem Türeci" />
          <meta name="citation_author_institution" content="Ganymed Pharmaceuticals AG, Mainz, Germany" />
      <meta name="citation_author" content="Mustafa Diken" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
      <meta name="citation_author" content="Cedrik Britten" />
          <meta name="citation_author_institution" content="Ribological GmbH, Mainz, Germany" />
      <meta name="citation_author" content="Sebastian Kreiter" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
      <meta name="citation_author" content="Michael Koslowski" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
      <meta name="citation_author" content="John C. Castle" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />
      <meta name="citation_author" content="Ugur Sahin" />
          <meta name="citation_author_institution" content="TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany" />

    <meta name="citation_date" content="2012/9/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pcbi.1002714.pdf" />

    <meta name="citation_journal_title" content="PLOS Computational Biology" />
    <meta name="citation_firstpage" content="e1002714"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="8"/>
    <meta name="citation_issn" content="1553-7358"/>

    <meta name="citation_journal_abbrev" content="PLoS Comput Biol" />

      <meta name="citation_reference" content="citation_title=Technology-specific error signatures in the 1000 Genomes Project data; citation_volume=130; citation_number=1; citation_pages=505-516; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Frequent pathway mutations of splicing machinery in myelodysplasia; citation_volume=478; citation_number=2; citation_pages=64-69; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Biases in Illumina transcriptome sequencing caused by random hexamer priming; citation_volume=38; citation_number=3; citation_pages=e131; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Overcoming bias and systematic errors in next generation sequencing data; citation_volume=2; citation_number=4; citation_pages=87; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Whole exome capture in solution with 3 Gbp of data; citation_volume=11; citation_number=5; citation_pages=R62; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Sequence-specific error profile of Illumina sequencers; citation_volume=39; citation_number=6; citation_pages=e90; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Improved base calling for the Illumina Genome Analyzer using machine learning strategies; citation_volume=10; citation_number=7; citation_pages=R83; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=SAMStat: monitoring biases in next generation sequencing data; citation_volume=27; citation_number=8; citation_pages=130-131; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Improving SNP discovery by base alignment quality; citation_volume=27; citation_number=9; citation_pages=1157-1158; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Patterns of nucleotide substitution, insertion and deletion in the human genome inferred from pseudogenes,; citation_volume=31; citation_number=10; citation_pages=5338-5348; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data; citation_volume=43; citation_number=11; citation_pages=491-498; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Identification and analysis of error types in high-throughput genotyping; citation_volume=67; citation_number=12; citation_pages=727-736; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Quantification of rare allelic variants from pooled genomic DNA; citation_volume=6; citation_number=13; citation_pages=263-265; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=A statistical method for the detection of variants from next-generation resequencing of DNA pools; citation_volume=26; citation_number=14; citation_pages=318-324; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries; citation_volume=5; citation_number=15; citation_pages=247-252; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_volume=25; citation_number=16; citation_pages=1754-1760; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_volume=10; citation_number=17; citation_pages=R25; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The Sequence alignment/map (SAM) format and SAMtools; citation_volume=25; citation_number=18; citation_pages=2078-2079; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=SomaticSniper: identification of somatic point mutations in whole genome sequencing data; citation_volume=28; citation_number=19; citation_pages=311-317; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Statistical Modeling: The Two Cultures; citation_volume=16; citation_number=20; citation_pages=199-231; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An introduction to ROC analysis; citation_volume=27; citation_number=21; citation_pages=861-874; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Accurate and comprehensive sequencing of personal genomes; citation_volume=21; citation_number=22; citation_pages=1498-1505; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing; citation_volume=27; citation_number=23; citation_pages=182-189; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and comparative analysis of the mouse genome; citation_volume=420; citation_number=24; citation_pages=520-562; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=dbSNP: the NCBI database of genetic variation; citation_volume=29; citation_number=25; citation_pages=308-311; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The UCSC Genome Browser database: extensions and updates 2011; citation_volume=39; citation_number=26; citation_pages=876-882; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; citation_volume=33; citation_number=27; citation_pages=501-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mapping and quantifying mammalian transcriptomes by rna-seq; citation_volume=5; citation_number=28; citation_pages=621-628; citation_date=2008; " />
      <meta name="citation_reference" content="citation_number=29; " />
      <meta name="citation_reference" content="citation_title=Classification and Regression by randomForest; citation_volume=2; citation_number=30; citation_pages=18-22; citation_date=2002; " />



<!--
<rdf:RDF xmlns="http://web.resource.org/cc/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<Work rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714">
   <license rdf:resource="http://creativecommons.org/licenses/by/2.5/" />
</Work>
<License rdf:about="http://creativecommons.org/licenses/by/2.5/">
   <permits rdf:resource="http://web.resource.org/cc/Reproduction" />
   <permits rdf:resource="http://web.resource.org/cc/Distribution" />
   <requires rdf:resource="http://web.resource.org/cc/Notice" />
   <requires rdf:resource="http://web.resource.org/cc/Attribution" />
   <permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" />
</License>
<rdf:Description
     rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714"
     dc:identifier="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714"
     dc:title="PLOS Computational Biology: Confidence-based Somatic Mutation Evaluation and Prioritization"
       trackback:ping="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714/trackback;jsessionid=10B3F584BF488C3BC70268A3ECAE261F"
     />
</rdf:RDF>

-->
</head>
<body>
<span property="dc:language" content="en"></span>
<!-- begin : container -->
<div id="container">
  <!-- begin : top banner external ad space -->
  <div id="topBanner">


<!-- begin : left banner slot -->
<div class="left">
  <iframe id='affe494b' name='affe494b'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=84&amp;cb=7107'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=affe494b&amp;cb=8948'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=84&amp;cb=1760&amp;n=affe494b'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : left banner slot -->
<!-- begin : right banner slot -->
<div class="right">
  <iframe id='af654bd5' name='af654bd5'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=85&amp;cb=2592'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=af654bd5&amp;cb=2666'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=85&amp;cb=3132&amp;n=af654bd5'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : right banner slot -->

  </div>
  <!-- end : top banner external ad space -->

  <!-- begin : header -->
  <div id="hdr" class="login">

  <!-- begin : logo -->
  <div id="logo" title="PLOS Computational Biology"><a href="/home.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" title="PLOS Computational Biology"><span>PLOS Computational Biology</span></a></div>
  <!-- end : logo -->
  <!-- begin : user controls -->





  <div id="user">
    <div>
      <ul>
        <li><a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002714" class="feedback"><strong>Login</strong></a> | </li>
        <li><a href="https://register.plos.org:443/ambra-registration/register.action">Create Account</a> | </li>
        <li class="feedback"><a href="/feedbackCreate.action?page=%252Farticle%252FfetchArticle.action%253FarticleURI%253Dinfo%25253Adoi%25252F10.1371%25252Fjournal.pcbi.1002714" title="Send us your feedback">Feedback</a></li>
      </ul>
    </div>
  </div>


  <!-- end : user controls -->
  <!-- begin search links -->
  <ul id="links"><li class="browse"><a href="/article/browse.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?field=date" title="Browse Articles">Browse</a></li><li class="rss"><a href="/static/rssFeeds.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" title="RSS Feeds">RSS</a></li></ul>
  <!-- end : search links -->
  <!-- begin : dashboard -->
  <div id="db">
    <form name="searchForm" action="/search/simpleSearch.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" method="get">
      <input type="hidden" name="from" value="globalSimpleSearch"/>
      <input type="hidden" name="filterJournals" value="PLoSCompBiol">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="wrap"><input id="search" type="text" name="query" value="Search articles..." onfocus="if(this.value=='Search articles...')value='';" onblur="if(this.value=='')value='Search articles...';" class="searchField" alt="Search articles..."/></div>
        <input src="/images/search_btn1.gif" onclick="submit();" value="ftsearch" alt="SEARCH" tabindex="3" class="button" type="image" />
      </fieldset>
    </form>
    <form name="gasf" action="/search/advancedSearch.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" method="get">
<input type="hidden" name="noSearchFlag" value="true" id="noSearchFlag"/><input type="hidden" name="query" value="" id="query"/>    </form>
    <a id="advSearch" href="#" onclick="if(document.searchForm.query.value!='Search articles...')document.gasf.query.value=document.searchForm.query.value;document.gasf.submit();return false;">Advanced Search</a>
  </div>
  <!-- end : dashboard -->
  <!-- begin : navigation -->
  <ul id="nav">
    <li><a href="/home.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" tabindex="101">Home</a></li>
     <!-- This is a temporary action to link to th static toc page -->
    <li><a href="/static/browse.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" tabindex="102">Browse Articles</a>
        <ul>
          <li><a href="/article/browseIssue.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?field=issue">Current Issue</a></li> <!-- Assuming dynamic TOC is in place -->
          <li><a href="/article/browseVolume.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?field=volume">Journal Archive</a></li> 
          <li><a href="/article/browse.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?field=date">By Publication Date</a></li>
          <li><a href="/article/browse.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">By Subject</a></li>
          <li><a href="http://www.ploscollections.org/static/pcbiCollections.action">Collections</a></li>
        </ul>
    </li>
    <li><a href="/static/about.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" tabindex="103">About</a>
        <ul>
          <li><a href="/static/information.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Journal Information</a></li>
          <li><a href="/static/edboard.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Editorial Board</a></li>
          <li><a href="/static/eic.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Editors-in-Chief</a></li>
          <li><a href="/static/almInfo.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Article-Level Metrics</a></li>
          <li><a href="/static/license.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Open-Access License</a></li>
          <li><a href="/static/contact.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Contact Us</a></li>
        </ul>
      </li>
    <li><a href="/static/users.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" tabindex="104">For Readers</a>
        <ul>
          <li><a href="/static/commentGuidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Guidelines for Notes, Comments, and Corrections</a></li>
          <li><a href="/static/ratingGuidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Guidelines for Rating</a></li>
          <li><a href="/static/help.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Help Using This Site</a></li>
          <li><a href="/static/downloads.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Media Downloads</a></li>
          <li><a href="/static/sitemap.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Site Map</a></li>
        </ul>
      </li>
    <li><a href="/static/authors.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" tabindex="105">For Authors and Reviewers</a>
        <ul>
          <li><a href="/static/policies.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Editorial and Publishing Policies</a></li>
          <li><a href="/static/guidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Author Guidelines</a></li>
          <li><a href="/static/figureGuidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Figure and Table Guidelines</a></li>
          <li><a href="/static/checklist.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Submit Your Paper</a></li>
          <li><a href="/static/reviewerGuidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">Reviewer Guidelines</a></li>
	   <li><a href="http://www.plos.org/journals/index.php">Submit to Other PLOS Journals</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://www.plos.org" title="PLOS" tabindex="110" class="drop">PLOS.org</a>
        <ul>
          <li><a href="http://www.plos.org/oa/index.html" title="Open Access Statement">Open Access</a></li>
          <li><a href="http://www.plos.org/support/donate.php" title="Join PLOS: Show Your Support">Join PLOS</a></li>
          <li><a href="http://blogs.plos.org/" title="PLOS Blogs">PLOS Blogs</a></li>
	  <li><a href="http://www.zazzle.com/plos?rf=238434112415374154" title="PLOS Store">PLOS Store</a></li>
          <li><a href="http://currents.plos.org" title="PLOS Currents">PLOS Currents</a></li>
          <li><a href="http://www.plos.org/connect.html" title="PLOS.org | Stay Connected">Stay Connected</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://hubs.plos.org" tabindex="109">Hubs</a>
        <ul>
          <li><a href="http://hubs.plos.org/biodiversity" title="PLOS Hubs: Biodiversity">Biodiversity</a></li>
          <li><a href="http://clinicaltrials.ploshubs.org" title="PLOS Hub for Clinical Trials">Clinical Trials</a></li>
        </ul>
    </li>
      <li class="journalnav"><a href="http://www.plosjournals.org" tabindex="108">Journals</a>
        <ul>
          <li><a href="http://www.plosbiology.org/" title="PLOSBiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org/" title="PLOSMedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org/" title="PLOSCompBiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://www.plosgenetics.org/" title="PLOSGenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org/" title="PLOSPathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org/" title="PLOSONE.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org/" title="PLOSNTDs.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </li>
    </ul>
  <!-- end : navigation -->
  </div>
  <!-- end : header --> 



<!-- begin : main content -->
<div id="content" class="article" style="visibility:visible;">

<!-- begin : right hand column -->

<div id="rhc" xpathLocation="noDialog">

  <div id="download" class="rhcBox_type1">
    <div class="wrap">
      <ul>
        <li class="download icon"><strong>Download:</strong>
          <a href="/article/fetchObjectAttachment.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714&amp;representation=PDF" title="Download article PDF">PDF</a> |
          <a href="/article/citationList.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714" title="Download citations">Citation</a> |
          <a href="/article/fetchObjectAttachment.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714&amp;representation=XML" title="Download Article XML">XML</a>
        </li>
        <li class="print icon"><a href="#" onclick="window.print();return false;" title="Print Article"><strong>Print article</strong></a></li>
        <li class="reprint icon"><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=3&doi=10.1371/journal.pcbi.1002714&volume=&issue=&title=Confidence-based Somatic Mutation Evaluation and Prioritization&author_name=Martin%20L%C3%B6wer%2C%20Bernhard%20Y.%20Renard%2C%20Jos%20de%20Graaf%2C%20Meike%20Wagner%2C%20Claudia%20Paret%2C%20Christoph%20Kneip%2C%20%C3%96zlem%20T%C3%BCreci%2C%20Mustafa%20Diken%2C%20Cedrik%20Britten%2C%20Sebastian%20Kreiter%2C%20Michael%20Koslowski%2C%20John%20C.%20Castle%2C%20Ugur%20Sahin&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a> New &amp; improved!</li>
      </ul>
    </div>
  </div>

  <div id="issues" class="rhcBox_type2">
    <p><strong>Published in the</strong>
    <a href="http://www.ploscompbiol.org/article/browseIssue.action?issue=info%3Adoi%2F10.1371%2Fissue.pcbi.v08.i09" title="Browse the Open-Access Issue">September 2012 Issue of <em>PLOS Computational Biology</em></a></p>
  </div>

  <div id="impact" class="rhcBox_type2">
    <h6>Metrics <a href="/static/help.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F#articleMetrics" class="replaced" id="info" title="More information"><span>info</span></a></h6>

    <div class="wrap">
      <img id="rhcDataSpinner" src="/images/loading_small.gif" class="loading" />
      <div id="totalDataInRHC"></div>
      <div id="pubDateInRHC"></div>
      <div id="relatedCitesInRHC"></div>
    </div>

    <div class="more clearfix"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" title="More information">More</a></div>
  </div>

  <div id="related" class="rhcBox_type2">
    <h6>Related Content</h6>
    <dl class="related">
      <dt>Related Articles on the Web</dt>
      <dd><a href="http://scholar.google.com/scholar?hl=en&lr=&q=related:http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002714&btnG=Search">Google Scholar</a></dd>
      <dd id="pubMedRelatedLIInRHC" style="display:none;"><a id="pubMedRelatedURLInRHC">PubMed</a></dd>
    </dl>
    <div class="more clearfix"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" title="More information">More</a></div>
  </div>

  <div id="share" class="rhcBox_type2">
    <h6>Share this Article <a href="/static/help.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F#socialBookmarkLinks" class="replaced" id="info" title="More information"><span>info</span></a></h6>
    <ul>
      <li class="bookmarklets">

        <a href="http://www.reddit.com/submit?url=http://dx.plos.org/10.1371/journal.pcbi.1002714"
           target="_new"
           id="reddit">
          <img src="http://www.reddit.com/static/spreddit4.gif" alt="submit to reddit" border="0"/> </a>
        <g:plusone size="small" annotation="none" href=http://dx.plos.org/10.1371/journal.pcbi.1002714></g:plusone>
        <script type="text/javascript">
          (function () {
            var po = document.createElement('script');
            po.type = 'text/javascript';
            po.async = true;
            po.src = 'https://apis.google.com/js/plusone.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(po, s);
          })();
        </script>
        <a href="http://www.stumbleupon.com/submit?url=http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714" target="_new"> <img border=0 src="http://cdn.stumble-upon.com/images/16x16_su_solid.gif" alt="StumbleUpon" title="Add to StumbleUpon"></a>
        <script>function fbs_click() {u='http://dx.plos.org/10.1371/journal.pcbi.1002714';t='Confidence-based%20Somatic%20Mutation%20Evaluation%20and%20Prioritization';window.open('http://www.facebook.com/sharer.php?u='+encodeURIComponent(u)+'&t='+encodeURIComponent(t),'sharer','toolbar=0,status=0,width=626,height=436');return false;}</script><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002714" onclick="return fbs_click()" target="_blank"><img src="http://static.ak.fbcdn.net/images/share/facebook_share_icon.gif" alt="Facebook" title="Add to Facebook" /></a>       <!-- for mor info, see http://www.facebook.com/share_partners.php -->
        <script type="text/javascript">
          function bookmark_in_connotea(u) {
            a=false; x=window; e=x.encodeURIComponent; d=document;
            w=open('http://www.connotea.org/addpopup?continue=confirm&uri='+e(u),
                'add', 'width=600, height=400, scrollbars, resizable');
            void(x.setTimeout('w.focus()',200));
          }
        </script>
        <a style='cursor: pointer;' onclick='javascript:bookmark_in_connotea("http://dx.plos.org/10.1371/journal.pcbi.1002714");'><img src='/images/icon_connotea_16x16.gif' alt="Connotea" title="Add to Connotea"/></a>
        <a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002714&title=Confidence-based%20Somatic%20Mutation%20Evaluation%20and%20Prioritization" target="_new"><img src='/images/icon_citeulike_16x16.gif' alt="CiteULike" title="Add to CiteULike" /></a>
        <a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002714" title="Add to Mendeley" target="_new"><img src="/images/icon_mendeley_16x16.gif" alt="Bibliography"></a>
        <script type="text/javascript">
          var twtTitle  = 'Confidence-based Somatic Mutation Evaluation and Prioritization';
          var twtUrl    = 'http://dx.plos.org/10.1371/journal.pcbi.1002714';
          var twtTag    = '#PLOSCompBio:'
          var maxLength = 140 - (twtUrl.length + 1);
          if (twtTitle.length > maxLength) {
            twtTitle = twtTitle.substr(0, (maxLength - 17))+'...';  //truncates title to include space for twtTag and ellipsis
          }
          var twtLink = 'http://twitter.com/intent/tweet?text='+encodeURIComponent(twtTag + ' ' + twtTitle + ' ' + twtUrl);
          document.write('<a href="'+twtLink+'" target="_blank"><img src="/images/icon_twitter.gif" border="0" alt="Twitter icon" title="Tweet This!" /></a>');
        </script>

      </li>
      <li class="email icon"><a href="/article/emailArticle.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714" title="Email this article">Email this article</a></li>
    </ul>
  </div>
  <div id="articleTwitterStream" style="display:none">
    <script src="http://widgets.twimg.com/j/2/widget.js"></script>
    <script>
      new TWTR.Widget({
        version: 2,
        type: 'search',
        search: encodeURIComponent('10.1371/journal.pcbi.1002714'),
        interval: 6000,
        title: '',
        subject: 'From Twitter',
        width: 300,
        height: 300,
        theme: {
          shell: {
            background: '#667893',
            color: '#ffffff'
          },
          tweets: {
            background: '#ffffff',
            color: '#444444',
            links: '#006699'
          }
        },
        features: {
          scrollbar: true,
          loop: false,
          live: true,
          hashtags: true,
          timestamp: true,
          avatars: true,
          toptweets: true,
          behavior: 'all'
        }
      }).render().start();
    </script>
  </div>
</div>
<!-- end : right hand column -->
<form name="articleInfo" id="articleInfo" method="" action="">
<input type="hidden" name="isAuthor" value="true" />
<input type="hidden" name="authorIdList" value="" />
<input type="hidden" name="userIdList" value="" />
<input type="hidden" name="otherIdList" value="" />
<input type="hidden" name="annotationId" value="" />
<input type="hidden" name="isResearchArticle" value="true" />
</form>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.XML" value="93107"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.PDF" value="1686480"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g001.TIF" value="883626"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g001.PNG_I" value="47562"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g001.PNG_S" value="13813"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g001.PNG_M" value="107230"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g001.PNG_L" value="446430"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g002.TIF" value="488326"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g002.PNG_I" value="44544"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g002.PNG_S" value="16049"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g002.PNG_M" value="89283"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g002.PNG_L" value="111093"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g003.TIF" value="546138"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g003.PNG_I" value="35240"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g003.PNG_S" value="11910"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g003.PNG_M" value="66719"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g003.PNG_L" value="248969"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e001.TIF" value="24926"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e001.PNG" value="11624"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.t001.TIF" value="102636"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.t001.PNG_I" value="17747"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.t001.PNG_S" value="3287"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.t001.PNG_M" value="39860"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.t001.PNG_L" value="34051"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g004.TIF" value="1804562"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g004.PNG_I" value="106704"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g004.PNG_S" value="19011"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g004.PNG_M" value="234025"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g004.PNG_L" value="1128853"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g005.TIF" value="264878"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g005.PNG_I" value="17198"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g005.PNG_S" value="7234"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g005.PNG_M" value="32602"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g005.PNG_L" value="95831"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g006.TIF" value="871330"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g006.PNG_I" value="50502"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g006.PNG_S" value="13703"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g006.PNG_M" value="111535"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.g006.PNG_L" value="433484"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e002.TIF" value="25262"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e002.PNG" value="11680"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e003.TIF" value="24926"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e003.PNG" value="11624"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e004.TIF" value="4170"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e004.PNG" value="2128"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e005.TIF" value="2342"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e005.PNG" value="1346"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e006.TIF" value="2224"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e006.PNG" value="1274"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e007.TIF" value="2076"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.e007.PNG" value="1201"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.s001.XLS" value="1954816"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.s002.XLS" value="29184"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.s003.XLS" value="27136"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.s004.XLS" value="45568"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002714.s005.PDF" value="930320"/>
<div id="articleContainer">

<span about="http://www.plos.org"  typeof="foaf:Organization" property="foaf:name" style="display:none; visibility:hidden;">PLOS</span>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader" xpathLocation="noSelect">
  <p>Open Access</p><p id="articleType">Research Article</p>
</div>

  <h1 xpathLocation="noSelect" property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">Confidence-based Somatic Mutation Evaluation and Prioritization</h1>
  <div property="dc:description" datatype="" style="display:none; visibility:hidden;">Author SummaryNext generation sequencing (NGS) has enabled unbiased, high throughput discovery of genetic variations and somatic mutations. However, the NGS platform is still prone to errors resulting in inaccurate mutation calls. A statistical measure of the confidence of putative mutation calls would enable researchers to prioritize and select mutations in a robust manner. Here we present our development of a confidence score for mutations calls and apply the method to the identification of somatic mutations in B16 melanoma. We use NGS exome resequencing to profile triplicates of both the reference C57BL/6 mice and the B16-F10 melanoma cells. These replicate data allow us to formulate the false discovery rate of somatic mutations as a statistical quantity. Using this method, we show that 50 of 50 high confidence mutation calls are correct while 0 of 44 low confidence mutations are correct, demonstrating that the method is able to correctly rank mutation calls.</div>
  <span property="dc:date" content="2012-09-27" datatype="xsd:date" rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pcbi.1002714"></span>
  <span property="dc:subject" content="Computational Biology"></span>
  <span property="dc:subject" content="Genetics and Genomics"></span>
<form action="">
  <input type="hidden" name="journalDisplayName" id="journalDisplayName" value="PLOS Computational Biology" />
  <input type="hidden" name="crossRefPageURL" id="crossRefPageURL" value="/article/crossref/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" />
  <input type="hidden" name="twitterPageURL" id="twitterPageURL" value="/article/twitter/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" />
  <input type="hidden" name="metricsTabURL" id="metricsTabURL" value="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" />
  <input type="hidden" name="doi" id="doi" value="info:doi/10.1371/journal.pcbi.1002714" />
  <input type="hidden" name="articleTitleUnformatted" id="articleTitleUnformatted" value="Confidence-based%20Somatic%20Mutation%20Evaluation%20and%20Prioritization" />
  <input type="hidden" name="articlePubDate" id="articlePubDate" value="1348729200000" />
</form>
<div class="horizontalTabs" xpathLocation="noSelect">
  <ul id="tabsContainer">
        <li id="article" class="active"><a href="/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenuBox" xpathLocation="noSelect">
    <div id="articleMenu">
      <div class="wrap">
        <ul>
          <li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" onclick="toggleAnnotation(this, 'public'); return false;" title="Click to turn notes on/off">Hide notes</a></li>
          <li class="discuss icon">
              <a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002714">Make a general comment</a>
          </li>
        </ul>
        <div id="sectionNavTopBox" style="display:none;">
          <p><strong>Jump to</strong></p>
          <div id="sectionNavTop" class="tools"></div>
        </div>
      </div>
    </div>
      <div id="lightBox" xpathLocation="noDialog" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714')">
      </div>
  </div>




<p class="authors" xpathLocation="noSelect"><span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Martin Löwer</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#equal-contrib">#</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Bernhard Y. Renard</span></span><sup><a href="#aff1">1</a></sup><sup>,</sup><sup><a href="#aff2">2</a></sup><sup><a href="#equal-contrib">#</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Jos de Graaf</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Meike Wagner</span></span><sup><a href="#aff1">1</a></sup><sup>,</sup><sup><a href="#aff3">3</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Claudia Paret</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Christoph Kneip</span></span><sup><a href="#aff4">4</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Özlem Türeci</span></span><sup><a href="#aff5">5</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Mustafa Diken</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Cedrik Britten</span></span><sup><a href="#aff6">6</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Sebastian Kreiter</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Michael Koslowski</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">John C. Castle</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#fn1">¶</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Ugur Sahin</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#fn1">¶</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup></p><p class="affiliations" xpathLocation="noSelect"><a name="aff1" id="aff1"></a><strong>1</strong> TRON - Translational Oncology at the Johannes Gutenberg University of Mainz Medicine, Mainz, Germany, <a name="aff2" id="aff2"></a><strong>2</strong> Research Group Bioinformatics (NG 4), Robert Koch-Institut, Berlin, Germany, <a name="aff3" id="aff3"></a><strong>3</strong> Department of Internal Medicine III, Division of Translational and Experimental Oncology, University Medical Center, Johannes Gutenberg University, Mainz, Germany, <a name="aff4" id="aff4"></a><strong>4</strong> Theracode GmbH, Mainz, Germany, <a name="aff5" id="aff5"></a><strong>5</strong> Ganymed Pharmaceuticals AG, Mainz, Germany, <a name="aff6" id="aff6"></a><strong>6</strong> Ribological GmbH, Mainz, Germany</p><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2 xpathLocation="noSelect">Abstract&nbsp;<a href="#top">Top</a></h2><p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]/p[1]">Next generation sequencing (NGS) has enabled high throughput discovery of somatic mutations. Detection depends on experimental design, lab platforms, parameters and analysis algorithms. However, NGS-based somatic mutation detection is prone to erroneous calls, with reported validation rates near 54% and congruence between algorithms less than 50%. Here, we developed an algorithm to assign a single statistic, a false discovery rate (FDR), to each somatic mutation identified by NGS. This FDR confidence value accurately discriminates true mutations from erroneous calls. Using sequencing data generated from triplicate exome profiling of C57BL/6 mice and B16-F10 melanoma cells, we used the existing algorithms GATK, SAMtools and SomaticSNiPer to identify somatic mutations. For each identified mutation, our algorithm assigned an FDR. We selected 139 mutations for validation, including 50 somatic mutations assigned a low FDR (high confidence) and 44 mutations assigned a high FDR (low confidence). All of the high confidence somatic mutations validated (50 of 50), none of the 44 low confidence somatic mutations validated, and 15 of 45 mutations with an intermediate FDR validated. Furthermore, the assignment of a single FDR to individual mutations enables statistical comparisons of lab and computation methodologies, including ROC curves and AUC metrics. Using the HiSeq 2000, single end 50 nt reads from replicates generate the highest confidence somatic mutation call set.</p>
</div><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]"><a id="abstract1" name="abstract1" toc="abstract1" title="Author Summary"></a>
<h2 xpathLocation="noSelect">Author Summary&nbsp;<a href="#top">Top</a></h2>
<p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]/p[1]">Next generation sequencing (NGS) has enabled unbiased, high throughput discovery of genetic variations and somatic mutations. However, the NGS platform is still prone to errors resulting in inaccurate mutation calls. A statistical measure of the confidence of putative mutation calls would enable researchers to prioritize and select mutations in a robust manner. Here we present our development of a confidence score for mutations calls and apply the method to the identification of somatic mutations in B16 melanoma. We use NGS exome resequencing to profile triplicates of both the reference C57BL/6 mice and the B16-F10 melanoma cells. These replicate data allow us to formulate the false discovery rate of somatic mutations as a statistical quantity. Using this method, we show that 50 of 50 high confidence mutation calls are correct while 0 of 44 low confidence mutations are correct, demonstrating that the method is able to correctly rank mutation calls.</p>
</div>


<div class="articleinfo" xpathLocation="noSelect"><p><strong>Citation: </strong>Löwer M, Renard BY, de Graaf J, Wagner M, Paret C, et al.  (2012) Confidence-based Somatic Mutation Evaluation and Prioritization. PLoS Comput Biol 8(9):
          e1002714.
            doi:10.1371/journal.pcbi.1002714</p><p><strong>Editor: </strong>Scott Markel, 
        Accelrys, United States of America
      </p><p></p><p><strong>Received:</strong> March 16, 2012; <strong>Accepted:</strong> August 18, 2012; <strong>Published:</strong> September 27, 2012</p><p><strong>Copyright:</strong> ©  Löwer et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>TRON is funded by the state of Rhineland-Palatine, the University of Mainz, and the University of Mainz Medical Center. No additional funding was received for this work. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p><p><a name="cor1"></a>* E-mail: <a href="mailto:john.castle@tron-mainz.de">john.castle@tron-mainz.de</a> (JCC); <a href="mailto:sahin@uni-mainz.de">sahin@uni-mainz.de</a> (US)</p><p><a name="equal-contrib"></a>#
              These authors contributed equally to this work.
          </p><p><a name="fn1"></a>¶ These authors are joint senior authors on this work.</p></div>





<div id="section1" xpathLocation="/article[1]/body[1]/sec[1]"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3 xpathLocation="noSelect">Introduction&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[1]/p[1]">Next generation sequencing (NGS) has revolutionized our ability to determine genomes and compare, for example, tumor to normal cells to identify somatic mutations. However, the platform is not error free and various experimental and algorithmic factors contribute to the false positive rate when identifying somatic mutations <a href="#pcbi.1002714-Nothnagel1">[1]</a>. Indeed, recent studies report validation rates of 54% <a href="#pcbi.1002714-Yoshida1">[2]</a>. Error sources include PCR artifacts, biases in priming <a href="#pcbi.1002714-Hansen1">[3]</a>, <a href="#pcbi.1002714-Taub1">[4]</a> and targeted enrichment <a href="#pcbi.1002714-Bainbridge1">[5]</a>, sequence effects <a href="#pcbi.1002714-Nakamura1">[6]</a>, base calling causing sequence errors <a href="#pcbi.1002714-Kircher1">[7]</a>, variations in coverage, and uncertainties in read alignments <a href="#pcbi.1002714-Lassmann1">[8]</a>, such as around insertions and deletions (indels) <a href="#pcbi.1002714-Li1">[9]</a>. Reflecting the rapid development of bench and computational methods, algorithms to identify somatic mutations from NGS data are still evolving rapidly. Remarkably, the congruence of identified mutations between current algorithms is less than 50% (below).</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[2]">Given the large discrepancies, one is left wondering which mutations to select, such as for clinical decision making or ranking for follow-up experiments. Ideal would be a statistical value, such as a p-value, indicating the confidence of each mutation call. Error sources have been addressed by examining bulk sets of mutations, such as computational methods to measure the expected amount of false positive mutation calls utilizing the transition/transversion ratio of a set of variations <a href="#pcbi.1002714-Zhang1">[10]</a>, <a href="#pcbi.1002714-DePristo1">[11]</a>, machine learning <a href="#pcbi.1002714-Ewen1">[12]</a> and inheritance errors when working with family genomes <a href="#pcbi.1002714-Druley1">[13]</a> or pooled samples <a href="#pcbi.1002714-Bansal1">[14]</a>, <a href="#pcbi.1002714-VanTassell1">[15]</a>. Druley <em>et al.</em> <a href="#pcbi.1002714-Druley1">[13]</a> optimized variation calls using short plasmid sequence fragments for optimization. The accuracy of calling germline variations, i.e. single nucleotide polymorphisms (SNPs), has been addressed by validating SNPs using other techniques such as genotyping microarrays <a href="#pcbi.1002714-VanTassell1">[15]</a>. Thus, these methods enable a comparison of methods to identify and characterize error sources, but they do not assign a ranking score to individual mutation.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[3]">Several NGS mutation identification algorithms do output multiple parameters for each mutation call, such as coverage, genotype quality and consensus quality. However, it is not clear if and how to interpret these metrics with regards to whether a mutation call is correct. Furthermore, multiple parameters are generated for each mutation call and thus one simply cannot rank or prioritize mutations using the values. Instead, researchers often rely on personal experience and arbitrary filtering thresholds to select mutations. In summary, a) there is a low level of congruence between somatic mutations identified by different algorithms and sequencing platforms and b) no method to assign a single accuracy estimate to individual mutations.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[4]">Here, we develop a methodology to assign a confidence value - a false discovery rate (FDR) - to individual identified mutations. This algorithm does not identify mutations but rather estimates the accuracy of each mutation. The method is applicable both to the selection and prioritization of mutations and to the development of algorithms and methods. Using Illumina HiSeq reads and the algorithms GATK, SAMtools and SomaticSNiPer, we identified 4,078 somatic mutations in B16 melanoma cells. We assigned a FDR to each mutation and show that 50 of 50 mutations with low FDR (high confidence) validated while 0 of 44 with high FDR (low confidence) validated.</p>
</div>

<div id="section2" xpathLocation="/article[1]/body[1]/sec[2]"><a id="s2" name="s2" toc="s2" title="Results"></a><h3 xpathLocation="noSelect">Results&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[2]/p[1]">Somatic mutation discovery involves the determination and comparison of two genomes: the “normal” germline genome from non-cancerous cells and the tumor genome. If one, however, sequences one sample multiple times, such as the normal genome, and then compares the replicates, one should identify no differences. Thus, any mutation detected in this “same versus same comparison” is a false positive. These can be generated during sample extraction, sample preparation, amplification and library construction, NGS sequencing and data analysis.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/p[2]">To determine the false discovery rate (FDR) for each somatic mutation detected in a tumor sample relative to a normal sample (“tumor versus normal comparison”), we first define and assign a quality score Q to each identified mutation. Then, we count the number of “same versus same” mutations (false positives) at the same or greater quality score (<a href="#pcbi-1002714-g001">Figure 1a</a>) and convert this number of false positives into a FDR. In summary, the use of a single quality score and a “same versus same” mutation profile allow us to determine the FDR as a function of Q and assign an FDR to each mutation.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/fig[1]"><a name="pcbi-1002714-g001" id="pcbi-1002714-g001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.g001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g001');"><span>Figure 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/fig[1]/caption[1]/title[1]">Schematic overview of FDR calculation method.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/fig[1]/caption[1]/p[1]"><strong>A</strong> Concept of FDR calculation which relies on the availability of a normal tissue replication experiment. <strong>B</strong> Examples of single nucleotide variations found: A somatic mutation found in all three B16 samples (left), a non-somatic mutation found in all B16 and black6 samples (middle) and a mutation found in only one black6 sample (right); this last variation would cause a raise in the FDR for all somatic mutations with a comparable or worse quality. <strong>C</strong> Process to generate FDRs for a set of somatic mutations and visualize the results. The FDR distribution is visualized as an average estimated ROC curve with the grey bars giving the 95% confidence interval for the mean in both dimensions at uniformly sampled positions. The mean was obtained from the distribution of estimated ROC curves of the FDRs for all possible 18 combinations of reference data sets (see text).</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002714.g001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002714.g001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002714.g001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002714.g001.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/p[3]"><a href="#pcbi-1002714-g001">Figure 1b</a> shows examples of variations found in sequences reads from B16F10 (B16) melanoma cells and the reference C57BL/6 (black6) mice. The variations include a somatic mutation found only in the tumor cells (left), a SNP found in both the black6 mice and B16 cells (middle) and probable false positive (right). We associate each mutation with a single quality score, the FDR, which defines the likelihood that the mutation call is inaccurate. Using the FDR, we plot receiver operating characteristic (ROC) curves and determine the corresponding area under curve (AUC) metric to compare the performance of different protocols and algorithms (<a href="#pcbi-1002714-g001">Figure 1c</a>).</p>

<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/title[1]">Mutation discovery</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]">To discover mutations, DNA from tail tissue of three black6 mice, all litter mates, and DNA from three B16 melanoma samples, was extracted and exon-encoding sequences were captured, resulting in six samples. RNA was extracted from B16 cells in triplicate. Single end 50 nt (1×50 nt) and paired end 100 nt (2×100 nt) reads were generated on an Illumina HiSeq 2000 (Supplementary Table S1 in <a href="#pcbi.1002714.s005">Text S1</a>). Each sample was sequenced on an individual lane, resulting in an average of 104 million reads per lane. DNA reads were aligned to the mouse reference genome using the Burrows-Wheeler Alignment Tool (bwa) <a href="#pcbi.1002714-Li2">[16]</a> and RNA reads were aligned with bowtie <a href="#pcbi.1002714-Langmead1">[17]</a>. Using the 1×50 nt reads, 97% of the targeted nucleotides were covered at least once, the mean/median targeted nucleotide coverage was 38×/30× and 70–73% of target nucleotides had 20× or higher coverage. Using the 2×100 nt reads, 98% of the targeted nucleotides were covered at least once, the mean/median targeted nucleotide coverage the was 165×/133× and 97% of target nucleotides had 20× coverage.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[2]">Somatic mutations were independently identified using the software packages SAMtools <a href="#pcbi.1002714-Li3">[18]</a>, GATK <a href="#pcbi.1002714-DePristo1">[11]</a> and SomaticSNiPer <a href="#pcbi.1002714-Larson1">[19]</a> (<a href="#pcbi-1002714-g002">Figure 2</a>) by comparing the single nucleotide variations found in B16 samples to the corresponding loci in the black6 samples (B16 cells were originally derived from a black6 mouse). The potential mutations were filtered according to recommendations from the respective software tools (SAMtools and GATK) or by selecting an appropriate threshold for the somatic score of SomaticSNiPer (<a href="#s4">Methods</a>). Considering only those mutations found in all tumor-normal pairings, the union of B16 somatic mutations identified by the three algorithms was 4,078 (<a href="#pcbi-1002714-g003">Figure 3a</a>). However, substantial differences between the sets of mutations identified by each program exist, even when considering those mutations found in all tumor-normal pairings (<a href="#pcbi-1002714-g003">Figure 3a</a>). While 1,355 mutations are identified by all three programs (33% of 4,078), the agreement between results is low. Of the 2,484 mutations identified by GATK, only 1,661 (67%) are identified by SAMtools and 1,469 (60%) are identified by SomaticSNiPer. Of the 3,109 mutations identified by SAMtools, only 53% and 66% are identified by GATK and SomaticSNiPer, respectively. Of the 2,302 mutation identified by SomaticSNiPer, only 64% and 89% are identified by GATK and SAMtools, respectively. The number of 1,355 mutations identified by all three algorithms reflects only 55% (GATK), 44% (SAMtools) and 59% (SomaticSNiPer) of the mutations found by the individual programs, respectively.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]"><a name="pcbi-1002714-g002" id="pcbi-1002714-g002" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g002');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g002');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.g002&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g002');"><span>Figure 2. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/title[1]">Overview of the process for finding somatic mutations in B16.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/p[1]">Numbers for the individual steps are given as an example for one B16 sample, compared to one black6 sample. “Exons” refers to the exon coordinates defined by all protein coding RefSeq transcripts.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002714.g002</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002714.g002/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g002/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002714.g002.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g002/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002714.g002.TIF"></span> TIFF)
                  </a></li></ul></div></div><div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]"><a name="pcbi-1002714-g003" id="pcbi-1002714-g003" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g003');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g003');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.g003&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g003');"><span>Figure 3. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]/caption[1]/title[1]">Process of selection of mutations for validation.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[2]/caption[1]/p[1]"><strong>A</strong> The Venn diagram shows the numbers of somatic variations in protein coding exons, found by the individual, two or all three software tools, respectively. The numbers were calculated after the recommended filtering procedures (see <a href="#s4">Methods</a> section) and represent the consensus of all three samples. <strong>B</strong> List of unfiltered somatic mutations found in the consensus of all three samples, sorted by FDR (low to high from top to bottom). Each row represents a predicted mutation and it is indicated which program did the prediction. <strong>C</strong> For each mutation a FDR can be calculated, which is used for prioritization of the validation experiment.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002714.g003</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002714.g003/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g003/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002714.g003.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g003/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002714.g003.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/title[1]">Mutations can be assigned a FDR confidence value</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[1]">We want to assign each somatic mutation a single quality score Q that could be used to rank mutations based on confidence. However, it is not straightforward to assign a single value since most mutation detection algorithms output multiple scores, each reflecting a different quality aspect. Thus, we generated a random forest classifier <a href="#pcbi.1002714-Breiman1">[20]</a> that combines multiple scores, resulting in a single quality score Q (<a href="#s4">Methods</a>). All identified somatic mutations, whether from the “same versus same” or “tumor versus normal” comparison, thus are assigned a single value predicting accuracy. Note that the classifier training needs to be performed separately for each program, due to the differences in the set of scores which are returned by the individual programs.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[2]">After defining a relevant quality score, we sought to re-define the score into a statistically relevant false discovery rate (FDR). We determined, at each Q value, the number of mutations with a better Q score in the “same versus same” and the number of mutations with a better Q score in the “tumor versus normal” pair. For a given mutation with quality score Q detected in the “tumor versus normal” comparison, we estimate the false discovery rate by computing the ratio of “same versus same” mutations with a score of Q or better to the overall number of mutations found in the tumor comparison with a score of Q or better.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[3]">A potential bias in comparing methods is differential coverage; we thus normalize the false discovery rate for the number of bases covered by NGS reads in each sample:<br><a name="pcbi.1002714.e001" id="pcbi.1002714.e001"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.e001&amp;representation=PNG"></span><br></p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[4]">We calculate the common coverage by counting all bases of the reference genome which are covered by data of the tumor and normal sample or by both “same versus same” samples, respectively. After assigning our FDR to each mutation, the FDR-sorted list of somatic mutations shows a clear preference of mutations found by three programs in the low FDR region (<a href="#pcbi-1002714-g003">Figure 3b</a>; see <a href="#pcbi.1002714.s001">Supplementary Dataset S1</a> for a complete list). This observation fits to the naïve assumption that the consensus of multiple different algorithms is likely to be correct.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[5]">We identified 50 mutations with a low FDR (high confidence) for validation, including 41 with an FDR less than 0.05 (<a href="#pcbi-1002714-g003">Figure 3c</a>). All 50 were validated by a combination of Sanger resequencing and inspection of the B16 RNA-Seq sequence reads. <a href="#pcbi-1002714-t001">Table 1</a> lists the ten somatic mutations with the best FDRs, all of which validated.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/table-wrap[1]"><a name="pcbi-1002714-t001" id="pcbi-1002714-t001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.t001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.t001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.t001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/table-wrap[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.t001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.t001');"><span>Table 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/table-wrap[1]/caption[1]/title[1]">Ten validated mutations with the lowest FDRs, selected out of a set of 2396 exonic variations which were found in duplicate in two B16 samples.</span></strong></p><span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002714.t001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002714.t001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.t001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002714.t001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.t001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002714.t001.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[6]">We selected 44 mutations identified by at least one detection algorithm, present in only one B16 sample and assigned a high FDR (&gt;0.5) by our algorithm (<a href="#pcbi-1002714-g003">Figure 3c</a>). In contrast to the low-FDR mutations, none of the 44 high FDR samples validated, neither by Sanger sequencing nor by inspection of the RNA alignments. 37 of those mutations were clear false positives (no mutation by Sanger or RNA-Seq) while the remaining seven loci neither yielded sequencing reactions nor were covered by RNA-Seq reads. <a href="#pcbi-1002714-g004">Figure 4</a> shows representative mutations together with the Sanger sequencing traces. In the case of the false positive mutation, the three used programs identified this in black6 as sequencing error (and did not output a mutation at this locus), but failed in the single B16 case (marked with the red box). If a real experiment would have included only this single sample, it would have produced a false positive mutation call, despite using the consensus of three programs.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]"><a name="pcbi-1002714-g004" id="pcbi-1002714-g004" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g004" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g004');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g004');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.g004&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g004" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g004');"><span>Figure 4. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/title[1]">Genome browser screen shot for triplicate black6 and B16 samples and associated Sanger sequencing traces for the black6 and B16 DNA for both a true positive (chr4:155261079) and a false positive (chr4:151534480) mutation call.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[1]/caption[1]/p[1]">Both mutations are predicted by GATK, SomaticSNiPer and SAMtools. The mean coverage is 54 (true positive) and 10 (false positive), respectively. Only four reads are shown for visual clarity. The red box marks the sample, in which the three mutation callers wrongly detected a SNV.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002714.g004</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002714.g004/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g004/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002714.g004.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g004/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002714.g004.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[7]">To test mutations with less extreme FDRs, we selected 45 somatic mutations, which were distributed evenly across the FDR spectrum from 0.1 to 0.6. Validation using both Sanger sequencing and inspection of the RNA-Seq reads resulted 15 positive (either Sanger sequencing or RNA-Seq reads), 22 negative validations (neither Sanger sequencing nor RNA-Seq reads) and 8 non-conclusive (failed sequencing reactions and no RNA-Seq coverage). See the <a href="#pcbi.1002714.s002">Supplementary Dataset S2</a> for a detailed table showing the results of the validation of those 45 mutations.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[8]">We computed a receiver operating characteristic (ROC) curve for all 131 validated mutations (<a href="#pcbi-1002714-g005">Figure 5a</a>), resulting in an area under the curve (AUC) <a href="#pcbi.1002714-Fawcett1">[21]</a> of 0.96. As this analysis might be biased due to the relatively large set sizes of the high and low FDR mutations, we randomly sampled 10 mutations each, added the 37 validated mutations with the intermediate FDRs, calculated the ROC-AUC and repeated this 1000 times in order to get a more robust performance estimate. The resulting mean AUC is 0.797 (+−0.002). A systematic test of FDR thresholds ranging from zero to one with a step size of 0.05 implies that an optimal threshold for using the FDR as a binary classifier should be ≤0.2.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[2]"><a name="pcbi-1002714-g005" id="pcbi-1002714-g005" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g005" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g005');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g005');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.g005&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[2]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g005" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g005');"><span>Figure 5. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[2]/caption[1]/title[1]">Results of mutation validation experiments.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/fig[2]/caption[1]/p[1]"><strong>A</strong> ROC curve for all 131 mutations with a successful validation (either positive or negative). 1-FDR was used as the probability of a mutation being a true call. <strong>B</strong> Relative amount of variations found for a given FDR cutoff in a set of 2396 exonic variations which were found in duplicate in two B16 samples (see also <a href="#pcbi-1002714-t001">Table 1</a>), plotted separately for all variants in the dataset and the 50 validated high confidence mutations. For visual clarity only values of 0 to 0.1 FDR are shown.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002714.g005</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002714.g005/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g005/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002714.g005.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g005/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002714.g005.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/title[1]">ROC curves can be used to compare methods</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[1]">ROC curves and the corresponding AUC are useful for comparing classifiers and visualizing their performance <a href="#pcbi.1002714-Fawcett1">[21]</a>. We extended this concept for evaluating the performance of experimental and computational procedures. However, plotting ROC graphs requires knowledge of all true and false positives (TP and FP) in a dataset, information which is usually not given and hard to establish for high throughput data (such as NGS data). Thus, we used the calculated FDRs to estimate the respective TP and FP rates and plot a ROC curve and calculate the AUC (<a href="#pcbi-1002714-g001">Figure 1c</a>). <a href="#pcbi-1002714-g005">Figure 5b</a> shows the ROC curve comparing the FDR versus the percent of 50 validated mutations and percent of total.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/title[1]">Benchmarking of experimental settings</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[1]">ROC curves and the associated AUC values can be compared across experiments, lab protocols, and algorithms. For the following comparisons, we used all somatic mutations found by any algorithm and in any tumor-normal pairing without applying any filter procedure. We considered only those mutations in target regions (exons).</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[2]">First, we tested the influence of the reference “same versus same” data on the calculation of the FDRs. Using the triplicate black6 and B16 sequencing runs, we created 18 triplets (combinations of “black6 versus black6” and “black6 versus B16”) to use for calculating the FDR. When comparing the resulting FDR distributions for the sets of somatic mutations, the results are consistent when the reference data sets are exchanged (<a href="#pcbi-1002714-g001">Figure 1c</a>, Supplementary Figure S2 in <a href="#pcbi.1002714.s005">Text S1</a>). This suggests that the method is robust with regards to the choice of the reference “same versus same” dataset. Thus, a “same versus same” duplicate profiling needs only be done once for a given lab platform and the resultant FDR(Q) reference function can be re-used for future profiling.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[3]">Using our definition of a false discovery rate, we have established a generic framework for evaluating the influence of numerous experimental and algorithmic parameters on the resulting set of somatic mutations. We apply this framework to study the influence of software tools, coverage, paired end sequencing and the number of technical replicates on somatic mutation identification.</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[4]">First, the choice of the software tool has a clear impact on the identified somatic mutations (<a href="#pcbi-1002714-g003">Figure 3</a>). On the tested data, SAMtools produces the highest enrichment of true positive somatic mutations (<a href="#pcbi-1002714-g006">Figure 6a</a>). We note that each tool has different parameters and quality scores for mutation detection; we used the default settings as specified by the algorithm developers.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]"><a name="pcbi-1002714-g006" id="pcbi-1002714-g006" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g006" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g006');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g006');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.g006&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002714?imageURI=info:doi/10.1371/journal.pcbi.1002714.g006" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002714', 'info:doi/10.1371/journal.pcbi.1002714.g006');"><span>Figure 6. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/title[1]">Comparison of different experimental settings and analysis procedures.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/p[1]"><strong>A</strong> Estimated ROC curves for the comparison of the three different software tools (duplicates, 38× coverage). <strong>B</strong> Estimated ROC curves for the comparison of different average sequencing depths (SAMtools, no replication). 38× denotes the coverage obtained by the experiment, while other coverages were down sampled starting with this data. <strong>C</strong> Estimated ROC curves visualizing the effect of experiment replication (38× coverage, SAMtools). <strong>D</strong> Estimated ROC curves for different sequencing protocols (SAMtools, no replication). The curves were calculated using the results of the 2×100 nt library (<strong>Note</strong>: A complete display of the results can be found in Supplementary Figures S2 and S3 in <a href="#pcbi.1002714.s005">Text S1</a>. Also, unscaled versions of the plots are shown in Supplementary Figure S8 in <a href="#pcbi.1002714.s005">Text S1</a>, giving an impression of the individual set sizes).</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002714.g006</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002714.g006/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g006/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002714.g006.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002714.g006/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002714.g006.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[5]">The impact of the coverage depth on whole genome SNV detection has been recently discussed <a href="#pcbi.1002714-Ajay1">[22]</a>. For the B16 sequencing experiment, we sequenced each sample in an individual flowcell lane and achieved a target region mean base coverage of 38 fold across target nucleotides. In order to study the effect of the coverage on exon capture data, we down-sampled the number of aligned sequence reads for every 1×50 nt library to generate a mean coverage of 5, 10 and 20 fold, respectively, and then reapplied the mutation identification algorithms. As expected, a higher coverage results in a better (i.e. fewer false positives) somatic mutation set, although the improvement from the 20 fold coverage to 38 fold is marginal for the B16 cells (<a href="#pcbi-1002714-g006">Figure 6b</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[6]">It is straightforward to simulate and rank other experimental settings using the available data and framework (<a href="#pcbi-1002714-g006">Figures 6c and d</a>). As we profiled each sample in triplicate, including three separate exon captures, we wanted to identify the impact of these replicates. Comparing duplicates to triplicates, triplicates do not offer a benefit compared to the duplicates (<a href="#pcbi-1002714-g006">Figure 6c</a>), while duplicates offer a clear improvement compared to a study without any replicates (indicated by the higher AUC). In terms of the ratio of somatic mutations at a FDR of 0.05 or less, we see enrichment from 24% for a run without replicates to 71% for duplicates and 86% for triplicates. These percentages correspond to 1441, 1549 and 1524 mutations, respectively. Using the intersection of triplicates removes more mutations with low FDRs than mutations with a high FDR, as indicated by the lower ROC AUC and the shift of the curve to the right (Supplementary Figure S7 in <a href="#pcbi.1002714.s005">Text S1</a>, <a href="#pcbi-1002714-g006">Figure 6c</a>): the specificity is slightly increased at the cost of a lower sensitivity, when assuming that removed low FDR mutations are true positives and the removed high FDR mutations are true negatives. This assumption is supported by our validation experiments, as true negative mutations are likely to get a high FDR (<a href="#pcbi-1002714-g005">Figure 5a</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[7]">The 2×100 nt library was used to create 6 libraries: a 2×100 nt library; a 1×100 nt library; a 1×50 nt library using the 50 nucleotides at the 5′ end of the first read; a 1×50 nt library using the nucleotides 51 to 100 at the 3′ end of the first read; a 2×50 nt read using nucleotides 1 to 50 of both reads; and a 2×50 nt library using nucleotides 51 to 100 of both reads. These libraries were compared using the calculated FDRs of predicted mutations (<a href="#pcbi-1002714-g006">Figure 6d</a>). The 1×50 3′ library performed worst, as expected due to the increasing error rate at the 3′ end of sequence reads. Despite the much higher median coverage (63–65 vs. 32), the somatic mutations found using the 2×50 5′ and 1×100 nt libraries have a smaller AUC than the 1×50 nt library. This surprising effect is a result of high FDR mutations in regions with low coverage (Supplementary <a href="#pcbi.1002714.s005">Text S1</a>). Indeed, the sets of low FDR mutations are highly similar. Thus, while the different read lengths and types identify non-identical mutations, the assigned FDR is nevertheless able to segregate true and false positives (Supplementary Figure S3 in <a href="#pcbi.1002714.s005">Text S1</a>).</p>

</div>

<div id="section3" xpathLocation="/article[1]/body[1]/sec[3]"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3 xpathLocation="noSelect">Discussion&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[3]/p[1]">NGS is a revolutionary platform for detecting somatic mutations. However, the error rates are not insignificant, with different detection algorithms identifying mutations with less than 50% congruence. Other high throughput genomic profiling platforms have developed methods to assign confidence values to each call, such as p-values associated with differential expression calls from oligonucleotide microarray data. Similarly, we developed here a method to assign a confidence value (FDR) to each identified mutation.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[2]">From the set of mutations identified by the different algorithms, the FDR accurately ranks mutations based on likelihood of being correct. Indeed, we selected 50 high confidence mutations and all 50 validated; we selected 45 intermediate confidence mutations and 15 validated, 22 were not present and 8 inconclusive; we selected 44 low confidence mutations and none validated. Again, all 139 mutations were identified by at least one of the detection algorithms. Unlike a consensus or majority voting approach, the assigned FDR not only effectively segregates true and false positives but also provides both the likelihood that the mutation is true and a statistically ranking. Also, our method allows the adjustment for a desired sensitivity or specificity which enables the detection of more true mutations than a consensus or majority vote, which report only 50 or 52 of all 65 validated mutations.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[3]">We applied the method to a set of B16 melanoma cell experiments. However, the method is not restricted to these data. The only requirement is the availability of a “same versus same” reference dataset, meaning at least a single replicate of a non-tumorous sample should be performed for each new protocol. Our experiments indicate that the method is robust with regard to the choice of the replicate, so that a replicate is not necessarily required in every single experiment. Once done, the derived FDR(Q) function can be reused when the Q scores are comparable (i.e. when the same program for mutation discovery was used). Here, we profiled all samples in triplicate; nevertheless, the method produces FDRs for each mutation from single-run tumor and normal profiles (non-replicates) using the FDR(Q) function. We do show, however, that duplicates improve data quality.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[4]">Furthermore, the framework enables one to define best practice procedures for the discovery of somatic mutations. For cell lines, at least 20-fold coverage and a replicate achieve close to the optimum results. A 1×50 nt library resulting in approximately 100 million reads is a pragmatic choice to achieve this coverage.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[5]">The possibility of using a reference data set to rank the results of another experiment can also be exploited to e.g. score somatic mutations found in different normal tissues by similar methods. Here, one would expect relatively few true mutations, so an independent set of reference data will improve the resolution of the FDR calculations.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[6]">While we define the optimum as the lowest number of false positive mutation calls, this definition might not suffice for other experiments, such as for genome wide association studies. However, our method allows the evaluation of the sensitivity and specificity of a given mutation set and we show application of the framework to four specific questions. The method is by no means limited to these parameters, but can be applied to study the influence of all experimental or algorithmic parameters, e.g. the influence of the alignment software, the choice of a mutation metric or the choice of vendor for exome selection.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/p[7]">In summary, we have pioneered a statistical framework for the assignment of a false-discovery-rate to the detection of somatic mutations. This framework allows for a generic comparison of experimental and computational protocol steps on generated quasi ground truth data. Furthermore, it is applicable for the diagnostic or therapeutic target selection as it is able to distinguish true mutations from false positives.</p>
</div>

<div id="section4" xpathLocation="/article[1]/body[1]/sec[4]"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3 xpathLocation="noSelect">Methods&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/title[1]">Library capture and sequencing</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]">Next-generation sequencing, DNA sequencing: Exome capture for DNA resequencing was performed using the Agilent Sure-Select solution-based capture assay <a href="#pcbi.1002714-Gnirke1">[23]</a>, in this case designed to capture all known mouse exons.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[2]">3 µg purified genomic DNA was fragmented to 150–200 nt using a Covaris S2 ultrasound device. gDNA fragments were end repaired using T4 DNA polymerase, Klenow DNA polymerase and 5′ phosphorylated using T4 polynucleotide kinase. Blunt ended gDNA fragments were 3′ adenylated using Klenow fragment (3′ to 5′ exo minus). 3′ single T-overhang Illumina paired end adapters were ligated to the gDNA fragments using a 10:1 molar ratio of adapter to genomic DNA insert using T4 DNA ligase. Adapter ligated gDNA fragments were enriched pre capture and flow cell specific sequences were added using Illumina PE PCR primers 1.0 and 2.0 and Herculase II polymerase (Agilent) using 4 PCR cycles.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[3]">500 ng of adapter ligated, PCR enriched gDNA fragments were hybridized to Agilent's SureSelect biotinylated mouse whole exome RNA library baits for 24 hrs at 65°C. Hybridized gDNA/RNA bait complexes where removed using streptavidin coated magnetic beads. gDNA/RNA bait complexes were washed and the RNA baits cleaved off during elution in SureSelect elution buffer leaving the captured adapter ligated, PCR enriched gDNA fragments. gDNA fragments were PCR amplified post capture using Herculase II DNA polymerase (Agilent) and SureSelect GA PCR Primers for 10 cycles. Cleanups were performed using 1.8× volume of AMPure XP magnetic beads (Agencourt). For quality controls we used Invitrogen's Qubit HS assay and fragment size was determined using Agilent's 2100 Bioanalyzer HS DNA assay. Exome enriched gDNA libraries were clustered on the cBot using Truseq SR cluster kit v2.5 using 7 pM and sequenced on the Illumina HiSeq2000 using Truseq SBS kit.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/title[1]">Exome data analysis</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/p[1]">Sequence reads were aligned using bwa (version 0.5.8c) <a href="#pcbi.1002714-Li2">[16]</a> using default options to the reference mouse genome assembly mm9 <a href="#pcbi.1002714-Mouse1">[24]</a>. Ambiguous reads – those reads mapping to multiple locations of the genome as provided by the bwa output - were removed (see <a href="#pcbi.1002714.s003">Supplementary Dataset S3</a> for the alignment statistics). The remaining alignments were sorted, indexed and converted to a binary and compressed format (BAM) and the read quality scores converted from the Illumina standard phred+64 to standard Sanger quality scores using shell scripts.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/p[2]">For each sequencing lane, mutations were identified using three software programs: SAMtools pileup (version 0.1.8) <a href="#pcbi.1002714-Li3">[18]</a>, GATK (version 1.0.4418) <a href="#pcbi.1002714-DePristo1">[11]</a> and SomaticSNiPer <a href="#pcbi.1002714-Larson1">[19]</a>. For SAMtools, the author-recommend options and filter criteria were used (<a href="http://sourceforge.net/apps/mediawiki/SAMtools/index.php?title=SAM_FAQ">http://sourceforge.net/apps/mediawiki/SA​Mtools/index.php?title=SAM_FAQ</a>; accessed September 2011), including first round filtering, maximum coverage 200. For SAMtools second round filtering, the point mutation minimum quality was 30. For GATK mutation calling, we followed the author-designed best practice guidelines presented on the GATK user manual (<a href="http://www.broadinstitute.org/gsa/wiki/index.php?title=Best_Practice_Variant_Detection_with_the_GATK_v2&amp;oldid=5207">http://www.broadinstitute.org/gsa/wiki/i​ndex.php?title=Best_Practice_Variant_Det​ection_with_the_GATK_v2&amp;oldid=5207</a>; accessed October 2010). For each sample a local realignment around indel sites followed by a base quality recalibration was performed. The Unified Genotyper module was applied to the resultant alignment data files. When needed, the known polymorphisms of the dbSNP <a href="#pcbi.1002714-Sherry1">[25]</a> (version 128 for mm9) were supplied to the individual steps. The variant score recalibration step was omitted and replaced by the hard-filtering option. For both SAMtools and GATK, potential indels were filtered out of the results before further processing and a mutation was accepted as somatic if it is present in the data for B16 but not in the black6 sample. Additionally, as a post filter, for each potentially mutated locus we required non-zero coverage in the normal tissue. This is intended to sort out mutations which only look to be somatic because of a not covered locus in the black6 samples. For SomaticSNiPer mutation calling, the default options were used and only predicted mutations with a “somatic score” of 30 or more were considered further (see Supplementary <a href="#pcbi.1002714.s005">Text S1</a> for a description of the cutoff selection). For all three programs, we removed all mutations located in repetitive sequences as defined by the RepeatMasker track of the UCSC Genome Browser <a href="#pcbi.1002714-Dreszer1">[26]</a> for the mouse genome assembly mm9.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/title[1]">RNA-Seq</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/p[1]">Barcoded mRNA-seq cDNA libraries were prepared from 5 ug of total RNA using a modified version of the Illumina mRNA-seq protocol. mRNA was isolated using SeramagOligo(dT) magnetic beads (Thermo Scientific). Isolated mRNA was fragmented using divalent cations and heat resulting in fragments ranging from 160–200 bp. Fragmented mRNA was converted to cDNA using random primers and SuperScriptII (Invitrogen) followed by second strand synthesis using DNA polymerase I and RNaseH. cDNA was end repaired using T4 DNA polymerase, Klenow DNA polymerase and 5′ phosphorylated using T4 polynucleotide kinase. Blunt ended cDNA fragments were 3′ adenylated using Klenow fragment (3′ to 5′ exo minus). 3′ single T-overhang Illumina multiplex specific adapters were ligated on the cDNA fragments using T4 DNA ligase. cDNA libraries were purified and size selected at 300 bp using the E-Gel 2% SizeSelect gel (Invitrogen). Enrichment, adding of Illumina six base index and flow cell specific sequences was done by PCR using Phusion DNA polymerase (Finnzymes). All cleanups were performed using 1.8× volume of Agencourt AMPure XP magnetic beads.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/p[2]">Barcoded RNA-seq libraries were clustered on the cBot using Truseq SR cluster kit v2.5 using 7 pM and sequenced on the Illumina HiSeq2000 using Truseq SBS kit.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/p[3]">The raw output data of the HiSeq was processed according to the Illumina standard protocol, including removal of low quality reads and demultiplexing. Sequence reads were then aligned to the reference genome sequence <a href="#pcbi.1002714-Mouse1">[24]</a> using bowtie <a href="#pcbi.1002714-Langmead1">[17]</a>. The alignment coordinates were compared to the exon coordinates of the RefSeq transcripts <a href="#pcbi.1002714-Pruitt1">[27]</a> and for each transcript the counts of overlapping alignments were recorded. Sequence reads not aligning to the genomic sequence were aligned to a database of all possible exon-exon junction sequences of the RefSeq transcripts <a href="#pcbi.1002714-Pruitt1">[27]</a>. The alignment coordinates were compared to RefSeq exon and junction coordinates, reads counted and normalized to RPKM (number of reads which map per nucleotide kilobase of transcript per million mapped reads <a href="#pcbi.1002714-Mortazavi1">[28]</a>) for each transcript.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/title[1]">Validation of SNVs</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/p[1]">We selected SNVs for validation by Sanger re-sequencing and RNA. SNVs were identified which were predicted by all three programs, non-synonymous and found in transcripts having a minimum of 10 RPKM. Of these, we selected the 50 with the highest SNP quality scores as provided by the programs. As a negative control, 44 SNVs were selected which have a FDR of 0.5 or more, are present in only one cell line sample and are predicted by only one mutation calling program. 45 mutations with intermediate FDR levels were selected. Using DNA, the selected variants were validated by PCR amplification of the regions using 50 ng of DNA (see <a href="#pcbi.1002714.s004">Supplementary Dataset S4</a> for the primer sequences and targeted loci), followed by Sanger sequencing (Eurofins MWG Operon, Ebersberg, Germany). The reactions were successful for 50, 32 and 37 loci of positive, negative and intermediate controls, respectively. Validation was also done by examination of the tumor RNA-Seq reads.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/title[1]">Calculation of FDRs and machine learning</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[1]">Random Forest Quality Score Computation: Commonly-used mutation calling algorithms (<a href="#pcbi.1002714-DePristo1">[11]</a>, <a href="#pcbi.1002714-Li3">[18]</a>, <a href="#pcbi.1002714-Larson1">[19]</a>) output multiple scores, which all are potentially influential for the quality of the mutation call. These include - but are not limited to - the quality of the base of interest as assigned by the instrument, the alignment quality and number of reads covering this position or a score for the difference between the two genomes compared at this position. For the computation of the false discovery rate we require an ordering of mutations, however this is not directly feasible for all mutations since we might have contradicting information from the various quality scores.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[2]">We use the following strategy to achieve a complete ordering. In a first step, we apply a very rigorous definition of superiority by assuming that a mutation has better quality than another if and only if it is superior in all categories. So a set of quality properties S = (s<sub>1</sub>,…,s<sub>n</sub>) is preferable to T = (t<sub>1</sub>,…,t<sub>n</sub>), denoted by S&gt;T, if s<sub>i</sub>&gt;t<sub>i</sub> for all i = 1,…,n. We define an intermediate FDR (IFDR) as follows<br><a name="pcbi.1002714.e002" id="pcbi.1002714.e002"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.e002&amp;representation=PNG"></span><br></p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[3]">However, we regard the IFDR only as an intermediate step since in many closely related cases, no comparison is feasible and we are thus not benefitting from the vast amount of data available. Thus, we take advantage of the good generalization property of random forest regression <a href="#pcbi.1002714-Breiman1">[20]</a> and train a random forest as implemented in R (<a href="#pcbi.1002714-R1">[29]</a>, <a href="#pcbi.1002714-Liaw1">[30]</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[4]">For <em>m</em> input mutations with <em>n</em> quality properties each, the value range for each property was determined and up to <em>p</em> values were sampled with uniform spacing out of this range; when the set of values for a quality property was smaller than <em>p</em>, this set was used instead of the sampled set. Then each possible combination of sampled or selected quality values was created, which resulted in a maximum of <em>p<sup>n</sup></em> data points in the <em>n</em>-dimensional quality space. A random sample of 1% of these points and the corresponding IFDR values were used as predictor and response, respectively, for the random forest training.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[5]">The resulting regression score is our generalized quality score Q; it can be regarded as a locally weighted combination of the individual quality scores. It allows direct, single value comparison of any two mutations and the computation of the actual false discovery rate:<br><a name="pcbi.1002714.e003" id="pcbi.1002714.e003"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.e003&amp;representation=PNG"></span><br></p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[6]">For the training of the random forest models used to create the results for this study, we calculate the sample IFDR on the somatic mutations of all samples before selecting the random 1% subset. This ensures the mapping of the whole available quality space to FDR values. We used the quality properties “SNP quality”, “coverage depth”, “consensus quality” and “RMS mapping quality” (SAMtools, <em>p</em> = 20); “SNP quality”, “coverage depth”, “Variant confidence/unfiltered depth” and “RMS mapping quality” (GATK, <em>p</em> = 20); or SNP quality”, “coverage depth”, “consensus quality”, “RMS mapping quality” and “somatic score” (SomaticSNiPer, <em>p</em> = 12), respectively. The different values of <em>p</em> ensure a set size of comparable magnitude.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[7]">To acquire the “same vs. same” and “same vs. different” data when calculating the FDRs for a given set of mutations, we use all variants generated by the different programs without any additional filtering.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[8]">Common coverage computation: The number of possible mutation calls can introduce a major bias in the definition of a false discovery rate. Only if we have the same number of possible locations for mutations to occur for our tumor comparison and for our “same vs. same” comparison, the number of called mutations is comparable and can serve as a basis for a false discovery rate computation. To correct for this potential bias, we use the common coverage ratio. As common coverage we define the number of bases with coverage of at least one in both samples which are used for the mutation calling. We compute the common coverage individually for the tumor comparison as well as for the “same vs. same” comparison.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/title[1]">ROC estimation</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[1]">The estimation of the ROC curves should satisfy the following criteria:</p>


<ol class="order">

<li xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/list[1]/list-item[1]">When all calculated FDRs are 0.5, one cannot use these rates to select true positive mutations. This should be reflected by a diagonal line from (0,0) to (1,1) in the ROC plot resulting in a ROC AUC of 0.5, which indicates a completely random prediction.</li>

<li xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/list[1]/list-item[2]">The normal calculation of ROC curves involves summing up the TP counts and FP counts, respectively, up to a given score threshold. Here, an individual mutation does not add one to the TP or FP count, but a fraction depending on the given FDR to both sums, respectively. Both fractions should add to one, then.</li>

</ol><p xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[2]">We start with two conditions; <a href="#pcbi.1002714.e004">Eq. [1]</a> is the definition of the FDR and <a href="#pcbi.1002714.e005">Eq. [2]</a> is needed to satisfy the criteria given above.<br><a name="pcbi.1002714.e004" id="pcbi.1002714.e004"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.e004&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[2]/disp-formula[1]/label[1]">(1)</span></span><br><br><a name="pcbi.1002714.e005" id="pcbi.1002714.e005"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.e005&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[2]/disp-formula[2]/label[1]">(2)</span></span><br><em>FPR</em> and <em>TPR</em> are the needed false positive true positive ratios, respectively, for the given mutation, defining the corresponding point in ROC space. <a href="#pcbi.1002714.e004">Eq. [1]</a> and <a href="#pcbi.1002714.e005">Eq. [2]</a> can be rearranged to <a href="#pcbi.1002714.e006">Eq. [3]</a> and <a href="#pcbi.1002714.e007">Eq. [4]</a>.<br><a name="pcbi.1002714.e006" id="pcbi.1002714.e006"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.e006&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[2]/disp-formula[3]/label[1]">(3)</span></span><br><br><a name="pcbi.1002714.e007" id="pcbi.1002714.e007"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002714.e007&amp;representation=PNG"><span class="note" xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[2]/disp-formula[4]/label[1]">(4)</span></span><br></p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[3]">To obtain an estimated ROC curve, the mutations in the dataset are sorted by FDR and for each mutation a point is plotted at the cumulative TPR and FPR values up to this mutation, divided by the sum of all TPR and FPR values, respectively. The AUC is calculated by summing up the areas of all consecutive trapezoids between the curve and the x-axis.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[4]">The program is implemented in R and is available from <a href="http://tron-mainz.de/tron-facilities/computational-medicine/">http://tron-mainz.de/tron-facilities/com​putational-medicine/</a>. The package allows convenient import and processing of variation calls in VCF files.</p>

</div>

<div id="section5" xpathLocation="/article[1]/body[1]/sec[5]"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3 xpathLocation="noSelect">Supporting Information&nbsp;<a href="#top">Top</a></h3><a name="pcbi.1002714.s001" id="pcbi.1002714.s001"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002714.s001">Dataset S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[1]">12,460 somatic mutations found in triplicate samples.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[2]">(XLS)</p>
<a name="pcbi.1002714.s002" id="pcbi.1002714.s002"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002714.s002">Dataset S2. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[1]">Validation results for mutations with an intermediate FDR.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[2]">(XLS)</p>
<a name="pcbi.1002714.s003" id="pcbi.1002714.s003"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002714.s003">Dataset S3. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[1]">Alignment statistics for all samples.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[2]">(XLS)</p>
<a name="pcbi.1002714.s004" id="pcbi.1002714.s004"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002714.s004">Dataset S4. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[1]">Primer sequences.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[2]">(XLS)</p>
<a name="pcbi.1002714.s005" id="pcbi.1002714.s005"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002714.s005">Text S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[1]">Selection of a filtering threshold for SomaticSNiPer, discussion of paired end library results and additional figures (S1, S2, S3, S4, S5, S6, S7, S8) and tables (S1, S2, S3).</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[2]">(PDF)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3 xpathLocation="noSelect">Author Contributions&nbsp;<a href="#top">Top</a></h3><p xpathLocation="noSelect">Conceived and designed the experiments: ÖT MD CB SK MK JCC US. Performed the experiments: JdG MW CP CK. Analyzed the data: ML BYR JCC. Contributed reagents/materials/analysis tools: MD. Wrote the paper: ML BYR JCC. Designed the software used in analysis: ML BYR JCC Implemented the software: ML BYR Processed the sequencing data: ML.</p></div><div xpathLocation="noSelect"><a id="references" name="references" toc="references" title="References"></a><h3 xpathLocation="noSelect">References&nbsp;<a href="#top">Top</a></h3><ol class="references" xpathLocation="noSelect"><li xpathLocation="noSelect"><a name="pcbi.1002714-Nothnagel1" id="pcbi.1002714-Nothnagel1"></a>Nothnagel M, Herrmann A, Wolf A, Schreiber S, Platzer M, et al. (2011) Technology-specific error signatures in the 1000 Genomes Project data. Hum Genet 130: 505–516.
        doi:
        <a href="http://dx.doi.org/10.1007/s00439-011-0971-3">10.1007/s00439-011-0971-3</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755191">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Yoshida1" id="pcbi.1002714-Yoshida1"></a>Yoshida K, Sanada M, Shiraishi Y, Nowak D, Nagata Y, et al. (2010) Frequent pathway mutations of splicing machinery in myelodysplasia. Nature 478: 64–69.
        doi:
        <a href="http://dx.doi.org/10.1038/nature10496">10.1038/nature10496</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755193">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Hansen1" id="pcbi.1002714-Hansen1"></a>Hansen KD, Brenner SE, Dudoit S (2010) Biases in Illumina transcriptome sequencing caused by random hexamer priming. Nucleic Acids Res 38: e131.
        doi:
        <a href="http://dx.doi.org/10.1093/nar/gkq224">10.1093/nar/gkq224</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755195">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Taub1" id="pcbi.1002714-Taub1"></a>Taub MA, Corrada Bravo H, Irizarry RA (2010) Overcoming bias and systematic errors in next generation sequencing data. Genome Med 2: 87.
        doi:
        <a href="http://dx.doi.org/10.1186/gm208">10.1186/gm208</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755197">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Bainbridge1" id="pcbi.1002714-Bainbridge1"></a>Bainbridge MN, Wang M, Burgess DL, Kovar C, Rodesch MJ, et al. (2010) Whole exome capture in solution with 3 Gbp of data. Genome Biol 11: R62.
        doi:
        <a href="http://dx.doi.org/10.1186/gb-2010-11-6-r62">10.1186/gb-2010-11-6-r62</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755199">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Nakamura1" id="pcbi.1002714-Nakamura1"></a>Nakamura K, Oshima T, Morimoto T, Ikeda S, Yoshikawa H, et al. (2011) Sequence-specific error profile of Illumina sequencers. Nucleic Acids Res 39: e90.
        doi:
        <a href="http://dx.doi.org/10.1093/nar/gkr344">10.1093/nar/gkr344</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755201">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Kircher1" id="pcbi.1002714-Kircher1"></a>Kircher M, Stenzel U, Kelso J (2009) Improved base calling for the Illumina Genome Analyzer using machine learning strategies. Genome Biol 10: R83.
        doi:
        <a href="http://dx.doi.org/10.1186/gb-2009-10-8-r83">10.1186/gb-2009-10-8-r83</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755203">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Lassmann1" id="pcbi.1002714-Lassmann1"></a>Lassmann T, Hayashizaki Y, Daub CO (2011) SAMStat: monitoring biases in next generation sequencing data. Bioinformatics 27: 130–131.
        doi:
        <a href="http://dx.doi.org/10.1093/bioinformatics/btq614">10.1093/bioinformatics/btq614</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755205">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Li1" id="pcbi.1002714-Li1"></a>Li H (2011) Improving SNP discovery by base alignment quality. Bioinformatics 27: 1157–1158.
        doi:
        <a href="http://dx.doi.org/10.1093/bioinformatics/btr076">10.1093/bioinformatics/btr076</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755207">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Zhang1" id="pcbi.1002714-Zhang1"></a>Zhang Z, Gerstein M (2003) Patterns of nucleotide substitution, insertion and deletion in the human genome inferred from pseudogenes,. Nucleic Acids Res 31: 5338–5348.
        doi:
        <a href="http://dx.doi.org/10.1093/nar/gkg745">10.1093/nar/gkg745</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755209">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-DePristo1" id="pcbi.1002714-DePristo1"></a>DePristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, et al. (2011) A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet 43: 491–498.
        doi:
        <a href="http://dx.doi.org/10.1038/ng.806">10.1038/ng.806</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755211">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Ewen1" id="pcbi.1002714-Ewen1"></a>Ewen KR, Bahlo M, Treloar SA, Levinson DF, Mowry B, et al. (2000) Identification and analysis of error types in high-throughput genotyping. Am J Hum Genet 67: 727–736.
        doi:
        <a href="http://dx.doi.org/10.1086/303048">10.1086/303048</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755213">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Druley1" id="pcbi.1002714-Druley1"></a>Druley TE, Vallania FL, Wegner DJ, Varley KE, Knowles OL, et al. (2009) Quantification of rare allelic variants from pooled genomic DNA. Nat Methods 6: 263–265.
        doi:
        <a href="http://dx.doi.org/10.1038/nmeth.1307">10.1038/nmeth.1307</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755215">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Bansal1" id="pcbi.1002714-Bansal1"></a>Bansal V (2010) A statistical method for the detection of variants from next-generation resequencing of DNA pools. Bioinformatics 26: 318–324.
        doi:
        <a href="http://dx.doi.org/10.1093/bioinformatics/btq214">10.1093/bioinformatics/btq214</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755217">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-VanTassell1" id="pcbi.1002714-VanTassell1"></a>Van Tassell CP, Smith TP, Matukumalli LK, Taylor JF, Schnabel RD, et al. (2008) SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries. Nat Methods 5: 247–252.
        doi:
        <a href="http://dx.doi.org/10.1038/nmeth.1185">10.1038/nmeth.1185</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755219">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Li2" id="pcbi.1002714-Li2"></a>Li H, Durbin R (2009) Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25: 1754–1760.
        doi:
        <a href="http://dx.doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755221">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Langmead1" id="pcbi.1002714-Langmead1"></a>Langmead B, Trapnell C, Pop M, Salzberg SL (2009) Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol 10: R25.
        doi:
        <a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25">10.1186/gb-2009-10-3-r25</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755223">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Li3" id="pcbi.1002714-Li3"></a>Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, et al. (2009) The Sequence alignment/map (SAM) format and SAMtools. Bioinformatics 25: 2078–2079. <a class="find" href="/article/findArticle.action?citedArticleID=5755225">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Larson1" id="pcbi.1002714-Larson1"></a>Larson DE, Harris CC, Chen K, Koboldt DC, Abbott TE, et al. (2012) SomaticSniper: identification of somatic point mutations in whole genome sequencing data. Bioinformatics 28: 311–317.
        doi:
        <a href="http://dx.doi.org/10.1093/bioinformatics/btr665">10.1093/bioinformatics/btr665</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755227">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Breiman1" id="pcbi.1002714-Breiman1"></a>Breiman L (2001) Statistical Modeling: The Two Cultures. Statist Sci 16: 199–231.
        doi:
        <a href="http://dx.doi.org/10.1214/ss/1009213726">10.1214/ss/1009213726</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755229">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Fawcett1" id="pcbi.1002714-Fawcett1"></a>Fawcett T (2006) An introduction to ROC analysis. Pattern Recogn Lett 27: 861–874.
        doi:
        <a href="http://dx.doi.org/10.1016/j.patrec.2005.10.010">10.1016/j.patrec.2005.10.010</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755231">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Ajay1" id="pcbi.1002714-Ajay1"></a>Ajay SS, Parker SC, OzelAbaan H, Fuentes Fajardo KV, Margulies EH (2011) Accurate and comprehensive sequencing of personal genomes. Genome Res 21: 1498–1505.
        doi:
        <a href="http://dx.doi.org/10.1101/gr.123638.111">10.1101/gr.123638.111</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755233">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Gnirke1" id="pcbi.1002714-Gnirke1"></a>Gnirke A, Melnikov A, Maguire J, Rogov P, LeProust EM, et al. (2009) Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing. Nat Biotechnol 27: 182–189.
        doi:
        <a href="http://dx.doi.org/10.1038/nbt.1523">10.1038/nbt.1523</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755235">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Mouse1" id="pcbi.1002714-Mouse1"></a>Mouse Genome Sequencing Consortium (2002) Initial sequencing and comparative analysis of the mouse genome. Nature 420: 520–562. <a class="find" href="/article/findArticle.action?citedArticleID=5755237">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Sherry1" id="pcbi.1002714-Sherry1"></a>Sherry ST, Ward MH, Kholodov M, Baker J, Phan L, et al. (2009) dbSNP: the NCBI database of genetic variation. Nucleic Acids Res 29: 308–311.
        doi:
        <a href="http://dx.doi.org/10.1093/nar/29.1.308">10.1093/nar/29.1.308</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755239">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Dreszer1" id="pcbi.1002714-Dreszer1"></a>Dreszer TR, Karolchik D, Zweig AS, Hinrichs AS, Raney BJ, et al. (2011) The UCSC Genome Browser database: extensions and updates 2011. Nucleic Acids Res 39: 876–882.
        doi:
        <a href="http://dx.doi.org/10.1093/nar/gkr1055">10.1093/nar/gkr1055</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755241">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Pruitt1" id="pcbi.1002714-Pruitt1"></a>Pruitt KD, Tatusova T, Maglott DR (2005) NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res 33: 501–504. <a class="find" href="/article/findArticle.action?citedArticleID=5755243">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-Mortazavi1" id="pcbi.1002714-Mortazavi1"></a>Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B (2008) Mapping and quantifying mammalian transcriptomes by rna-seq. Nat Methods 5: 621–628.
        doi:
        <a href="http://dx.doi.org/10.1038/nmeth.1226">10.1038/nmeth.1226</a>.
       <a class="find" href="/article/findArticle.action?citedArticleID=5755245">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002714-R1" id="pcbi.1002714-R1"></a>R Development Core Team (2010) R: A language and environment for statistical computing. Vienna: R Foundation for Statistical Computing. </li><li xpathLocation="noSelect"><a name="pcbi.1002714-Liaw1" id="pcbi.1002714-Liaw1"></a>Liaw A, Wiener M (2002) Classification and Regression by randomForest. R News 2: 18–22. <a class="find" href="/article/findArticle.action?citedArticleID=5755249">
                      Find this article online
                    </a></li></ol></div>

</div></div>
<div style="display:none">
<div dojoType="ambra.widget.RegionalDialog" id="AnnotationDialog" style="padding:0;margin:0;">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Post Your Note (For Public Viewing)</span></h5>
      <div class="posting pane">
        <form name="createAnnotation" id="createAnnotation" method="post" action="">
          <input type="hidden" name="target" value="info:doi/10.1371/journal.pcbi.1002714" />
          <input type="hidden" name="startPath" value="" />
          <input type="hidden" name="startOffset" value="" />
          <input type="hidden" name="endPath" value="" />
          <input type="hidden" name="endOffset" value="" />
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="false" />
          <input type="hidden" name="noteType" id="noteType" value="" />
          <fieldset>
            <legend>Compose Your Note</legend>
            <span id="submitMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td>
                  <label for="cNoteType">This is a </label><select name="cNoteType" id="cNoteType"><option value="note">note</option><option value="correction">correction</option></select>
                  <span id="cdls" style="visibility:hidden;margin-left:0.3em; white-space:nowrap;"><a href="/static/commentGuidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F?target=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002714#corrections">What are corrections?</a></span>
                  <label for="cTitle"><span class="none">Enter your note title</span><!-- error message text <em>A title is required for all public notes</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your note title..." class="title" alt="Enter your note title..." />
                  <label for="cArea"><span class="none">Enter your note</span><!-- error message text <em>Please enter your note</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your note..." alt="Enter your note...">Enter your note...</textarea>
                  <input type="hidden" name="isPublic" value="true" />
                </td>
                <td>&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" checked="checked" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." alt="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td colspan="3" class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel"/>
                  <input type="button" value="Submit" title="Click to post your note publicly" id="btn_post" class="primary"/>
                </td>
              </tr>
              <tr>
                <td colspan="3">
                  <p>Notes and Corrections can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p>
                </td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
    <div class="tip" id="dTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialog" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="caTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      Please follow our <a href="/static/commentGuidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">guidelines for notes and comments</a> and review our <a href="/static/competing.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
      <ul>
        <li>Remarks that could be interpreted as allegations of misconduct</li>
        <li>Unsupported assertions or statements</li>
        <li>Inflammatory or insulting language</li>
      </ul>
      <form name="contextActionForm" id="contextActionForm" class="clearfix buttons" method="post" action="">
        <input type="button" name="Continue" value="Continue" id="ContextActionDialogContinueButton" onmouseup="ambra.displayAnnotationContext.startComment(event);" title="Add a note to this text" class="primary"/>
        <input type="button" name="Cancel" value="Cancel" id="ContextActionDialogCancelButton" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);" title="Close this Window"/>
      </form>
    </div>
    <div class="tip" id="caTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogNotLogged" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canlTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You must be logged in to add a note to an article.
      You may log in by <a onmousedown="ambra.displayAnnotationContext.disconnect(event);" href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002714">clicking here</a> or <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">cancel this note</a>.
    </div>
    <div class="tip" id="canlTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canBDTipu"></div>
    <div class="contextActionContent">
      <h5 class="annotation icon"><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot annotate this area of the document. <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canBDTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadRangeSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canbrTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot create an annotation that spans different sections of the document; please adjust your selection.<br/>
      <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canbrTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialog" style="padding:0;margin:0;">
  <div class="dialog preview">
    <div class="tipu" id="cTipu"></div>
    <div class="btn close" id="btn_close" title="Click to close"><a title="Click to close">Close</a></div>
    <div id="cmtContainer" class="comment">
      <h6 id="viewCmtTitle"></h6>
      <div class="detail" id="viewCmtDetail"></div>
      <div class="contentwrap" id="viewComment"></div>
      <div class="contentwrap" id="viewCIStatement"></div>
      <div class="detail" id="viewLink">
        <!--<a href="#" class="commentary icon" title="Click to view full thread and respond">View all responses</a>
        <a href="#" class="respond tooltip" title="Click to respond to this posting">Respond to this</a>-->
      </div>
    </div>
    <div class="tip" id="cTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialogMultiple" style="padding:0;margin:0;">
  <div class="dialog multiple preview">
    <div class="tipu" id="mTipu"></div>
    <div class="btn close" id="btn_close_multi" title="Click to close"><a title="Click to close">Close</a></div>
    <ol id="multilist"></ol>
    <br/>
    <div id="multidetail"></div>
    <div class="tip" id="mTip"></div>
  </div>
</div>


<div dojoType="dijit.Dialog" id="Rating">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Rate This Article</span></h5>
      <div class="instructions">Please follow our <a  href="/static/ratingGuidelines.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">guidelines for rating</a> and review our <a href="/static/competing.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
        <ol>
          <li>Remarks that could be interpreted as allegations of misconduct</li>
          <li>Unsupported assertions or statements</li>
          <li>Inflammatory or insulting language</li>
        </ol>
      </div>
      <div class="posting pane">
        <form name="ratingForm" id="ratingForm" method="post" action="">
          <input type="hidden" name="articleURI" value="info:doi/10.1371/journal.pcbi.1002714" />  
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="" />
          <fieldset>
            <legend>Compose Your Annotation</legend>
            <span id="submitRatingMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td rowspan="2">
                  <label for="insight">Insight</label>
                  <ul class="star-rating rating edit" title="Rate insight" id="rateInsight">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Bland" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'insight', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Profound" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="insight" title="insight" value="" />
                  <label for="reliability">Reliability</label>
                  <ul class="star-rating rating edit" title="Rate reliability" id="rateReliability">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Tenuous" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'reliability', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Unassailable" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="reliability" title="reliability" value="" />
                  <label for="style">Style</label>
                  <ul class="star-rating rating edit" title="Rate style" id="rateStyle">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Crude" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'style', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Elegant" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="style" title="style" value="" />
                  <label for="cTitle"><span class="none">Enter your comment title</span><!-- error message text <em>A title is required for all public annotations</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your comment title..." class="title" alt="Enter your comment title..." />
                  <label for="cArea"><span class="none">Enter your comment</span><!-- error message text <em>Please enter your annotation</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your comment..." alt="Enter your comment...">Enter your comment...</textarea>
                </td>
                <td rowspan="2">&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." title="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel_rating"/>
                  <input type="button" value="Submit" title="Click to post your annotation publicly" id="btn_post_rating" class="primary"/>
                </td>
              </tr>
              <tr>
                  <td colspan="3"><p>Ratings can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p></td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
<div dojoType="ambra.widget.LoadingCycle" id="LoadingCycle" class="loadingCycler">
  <img src="/images/loading.gif" width="58" height="58" title="Loading..." />
</div>
<div dojoType="dijit.Dialog" id="LightBox">

    <!-- start main window wrapper-->
    <div class="figure-window-wrapper">

      <!--start film strip Wrapper-->
      <div class="figure-window-nav-wrapper">
        <div id="figure-window-nav"></div>
      </div><!--end figure-window-nav-wrapper-->

      <!-- start figure container-->
      <div class="figure-window-container">

        <!-- start/end figure viewer-->
        <div id="figure-window-viewer">
          <img id="figure-window-img" class="large"/>
        </div>

        <!-- start/end figure doi-->
        <div id="figure-window-doi"></div>

        <!--start figure title-box-->
        <div id="figure-window-title-box" class="figure-window-title-less">
          <span id="figure-window-title"></span>
          <div id="figure-window-more" class="figure-window-more" onclick="return ambra.lightBox.showMoreOrLess();"></div>
          <div id="figure-window-description"></div>
        </div><!--end figure title-box-->

        <!--start figure download-->
        <div class="figure-window-download">
          <ul>
            <li class="download icon"><strong>Download:</strong>
              <a id="figure-window-ppt" title="Click to download PowerPoint slide of this image">PowerPoint
                slide</a> |
              <a id="figure-window-large" title="Click to download a larger version of this image">larger image
                (<span id="figure-window-large-size"></span>)</a> |
              <a id="figure-window-tiff" title="Click to download a original version of this image">original image
                (<span id="figure-window-tiff-size"></span>)</a>
            </li>
          </ul>
        </div><!--end figure download-->

        <!--start/end of close, context, next and previous button-->
        <div class="figure-window-close" onclick="return ambra.lightBox.hide();"></div>
        <div class="figure-window-context" onclick="return ambra.lightBox.showInContext();"></div>
        <div id="figure-window-previous" onclick="return ambra.lightBox.showPrevious();"></div>
        <div id="figure-window-next" onclick="return ambra.lightBox.showNext();"></div>

      </div><!--end figure-window-container-->

    </div><!--end figure-window-wrapper-->

</div><!--end dijit.Dialog-->
</div>
</div>
<!-- end : main contents -->

</div>
<!-- end : container -->	

<!-- begin : footer -->
<div id="ftr">
  <p><span>All site content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License 2.5" tabindex="200">Creative Commons Attribution License</a>.</span></p>
  <ul>
    <li><a href="/static/privacy.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" title="PLOS Privacy Statement" tabindex="501">Privacy Statement</a></li>
    <li><a href="/static/terms.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" title="PLOS Terms of Use" tabindex="502">Terms of Use</a></li>
    <li><a href="http://www.plos.org/advertise/" title="Advertise With PLOS" tabindex="503">Advertise</a></li>
    <li><a href="http://www.plos.org/about/media-inquiries/" title="PLOS Embargo Policy" tabindex="504">Media Inquiries</a></li>
    <li><a href="http://www.plos.org/publish/order-reprints/" title="PLOS in Print" tabindex="505">PLOS in Print</a></li>
    <li><a href="/static/sitemap.action;jsessionid=10B3F584BF488C3BC70268A3ECAE261F" title="Site Map" tabindex="506">Site Map</a></li>
    <li><a href="http://www.plos.org" title="PLOS.org" tabindex="507">PLOS.org</a></li>
  </ul>
  <div class="powered">
  <ul>
    <li><a href="/static/releaseNotes.action" title="Ambra | Release Notes">Ambra 2.4.9</a></li>
    <li>Managed Colocation provided by <a href="http://www.isc.org/" title="Internet Systems Consortium">Internet Systems Consortium</a>.</li>
  </ul>
  </div>
</div>
<!-- end : footer -->
<script type="text/javascript">
  var _namespace="";
    var loggedIn = false;

  var almHost = "http://alm.plos.org";
  var solrHost = "http://api.plos.org/search";
  var pubGetHost = "http://pubget.com/widgetizer/links_hash";
  var appContext = "";

  // Safari v3.1.1 "console.debug" issue (http://trac.dojotoolkit.org/ticket/6849) workaround
  if (/3[\.0-9]+ Safari/.test(navigator.appVersion)) {
    window.console = {
      origConsole: window.console,
      log: function(s){ this.origConsole.log(s); },
      info: function(s){ this.origConsole.info(s); },
      error: function(s){ this.origConsole.error(s); },
      warn: function(s){ this.origConsole.warn(s); }
    };
  }

  var djConfig = {
    // don't debug for IE - as dojo's firebug lite module is error prone in IE
    isDebug: false,
    parseOnLoad: true
  };

  var bamGAID = 'UA-338393-1';
  var bamGAVPV = location.pathname + location.search + location.hash;
</script>
<script type="text/javascript" src="/javascript/dojo/dojo/dojo.js?v=pzlwVCPPo+oJC12upPgJrJ5EG_k"></script>
<script type="text/javascript" src="/javascript/dojo/dojo/ambra.js?v=Y8JM0CVD9YwjC99M_jvbcRTiGhY"></script>
<script type="text/javascript" src="/javascript/init_global.js?v=fF03pfG3r+Nkj_czZ8ORg8o5sNM"></script>
<script type="text/javascript" src="/javascript/init_article.js?v=qzBD_a7nKbeMM2xNUtB3vckN+Zc"></script>
<script type="text/javascript" src="/javascript/init_ratings.js?v=seKykuS0X33CQGS7Lj+8FtBy5Uw"></script>
<script type="text/javascript" src="/javascript/init_lightBox.js?v=vJRLPy7RRX8Zp31Ogcm3i+TA4f8"></script>
<script type="text/javascript" src="/javascript/init_article_body.js?v=YgFei2xCi50CahwaPbSsxUxUsmw"></script>
<script type="text/javascript" src="/javascript/init_article_rhc.js?v=djtbTMOagorKE6ZPrkCwufAfWWI"></script>
<script type="text/javascript" src="/javascript/alm.js?v=1tH9JZ8uDtXspXssGJDr8O+jBOs"></script>
<script type="text/javascript" src="/javascript/reporting/articleViewsCumulative.js?v=RQ+gCrcAl0dBeLbmokxgQZTGYYA"></script>
<script type="text/javascript" src="/javascript/check_twitter.js?v=Hc0sH1W9xIu0qhqxWPD4c1u50K4"></script>
<script type="text/javascript" src="/javascript/init_googleAnalytics.js?v=yCD4h9PHGI0dsefmuWDlMn7pHog"></script>

</body>
</html>

