

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article">

<head>
   <title>PLOS Computational Biology: Weakly Circadian Cells Improve Resynchrony</title>



<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9" />

<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
<link rel="home" title="home" href="/home.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" />
<link rel="alternate" type="application/rss+xml"
  title="PLOS Computational Biology: New Articles"
  href="http://www.ploscompbiol.org/article/feed" />

<link rel="pingback" href="http://www.ploscompbiol.org/pingback" />


<!-- global_css.ftl -->
<link rel="stylesheet" type="text/css" media="screen" href="/css/dojo/dijit/themes/tundra/tundra.css?v=zwugcxLdpBWmb72qqlHgGyeNAGM" />
<link rel="stylesheet" type="text/css" media="all" href="/css/star_rating.css?v=Mm3HyaUPUahAJAq6xG5W3tdF7Hk" />
<link rel="stylesheet" type="text/css" media="all" href="/css/rating.css?v=AOT9rozE9sYVjGTdeiyxHf18Ykw" />
<link rel="stylesheet" type="text/css" media="all" href="/css/screen.css?v=KBNFqBmhD635U0NgiM5tf9dtxRE" />
<link rel="stylesheet" type="text/css" media="all" href="/css/iepc.css?v=FIjyR83BhSovq_5yNVk1Sih60AI" />
<link rel="stylesheet" type="text/css" media="all" href="/css/journal.css?v=SiREXztjNSUsOyrlkE9sv028Y+0" />
<link rel="stylesheet" type="text/css" media="all" href="/css/lightBox.css?v=sDgsUXamFoYAakEshdj0YWrZiiQ" />
<link rel="stylesheet" type="text/css" media="print" href="/css/print.css?v=0T+LTV7Gd0jtHEhqEvBfSrmHUGg" />

<!--chartbeat --> 
<script type="text/javascript">var _sf_startpt=(new Date()).getTime()</script>


<!--[if IE]><![endif]-->
<meta name="description" content="PLOS Computational Biology is an open-access" />

<meta name="keywords" content="plos computational biology" />

  <meta name="citation_publisher" content="Public Library of Science" />
  <meta name="citation_doi" content="10.1371/journal.pcbi.1002787" />
    <meta name="citation_title" content="Weakly Circadian Cells Improve Resynchrony"/>
    <meta itemprop="name" content="Weakly Circadian Cells Improve Resynchrony"/>

      <meta name="citation_author" content="Alexis B. Webb" />
          <meta name="citation_author_institution" content="Department of Biology, Washington University, St. Louis, Missouri, United States of America" />
      <meta name="citation_author" content="Stephanie R. Taylor" />
          <meta name="citation_author_institution" content="Department of Computer Science, Colby College, Waterville, Maine, United States of America" />
      <meta name="citation_author" content="Kurt A. Thoroughman" />
          <meta name="citation_author_institution" content="Department of Biomedical Engineering, Washington University, St. Louis, Missouri, United States of America" />
      <meta name="citation_author" content="Francis J. Doyle" />
          <meta name="citation_author_institution" content="Department of Chemical Engineering, University of California Santa Barbara, Santa Barbara, California, United States of America" />
      <meta name="citation_author" content="Erik D. Herzog" />
          <meta name="citation_author_institution" content="Department of Biology, Washington University, St. Louis, Missouri, United States of America" />

    <meta name="citation_date" content="2012/11/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pcbi.1002787.pdf" />

    <meta name="citation_journal_title" content="PLOS Computational Biology" />
    <meta name="citation_firstpage" content="e1002787"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="8"/>
    <meta name="citation_issn" content="1553-7358"/>

    <meta name="citation_journal_abbrev" content="PLoS Comput Biol" />

      <meta name="citation_reference" content="citation_title=Molecular bases for circadian clocks; citation_volume=96; citation_number=1; citation_pages=271-290; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Transplanted suprachiasmatic nucleus determines circadian period; citation_volume=247; citation_number=2; citation_pages=975-978; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Individual neurons dissociated from rat suprachiasmatic nucleus express independently phased circadian firing rhythms; citation_volume=14; citation_number=3; citation_pages=697-706; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Cellular construction of a circadian clock: period determination in the suprachiasmatic nuclei; citation_volume=91; citation_number=4; citation_pages=855-860; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Clock controls circadian period in isolated suprachiasmatic nucleus neurons; citation_volume=1; citation_number=5; citation_pages=708-713; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Synchronization of cellular clocks in the suprachiasmatic nucleus; citation_volume=302; citation_number=6; citation_pages=1408-1412; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Coordination of circadian timing in mammals; citation_volume=418; citation_number=7; citation_pages=935-941; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genetics of the human circadian clock; citation_volume=74; citation_number=8; citation_pages=231-247; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Multiscale complexity in the mammalian circadian clock; citation_volume=20; citation_number=9; citation_pages=626-633; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Intrinsic, nondeterministic circadian rhythm generation in identified mammalian neurons; citation_volume=106; citation_number=10; citation_pages=16493-16498; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Emergence of noise-induced oscillations in the central circadian pacemaker; citation_volume=8; citation_number=11; citation_pages=e1000513; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Intrinsic regulation of spatiotemporal organization within the suprachiasmatic nucleus; citation_volume=6; citation_number=12; citation_pages=e15869; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Characterization of orderly spatiotemporal patterns of clock gene activation in mammalian suprachiasmatic nucleus; citation_volume=33; citation_number=13; citation_pages=1851-1865; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Gradients in the circadian expression of Per1 and Per2 genes in the rat suprachiasmatic nucleus; citation_volume=15; citation_number=14; citation_pages=1153-1162; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The biological clock nucleus: a multiphasic oscillator network regulated by light; citation_volume=23; citation_number=15; citation_pages=8070-8076; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A diversity of paracrine signals sustains molecular circadian cycling in suprachiasmatic nucleus circuits; citation_volume=108; citation_number=16; citation_pages=14306-11; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Visualizing jet lag in the mouse suprachiasmatic nucleus and peripheral circadian timing system; citation_volume=29; citation_number=17; citation_pages=171-80; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Toward a detailed computational model for the mammalian circadian clock; citation_volume=100; citation_number=18; citation_pages=7051-7056; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stochastic simulation of the mammalian circadian clock; citation_volume=102; citation_number=19; citation_pages=321-324; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modeling the mammalian circadian clock: Sensitivity analysis and multiplicity of oscillatory mechanisms; citation_volume=230; citation_number=20; citation_pages=541-562; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A Molecular Model for Intercellular Synchronization in the Mammalian Circadian Clock; citation_volume=92; citation_number=21; citation_pages=3792-3803; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Small-World Network Models of Intercellular Coupling Predict Enhanced Synchronization in the Suprachiasmatic Nucleus; citation_volume=24; citation_number=22; citation_pages=243-254; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Spontaneous synchronization of coupled circadian oscillators; citation_volume=89; citation_number=23; citation_pages=120-129; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Synchronization-Induced Rhythmicity of Circadian Oscillators in the Suprachiasmatic Nucleus; citation_volume=3; citation_number=24; citation_pages=e68; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A multicellular model for differential regulation of circadian signals in the core and shell regions of the suprachiasmatic nucleus; citation_volume=288; citation_number=25; citation_pages=44-56; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Effect of network architecture on synchronization and entrainment properties of the circadian oscillations in the suprachiasmatic nucleus; citation_volume=8; citation_number=26; citation_pages=e1002419; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Transcription-based oscillator model for light-induced splitting as antiphase circadian gene expression in the suprachiasmatic nuclei; citation_volume=27; citation_number=27; citation_pages=79-90; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Quantification of circadian rhythms in single cells; citation_volume=5; citation_number=28; citation_pages=e1000580; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=How to achieve fast entrainment? The timescale to synchronization; citation_volume=4; citation_number=29; citation_pages=e7057; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=Persistent cell-autonomous circadian oscillations in fibroblasts revealed by six-week single-cell imaging of PER2:: LUC bioluminescence; citation_volume=7; citation_number=30; citation_pages=e33334; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=Sensitivity Measures for Oscillating Systems: Application to Mammalian Circadian Gene Network; citation_volume=53; citation_number=31; citation_pages=177-188; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Velocity response curves support the role of continuous entrainment in circadian clocks; citation_volume=25; citation_number=32; citation_pages=138-149; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Coupling governs entrainment range of circadian clocks; citation_volume=6; citation_number=33; citation_pages=438; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Temperature as a universal resetting cue for mammalian circadian oscillators; citation_volume=330; citation_number=34; citation_pages=379-385; citation_date=2010; " />
      <meta name="citation_reference" content="citation_title=Period2:: luciferase real-time reporting of circadian dynamics reveals persistent circadian oscillations in mouse peripheral tissues; citation_volume=101; citation_number=35; citation_pages=5339-5346; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structure and visualization of high-dimensional conductance spaces; citation_volume=96; citation_number=36; citation_pages=891-905; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Vasoactive intestinal polypeptide mediates circadian rhythmicity and synchrony in mammalian clock neurons; citation_volume=8; citation_number=37; citation_pages=476-483; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Separate oscillating cell groups in mouse suprachiasmatic nucleus couple photoperiodically to the onset and end of daily activity; citation_volume=104; citation_number=38; citation_pages=7664-7669; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Heterogeneity of rhythmic suprachiasmatic nucleus neurons: Implications for circadian waveform and photoperiodic encoding; citation_volume=100; citation_number=39; citation_pages=15994-15999; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Exploring spatiotemporal organization of SCN circuits; citation_volume=72; citation_number=40; citation_pages=527-541; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Intercellular Coupling Confers Robustness against Mutations in the SCN Circadian Clock Network; citation_volume=129; citation_number=41; citation_pages=605-616; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Effects of vasoactive intestinal peptide genotype on circadian gene expression in the suprachiasmatic nucleus and peripheral organs; citation_volume=26; citation_number=42; citation_pages=200-209; citation_date=2011; " />
      <meta name="citation_reference" content="citation_title=Reversible Protein Phosphorylation Regulates Circadian Rhythms; citation_volume=72; citation_number=43; citation_pages=413-420; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Global parameter search reveals design principles of the mammalian circadian clock; citation_volume=2; citation_number=44; citation_pages=22; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Gates and oscillators: a network model of the brain clock; citation_volume=18; citation_number=45; citation_pages=339-350; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gates and Oscillators II: Zeitgebers and the Network Model of the Brain Clock; citation_volume=22; citation_number=46; citation_pages=14-25; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Bioluminescence imaging of period1 gene expression in utero; citation_volume=6; citation_number=47; citation_pages=68-72; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Circadian system from conception till adulthood; citation_volume=199; citation_number=48; citation_pages=83-103; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=The rabbit pup, a natural model of nursing-anticipatory activity; citation_volume=30; citation_number=49; citation_pages=1697-1706; citation_date=2009; " />
      <meta name="citation_reference" content="citation_title=The chi square periodogram: its utility for analysis of circadian rhythms; citation_volume=72; citation_number=50; citation_pages=131-160; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Quantitative analysis of Drosophila period gene transcription in living animals; citation_volume=12; citation_number=51; citation_pages=204-217; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=WAVOS: a MATLAB toolkit for wavelet analysis and visualization of oscillatory systems; citation_volume=5; citation_number=52; citation_pages=163; citation_date=2012; " />
      <meta name="citation_reference" content="citation_title=From Kuramoto to Crawford: exploring the onset of synchronization in populations of coupled oscillators; citation_volume=143; citation_number=53; citation_pages=1-20; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Wavelet measurement suggests cause of period instability in mammalian circadian neurons; citation_volume=26; citation_number=54; citation_pages=353-362; citation_date=2011; " />



<!--
<rdf:RDF xmlns="http://web.resource.org/cc/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<Work rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787">
   <license rdf:resource="http://creativecommons.org/licenses/by/2.5/" />
</Work>
<License rdf:about="http://creativecommons.org/licenses/by/2.5/">
   <permits rdf:resource="http://web.resource.org/cc/Reproduction" />
   <permits rdf:resource="http://web.resource.org/cc/Distribution" />
   <requires rdf:resource="http://web.resource.org/cc/Notice" />
   <requires rdf:resource="http://web.resource.org/cc/Attribution" />
   <permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" />
</License>
<rdf:Description
     rdf:about="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787"
     dc:identifier="http://www.ploscompbiol.org/article/fetchArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787"
     dc:title="PLOS Computational Biology: Weakly Circadian Cells Improve Resynchrony"
       trackback:ping="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787/trackback;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3"
     />
</rdf:RDF>

-->
</head>
<body>
<span property="dc:language" content="en"></span>
<!-- begin : container -->
<div id="container">
  <!-- begin : top banner external ad space -->
  <div id="topBanner">


<!-- begin : left banner slot -->
<div class="left">
  <iframe id='affe494b' name='affe494b'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=84&amp;cb=6727'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=affe494b&amp;cb=5200'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=84&amp;cb=4729&amp;n=affe494b'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : left banner slot -->
<!-- begin : right banner slot -->
<div class="right">
  <iframe id='af654bd5' name='af654bd5'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=85&amp;cb=5414'
    frameborder='0' scrolling='no' width='468' height='60'><a
    href='http://ads.plos.org/www/delivery/ck.php?n=af654bd5&amp;cb=8021'
    target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=85&amp;cb=6120&amp;n=af654bd5'
    border='0' alt='' /></a></iframe>
</div>
<!-- end : right banner slot -->

  </div>
  <!-- end : top banner external ad space -->

  <!-- begin : header -->
  <div id="hdr" class="login">

  <!-- begin : logo -->
  <div id="logo" title="PLOS Computational Biology"><a href="/home.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" title="PLOS Computational Biology"><span>PLOS Computational Biology</span></a></div>
  <!-- end : logo -->
  <!-- begin : user controls -->





  <div id="user">
    <div>
      <ul>
        <li><a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002787" class="feedback"><strong>Login</strong></a> | </li>
        <li><a href="https://register.plos.org:443/ambra-registration/register.action">Create Account</a> | </li>
        <li class="feedback"><a href="/feedbackCreate.action?page=%252Farticle%252FfetchArticle.action%253FarticleURI%253Dinfo%25253Adoi%25252F10.1371%25252Fjournal.pcbi.1002787" title="Send us your feedback">Feedback</a></li>
      </ul>
    </div>
  </div>


  <!-- end : user controls -->
  <!-- begin search links -->
  <ul id="links"><li class="browse"><a href="/article/browse.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?field=date" title="Browse Articles">Browse</a></li><li class="rss"><a href="/static/rssFeeds.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" title="RSS Feeds">RSS</a></li></ul>
  <!-- end : search links -->
  <!-- begin : dashboard -->
  <div id="db">
    <form name="searchForm" action="/search/simpleSearch.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" method="get">
      <input type="hidden" name="from" value="globalSimpleSearch"/>
      <input type="hidden" name="filterJournals" value="PLoSCompBiol">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="wrap"><input id="search" type="text" name="query" value="Search articles..." onfocus="if(this.value=='Search articles...')value='';" onblur="if(this.value=='')value='Search articles...';" class="searchField" alt="Search articles..."/></div>
        <input src="/images/search_btn1.gif" onclick="submit();" value="ftsearch" alt="SEARCH" tabindex="3" class="button" type="image" />
      </fieldset>
    </form>
    <form name="gasf" action="/search/advancedSearch.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" method="get">
<input type="hidden" name="noSearchFlag" value="true" id="noSearchFlag"/><input type="hidden" name="query" value="" id="query"/>    </form>
    <a id="advSearch" href="#" onclick="if(document.searchForm.query.value!='Search articles...')document.gasf.query.value=document.searchForm.query.value;document.gasf.submit();return false;">Advanced Search</a>
  </div>
  <!-- end : dashboard -->
  <!-- begin : navigation -->
  <ul id="nav">
    <li><a href="/home.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" tabindex="101">Home</a></li>
     <!-- This is a temporary action to link to th static toc page -->
    <li><a href="/static/browse.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" tabindex="102">Browse Articles</a>
        <ul>
          <li><a href="/article/browseIssue.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?field=issue">Current Issue</a></li> <!-- Assuming dynamic TOC is in place -->
          <li><a href="/article/browseVolume.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?field=volume">Journal Archive</a></li> 
          <li><a href="/article/browse.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?field=date">By Publication Date</a></li>
          <li><a href="/article/browse.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">By Subject</a></li>
          <li><a href="http://www.ploscollections.org/static/pcbiCollections.action">Collections</a></li>
        </ul>
    </li>
    <li><a href="/static/about.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" tabindex="103">About</a>
        <ul>
          <li><a href="/static/information.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Journal Information</a></li>
          <li><a href="/static/edboard.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Editorial Board</a></li>
          <li><a href="/static/eic.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Editors-in-Chief</a></li>
          <li><a href="/static/almInfo.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Article-Level Metrics</a></li>
          <li><a href="/static/license.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Open-Access License</a></li>
          <li><a href="/static/contact.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Contact Us</a></li>
        </ul>
      </li>
    <li><a href="/static/users.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" tabindex="104">For Readers</a>
        <ul>
          <li><a href="/static/commentGuidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Guidelines for Notes, Comments, and Corrections</a></li>
          <li><a href="/static/ratingGuidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Guidelines for Rating</a></li>
          <li><a href="/static/help.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Help Using This Site</a></li>
          <li><a href="/static/downloads.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Media Downloads</a></li>
          <li><a href="/static/sitemap.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Site Map</a></li>
        </ul>
      </li>
    <li><a href="/static/authors.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" tabindex="105">For Authors and Reviewers</a>
        <ul>
          <li><a href="/static/policies.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Editorial and Publishing Policies</a></li>
          <li><a href="/static/guidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Author Guidelines</a></li>
          <li><a href="/static/figureGuidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Figure and Table Guidelines</a></li>
          <li><a href="/static/checklist.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Submit Your Paper</a></li>
          <li><a href="/static/reviewerGuidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">Reviewer Guidelines</a></li>
	   <li><a href="http://www.plos.org/journals/index.php">Submit to Other PLOS Journals</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://www.plos.org" title="PLOS" tabindex="110" class="drop">PLOS.org</a>
        <ul>
          <li><a href="http://www.plos.org/oa/index.html" title="Open Access Statement">Open Access</a></li>
          <li><a href="http://www.plos.org/support/donate.php" title="Join PLOS: Show Your Support">Join PLOS</a></li>
          <li><a href="http://blogs.plos.org/" title="PLOS Blogs">PLOS Blogs</a></li>
	  <li><a href="http://www.zazzle.com/plos?rf=238434112415374154" title="PLOS Store">PLOS Store</a></li>
          <li><a href="http://currents.plos.org" title="PLOS Currents">PLOS Currents</a></li>
          <li><a href="http://www.plos.org/connect.html" title="PLOS.org | Stay Connected">Stay Connected</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://hubs.plos.org" tabindex="109">Hubs</a>
        <ul>
          <li><a href="http://hubs.plos.org/biodiversity" title="PLOS Hubs: Biodiversity">Biodiversity</a></li>
          <li><a href="http://clinicaltrials.ploshubs.org" title="PLOS Hub for Clinical Trials">Clinical Trials</a></li>
        </ul>
    </li>
      <li class="journalnav"><a href="http://www.plosjournals.org" tabindex="108">Journals</a>
        <ul>
          <li><a href="http://www.plosbiology.org/" title="PLOSBiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org/" title="PLOSMedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org/" title="PLOSCompBiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://www.plosgenetics.org/" title="PLOSGenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org/" title="PLOSPathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org/" title="PLOSONE.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org/" title="PLOSNTDs.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </li>
    </ul>
  <!-- end : navigation -->
  </div>
  <!-- end : header --> 



<!-- begin : main content -->
<div id="content" class="article" style="visibility:visible;">

<!-- begin : right hand column -->

<div id="rhc" xpathLocation="noDialog">

  <div id="download" class="rhcBox_type1">
    <div class="wrap">
      <ul>
        <li class="download icon"><strong>Download:</strong>
          <a href="/article/fetchObjectAttachment.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787&amp;representation=PDF" title="Download article PDF">PDF</a> |
          <a href="/article/citationList.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787" title="Download citations">Citation</a> |
          <a href="/article/fetchObjectAttachment.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?uri=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787&amp;representation=XML" title="Download Article XML">XML</a>
        </li>
        <li class="print icon"><a href="#" onclick="window.print();return false;" title="Print Article"><strong>Print article</strong></a></li>
        <li class="reprint icon"><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=3&doi=10.1371/journal.pcbi.1002787&volume=&issue=&title=Weakly Circadian Cells Improve Resynchrony&author_name=Alexis%20B.%20Webb%2C%20Stephanie%20R.%20Taylor%2C%20Kurt%20A.%20Thoroughman%2C%20Francis%20J.%20Doyle%2C%20Erik%20D.%20Herzog&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a> New &amp; improved!</li>
      </ul>
    </div>
  </div>

  <div id="issues" class="rhcBox_type2">
    <p><strong>Published in the</strong>
    <a href="http://www.ploscompbiol.org/article/browseIssue.action?issue=info%3Adoi%2F10.1371%2Fissue.pcbi.v08.i11" title="Browse the Open-Access Issue">November 2012 Issue of <em>PLOS Computational Biology</em></a></p>
  </div>

  <div id="impact" class="rhcBox_type2">
    <h6>Metrics <a href="/static/help.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3#articleMetrics" class="replaced" id="info" title="More information"><span>info</span></a></h6>

    <div class="wrap">
      <img id="rhcDataSpinner" src="/images/loading_small.gif" class="loading" />
      <div id="totalDataInRHC"></div>
      <div id="pubDateInRHC"></div>
      <div id="relatedCitesInRHC"></div>
    </div>

    <div class="more clearfix"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" title="More information">More</a></div>
  </div>

  <div id="related" class="rhcBox_type2">
    <h6>Related Content</h6>
    <dl class="related">
      <dt>Related Articles on the Web</dt>
      <dd><a href="http://scholar.google.com/scholar?hl=en&lr=&q=related:http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002787&btnG=Search">Google Scholar</a></dd>
      <dd id="pubMedRelatedLIInRHC" style="display:none;"><a id="pubMedRelatedURLInRHC">PubMed</a></dd>
    </dl>
    <div class="more clearfix"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" title="More information">More</a></div>
  </div>

  <div id="share" class="rhcBox_type2">
    <h6>Share this Article <a href="/static/help.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3#socialBookmarkLinks" class="replaced" id="info" title="More information"><span>info</span></a></h6>
    <ul>
      <li class="bookmarklets">

        <a href="http://www.reddit.com/submit?url=http://dx.plos.org/10.1371/journal.pcbi.1002787"
           target="_new"
           id="reddit">
          <img src="http://www.reddit.com/static/spreddit4.gif" alt="submit to reddit" border="0"/> </a>
        <g:plusone size="small" annotation="none" href=http://dx.plos.org/10.1371/journal.pcbi.1002787></g:plusone>
        <script type="text/javascript">
          (function () {
            var po = document.createElement('script');
            po.type = 'text/javascript';
            po.async = true;
            po.src = 'https://apis.google.com/js/plusone.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(po, s);
          })();
        </script>
        <a href="http://www.stumbleupon.com/submit?url=http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787" target="_new"> <img border=0 src="http://cdn.stumble-upon.com/images/16x16_su_solid.gif" alt="StumbleUpon" title="Add to StumbleUpon"></a>
        <script>function fbs_click() {u='http://dx.plos.org/10.1371/journal.pcbi.1002787';t='Weakly%20Circadian%20Cells%20Improve%20Resynchrony';window.open('http://www.facebook.com/sharer.php?u='+encodeURIComponent(u)+'&t='+encodeURIComponent(t),'sharer','toolbar=0,status=0,width=626,height=436');return false;}</script><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002787" onclick="return fbs_click()" target="_blank"><img src="http://static.ak.fbcdn.net/images/share/facebook_share_icon.gif" alt="Facebook" title="Add to Facebook" /></a>       <!-- for mor info, see http://www.facebook.com/share_partners.php -->
        <script type="text/javascript">
          function bookmark_in_connotea(u) {
            a=false; x=window; e=x.encodeURIComponent; d=document;
            w=open('http://www.connotea.org/addpopup?continue=confirm&uri='+e(u),
                'add', 'width=600, height=400, scrollbars, resizable');
            void(x.setTimeout('w.focus()',200));
          }
        </script>
        <a style='cursor: pointer;' onclick='javascript:bookmark_in_connotea("http://dx.plos.org/10.1371/journal.pcbi.1002787");'><img src='/images/icon_connotea_16x16.gif' alt="Connotea" title="Add to Connotea"/></a>
        <a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002787&title=Weakly%20Circadian%20Cells%20Improve%20Resynchrony" target="_new"><img src='/images/icon_citeulike_16x16.gif' alt="CiteULike" title="Add to CiteULike" /></a>
        <a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pcbi.1002787" title="Add to Mendeley" target="_new"><img src="/images/icon_mendeley_16x16.gif" alt="Bibliography"></a>
        <script type="text/javascript">
          var twtTitle  = 'Weakly Circadian Cells Improve Resynchrony';
          var twtUrl    = 'http://dx.plos.org/10.1371/journal.pcbi.1002787';
          var twtTag    = '#PLOSCompBio:'
          var maxLength = 140 - (twtUrl.length + 1);
          if (twtTitle.length > maxLength) {
            twtTitle = twtTitle.substr(0, (maxLength - 17))+'...';  //truncates title to include space for twtTag and ellipsis
          }
          var twtLink = 'http://twitter.com/intent/tweet?text='+encodeURIComponent(twtTag + ' ' + twtTitle + ' ' + twtUrl);
          document.write('<a href="'+twtLink+'" target="_blank"><img src="/images/icon_twitter.gif" border="0" alt="Twitter icon" title="Tweet This!" /></a>');
        </script>

      </li>
      <li class="email icon"><a href="/article/emailArticle.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?articleURI=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787" title="Email this article">Email this article</a></li>
    </ul>
  </div>
  <div id="articleTwitterStream" style="display:none">
    <script src="http://widgets.twimg.com/j/2/widget.js"></script>
    <script>
      new TWTR.Widget({
        version: 2,
        type: 'search',
        search: encodeURIComponent('10.1371/journal.pcbi.1002787'),
        interval: 6000,
        title: '',
        subject: 'From Twitter',
        width: 300,
        height: 300,
        theme: {
          shell: {
            background: '#667893',
            color: '#ffffff'
          },
          tweets: {
            background: '#ffffff',
            color: '#444444',
            links: '#006699'
          }
        },
        features: {
          scrollbar: true,
          loop: false,
          live: true,
          hashtags: true,
          timestamp: true,
          avatars: true,
          toptweets: true,
          behavior: 'all'
        }
      }).render().start();
    </script>
  </div>
</div>
<!-- end : right hand column -->
<form name="articleInfo" id="articleInfo" method="" action="">
<input type="hidden" name="isAuthor" value="true" />
<input type="hidden" name="authorIdList" value="" />
<input type="hidden" name="userIdList" value="" />
<input type="hidden" name="otherIdList" value="" />
<input type="hidden" name="annotationId" value="" />
<input type="hidden" name="isResearchArticle" value="true" />
</form>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.XML" value="112485"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.PDF" value="463925"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g001.TIF" value="719266"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g001.PNG_I" value="70838"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g001.PNG_S" value="19267"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g001.PNG_M" value="160531"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g001.PNG_L" value="263601"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g002.TIF" value="641860"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g002.PNG_I" value="76198"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g002.PNG_S" value="21749"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g002.PNG_M" value="133515"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g002.PNG_L" value="201043"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g003.TIF" value="466084"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g003.PNG_I" value="46597"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g003.PNG_S" value="15647"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g003.PNG_M" value="98738"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.g003.PNG_L" value="155770"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.e001.TIF" value="4374"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.e001.PNG" value="2262"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s001.EPS" value="2409395"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s002.EPS" value="5440055"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s003.EPS" value="3512835"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s004.EPS" value="439523"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s005.EPS" value="456600"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s006.EPS" value="309099"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s007.PDF" value="326045"/>
  <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pcbi.1002787.s008.PDF" value="582388"/>
<div id="articleContainer">

<span about="http://www.plos.org"  typeof="foaf:Organization" property="foaf:name" style="display:none; visibility:hidden;">PLOS</span>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader" xpathLocation="noSelect">
  <p>Open Access</p><p id="articleType">Research Article</p>
</div>

  <h1 xpathLocation="noSelect" property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">Weakly Circadian Cells Improve Resynchrony</h1>
  <div property="dc:description" datatype="" style="display:none; visibility:hidden;">Author SummaryCircadian rhythms are daily, near 24-h oscillations in biological processes that nearly all organisms on Earth experience. Single cells contain a molecular clock that drives circadian rhythms in physiology and, when many cells synchronize in a population, daily behaviors. We hypothesized that small differences in intrinsic cellular properties allow for a diversity of circadian periods and amplitudes across cells. We observed circadian cells and their synchrony before, during, and after limiting communication between cells and then compared their intrinsic properties to their resynchronization behavior. We found that arrhythmic, weakly oscillating, and self-sustained circadian cells rejoined the rhythmic population independent of their cell-intrinsic oscillations. Using a mechanistic computational model of circadian cells, we found that resynchronization could be enhanced by including more weak oscillators or by placing weak oscillators at more connected nodes in the network. We conclude that intrinsic properties (e.g. oscillator weakness and responsiveness) and network structure (e.g. positions of weak oscillators) can independently buffer tissue rhythms from perturbations. This reveals how cellular and network properties impose rules on systems of circadian cells that must achieve synchrony from a desynchronized state, for example during perinatal development or when forced to overcome societal constraints on sleep-wake behavior, such as working early or late shifts.</div>
  <span property="dc:date" content="2012-11-29" datatype="xsd:date" rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pcbi.1002787"></span>
  <span property="dc:subject" content="Computational Biology"></span>
  <span property="dc:subject" content="Neuroscience"></span>
<form action="">
  <input type="hidden" name="journalDisplayName" id="journalDisplayName" value="PLOS Computational Biology" />
  <input type="hidden" name="crossRefPageURL" id="crossRefPageURL" value="/article/crossref/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" />
  <input type="hidden" name="twitterPageURL" id="twitterPageURL" value="/article/twitter/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" />
  <input type="hidden" name="metricsTabURL" id="metricsTabURL" value="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" />
  <input type="hidden" name="doi" id="doi" value="info:doi/10.1371/journal.pcbi.1002787" />
  <input type="hidden" name="articleTitleUnformatted" id="articleTitleUnformatted" value="Weakly%20Circadian%20Cells%20Improve%20Resynchrony" />
  <input type="hidden" name="articlePubDate" id="articlePubDate" value="1354176000000" />
</form>
<div class="horizontalTabs" xpathLocation="noSelect">
  <ul id="tabsContainer">
        <li id="article" class="active"><a href="/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenuBox" xpathLocation="noSelect">
    <div id="articleMenu">
      <div class="wrap">
        <ul>
          <li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" onclick="toggleAnnotation(this, 'public'); return false;" title="Click to turn notes on/off">Hide notes</a></li>
          <li class="discuss icon">
              <a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002787">Make a general comment</a>
          </li>
        </ul>
        <div id="sectionNavTopBox" style="display:none;">
          <p><strong>Jump to</strong></p>
          <div id="sectionNavTop" class="tools"></div>
        </div>
      </div>
    </div>
      <div id="lightBox" xpathLocation="noDialog" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787')">
      </div>
  </div>




<p class="authors" xpathLocation="noSelect"><span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Alexis B. Webb</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#equal-contrib">#</a></sup><sup><a href="#fn1">¤</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Stephanie R. Taylor</span></span><sup><a href="#aff2">2</a></sup><sup><a href="#equal-contrib">#</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Kurt A. Thoroughman</span></span><sup><a href="#aff3">3</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Francis J. Doyle III</span></span><sup><a href="#aff4">4</a></sup>, <span rel="dc:creator"><span class="person" property="foaf:name" typeof="foaf:Person">Erik D. Herzog</span></span><sup><a href="#aff1">1</a></sup></p><p class="affiliations" xpathLocation="noSelect"><a name="aff1" id="aff1"></a><strong>1</strong> Department of Biology, Washington University, St. Louis, Missouri, United States of America, <a name="aff2" id="aff2"></a><strong>2</strong> Department of Computer Science, Colby College, Waterville, Maine, United States of America, <a name="aff3" id="aff3"></a><strong>3</strong> Department of Biomedical Engineering, Washington University, St. Louis, Missouri, United States of America, <a name="aff4" id="aff4"></a><strong>4</strong> Department of Chemical Engineering, University of California Santa Barbara, Santa Barbara, California, United States of America</p><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2 xpathLocation="noSelect">Abstract&nbsp;<a href="#top">Top</a></h2><p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[1]/p[1]">The mammalian suprachiasmatic nuclei (SCN) contain thousands of neurons capable of generating near 24-h rhythms. When isolated from their network, SCN neurons exhibit a range of oscillatory phenotypes: sustained or damping oscillations, or arrhythmic patterns. The implications of this variability are unknown. Experimentally, we found that cells within SCN explants recover from pharmacologically-induced desynchrony by re-establishing rhythmicity and synchrony in waves, independent of their intrinsic circadian period We therefore hypothesized that a cell's location within the network may also critically determine its resynchronization. To test this, we employed a deterministic, mechanistic model of circadian oscillators where we could independently control cell-intrinsic and network-connectivity parameters. We found that small changes in key parameters produced the full range of oscillatory phenotypes seen in biological cells, including similar distributions of period, amplitude and ability to cycle. The model also predicted that weaker oscillators could adjust their phase more readily than stronger oscillators. Using these model cells we explored potential biological consequences of their number and placement within the network. We found that the population synchronized to a higher degree when weak oscillators were at highly connected nodes within the network. A mathematically independent phase-amplitude model reproduced these findings. Thus, small differences in cell-intrinsic parameters contribute to large changes in the oscillatory ability of a cell, but the location of weak oscillators within the network also critically shapes the degree of synchronization for the population.</p>
</div><div class="abstract" xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]"><a id="abstract1" name="abstract1" toc="abstract1" title="Author Summary"></a>
<h2 xpathLocation="noSelect">Author Summary&nbsp;<a href="#top">Top</a></h2>
<p xpathLocation="/article[1]/front[1]/article-meta[1]/abstract[2]/p[1]">Circadian rhythms are daily, near 24-h oscillations in biological processes that nearly all organisms on Earth experience. Single cells contain a molecular clock that drives circadian rhythms in physiology and, when many cells synchronize in a population, daily behaviors. We hypothesized that small differences in intrinsic cellular properties allow for a diversity of circadian periods and amplitudes across cells. We observed circadian cells and their synchrony before, during, and after limiting communication between cells and then compared their intrinsic properties to their resynchronization behavior. We found that arrhythmic, weakly oscillating, and self-sustained circadian cells rejoined the rhythmic population independent of their cell-intrinsic oscillations. Using a mechanistic computational model of circadian cells, we found that resynchronization could be enhanced by including more weak oscillators or by placing weak oscillators at more connected nodes in the network. We conclude that intrinsic properties (e.g. oscillator weakness and responsiveness) and network structure (e.g. positions of weak oscillators) can independently buffer tissue rhythms from perturbations. This reveals how cellular and network properties impose rules on systems of circadian cells that must achieve synchrony from a desynchronized state, for example during perinatal development or when forced to overcome societal constraints on sleep-wake behavior, such as working early or late shifts.</p>
</div>


<div class="articleinfo" xpathLocation="noSelect"><p><strong>Citation: </strong>Webb AB, Taylor SR, Thoroughman KA, Doyle FJ III, Herzog ED (2012) Weakly Circadian Cells Improve Resynchrony. PLoS Comput Biol 8(11):
          e1002787.
            doi:10.1371/journal.pcbi.1002787</p><p><strong>Editor: </strong>Olaf Sporns, 
        Indiana University, United States of America
      </p><p></p><p><strong>Received:</strong> May 18, 2012; <strong>Accepted:</strong> October 5, 2012; <strong>Published:</strong> November 29, 2012</p><p><strong>Copyright:</strong> © 2012 Webb et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>ABW was supported by an NSF Graduate Research Fellowship and a Washington University Vision Sciences training grant. KAT was supported by NIH awards R01 HD055851 and R01 NS057813. SRT was supported by the Clare Boothe Luce Program of the Henry Luce Foundation. SRT, FJD and EDH were supported by Award Number R01GM096873 from the National Institute of General Medical Sciences. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p><p><a name="cor1"></a>* E-mail: <a href="mailto:srtaylor@colby.edu">srtaylor@colby.edu</a></p><p><a name="equal-contrib"></a>#
              These authors contributed equally to this work.
          </p><p><a name="fn1"></a>¤Current address: Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, Germany</p></div>





<div id="section1" xpathLocation="/article[1]/body[1]/sec[1]"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3 xpathLocation="noSelect">Introduction&nbsp;<a href="#top">Top</a></h3><p xpathLocation="/article[1]/body[1]/sec[1]/p[1]">Circadian clocks generate the near 24-h oscillations that orchestrate daily behaviors in organisms throughout the kingdoms of life <a href="#pcbi.1002787-Dunlap1">[1]</a>. In mammals, the suprachiasmatic nucleus (SCN), a bilateral structure of 20,000 neurons in the ventral hypothalamus, functions as the master pacemaker with circadian cells driving rhythms in behavior and physiological processes, such as sleep-wake, locomotor activity, temperature, and hormone release <a href="#pcbi.1002787-Ralph1">[2]</a>. It was hypothesized that every SCN neuron acts an autonomous clock, using molecular feedback loops to generate daily rhythms in gene expression and cellular output in the absence of external signals <a href="#pcbi.1002787-Welsh1">[3]</a>, <a href="#pcbi.1002787-Liu1">[4]</a>, <a href="#pcbi.1002787-Herzog1">[5]</a>, <a href="#pcbi.1002787-Yamaguchi1">[6]</a>, <a href="#pcbi.1002787-Reppert1">[7]</a>. For example, the <em>Period2</em> (<em>Per2</em>) gene, a clock gene found in humans and other animals, shows daily rhythms in transcription that appear to depend on daily repression by complexes including its protein (PER2) <a href="#pcbi.1002787-Zhang1">[8]</a>, <a href="#pcbi.1002787-Yamada1">[9]</a>. Recent data, however, highlight that, when isolated, SCN neurons exhibit a range of behaviors including damped or unstable circadian oscillations <a href="#pcbi.1002787-Webb1">[10]</a>, <a href="#pcbi.1002787-Ko1">[11]</a>. Therefore, although all cells may be capable of autonomous rhythmicity, they require stabilization from the SCN network to function as robust circadian oscillators. The potential source or sources of this cell-intrinsic variability, as well as its potential impact, are unknown. Whether the intrinsic properties of SCN oscillators independent of, or interactions amongst groups of oscillators within, the SCN network, or both, are responsible for the overall behavior is a current area of research <a href="#pcbi.1002787-Evans1">[12]</a>, <a href="#pcbi.1002787-Foley1">[13]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[2]">To first test the hypothesis that intrinsic differences between cells may affect how they resynchronize to each other, we followed daily oscillations of PERIOD2 protein levels in single SCN cells before, during, and after pharmacological blockade of intercellular signaling. The results revealed individual cells that differed in their intrinsic amplitude, level of gene expression, circadian period and ability to sustain rhythmicity, none of which predicted the cells' behaviors as they resynchronized to the population. Instead, we found that oscillations resumed and cells joined the rhythmic population at specific circadian phases, ultimately revealing the previously described daily waves of gene expression across the SCN <a href="#pcbi.1002787-Yamaguchi1">[6]</a>, <a href="#pcbi.1002787-Yan1">[14]</a>, <a href="#pcbi.1002787-Quintero1">[15]</a>. Recent work has further suggested that the phase relationships of SCN cells across the network could be important for robust rhythmic behavior of the tissue <a href="#pcbi.1002787-Foley1">[13]</a>, <a href="#pcbi.1002787-Maywood1">[16]</a>, <a href="#pcbi.1002787-Davidson1">[17]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[3]">To understand the complex behaviors of SCN cells, many studies have employed computational models. Both deterministic models detailing the molecular processes driving oscillations in single cells and stochastic models investigating the effects of noise on the system have aided in the understanding of mechanisms generating circadian rhythmicity in mammals <a href="#pcbi.1002787-Leloup1">[18]</a>, <a href="#pcbi.1002787-Forger1">[19]</a>, <a href="#pcbi.1002787-Leloup2">[20]</a>. Multi-cellular network models have been constructed from these single oscillators to describe synchrony across the SCN tissue, entrainment to light-dark cycles, and phase shifting behavior <a href="#pcbi.1002787-To1">[21]</a>, <a href="#pcbi.1002787-Vasalou1">[22]</a>, <a href="#pcbi.1002787-Gonze1">[23]</a>, <a href="#pcbi.1002787-Bernard1">[24]</a>. Network models have also probed regional differences in the SCN <a href="#pcbi.1002787-Vasalou2">[25]</a>, <a href="#pcbi.1002787-Hafner1">[26]</a> and the phenomenon of splitting, in which synchronized regions in the SCN can oscillate with the same period but opposite phases <a href="#pcbi.1002787-Schroder1">[27]</a>. We were interested in the relationship between cell-intrinsic rhythmicity and tissue synchronization, and found two major implications in the literature. First is that “smaller is better”: damped oscillators <a href="#pcbi.1002787-Gonze1">[23]</a>, <a href="#pcbi.1002787-Bernard1">[24]</a>, <a href="#pcbi.1002787-Westermark1">[28]</a> and oscillators with short relaxation times <a href="#pcbi.1002787-Granada1">[29]</a> synchronize efficiently. Additionally, a recent study of fibroblast cells shows that cellular oscillators have small, but sustained amplitudes, and that their proximity to a bifurcation allows them greater control over their period <a href="#pcbi.1002787-Leise1">[30]</a>. The authors note that this could be advantageous for peripheral oscillators that need to be entrained by the pacemaker and suggest that similar properties in pacemaker cells could aid synchrony. Second, network topology also affects the quality of synchrony, and specifically, small-world type network topologies are beneficial for synchrony <a href="#pcbi.1002787-Vasalou1">[22]</a>, <a href="#pcbi.1002787-Hafner1">[26]</a>. It has not been shown, however, why small oscillations are good for synchrony or how cell-intrinsic behaviors and network topology together affect synchrony. Using a mathematical model provides us the flexibility to explain biological phenomena without constraints found in the physiology, e.g. the type, number, and location of oscillators within a network. We sought to address this by first assessing the roles of intracellular processes on intrinsic properties, such as rhythmic ability and phase-responsiveness. Next we assessed the effects of individual cell properties on network synchronization, and finally, how the location of key cells within the network affects synchrony.</p>
<p xpathLocation="/article[1]/body[1]/sec[1]/p[4]">We hypothesized that intracellular properties and intercellular interactions contribute to the resynchronization behaviors we observed in the tissue data. To test this prediction, we used a computational model to simulate clock gene transcription-translation feedback loops in single cells and found that small changes in parameter combinations produce the range of intrinsic oscillations observed in SCN cells. When placed in a network, these cells were able to synchronize, meaning that they were capable of adjusting their phases to align with the population. To understand this phenomenon, we computed velocity response curves (VRCs) for these cells <a href="#pcbi.1002787-Taylor1">[31]</a>, <a href="#pcbi.1002787-Taylor2">[32]</a>. VRCs predict the phase velocity, i.e. how fast phase changes in response to intercellular signals. For our model, the VRCs suggested that cells with weaker oscillations could adjust their phase velocity more readily than cells with strong oscillations. These results were consistent with previous results that “smaller” is better to initiate synchrony, but with an alternative definition of smaller – we studied the effects of rhythmic, but low-amplitude (weak) cells, rather than initially rhythmic cells that lose amplitude, and eventually, all rhythmic ability, over the long-term (damped). We therefore tested the prediction that inclusion of weak circadian cells, which are highly responsive when isolated, would improve a network's ability to synchronize. We hypothesized that as weak cells establish rhythmicity and synchrony in the network, they lose responsiveness, becoming strong oscillators when coupled. By using a model of 400 coupled, heterogeneously oscillating cells, we found that increasing the proportion of weak oscillators or placing weak oscillators at more connected nodes in the network allowed for improved resynchronization.</p>
</div>

<div id="section2" xpathLocation="/article[1]/body[1]/sec[2]"><a id="s2" name="s2" toc="s2" title="Results"></a><h3 xpathLocation="noSelect">Results&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/title[1]">Circadian cells desynchronize similarly, but resynchronize differently</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]">Recent reports have shown that when SCN explants are treated with tetrodotoxin (TTX), a blocker of voltage-gated Na+ channels, the circadian rhythms of single cells gradually drift out of phase from one another <a href="#pcbi.1002787-Yamaguchi1">[6]</a>, <a href="#pcbi.1002787-Webb1">[10]</a>, <a href="#pcbi.1002787-Abraham1">[33]</a>, <a href="#pcbi.1002787-Buhr1">[34]</a>. To understand the relative contributions of cell-intrinsic and network properties to these synchronization dynamics, we examined the bioluminescence recorded from single cells (n = 123 across two nuclei, slice 1; n = 90 within one nucleus, slice 2; for details see <a href="#pcbi.1002787.s007">Text S1</a>) in SCN explants from homozygous PERIOD2::LUCIFERASE (PER2::LUC) knock-in mice <a href="#pcbi.1002787-Yoo1">[35]</a> during and after TTX treatment (<a href="#pcbi-1002787-g001">Fig. 1</a>). Although all cells appeared to gradually drift out of phase, only some expressed sustained circadian rhythms while others slowly or rapidly lost rhythmicity until the TTX was removed, at which point they began to regain rhythmicity and, eventually, synchrony. Looking at the timing of recovery of oscillations in slice 1, we found that approximately one-third of the cells that regained rhythms showed significant circadian oscillations within the first 35 h after removal of TTX. During the next 10 h another group of cells, similar in number, became circadian and began to synchronize to the first group. The remaining cells showed significant circadian rhythms starting around 45 h after removal of TTX, with the final cells entering by 96 h. Interestingly, the initial cohort of cells regained rhythmicity closely in phase while later cells regained rhythmicity with more broadly dispersed phases (<a href="#pcbi-1002787-g001">Figs. 1B</a>; <a href="#pcbi.1002787.s007">Text S1</a>; Rayleigh tests performed at the entrance time of the last cell in each cohort; Cohort 1, n = 39 cells, r = 0.68; Cohort 2, n = 43 cells, r = 0.55; Cohort 3, n = 32 cells, r = 0.43). In the second explant, we found a similar gradual restoration of rhythmicity to individual cells after TTX was removed (slice 2; <a href="#pcbi.1002787.s007">Text S1</a>). There was also a spatial pattern in each nucleus of the slices: lateral cells regain rhythmicity earlier than or phase lead medial cells (see Table S2 and Figure S10 in <a href="#pcbi.1002787.s007">Text S1</a>). In addition, lateral cells are, on average, smaller in amplitude than medial cells. This suggested a spatial organization of amplitude in the network during synchrony recovery. This led us to ask if there was something intrinsically different about the oscillations in cells that became rhythmic earlier or later after TTX was removed.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]"><a name="pcbi-1002787-g001" id="pcbi-1002787-g001" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002787?imageURI=info:doi/10.1371/journal.pcbi.1002787.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g001');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g001');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002787.g001&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002787?imageURI=info:doi/10.1371/journal.pcbi.1002787.g001" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g001');"><span>Figure 1. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/title[1]">TTX-treatment and wash-out in SCN explants reveals differences in the dynamics of desynchrony and resynchrony of single cells.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/p[1]">(<strong>A</strong>) Raster plot of bioluminescence (a. u.) from single neurons in an organotypic SCN explant before (days 1–6), during (days 7–12), and following treatment with 0.5 µM tetrodotoxin (TTX, days 13–18). Time of TTX treatment (days 7–12) is indicated by the shaded region of the bar along the top of the raster plot. TTX minimizes cell-cell communication and reveals intrinsic circadian properties. Here we plot traces from 123 neurons that show a range of oscillatory behaviors in TTX, all of which are rhythmic when coupling is in place before treatment. Following washout, rhythms return to all cells, but at different rates. The intensity of the white is proportional to the bioluminescence in each experimental condition. (<strong>B</strong>) Preliminary data suggests a small population of neurons resynchronizes first and then a slow recruitment of the remaining population is observed following TTX washout. Here we include the SI for the entire population (black), as well as the SI for each cohort of cells. Cohorts are determined by when cells become rhythmic after the wash; cells in cohort 1 (blue) become rhythmic first (hours 31–35), cells in cohort 2 (green) become rhythmic second (hours 36–45), and cells in cohort 3 (red) become rhythmic after hour 46 of the wash. (<strong>C</strong>) The location of single cells recorded in the SCN explant. Cell position is color-coded based on the resynchronization cohort.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002787.g001</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002787.g001/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002787.g001/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002787.g001.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002787.g001/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002787.g001.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[1]/p[2]">We reported previously that SCN cells uncoupled by TTX display diverse circadian behaviors both in terms of amplitude and period <a href="#pcbi.1002787-Webb1">[10]</a>. We acknowledge the possibility that TTX treatment itself can alter a cell's amplitude; however, we will assume that amplitude during TTX is reflective of amplitude that is independent of other feedback from other cells, and as such, is intrinsic to a cell. To determine whether or not intrinsic behaviors explain early or late restoration of rhythms, we compared amplitudes and periods of cells in TTX-treated SCN explants to the time when their rhythms reemerged and to the quality of synchrony within the group of circadian cells. In both slices, we found no significant correlations (R<sup>2</sup> values of &lt;0.2, <a href="#pcbi.1002787.s007">Text S1</a>) between intrinsic circadian properties, such as mean bioluminescence, total bioluminescence, bioluminescence amplitude and period, and when a cell joined in oscillations within a resynchronizing SCN network. We conclude that intrinsic properties alone fail to explain the dynamic emergence of rhythms and resynchrony of individual cells. Therefore network properties likely participate along with these intrinsic behaviors in synchrony. To explore the relationship, if any, between the intrinsic properties of the cells within the context of the network, we implemented a mathematical model.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/title[1]">Small changes in key parameters can create diverse oscillator types</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[1]">First, we sought to reproduce the diversity of characteristics of isolated cells (i.e. PER-driven bioluminescence with patterns that could be described as strongly rhythmic, weakly rhythmic or arrhythmic over multiple days) by identifying potential molecular determinants of these circadian phenotypes. We utilized an existing model of the mammalian molecular clock to simulate SCN neurons <a href="#pcbi.1002787-Leloup1">[18]</a> and focused on four parameters that regulate the output we recorded in the biological data (PER2::LUC): the rate of transcription of the <em>Period</em> (<em>Per</em>) gene, or translation, phosphorylation or degradation of the PERIOD (PER) protein. We categorized each cell as arrhythmic, weak (rhythmic but low in amplitude), or strong (rhythmic and high in amplitude; see <a href="#s4">Materials and Methods</a>). We found that changing any of the four parameters by at least 10% moved simulated cells from arrhythmic to weakly rhythmic to sustained circadian gene expression (<a href="#pcbi.1002787.s002">Fig. S2A</a>). Regardless of whether they were varied alone or in combination, these parameters recapitulated the phenotypes found in SCN explants (<a href="#pcbi.1002787.s001">Figs. S1</a>, <a href="#pcbi.1002787.s002">S2</a>, <a href="#pcbi.1002787.s003">S3</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[2]">We used a multi-dimensional visualization technique to evaluate the relative contributions of the four parameters to rhythm generation <a href="#pcbi.1002787-Taylor3">[36]</a>, providing a novel analysis of sensitivity of strength and sustainability of circadian oscillations to specific parameter combinations. By nesting parameter combinations into stacks, we arranged our data set with a large number of parameters in two dimensions that could be displayed easily (<a href="#s4">Materials and Methods</a>). Based on the position of cells across the parameter space visualization, we found that small changes in rates of transcription of <em>Per</em> mRNA and degradation of PER protein produced larger effects than changes in translation and phosphorylation of PER on the circadian phenotype of simulated cells (<a href="#pcbi.1002787.s001">Figs. S1</a> and <a href="#pcbi.1002787.s002">S2B</a>). <em>Per</em> rhythmicity was similarly more sensitive to <em>Bmal1</em> transcription and BMAL1 degradation than BMAL1 translation and phosphorylation (<a href="#pcbi.1002787.s002">Fig. S2B</a>).</p>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[2]/p[3]">We ensured that individual model cells accurately represented individual cells from the slice. Amplitude was of particular importance because here we tested the effect of weak oscillators for the first time. When we compared the circadian periods and amplitudes of simulated and recorded cells we found no correlation between period and amplitude for either the model or the slices (<a href="#pcbi.1002787.s004">Fig. S4</a>; R<sup>2</sup>&lt;0.02 for all). Further, the periods were similarly distributed (slice 1 std. dev. = 2.1 h, slice 2 std. dev. = 2.1 h, model std. dev. = 2.1 h) and the amplitude distributions were dominated by small values in both the model and the slices (<a href="#pcbi.1002787.s004">Fig. S4</a>, <a href="#pcbi.1002787.s007">Text S1</a>). This suggests that the period and amplitude values in model cells faithfully mimic behaviors we observe in the slice during TTX treatment. Neither this independence of period and amplitude, nor the dominance of small amplitudes has been described in other computational models. Here we are explicit in our modeling that the intrinsic amplitude is much smaller than the in-network amplitude. We concluded that by specifying small differences in key circadian parameters between cells, our simulated cells accurately represented the diverse rhythmic abilities, as well as realistic intrinsic properties such as period and amplitude, of SCN cells.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/title[1]">Weak oscillators predicted to show greater shifts following perturbation</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/p[1]">Another relevant property of a circadian oscillator is how it will adjust its phase velocity (speed) following a perturbation. We calculated the velocity response properties of the simulated cell set, including both strong and weak cells. <a href="#pcbi-1002787-g002">Fig. 2C</a> shows representative velocity response curves (VRCs) to a signal, where curves are plotted as a function of phase of oscillation. From the curve, we see that if the signal arrives early in the day (around circadian time, CT, 0) the cell will speed up, and if it arrives late in the day (between CT6 and CT12), the cell will slow down. To measure the cell's ability to shift, we computed the area under the absolute value of the VRC. We compared this VRC area to intrinsic oscillator amplitude (the sum of the peak to trough amplitude of all states) and found it inversely correlated with velocity response (<a href="#pcbi-1002787-g002">Fig. 2D</a>; R<sup>2</sup> = 0.85). This indicates that oscillators with small intrinsic amplitude are more likely to have larger velocity response and therefore greater phase shifting ability compared to high-amplitude cells. Interestingly, we found that small oscillators in both the simulation and slice 1 have a broader distribution of periods compared to strong cells. The VRC results suggest a functional strategy to overcome this period variability: weaker cells are better at shifting their phase.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]"><a name="pcbi-1002787-g002" id="pcbi-1002787-g002" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002787?imageURI=info:doi/10.1371/journal.pcbi.1002787.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g002');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g002');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002787.g002&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002787?imageURI=info:doi/10.1371/journal.pcbi.1002787.g002" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g002');"><span>Figure 2. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/title[1]">Mechanistic model produces diverse cells with varying response properties similar to SCN cells treated with TTX.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/p[1]">(<strong>A</strong>) Bioluminescence traces from cells classified as strong (left), weak (center) or arrhythmic (right) based on their amplitude in their final cycle during TTX treatment. We show the last peak before TTX treatment and then 5 cycles during treatment. (<strong>B</strong>) Simulated traces using a mechanistic model from strong, weak, and arrhythmic cells show similar oscillation qualities to the real cells. Simulated cells are classified according to their limit cycle's “cumulative amplitude”, which is the sum of peak-to-trough amplitudes of all modeled messages and proteins. (<strong>C</strong>) The range of amplitudes and phases of velocity response curves (VRCs) from strong (left) and weak (right) simulated cells; VRCs are not computable for arrhythmic simulated cells. The shaded regions indicate the areas in which model VRCs peak and trough. Two representative VRC's are shown in each plot, each with a trough and peak falling in a different place in the regions. Circadian time is defined relative to peak <em>Per</em> mRNA expression, which is at circadian time (CT) 7. Arrows indicate the VRC's for the cells shown in (<strong>B</strong>). By computing the area under the absolute value of the VRC, we determine each cell's ability to speed up or slow down when signaled (greater area indicates greater “shiftability”). (<strong>D</strong>) We plot the VRC area (log scale) vs. the cumulative amplitude (linear scale). Arrows indicate the data points for the example cells in (<strong>B</strong>). We include the correlation coefficient for the amplitude and natural logarithm of the VRC area. There is a negative correlation between the VRC area and the oscillatory behavior of simulated cells, indicating that weak cells are more shiftable.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002787.g002</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002787.g002/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002787.g002/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002787.g002.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002787.g002/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002787.g002.TIF"></span> TIFF)
                  </a></li></ul></div></div>

<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/title[1]">Networks including weak oscillators in greater proportions or at more highly connected nodes reach higher synchrony</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[1]">To test empirically if and how the proportion of weak oscillators contribute to the synchronization properties of a network like the SCN, we modeled a network of 400 SCN cells with diverse oscillatory abilities, including different periods and amplitudes, as well as network connections. Specifically, each cell had a unique set of parameters selected randomly to establish a population with defined proportions of arrhythmic, weak and sustained oscillators. We chose to include both local and global coupling between cells based on recent theoretical work <a href="#pcbi.1002787-Vasalou1">[22]</a>. Each cell was connected to its four nearest neighbors and 20% of cells connected to cells beyond their immediate neighbors (<a href="#pcbi.1002787.s005">Fig. S5</a>). Coupling was achieved in the model by simulating release of vasoactive intestinal polypeptide (VIP), a known synchronizer in the SCN <a href="#pcbi.1002787-Aton1">[37]</a>, from all cells. Each network (n = 56 independent runs for each condition) was populated with 400 characterized cells and its overall response to uncoupling and recoupling was measured by calculating the synchronization index (SI) of all cells (see <a href="#s4">Materials and Methods</a>). On average, we found that networks with more weak oscillators (total oscillator amplitude &lt; = 8.4 a.u.) reliably reached higher levels of synchrony (SI&gt; = 0.7 at days 15–18) with approximately 5-fold higher synchrony in networks comprised of mostly weak, compared to mostly strong, oscillators (<a href="#pcbi-1002787-g003">Fig. 3A–B</a>; ANOVA between populations, p&lt;0.001). We found that networks with only strong oscillators failed to resynchronize (<a href="#pcbi-1002787-g003">Fig. 3A–B</a>; SI = 0.2 at days 15–18). We conclude that weak, highly shiftable cells can promote synchrony.</p>
<div class="figure" xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]"><a name="pcbi-1002787-g003" id="pcbi-1002787-g003" title="Click for larger image " href="/article/info:doi/10.1371/journal.pcbi.1002787?imageURI=info:doi/10.1371/journal.pcbi.1002787.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g003');"><div class="expand-link" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g003');"></div><img xpathLocation="noSelect" border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002787.g003&amp;representation=PNG_I" align="left" alt="thumbnail" class="thumbnail"></a><p><strong xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/label[1]"><a xpathLocation="noSelect" href="/article/info:doi/10.1371/journal.pcbi.1002787?imageURI=info:doi/10.1371/journal.pcbi.1002787.g003" onclick="return ambra.lightBox.show('info:doi/10.1371/journal.pcbi.1002787', 'info:doi/10.1371/journal.pcbi.1002787.g003');"><span>Figure 3. </span></a> <span xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/title[1]">Networks with more weak oscillators resynchronize better.</span></strong></p><p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/p[1]">We measured the synchronization index (SI) of the modeled population in the coupled (days 1–6), TTX-uncoupled (days 7–12), and recovery (days 13–18) conditions and compare the results across different population and network configurations. We show the effects of varying the percentage of strong oscillators (top row) and of varying the network topology (bottom row). For each configuration-type, we run 56 simulations. (<strong>A</strong>) The mean SI is shown over time for populations of 0% (cyan), 20%, 35%, 50%, 65%, 80%, and 100% (magenta) strong cells. (<strong>B</strong>) For each simulation, we compute the mean SI for days 3 to 5 after the wash (days 15 to 18). Here we show the mean and standard deviation across simulations using the same color scheme. (<strong>C</strong>) We simulated all networks with 50% strong cells and show the mean SI is shown for populations with weak cells at hubs (squares), both strong and weak cells placed randomly at hubs (circles), and strong cells at hubs (triangles) The effect of network configuration on quality of synchrony is shown in (<strong>D</strong>) and is labeled by the type of cell at network hubs.</p>
<span xpathLocation="noSelect">doi:10.1371/journal.pcbi.1002787.g003</span><div class="clearer"></div><div class="figure-inline-download"><ul><li><strong>Download: </strong><a href="/article/info:doi/10.1371/journal.pcbi.1002787.g003/powerpoint">PowerPoint slide</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002787.g003/largerimage">
                    larger image (<span id="info:doi/10.1371/journal.pcbi.1002787.g003.PNG_L"></span> PNG)</a> |
                  <a href="/article/info:doi/10.1371/journal.pcbi.1002787.g003/originalimage">
                    original image (<span id="info:doi/10.1371/journal.pcbi.1002787.g003.TIF"></span> TIFF)
                  </a></li></ul></div></div><p xpathLocation="/article[1]/body[1]/sec[2]/sec[4]/p[2]">To test the importance of location within the network, we evaluated synchrony in networks of 50% weak and 50% sustained oscillators in which weak oscillators were assigned to hubs, i.e. nodes with more than the average of 10 outputs (range = 4–39 outputs). We found that when weak cells were placed in the more connected nodes of the network (n = 56 independent network runs for each condition), the population reached approximately 5-fold greater synchrony compared to networks with strong cells at these nodes or networks with oscillators distributed to nodes randomly (<a href="#pcbi-1002787-g003">Fig. 3C–D</a> ANOVA between populations, p&lt;0.001). The quality of resynchrony, therefore, depended on both the number and placement of weak, shiftable oscillators in the network.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/title[1]">Weak, low-amplitude oscillators enhance network synchrony similarly to damped oscillators</h4>
<p xpathLocation="/article[1]/body[1]/sec[2]/sec[5]/p[1]">To test whether our findings regarding weak oscillators extend to damped oscillators, we repeated the simulations using cells that lose amplitude, and eventually, all rhythmic ability (see <a href="#s4">Materials and Methods</a>). We found that the effects of weak circadian cells and of damped cells were nearly identical. For example, increasing the percentage of damped cells (<a href="#pcbi.1002787.s006">Fig. S6A</a>) or placing damped cells at network hubs (<a href="#pcbi.1002787.s006">Fig. S6B</a>) enhanced synchrony. Further, we verified that our results were not sensitive to our definition of weak. For the simulations used to generate <a href="#pcbi-1002787-g003">Fig. 3</a> and <a href="#pcbi.1002787.s006">Fig. S6</a>, the weak cells were the smallest 30% in intrinsic amplitude. We repeated all simulations varying the percentage of rhythmic cells classified as weak. For each of these cut-offs, we measured the largest difference in SI between weak and damped cells (range = +0.07–0.69). The closer the weakly circadian cells were to the bifurcation, the more they acted like damped cells. We observed that as long as the cut-off is less than 50%, weak cells are similar to damped cells. To demonstrate that these behaviors could be generalized to other oscillatory systems, we constructed a phase-amplitude model, which functions as a reduced version of our mechanistic model (<a href="#pcbi.1002787.s008">Text S2</a>). We wanted to know if the benefit of weak cells for synchrony was evident in simpler systems and if a reduced model could further our understanding. The reduced model also showed that inclusion of more low amplitude, or small, oscillators or strategically placing them at more highly connected nodes increased synchrony. Thus, these results were robust across model compositions and types, and indicate that larger phase adjustments by small oscillators will, in general, produce higher synchrony.</p>

</div>

<div id="section3" xpathLocation="/article[1]/body[1]/sec[3]"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3 xpathLocation="noSelect">Discussion&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/title[1]">Potential sources and consequences of oscillator heterogeneity in a circadian network</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[1]">Although physiologists and anatomists have described differences between SCN cells including their circadian amplitude, phase and waveform <a href="#pcbi.1002787-Inagaki1">[38]</a>, <a href="#pcbi.1002787-Schaap1">[39]</a>, <a href="#pcbi.1002787-Yan2">[40]</a>, the functional role of oscillator heterogeneity has been little studied. For example, the intrinsic daily oscillations of SCN neurons can be sustained, damped, or absent <a href="#pcbi.1002787-Webb1">[10]</a>, <a href="#pcbi.1002787-Ko1">[11]</a>, <a href="#pcbi.1002787-Liu2">[41]</a>, but the consequences of these diverse circadian phenotypes remain unknown. Here, we found that the resynchronization of SCN cells following pharmacological blockade of cell-cell signaling involves waves of cells becoming rhythmic and adjusting their phases to join the daily cycling of the population. Previous theoretical studies have suggested that damped cells can aid network synchrony by entraining to a wider range of periods <a href="#pcbi.1002787-Gonze1">[23]</a>, <a href="#pcbi.1002787-Bernard1">[24]</a> and relaxation oscillators can entrain faster if they have shorter relaxation rates or more spike-like waveforms <a href="#pcbi.1002787-Granada1">[29]</a>, but have also highlighted that it is not yet possible to distinguish whether SCN cells should be modeled as damped oscillators or low amplitude, sustained oscillators <a href="#pcbi.1002787-Westermark1">[28]</a>. What are the potential sources of these differences? By tuning a computational model, we found that small changes in a small set of parameters could produce a realistic distribution of cells that varied not only in period length, as has been generated previously <a href="#pcbi.1002787-Leloup1">[18]</a>, <a href="#pcbi.1002787-To1">[21]</a>, but in qualities of the oscillations themselves. Using non-biased minimization techniques to represent multi-dimensional parameter space, we found that parameters associated with transcription rate and protein degradation of the <em>Period</em> gene were more likely to contribute to circadian changes than other parameters. We speculate that genetic differences in and the environmental modulation of these key rate constants between SCN cells could underlie the heterogeneity in their circadian properties. For example, it has been shown that the amplitude of <em>Per</em> transcription is altered in the absence of VIP <a href="#pcbi.1002787-Loh1">[42]</a> and that the stability of PER protein against degradation affects circadian period <a href="#pcbi.1002787-Virshup1">[43]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[2]">Because we found that the amplitude of our model cells is reliably and inversely related to their ability to adjust their phase velocities in response to natural signals, we tested the impact of both low (weak) and high (strong) amplitude oscillators on network synchrony. Previously, Bernard and colleagues suggested that a network comprised of damped circadian oscillators is capable of synchronizing and maintaining rhythmicity, and hypothesized that damped oscillators, when synchronized, induced rhythmicity in the population <a href="#pcbi.1002787-Bernard1">[24]</a>. Locke and colleagues then performed a parameter optimization, searching to maximize the ability of a network of oscillators to synchronize. The best-synchronized networks were composed of damped cells <a href="#pcbi.1002787-Locke1">[44]</a>. Together, these results suggested that the driving force coupling cells together could arise from some inherent property found in damped cells.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[3]">In a similar fashion, we sought to identify inherent characteristics in both biological and modeled weak oscillators, including relationships between intrinsic amplitude and intrinsic shiftability. Published models using damped oscillators have been unable to mathematically quantify shiftability. By studying weak circadian oscillators, we measured larger changes in oscillator speed for smaller amplitude cells. Measurements of shiftability now provide a tool to study for the first time the kinetics of resynchronization. We posit that weakly oscillatory cells can send signals to other weakly oscillatory cells to readily adjust their phases. As the system synchronizes, the cells gain amplitude and thus lose the ability to make dramatic shifts. This suggests a strategy for neurons to resynchronize. The system can move from being sensitive to perturbations to being robust against them through a process of cell-cell amplification of rhythm amplitudes <a href="#pcbi.1002787-Abraham1">[33]</a>.</p>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[1]/p[4]">In our model networks we demonstrated that the total number of a specific oscillator type is critical and that there is an effect of the degree of connectivity of certain oscillator types on synchrony, such that, networks with more and more highly connected weak oscillators have improved synchrony during the recovery period following a perturbation. We concluded that heterogeneity arises from both cell intrinsic and network contributions, including the network topology and number of weakly circadian cells. The model does not account for all dynamics of resynchrony that we observed in the data, which will be addressed in the future. For example, though we observed populations of cells consistently ahead of or behind the mean phase of the network simulations, we observed no spatial pattern in these phase differences; the more homogeneous connections in the model networks led to most cells becoming rhythmic at the same time and together tighten in phase. Future work will use modeling to understand if and how spatial heterogeneity in network connections causes spatial patterns in the phase of oscillators across the slice. Future work will also take into account stochasticity in cell behavior. Preliminary results (data not shown) indicate that incorporating white noise into tissue simulations has no effect on the role of weak oscillators.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/title[1]">Is the tissue the issue?</h4>
<p xpathLocation="/article[1]/body[1]/sec[3]/sec[2]/p[1]">How the evolving differences within oscillators and amongst oscillator populations carry over to the behavior of networks is an open question for investigation. We return to the issue of whether rhythmicity and synchrony are due to intrinsic cell properties or are dependent on cell location and network structure. Recent studies have examined phase heterogeneity within the SCN <a href="#pcbi.1002787-Evans1">[12]</a>, <a href="#pcbi.1002787-Foley1">[13]</a> and have concluded it is not a function of cellular properties. Foley and colleagues summarized their findings as “the tissue is the issue” – that placement within the SCN network (based on assigned phase) dictates whether and how an SCN neuron will oscillate <a href="#pcbi.1002787-Foley1">[13]</a>. We extend this to hypothesize specifically that cells, which are intrinsically different in their ability to maintain strong or weak rhythms, will impact the population rhythm differentially (e.g. the quality of synchronization increases with more weak oscillators), but also depending on their location within the network (e.g. cells at hubs have greater influence). Other theoretical studies have emphasized that the number of connections between cells could modulate the degree of synchrony in the network and argued for region-specific placement of particular oscillator types (e.g. sustained cells in the dorsal SCN and arrhythmic or gated cells in the ventral SCN) <a href="#pcbi.1002787-Vasalou2">[25]</a>, <a href="#pcbi.1002787-Antle1">[45]</a>, <a href="#pcbi.1002787-Antle2">[46]</a>. We find no evidence for specialized, localized populations of oscillators in the resynchronizing SCN slice following the removal of TTX. In contrast, our model shows the importance of weakly rhythmic, highly responsive oscillators at hubs where they can send coordinated phase information broadly throughout the network, becoming less responsive as they increase in amplitude, and that this is critical for improved synchrony. It is thought that SCN neurons establish rhythmicity and synchrony amongst each other and with the external light-dark environment late in gestation <a href="#pcbi.1002787-Saxena1">[47]</a>, <a href="#pcbi.1002787-Sumova1">[48]</a>. Because these features are likely critical for the survival <a href="#pcbi.1002787-Caba1">[49]</a>, we posit that the composition of the SCN, including a continuum of oscillator behaviors and connections, allows the tissue to adjust to shifts in environmental timing cues. These properties may be universal to all networks that include weak oscillators.</p>

</div>

<div id="section4" xpathLocation="/article[1]/body[1]/sec[4]"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3 xpathLocation="noSelect">Materials and Methods&nbsp;<a href="#top">Top</a></h3>
<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/title[1]">SCN cell culture</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]">Single cells measured in SCN slices reported in this study were recorded as previously published <a href="#pcbi.1002787-Webb1">[10]</a>. Briefly, SCN explants from neonatal PER2::LUC mice were cultured for 3 days on MilliCell-CM (Millipore) membrane pieces in CO<sub>2</sub>-buffered medium supplemented with 10% newborn calf serum (Invitrogen) before being inverted onto polylysine/laminin coated coverslip dishes. All procedures were approved by the Washington University Animal Studies Committee and complied with NIH guidelines.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/title[1]">Bioluminescence recording</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[2]/p[1]">We conducted recordings in air-buffered medium containing 0.1 mM beetle luciferin (BioThema) at 37°C beginning at day 2 after slice transfer to coverslip dishes We temporally (1 h integration time) and spatially (2×2 pixel resolution) bioluminescence counts using a Versarray 1024 CCD camera (Princeton Instruments).</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/title[1]">TTX treatment and washout</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[3]/p[1]">Following 6 days of baseline recording, we treated organotypic SCN explants with 0.5 µM tetrodotoxin (TTX, Sigma) as previously described <a href="#pcbi.1002787-Webb1">[10]</a>. TTX remained in the medium for 6 days before the medium was removed and we washed explants with 1 full volume exchange of fresh medium. Recording then continued for at least 6 days to examine rhythms as cells resynchronized after the restoration of cell-cell communication.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/title[1]">Analysis of rhythms</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[4]/p[1]">We used NIH ImageJ software to process all images by first subtracting background levels and then measuring pixel intensity over time in a region of interest above each cell. Cells were tracked manually from frame-to-frame and across treatments to account for any tissue movement. Cells were initially scored as rhythmic or arrhythmic if their gene expression rhythm oscillated with a period between 15 and 35 hours that was statistically significant by both Chi-squared periodogram <a href="#pcbi.1002787-Sokolove1">[50]</a> and FFT-NLLS <a href="#pcbi.1002787-Plautz1">[51]</a>. We also used Wavos to determine period and phase information from the single cell traces <a href="#pcbi.1002787-Harang1">[52]</a>.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/title[1]">Mechanistic model of molecular clock in single SCN cells</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[5]/p[1]">A version of a previously published 16-ordinary differential equation model of the mammalian circadian clock was used to simulate rhythms in single model cells <a href="#pcbi.1002787-Leloup1">[18]</a>. We altered parameters for rates of transcription, translation, phosphorylation, and degradation of either <em>Period</em> or <em>Bmal1</em> genes, leaving 50 other parameters set to published basal values <a href="#pcbi.1002787-Leloup1">[18]</a>, and measured rhythms in gene output. We simulated 720 hours of gene expression from each cell, using initial conditions from a representative, high-amplitude sustained cell.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/title[1]">Visualization of parameter space</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[6]/p[1]">To measure the sensitivity of circadian cycling to clock gene parameters, we organized results from single cell simulations using clutter based dimensional reordering (CBDR), which applies minimization and dimensional stacking algorithms described below. These methods allow visualization of the underlying structure of clock parameter space and gauge the influence of tested parameters relative to output behavior. We utilized published Matlab code <a href="#pcbi.1002787-Taylor3">[36]</a> to minimize differences between output scores (strong, weak, arrhythmic) and cluster behaviors together. The code arranged parameter combinations iteratively until the minimization requirement, i.e. cells with like behavior, were clustered together, was fulfilled. First, the code scans one pair of parameters over a range of values while the remaining parameters are set to basal values. Then we label this grid based on the output for each combination and add it to a larger montage of other parameter pairs. For a useful visualization, the code orders these dimensional stacks to group similar outputs together. Given a unique behavior and parameter combination for each cell, we minimize the stack so that differences between regions of varying outputs are small (in this case, strong, weak, or arrhythmic patterns in gene expression), and this provides an order ranking of “higher” versus “lower” parameters in the stack. Changes in parameter value that produce larger effects in output phenotype are higher in the stack order.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[7]/title[1]">Velocity response curves</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[7]/p[1]">A velocity response curve (VRC) predicts the effect of parametric perturbation on the phase velocity of the oscillator. For a cell in the SCN, there is a single parameter (vsP) that is manipulated by VIP signaling. Hence, we consider the VRC associated with vsP, mapping the circadian time of VIP signaling to its effect on the phase velocity. Cells with higher-magnitude VRCs can be sped up or slowed down more by VIP signals than cells with lower magnitude VRCs. To quantify the “shiftability” of a cell, we compute the area under the absolute value of the VRC.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[7]/p[2]">A VRC may be computed for any cell with a parameter set allowing for limit cycle oscillations. For details regarding computation, see <a href="#pcbi.1002787-Taylor1">[31]</a>.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[8]/title[1]">Categorization of simulated cell types</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[8]/p[1]">Mathematically, the individual cells we have modeled can be categorized as rhythmic (those that converge to a periodic orbit) or arrhythmic (those that converge to a steady-state solution). However, simulations of single cells display a spectrum of behaviors, with some showing lower or higher amplitude than others. Using the peak to trough amplitudes of all model components (i.e. by summing the amplitudes of all states), we separated the rhythmic cells into two categories: 1.Weak cells are rhythmic with small amplitudes and 2. Strong cells are rhythmic with larger amplitudes.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[8]/p[2]">For some simulations, we needed damped oscillators, which form a subset of the arrhythmic oscillators. We used the total amplitude at the end of a 15-day simulation (starting from high-amplitude initial conditions) to create the additional categories: 1. Flat cells are arrhythmic and have the smallest amplitudes in their final pseudo-cycle and 2. Damped cells are arrhythmic and have the largest amplitudes in their final pseudo-cycle.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[8]/p[3]">For most simulations in the paper, we defined the smallest 30% (n = 228) of the oscillatory cells as weak and the remaining 70% (n = 595) as strong. For simulations that needed to distinguish between flat and damped, we chose the cut-off so there would be the same number of damped cells and weak cells.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[9]/title[1]">Model of SCN coupling and network topology</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[9]/p[1]">Model cells were coupled together by VIP signaling, simulated as a drive on the rate of <em>Per</em> transcription, as previously published <a href="#pcbi.1002787-To1">[21]</a>. In our model, 20 percent of the 400 neurons were capable of sending a VIP signal and all neurons could respond to VIP. Connections between cells were organized with a small world network topology as in <a href="#pcbi.1002787-Vasalou1">[22]</a> where each VIP cell was coupled to its four nearest neighbors and then had a probability of sending unidirectional long-range connections to other cells in the network. We set the connection probability to p = 0.05, resulting in a synchronized system with a range of 4 to ~40 outgoing connections in most networks. To mimic the TTX experiments, we simulated 6 days with VIP-mediated coupling followed by 6 days with coupling eliminated and then reinstated for 6 days. We assessed the intrinsic circadian expression of each cell as well as the rate of resynchronization of each cell and the ensemble. The network connections and parameter values for each cell did not change throughout the simulation.</p>


<h4 xpathLocation="/article[1]/body[1]/sec[4]/sec[10]/title[1]">Calculation of Sync Index</h4>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[10]/p[1]">The synchronization index (SI) provides a real-time measure of the phase dispersion across a population of oscillators, which ranges from 1 (all cells peak in phase) to 0 (all cells peak at uniformly-distributed times of the day). We defined SI at each time <em>t</em> by the radius <em>r</em> of the complex order parameter <a href="#pcbi.1002787-Strogatz1">[53]</a> according to<br><a name="pcbi.1002787.e001" id="pcbi.1002787.e001"></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pcbi.1002787.e001&amp;representation=PNG"></span><br>where <em>N</em> is the number of cells, <em>φ<sub>j</sub>(t)</em> is the phase of the <em>j<sup>th</sup></em> cell at time <em>t</em>, and <em>ψ(t)</em> is the average phase of all cells. We compute the instantaneous phase of each cell (simulated or real) by applying the continuous wavelet transform using a Morlet wavelet <a href="#pcbi.1002787-Meeker1">[54]</a> to its trace of <em>Period</em> mRNA. The phase of the cell over time may be recovered from the ridges of the transform, which are extracted using a straight-forward algorithm (<a href="#pcbi.1002787-Harang1">[52]</a>, <a href="#pcbi.1002787-Meeker1">[54]</a>; Wavos Package). Briefly, the continuous wavelet transform (CWT) produces a complex-valued field over scales (which may be mapped to instantaneous frequency) and translations (which may be mapped to time). The magnitude of the complex number at a given translation and scale may be interpreted as the strength of oscillation of the signal at the frequency given by the scale and the time given by the translation. The phase of the complex number at a given translation and scale gives the phase of that oscillatory component. By selecting points with contiguous scales across a range of translations that maximize the magnitude of the CWT (the “wavelet ridge”), we may extract the dominant frequency of the oscillator over time, and from those points extract the phase evolution of the oscillator from the angles of the CWT coefficients.</p>
<p xpathLocation="/article[1]/body[1]/sec[4]/sec[10]/p[2]">Because the wavelet analysis requires a window (in time) around the model state in question, it is unable to calculate the phase during the first and last 34 hours of each simulation. We treat each experimental condition separately, which mean there are gaps in the SI plotted in <a href="#pcbi-1002787-g003">Fig. 3</a>.</p>

</div>

<div id="section5" xpathLocation="/article[1]/body[1]/sec[5]"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3 xpathLocation="noSelect">Supporting Information&nbsp;<a href="#top">Top</a></h3><a name="pcbi.1002787.s001" id="pcbi.1002787.s001"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s001">Figure S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[1]"><strong>Small changes in parameter values of a mechanistic model produce a range of oscillatory behaviors.</strong> We sought to determine whether certain molecular events were more likely to contribute to the range of circadian phenotypes observed. Using a minimization algorithm we arrange each simulated cell (small box), representing one of 1296 potential combinations of parameters for transcription, translation, phosphorylation, and degradation of Period varied over 6 equivalent steps, to generate a two-dimensional visualization of multi-dimensional parameter space. This non-biased algorithm <a href="#pcbi.1002787-Taylor3">[36]</a> groups cells into distinct areas of strong (blue), weak (cyan) and arrhythmic (black) outputs and ranking the contribution of each of these four parameters to rhythm generation. Axes for small boxes are determined by the lower order or less sensitive parameters, and these are then assembled to form larger, 6×6 grids, which vary with the higher order parameters. Rates for transcription and degradation appear on the larger axes, indicating that these are higher order parameters whose values produce a more dramatic shift in phenotype when changed. The red arrows emphasize how in three small steps within parameter space output transitions from arrhythmic to weak to strong, sustained circadian oscillations. While populations of strong and arrhythmic cells cover large ranges of parameters values, the number of weak cells at the transitions between these groups is much smaller.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[2]">(EPS)</p>
<a name="pcbi.1002787.s002" id="pcbi.1002787.s002"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s002">Figure S2. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[1]"><strong>Parameter values in the mechanistic model straddle bifurcations, producing a spectrum of oscillator types.</strong> (A) To produce a variety of circadian phenotypes in model cells, we probed parameters individually and selected ranges that produced a spectrum of behaviors when all other values were set to their published basal levels (with the exception of the maximal rate of per transcription, which is moved to 1.01). Here we show how the cumulative oscillator amplitude changes from strong (blue) to weak (cyan) to arrhythmic (gray) as a function of parameter value for the transcription, translation, phosphorylation and degradation of Per (left) and Bmal1 (right). (B) To determine whether another set of parameters produced similar phenotypes, we varied rates of transcription of <em>Bmal1</em>, or translation, phosphorylation and degradation of BMAL1 protein. Each plot visualizes areas of parameter space found by varying rates of transcription and translation (x-axis), phosphorylation, and degradation (y-axis) for both Period (left, replotted from Supp. Figure 1) and Bmal1 parameter sets (right), respectively.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[2]">(EPS)</p>
<a name="pcbi.1002787.s003" id="pcbi.1002787.s003"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s003">Figure S3. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[1]"><strong>The mechanistic model produces cells with a spectrum of period and amplitude values consistent with SCN data.</strong> When we examined the period (left) and cumulative oscillator amplitude (right) values of simulated cells, here plotted using the same axes as Supp. Figs. 1 and 2, we found that cells falling in the region of parameter space corresponding to weak oscillators show a range of period values (18 to 30 h) similar to what was observed in functionally isolated SCN neurons <a href="#pcbi.1002787-Webb1">[10]</a>. The range of final amplitude values of weak cells was lower than that of strong cells (range 2–10 arbitrary units vs. range 12–28 a.u.). We hypothesized that a heterogeneous mix of oscillators with periods and amplitudes that are easily tuned are beneficial to resynchrony.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[2]">(EPS)</p>
<a name="pcbi.1002787.s004" id="pcbi.1002787.s004"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s004">Figure S4. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[1]"><strong>Intrinsic amplitude and period do not correlate.</strong> We plot the intrinsic period against the intrinsic amplitude (normalized to the largest value) for slice 1 (A), slice 2 (B), and the model (C). The model tissue is composed of 80% weak and 20% strong cells. The slice amplitude is measured as the amplitude during the final day of TTX treatment and its period is the mean value of the Wavos-computed period throughout TTX treatment. In all cases, there is similar period variability and the period and amplitude are un-correlated.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[2]">(EPS)</p>
<a name="pcbi.1002787.s005" id="pcbi.1002787.s005"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s005">Figure S5. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[1]"><strong>Three 100-cell networks comprised of 50% sustained cells (blue) and 50% weak cells (cyan).</strong> All are connected using a small world network, but with different classes of cells at the “hubs” - (<strong>A</strong>) sustained cells are placed at hubs, (<strong>B</strong>) no cell class is given preference, and (<strong>C</strong>) weak cells are placed at hubs. The size of the cell is proportional to the number of cells it connects to. Most network connections are shown in light gray. One “hub” cell is highlighted in each network, and its connections are dark gray.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[2]">(EPS)</p>
<a name="pcbi.1002787.s006" id="pcbi.1002787.s006"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s006">Figure S6. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[6]/caption[1]/p[1]"><strong>Damped and weak cells have similar effects on the network.</strong> We show heat plots of the quality of synchrony after the wash for varying demographics (<strong>A</strong>) and varying types of cells at network hubs when 50% of the cells are strong (<strong>B</strong>), with red indicating perfect synchrony and blue indicating no synchrony. The y-axis of both plots indicates the fraction of non-strong (i.e. weak and damped) cells that is weak. The top row in both plots corresponds to the data in <a href="#pcbi-1002787-g003">Figure 3</a>. In (<strong>A</strong>), we show that when there are no strong cells (leftmost column), synchrony is high, regardless of whether the population is composed of all damped cells (bottom), a mix of damped and weak cells (middle), or all weak cells (top). For each column, the color remains mostly constant, indicating that the quality of sync is a function of the fraction of oscillators that are strong, and that weak and damped oscillators are interchangeable. In (<strong>B</strong>), we show that placing weak or damped cells at the hubs allows for better synchrony than placing mixed or strong cells at the hubs. Note that placing weak cells at hubs in populations with no weak cells is equivalent to placing mixed cells at hubs. Likewise, placing damped cells at hubs when there are no damped cells is equivalent to placing mixed cells at hubs.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[6]/caption[1]/p[2]">(EPS)</p>
<a name="pcbi.1002787.s007" id="pcbi.1002787.s007"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s007">Text S1. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[7]/caption[1]/p[1]"><strong>Statistical analysis.</strong> We examine two SCN explants and corresponding simulation data for evidence that intrinsic properties of cells predict emergent behaviors in coupled conditions. We find no correlation between any intrinsic property and a cell's behavior after coupling has been established, for either the model or the explants. For the explants, however, we find evidence of a spatial pattern, in which weak, lateral cells phase lead strong, medial cells.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[7]/caption[1]/p[2]">(PDF)</p>
<a name="pcbi.1002787.s008" id="pcbi.1002787.s008"></a><p><strong xPathLocation="noSelect"><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pcbi.1002787.s008">Text S2. </a></strong></p><p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[8]/caption[1]/p[1]"><strong>Reduced model details.</strong> We define the reduced model and demonstrate that it accurately captures the behavior of the mechanistic model used in the main text.</p>
<p xpathLocation="/article[1]/body[1]/sec[5]/supplementary-material[8]/caption[1]/p[2]">(PDF)</p>
</div>





<div xpathLocation="noSelect"><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3 xpathLocation="noSelect">Acknowledgments&nbsp;<a href="#top">Top</a></h3>
<p xpathLocation="/article[1]/back[1]/ack[1]/p[1]">The authors would like to thank members of the Washington University – UCSB – UMass – Colby Sync Group for helpful discussions throughout the development of this work. We are also grateful to Panagiota Foteinou, Peter St. John, and Luis Morelli for helpful comments on the manuscript. SRT is grateful to Randall Downer for his expertise running the compute cluster and to James Scott for helpful discussion about data analyses.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3 xpathLocation="noSelect">Author Contributions&nbsp;<a href="#top">Top</a></h3><p xpathLocation="noSelect">Conceived and designed the experiments: ABW SRT KAT FJD EDH. Performed the experiments: ABW SRT EDH. Analyzed the data: ABW SRT EDH. Contributed reagents/materials/analysis tools: SRT FJD EDH. Wrote the paper: ABW SRT EDH.</p></div><div xpathLocation="noSelect"><a id="references" name="references" toc="references" title="References"></a><h3 xpathLocation="noSelect">References&nbsp;<a href="#top">Top</a></h3><ol class="references" xpathLocation="noSelect"><li xpathLocation="noSelect"><a name="pcbi.1002787-Dunlap1" id="pcbi.1002787-Dunlap1"></a>Dunlap JC (1999) Molecular bases for circadian clocks. Cell 96: 271–290. <a class="find" href="/article/findArticle.action?citedArticleID=6231746">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Ralph1" id="pcbi.1002787-Ralph1"></a>Ralph MR, Foster RG, Davis FC, Menaker M (1990) Transplanted suprachiasmatic nucleus determines circadian period. Science 247: 975–978. <a class="find" href="/article/findArticle.action?citedArticleID=6231748">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Welsh1" id="pcbi.1002787-Welsh1"></a>Welsh DK, Logothetis DE, Meister M, Reppert SM (1995) Individual neurons dissociated from rat suprachiasmatic nucleus express independently phased circadian firing rhythms. Neuron 14: 697–706. <a class="find" href="/article/findArticle.action?citedArticleID=6231750">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Liu1" id="pcbi.1002787-Liu1"></a>Liu C, Weaver DR, Strogatz SH, Reppert SM (1997) Cellular construction of a circadian clock: period determination in the suprachiasmatic nuclei. Cell 91: 855–860. <a class="find" href="/article/findArticle.action?citedArticleID=6231752">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Herzog1" id="pcbi.1002787-Herzog1"></a>Herzog ED, Takahashi JS, Block GD (1998) Clock controls circadian period in isolated suprachiasmatic nucleus neurons. Nat Neurosci 1: 708–713. <a class="find" href="/article/findArticle.action?citedArticleID=6231754">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Yamaguchi1" id="pcbi.1002787-Yamaguchi1"></a>Yamaguchi S, Isejima H, Matsuo T, Okura R, Yagita K, et al. (2003) Synchronization of cellular clocks in the suprachiasmatic nucleus. Science 302: 1408–1412. <a class="find" href="/article/findArticle.action?citedArticleID=6231756">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Reppert1" id="pcbi.1002787-Reppert1"></a>Reppert SM, Weaver DR (2002) Coordination of circadian timing in mammals. Nature 418: 935–941. <a class="find" href="/article/findArticle.action?citedArticleID=6231758">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Zhang1" id="pcbi.1002787-Zhang1"></a>Zhang L, Jones CR, Ptacek LJ, Fu YH (2011) The genetics of the human circadian clock. Adv Genet 74: 231–247. <a class="find" href="/article/findArticle.action?citedArticleID=6231760">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Yamada1" id="pcbi.1002787-Yamada1"></a>Yamada Y, Forger D (2010) Multiscale complexity in the mammalian circadian clock. Curr Opin Genet Dev 20: 626–633. <a class="find" href="/article/findArticle.action?citedArticleID=6231762">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Webb1" id="pcbi.1002787-Webb1"></a>Webb AB, Angelo N, Huettner JE, Herzog ED (2009) Intrinsic, nondeterministic circadian rhythm generation in identified mammalian neurons. Proc Natl Acad Sci U S A 106: 16493–16498. <a class="find" href="/article/findArticle.action?citedArticleID=6231764">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Ko1" id="pcbi.1002787-Ko1"></a>Ko CH, Yamada YR, Welsh DK, Buhr ED, Liu AC, et al. (2010) Emergence of noise-induced oscillations in the central circadian pacemaker. PLoS Biol 8: e1000513. <a class="find" href="/article/findArticle.action?citedArticleID=6231766">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Evans1" id="pcbi.1002787-Evans1"></a>Evans JA, Leise TL, Castanon-Cervantes O, Davidson AJ (2011) Intrinsic regulation of spatiotemporal organization within the suprachiasmatic nucleus. PLoS One 6: e15869. <a class="find" href="/article/findArticle.action?citedArticleID=6231768">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Foley1" id="pcbi.1002787-Foley1"></a>Foley NC, Tong TY, Foley D, Lesauter J, Welsh DK, et al. (2011) Characterization of orderly spatiotemporal patterns of clock gene activation in mammalian suprachiasmatic nucleus. Eur J Neurosci 33: 1851–1865. <a class="find" href="/article/findArticle.action?citedArticleID=6231770">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Yan1" id="pcbi.1002787-Yan1"></a>Yan L, Okamura H (2002) Gradients in the circadian expression of Per1 and Per2 genes in the rat suprachiasmatic nucleus. Eur J Neurosci 15: 1153–1162. <a class="find" href="/article/findArticle.action?citedArticleID=6231772">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Quintero1" id="pcbi.1002787-Quintero1"></a>Quintero JE, Kuhlman SJ, McMahon DG (2003) The biological clock nucleus: a multiphasic oscillator network regulated by light. J Neurosci 23: 8070–8076. <a class="find" href="/article/findArticle.action?citedArticleID=6231774">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Maywood1" id="pcbi.1002787-Maywood1"></a>Maywood ES, Chesham JE, O'Brien JA, Hastings MH (2011) A diversity of paracrine signals sustains molecular circadian cycling in suprachiasmatic nucleus circuits. Proc Natl Acad Sci U S A 108: 14306–11. <a class="find" href="/article/findArticle.action?citedArticleID=6231776">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Davidson1" id="pcbi.1002787-Davidson1"></a>Davidson AJ, Castanon-Cervantes O, Leise TL, Molyneux PC, Harrington ME (2008) Visualizing jet lag in the mouse suprachiasmatic nucleus and peripheral circadian timing system. Eur J Neurosci 29: 171–80. <a class="find" href="/article/findArticle.action?citedArticleID=6231778">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Leloup1" id="pcbi.1002787-Leloup1"></a>Leloup JC, Goldbeter A (2003) Toward a detailed computational model for the mammalian circadian clock. Proc Natl Acad Sci U S A 100: 7051–7056. <a class="find" href="/article/findArticle.action?citedArticleID=6231780">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Forger1" id="pcbi.1002787-Forger1"></a>Forger DB, Peskin CS (2004) Stochastic simulation of the mammalian circadian clock. Proc Natl Acad Sci U S A 102: 321–324. <a class="find" href="/article/findArticle.action?citedArticleID=6231782">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Leloup2" id="pcbi.1002787-Leloup2"></a>Leloup JC, Goldbeter A (2004) Modeling the mammalian circadian clock: Sensitivity analysis and multiplicity of oscillatory mechanisms. J Theor Biol 230: 541–562. <a class="find" href="/article/findArticle.action?citedArticleID=6231784">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-To1" id="pcbi.1002787-To1"></a>To TL, Henson MA, Herzog ED, Doyle FJ (2007) A Molecular Model for Intercellular Synchronization in the Mammalian Circadian Clock. Biophys J 92: 3792–3803. <a class="find" href="/article/findArticle.action?citedArticleID=6231786">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Vasalou1" id="pcbi.1002787-Vasalou1"></a>Vasalou C, Herzog ED, Henson MA (2009) Small-World Network Models of Intercellular Coupling Predict Enhanced Synchronization in the Suprachiasmatic Nucleus. J Biol Rhythms 24: 243–254. <a class="find" href="/article/findArticle.action?citedArticleID=6231788">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Gonze1" id="pcbi.1002787-Gonze1"></a>Gonze D, Bernard S, Waltermann C, Kramer A, Herzel H (2005) Spontaneous synchronization of coupled circadian oscillators. Biophys J 89: 120–129. <a class="find" href="/article/findArticle.action?citedArticleID=6231790">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Bernard1" id="pcbi.1002787-Bernard1"></a>Bernard S, Gonze D, Cajavec B, Herzel H, Kramer A (2007) Synchronization-Induced Rhythmicity of Circadian Oscillators in the Suprachiasmatic Nucleus. PLoS Comput Biol 3: e68. <a class="find" href="/article/findArticle.action?citedArticleID=6231792">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Vasalou2" id="pcbi.1002787-Vasalou2"></a>Vasalou C, Henson MA (2011) A multicellular model for differential regulation of circadian signals in the core and shell regions of the suprachiasmatic nucleus. J Theor Biol 288: 44–56. <a class="find" href="/article/findArticle.action?citedArticleID=6231794">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Hafner1" id="pcbi.1002787-Hafner1"></a>Hafner M, Koeppl H, Gonze D (2012) Effect of network architecture on synchronization and entrainment properties of the circadian oscillations in the suprachiasmatic nucleus. PLoS Comput Biol 8: e1002419. <a class="find" href="/article/findArticle.action?citedArticleID=6231796">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Schroder1" id="pcbi.1002787-Schroder1"></a>Schroder S, Herzog ED, Kiss IZ (2012) Transcription-based oscillator model for light-induced splitting as antiphase circadian gene expression in the suprachiasmatic nuclei. J Biol Rhythms 27: 79–90. <a class="find" href="/article/findArticle.action?citedArticleID=6231798">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Westermark1" id="pcbi.1002787-Westermark1"></a>Westermark PO, Welsh DK, Okamura H, Herzel H (2009) Quantification of circadian rhythms in single cells. PLoS Comput Biol 5: e1000580. <a class="find" href="/article/findArticle.action?citedArticleID=6231800">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Granada1" id="pcbi.1002787-Granada1"></a>Granada AE, Herzel H (2009) How to achieve fast entrainment? The timescale to synchronization. PLoS One 4: e7057. <a class="find" href="/article/findArticle.action?citedArticleID=6231802">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Leise1" id="pcbi.1002787-Leise1"></a>Leise TL, Wang CW, Gitis PJ, Welsh DK (2012) Persistent cell-autonomous circadian oscillations in fibroblasts revealed by six-week single-cell imaging of PER2:: LUC bioluminescence. PLoS One 7: e33334. <a class="find" href="/article/findArticle.action?citedArticleID=6231804">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Taylor1" id="pcbi.1002787-Taylor1"></a>Taylor SR, Gunawan R, Petzold LR, Doyle FJ (2008) Sensitivity Measures for Oscillating Systems: Application to Mammalian Circadian Gene Network. IEEE Trans Automat Contr 53: 177–188. <a class="find" href="/article/findArticle.action?citedArticleID=6231806">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Taylor2" id="pcbi.1002787-Taylor2"></a>Taylor SR, Webb AB, Smith KS, Petzold LR, Doyle FJ (2010) Velocity response curves support the role of continuous entrainment in circadian clocks. J Biol Rhythms 25: 138–149. <a class="find" href="/article/findArticle.action?citedArticleID=6231808">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Abraham1" id="pcbi.1002787-Abraham1"></a>Abraham U, Granada AE, Westermark PO, Heine M, Kramer A, et al. (2010) Coupling governs entrainment range of circadian clocks. Mol Syst Biol 6: 438. <a class="find" href="/article/findArticle.action?citedArticleID=6231810">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Buhr1" id="pcbi.1002787-Buhr1"></a>Buhr ED, Yoo SH, Takahashi JS (2010) Temperature as a universal resetting cue for mammalian circadian oscillators. Science 330: 379–385. <a class="find" href="/article/findArticle.action?citedArticleID=6231812">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Yoo1" id="pcbi.1002787-Yoo1"></a>Yoo SH, Yamazaki S, Lowrey PL, Shimomura K, Ko CH, et al. (2004) Period2:: luciferase real-time reporting of circadian dynamics reveals persistent circadian oscillations in mouse peripheral tissues. Proc Natl Acad Sci U S A 101: 5339–5346. <a class="find" href="/article/findArticle.action?citedArticleID=6231814">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Taylor3" id="pcbi.1002787-Taylor3"></a>Taylor AL, Hickey TJ, Prinz AA, Marder E (2006) Structure and visualization of high-dimensional conductance spaces. J Neurophysiol 96: 891–905. <a class="find" href="/article/findArticle.action?citedArticleID=6231816">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Aton1" id="pcbi.1002787-Aton1"></a>Aton SJ, Colwell CS, Harmar AJ, Waschek J, Herzog ED (2005) Vasoactive intestinal polypeptide mediates circadian rhythmicity and synchrony in mammalian clock neurons. Nat Neurosci 8: 476–483. <a class="find" href="/article/findArticle.action?citedArticleID=6231818">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Inagaki1" id="pcbi.1002787-Inagaki1"></a>Inagaki N, Honma S, Ono D, Tanahashi Y, Honma KI (2007) Separate oscillating cell groups in mouse suprachiasmatic nucleus couple photoperiodically to the onset and end of daily activity. Proc Natl Acad Sci U S A 104: 7664–7669. <a class="find" href="/article/findArticle.action?citedArticleID=6231820">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Schaap1" id="pcbi.1002787-Schaap1"></a>Schaap J, Albus H, Tjebbe VH, Eilers PH, Detari L, et al. (2003) Heterogeneity of rhythmic suprachiasmatic nucleus neurons: Implications for circadian waveform and photoperiodic encoding. Proc Natl Acad Sci USA 100: 15994–15999. <a class="find" href="/article/findArticle.action?citedArticleID=6231822">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Yan2" id="pcbi.1002787-Yan2"></a>Yan L, Karatsoreos I, Lesauter J, Welsh DK, Kay S, et al. (2007) Exploring spatiotemporal organization of SCN circuits. Cold Spring Harb Symp Quant Biol 72: 527–541. <a class="find" href="/article/findArticle.action?citedArticleID=6231824">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Liu2" id="pcbi.1002787-Liu2"></a>Liu AC, Welsh DK, Ko CH, Tran HG, Zhang EE, et al. (2007) Intercellular Coupling Confers Robustness against Mutations in the SCN Circadian Clock Network. Cell 129: 605–616. <a class="find" href="/article/findArticle.action?citedArticleID=6231826">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Loh1" id="pcbi.1002787-Loh1"></a>Loh DH, Dragich JM, Kudo T, Schroeder AM, Nakamura TJ, et al. (2011) Effects of vasoactive intestinal peptide genotype on circadian gene expression in the suprachiasmatic nucleus and peripheral organs. J Biol Rhythms 26: 200–209. <a class="find" href="/article/findArticle.action?citedArticleID=6231828">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Virshup1" id="pcbi.1002787-Virshup1"></a>Virshup DM, Eide EJ, Forger DB, Gallego M, Harnish EV (2007) Reversible Protein Phosphorylation Regulates Circadian Rhythms. Cold Spring Harb Symp Quant Biol 72: 413–420. <a class="find" href="/article/findArticle.action?citedArticleID=6231830">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Locke1" id="pcbi.1002787-Locke1"></a>Locke JC, Westermark PO, Kramer A, Herzel H (2008) Global parameter search reveals design principles of the mammalian circadian clock. BMC Syst Biol 2: 22. <a class="find" href="/article/findArticle.action?citedArticleID=6231832">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Antle1" id="pcbi.1002787-Antle1"></a>Antle MC, Foley DK, Foley NC, Silver R (2003) Gates and oscillators: a network model of the brain clock. J Biol Rhythms 18: 339–350. <a class="find" href="/article/findArticle.action?citedArticleID=6231834">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Antle2" id="pcbi.1002787-Antle2"></a>Antle MC, Foley NC, Foley DK, Silver R (2007) Gates and Oscillators II: Zeitgebers and the Network Model of the Brain Clock. J Biol Rhythms 22: 14–25. <a class="find" href="/article/findArticle.action?citedArticleID=6231836">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Saxena1" id="pcbi.1002787-Saxena1"></a>Saxena MT, Aton SJ, Hildebolt C, Prior JL, Abraham U, et al. (2007) Bioluminescence imaging of period1 gene expression in utero. Mol Imaging 6: 68–72. <a class="find" href="/article/findArticle.action?citedArticleID=6231838">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Sumova1" id="pcbi.1002787-Sumova1"></a>Sumova A, Sladek M, Polidarova L, Novakova M, Houdek P (2012) Circadian system from conception till adulthood. Prog Brain Res 199: 83–103. <a class="find" href="/article/findArticle.action?citedArticleID=6231840">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Caba1" id="pcbi.1002787-Caba1"></a>Caba M, Gonzalez-Mariscal G (2009) The rabbit pup, a natural model of nursing-anticipatory activity. Eur J Neurosci 30: 1697–1706. <a class="find" href="/article/findArticle.action?citedArticleID=6231842">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Sokolove1" id="pcbi.1002787-Sokolove1"></a>Sokolove PG, Bushell WN (1978) The chi square periodogram: its utility for analysis of circadian rhythms. J Theor Biol 72: 131–160. <a class="find" href="/article/findArticle.action?citedArticleID=6231844">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Plautz1" id="pcbi.1002787-Plautz1"></a>Plautz JD, Straume M, Stanewsky R, Jamison CF, Brandes C, et al. (1997) Quantitative analysis of Drosophila period gene transcription in living animals. J Biol Rhythms 12: 204–217. <a class="find" href="/article/findArticle.action?citedArticleID=6231846">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Harang1" id="pcbi.1002787-Harang1"></a>Harang RE, Bonnet G, Petzold LR (2012) WAVOS: a MATLAB toolkit for wavelet analysis and visualization of oscillatory systems. BMC research notes 5: 163. <a class="find" href="/article/findArticle.action?citedArticleID=6231848">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Strogatz1" id="pcbi.1002787-Strogatz1"></a>Strogatz SH (2000) From Kuramoto to Crawford: exploring the onset of synchronization in populations of coupled oscillators. Physica D 143: 1–20. <a class="find" href="/article/findArticle.action?citedArticleID=6231850">
                      Find this article online
                    </a></li><li xpathLocation="noSelect"><a name="pcbi.1002787-Meeker1" id="pcbi.1002787-Meeker1"></a>Meeker K, Harang R, Webb AB, Welsh DK, Doyle III FJ, et al. (2011) Wavelet measurement suggests cause of period instability in mammalian circadian neurons. J Biol Rhythms 26: 353–362. <a class="find" href="/article/findArticle.action?citedArticleID=6231852">
                      Find this article online
                    </a></li></ol></div>

</div></div>
<div style="display:none">
<div dojoType="ambra.widget.RegionalDialog" id="AnnotationDialog" style="padding:0;margin:0;">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Post Your Note (For Public Viewing)</span></h5>
      <div class="posting pane">
        <form name="createAnnotation" id="createAnnotation" method="post" action="">
          <input type="hidden" name="target" value="info:doi/10.1371/journal.pcbi.1002787" />
          <input type="hidden" name="startPath" value="" />
          <input type="hidden" name="startOffset" value="" />
          <input type="hidden" name="endPath" value="" />
          <input type="hidden" name="endOffset" value="" />
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="false" />
          <input type="hidden" name="noteType" id="noteType" value="" />
          <fieldset>
            <legend>Compose Your Note</legend>
            <span id="submitMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td>
                  <label for="cNoteType">This is a </label><select name="cNoteType" id="cNoteType"><option value="note">note</option><option value="correction">correction</option></select>
                  <span id="cdls" style="visibility:hidden;margin-left:0.3em; white-space:nowrap;"><a href="/static/commentGuidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3?target=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002787#corrections">What are corrections?</a></span>
                  <label for="cTitle"><span class="none">Enter your note title</span><!-- error message text <em>A title is required for all public notes</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your note title..." class="title" alt="Enter your note title..." />
                  <label for="cArea"><span class="none">Enter your note</span><!-- error message text <em>Please enter your note</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your note..." alt="Enter your note...">Enter your note...</textarea>
                  <input type="hidden" name="isPublic" value="true" />
                </td>
                <td>&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" checked="checked" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." alt="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td colspan="3" class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel"/>
                  <input type="button" value="Submit" title="Click to post your note publicly" id="btn_post" class="primary"/>
                </td>
              </tr>
              <tr>
                <td colspan="3">
                  <p>Notes and Corrections can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p>
                </td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
    <div class="tip" id="dTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialog" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="caTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      Please follow our <a href="/static/commentGuidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">guidelines for notes and comments</a> and review our <a href="/static/competing.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
      <ul>
        <li>Remarks that could be interpreted as allegations of misconduct</li>
        <li>Unsupported assertions or statements</li>
        <li>Inflammatory or insulting language</li>
      </ul>
      <form name="contextActionForm" id="contextActionForm" class="clearfix buttons" method="post" action="">
        <input type="button" name="Continue" value="Continue" id="ContextActionDialogContinueButton" onmouseup="ambra.displayAnnotationContext.startComment(event);" title="Add a note to this text" class="primary"/>
        <input type="button" name="Cancel" value="Cancel" id="ContextActionDialogCancelButton" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);" title="Close this Window"/>
      </form>
    </div>
    <div class="tip" id="caTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogNotLogged" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canlTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You must be logged in to add a note to an article.
      You may log in by <a onmousedown="ambra.displayAnnotationContext.disconnect(event);" href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002787">clicking here</a> or <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">cancel this note</a>.
    </div>
    <div class="tip" id="canlTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canBDTipu"></div>
    <div class="contextActionContent">
      <h5 class="annotation icon"><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot annotate this area of the document. <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canBDTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.ContextAction" id="ContextActionDialogBadRangeSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canbrTipu"></div>
    <div class="contextActionContent">
      <h5><img src="/images/tooltip_addannotation.gif" /> Add a note to this text.</h5>
      You cannot create an annotation that spans different sections of the document; please adjust your selection.<br/>
      <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canbrTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialog" style="padding:0;margin:0;">
  <div class="dialog preview">
    <div class="tipu" id="cTipu"></div>
    <div class="btn close" id="btn_close" title="Click to close"><a title="Click to close">Close</a></div>
    <div id="cmtContainer" class="comment">
      <h6 id="viewCmtTitle"></h6>
      <div class="detail" id="viewCmtDetail"></div>
      <div class="contentwrap" id="viewComment"></div>
      <div class="contentwrap" id="viewCIStatement"></div>
      <div class="detail" id="viewLink">
        <!--<a href="#" class="commentary icon" title="Click to view full thread and respond">View all responses</a>
        <a href="#" class="respond tooltip" title="Click to respond to this posting">Respond to this</a>-->
      </div>
    </div>
    <div class="tip" id="cTip"></div>
  </div>
</div>
<div dojoType="ambra.widget.RegionalDialog" id="CommentDialogMultiple" style="padding:0;margin:0;">
  <div class="dialog multiple preview">
    <div class="tipu" id="mTipu"></div>
    <div class="btn close" id="btn_close_multi" title="Click to close"><a title="Click to close">Close</a></div>
    <ol id="multilist"></ol>
    <br/>
    <div id="multidetail"></div>
    <div class="tip" id="mTip"></div>
  </div>
</div>


<div dojoType="dijit.Dialog" id="Rating">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Rate This Article</span></h5>
      <div class="instructions">Please follow our <a  href="/static/ratingGuidelines.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">guidelines for rating</a> and review our <a href="/static/competing.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
        <ol>
          <li>Remarks that could be interpreted as allegations of misconduct</li>
          <li>Unsupported assertions or statements</li>
          <li>Inflammatory or insulting language</li>
        </ol>
      </div>
      <div class="posting pane">
        <form name="ratingForm" id="ratingForm" method="post" action="">
          <input type="hidden" name="articleURI" value="info:doi/10.1371/journal.pcbi.1002787" />  
          <input type="hidden" name="commentTitle" id="commentTitle" value="" />
          <input type="hidden" name="comment" id="commentArea" value="" />
          <input type="hidden" name="ciStatement" id="statementArea" value="" />
          <input type="hidden" name="isCompetingInterest" id="isCompetingInterest" value="" />
          <fieldset>
            <legend>Compose Your Annotation</legend>
            <span id="submitRatingMsg" class="error" style="display:none;"></span>
            <table class="layout">
              <tr>
                <td rowspan="2">
                  <label for="insight">Insight</label>
                  <ul class="star-rating rating edit" title="Rate insight" id="rateInsight">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Bland" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'insight', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Profound" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="insight" title="insight" value="" />
                  <label for="reliability">Reliability</label>
                  <ul class="star-rating rating edit" title="Rate reliability" id="rateReliability">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Tenuous" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'reliability', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Unassailable" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="reliability" title="reliability" value="" />
                  <label for="style">Style</label>
                  <ul class="star-rating rating edit" title="Rate style" id="rateStyle">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Crude" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'style', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Elegant" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 5);">5</a></li>
                  </ul>    
                  <input type="hidden" name="style" title="style" value="" />
                  <label for="cTitle"><span class="none">Enter your comment title</span><!-- error message text <em>A title is required for all public annotations</em>--></label>
                  <input type="text" name="cTitle" id="cTitle" value="Enter your comment title..." class="title" alt="Enter your comment title..." />
                  <label for="cArea"><span class="none">Enter your comment</span><!-- error message text <em>Please enter your annotation</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your comment..." alt="Enter your comment...">Enter your comment...</textarea>
                </td>
                <td rowspan="2">&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" type="radio" name="competingInterest" value="false"  /> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" type="radio" name="competingInterest" value="true"  /> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled value="Enter your competing interests..." title="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td class="buttons">
                  <input type="button" value="Cancel" title="Click to close and cancel" id="btn_cancel_rating"/>
                  <input type="button" value="Submit" title="Click to post your annotation publicly" id="btn_post_rating" class="primary"/>
                </td>
              </tr>
              <tr>
                  <td colspan="3"><p>Ratings can include the following markup tags:</p>
                  <p><strong>Emphasis:</strong> ''<em>italic</em>''&nbsp;&nbsp;'''<strong>bold</strong>'''&nbsp;&nbsp;'''''<strong><em>bold italic</em></strong>'''''</p>
                  <p><strong>Other:</strong> ^^<sup>superscript</sup>^^&nbsp;&nbsp;~~<sub>subscript</sub>~~</p></td>
              </tr>
            </table>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
<div dojoType="ambra.widget.LoadingCycle" id="LoadingCycle" class="loadingCycler">
  <img src="/images/loading.gif" width="58" height="58" title="Loading..." />
</div>
<div dojoType="dijit.Dialog" id="LightBox">

    <!-- start main window wrapper-->
    <div class="figure-window-wrapper">

      <!--start film strip Wrapper-->
      <div class="figure-window-nav-wrapper">
        <div id="figure-window-nav"></div>
      </div><!--end figure-window-nav-wrapper-->

      <!-- start figure container-->
      <div class="figure-window-container">

        <!-- start/end figure viewer-->
        <div id="figure-window-viewer">
          <img id="figure-window-img" class="large"/>
        </div>

        <!-- start/end figure doi-->
        <div id="figure-window-doi"></div>

        <!--start figure title-box-->
        <div id="figure-window-title-box" class="figure-window-title-less">
          <span id="figure-window-title"></span>
          <div id="figure-window-more" class="figure-window-more" onclick="return ambra.lightBox.showMoreOrLess();"></div>
          <div id="figure-window-description"></div>
        </div><!--end figure title-box-->

        <!--start figure download-->
        <div class="figure-window-download">
          <ul>
            <li class="download icon"><strong>Download:</strong>
              <a id="figure-window-ppt" title="Click to download PowerPoint slide of this image">PowerPoint
                slide</a> |
              <a id="figure-window-large" title="Click to download a larger version of this image">larger image
                (<span id="figure-window-large-size"></span>)</a> |
              <a id="figure-window-tiff" title="Click to download a original version of this image">original image
                (<span id="figure-window-tiff-size"></span>)</a>
            </li>
          </ul>
        </div><!--end figure download-->

        <!--start/end of close, context, next and previous button-->
        <div class="figure-window-close" onclick="return ambra.lightBox.hide();"></div>
        <div class="figure-window-context" onclick="return ambra.lightBox.showInContext();"></div>
        <div id="figure-window-previous" onclick="return ambra.lightBox.showPrevious();"></div>
        <div id="figure-window-next" onclick="return ambra.lightBox.showNext();"></div>

      </div><!--end figure-window-container-->

    </div><!--end figure-window-wrapper-->

</div><!--end dijit.Dialog-->
</div>
</div>
<!-- end : main contents -->

</div>
<!-- end : container -->	

<!-- begin : footer -->
<div id="ftr">
  <p><span>All site content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License 2.5" tabindex="200">Creative Commons Attribution License</a>.</span></p>
  <ul>
    <li><a href="/static/privacy.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" title="PLOS Privacy Statement" tabindex="501">Privacy Statement</a></li>
    <li><a href="/static/terms.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" title="PLOS Terms of Use" tabindex="502">Terms of Use</a></li>
    <li><a href="http://www.plos.org/advertise/" title="Advertise With PLOS" tabindex="503">Advertise</a></li>
    <li><a href="http://www.plos.org/about/media-inquiries/" title="PLOS Embargo Policy" tabindex="504">Media Inquiries</a></li>
    <li><a href="http://www.plos.org/publish/order-reprints/" title="PLOS in Print" tabindex="505">PLOS in Print</a></li>
    <li><a href="/static/sitemap.action;jsessionid=7A02DD243343C98EF23CCD1CF348B2C3" title="Site Map" tabindex="506">Site Map</a></li>
    <li><a href="http://www.plos.org" title="PLOS.org" tabindex="507">PLOS.org</a></li>
  </ul>
  <div class="powered">
  <ul>
    <li><a href="/static/releaseNotes.action" title="Ambra | Release Notes">Ambra 2.4.9</a></li>
    <li>Managed Colocation provided by <a href="http://www.isc.org/" title="Internet Systems Consortium">Internet Systems Consortium</a>.</li>
  </ul>
  </div>
</div>
<!-- end : footer -->
<script type="text/javascript">
  var _namespace="";
    var loggedIn = false;

  var almHost = "http://alm.plos.org";
  var solrHost = "http://api.plos.org/search";
  var pubGetHost = "http://pubget.com/widgetizer/links_hash";
  var appContext = "";

  // Safari v3.1.1 "console.debug" issue (http://trac.dojotoolkit.org/ticket/6849) workaround
  if (/3[\.0-9]+ Safari/.test(navigator.appVersion)) {
    window.console = {
      origConsole: window.console,
      log: function(s){ this.origConsole.log(s); },
      info: function(s){ this.origConsole.info(s); },
      error: function(s){ this.origConsole.error(s); },
      warn: function(s){ this.origConsole.warn(s); }
    };
  }

  var djConfig = {
    // don't debug for IE - as dojo's firebug lite module is error prone in IE
    isDebug: false,
    parseOnLoad: true
  };

  var bamGAID = 'UA-338393-1';
  var bamGAVPV = location.pathname + location.search + location.hash;
</script>
<script type="text/javascript" src="/javascript/dojo/dojo/dojo.js?v=pzlwVCPPo+oJC12upPgJrJ5EG_k"></script>
<script type="text/javascript" src="/javascript/dojo/dojo/ambra.js?v=Y8JM0CVD9YwjC99M_jvbcRTiGhY"></script>
<script type="text/javascript" src="/javascript/init_global.js?v=fF03pfG3r+Nkj_czZ8ORg8o5sNM"></script>
<script type="text/javascript" src="/javascript/init_article.js?v=qzBD_a7nKbeMM2xNUtB3vckN+Zc"></script>
<script type="text/javascript" src="/javascript/init_ratings.js?v=seKykuS0X33CQGS7Lj+8FtBy5Uw"></script>
<script type="text/javascript" src="/javascript/init_lightBox.js?v=vJRLPy7RRX8Zp31Ogcm3i+TA4f8"></script>
<script type="text/javascript" src="/javascript/init_article_body.js?v=YgFei2xCi50CahwaPbSsxUxUsmw"></script>
<script type="text/javascript" src="/javascript/init_article_rhc.js?v=djtbTMOagorKE6ZPrkCwufAfWWI"></script>
<script type="text/javascript" src="/javascript/alm.js?v=1tH9JZ8uDtXspXssGJDr8O+jBOs"></script>
<script type="text/javascript" src="/javascript/reporting/articleViewsCumulative.js?v=RQ+gCrcAl0dBeLbmokxgQZTGYYA"></script>
<script type="text/javascript" src="/javascript/check_twitter.js?v=Hc0sH1W9xIu0qhqxWPD4c1u50K4"></script>
<script type="text/javascript" src="/javascript/init_googleAnalytics.js?v=yCD4h9PHGI0dsefmuWDlMn7pHog"></script>

</body>
</html>

